{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# **Mutated-gene Genome-Wide Association Study (MugGWAS)**"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "MugGWAS offers an non-model-organism-friendly pipeline to infer gene-trait association. It tackles common challenges faced by non-model organism research—such as lack of readily available annotation databases and small sample sizes—by integrating a simple annotation pipeline with mutation-centric GWAS analysis. Users can identify putative gene mutations that drive phenotypic change without the large statistical power required to perform genome-wide tests on a per nucleotide basis."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## **Motivation**\n",
    "\n",
    "Understanding the genetic basis of a trait is central to many biological research that help gain mechanismtic insight into complex biological phenomena. For instance, the discovery of genetic loci that contribute to antibiotic resistance could better our knowledge for how important pathogens evolve to escape medicinal application. \n",
    "\n",
    "Most association tools classify raw genetic variants (like single nucleotide polymorphisms or structural variants) into reference or alternative genotypes, then perform statistical tests to infer associations between particular variants and phenotypes. However, genome-wide analyses typically require large sample sizes for statistical power—a major challenge for organisms that are difficult to culture or traits that are challenging to measure. Alternative approaches test associations between phenotypes and gene presence/absence, but these methods don't capture finer details like important mutations that potentially disrupt gene function.\n",
    "\n",
    "Mutated-gene Genome-Wide Association Study (MugGWAS) addresses these limitations by offering a non-model-organism-friendly pipeline to infer gene-trait associations. By annotating disruptive mutation types (nonsense, missense, stopgain, or nonstop), MugGWAS identifies putative gene dysfunctions associated with phenotypic changes. This approach conserves statistical power by avoiding tests on variants that either (1) don't affect function at the gene level or (2) result in disruption of the same gene, allowing users to identify putative gene mutations driving phenotypic changes without the large statistical power required for per-nucleotide genome-wide tests."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## **Short Guide:**\n",
    "MugGWAS offers an non-model-organism-friendly pipeline to infer gene-trait association.\n",
    "\n",
    "**Common GWAS tools:**\n",
    "- Classify raw genetic variants (like SNPs or Indels) into reference or alternative genotypes.\n",
    "- Caveats:\n",
    "    1. require large sample sizes for statistical power—a major challenge.\n",
    "    2. hard for organisms that are difficult to culture or traits that are challenging to measure.\n",
    "    3. most variants don't affect functions on the gene level, i.e. synonymous mutations.\n",
    "\n",
    "**Mutated-gene (Mug) GWAS:**\n",
    "- Tackles common challanges for non-model organisms.\n",
    "- Integrates a simple annotation pipeline with mutation-centric GWAS analysis.\n",
    "- Identify putative gene mutations associated with phenotypic change."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## **Prerquisites**\n",
    "MugGWAS uses the external tool-**ANNOVAR** to annotate variants. Please follow the [instructions](http://annovar.openbioinformatics.org/en/latest/) to download **ANNOVAR** (_registration is required_), and specify the path to ANNOVAR scripts for MugGWAS. More instructions are provided below.\n",
    "\n",
    "This package uses the fixed effect model in [**pyseer**](https://pyseer.readthedocs.io/en/master/index.html) for association analysis. Users don't need to download **Pyseer** and the [prerequisites](https://pyseer.readthedocs.io/en/master/installation.html#prerequisites) themselves. They will be included in the conda environments following the instructions below."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## **Installation Guide**\n",
    "\n",
    "The easiest way to install MugGWAS and fulfill the prerequites is through this path:\n",
    "\n",
    "#### **Build Conda Environment**\n",
    "\n",
    "Download the [environmental.yaml](https://github.com/lyuchengmarvin/MugGWAS/blob/main/envs/environment.yaml) file for setting up your conda environement. You can download it using this command in your terminal.\n",
    "\n",
    "```{command line}\n",
    "wget https://github.com/lyuchengmarvin/MugGWAS/blob/main/envs/environment.yaml \n",
    "```\n",
    "\n",
    "Build the environment to satisfy software prerequisites and ensure applicability.\n",
    "\n",
    "```{command line}\n",
    "conda env create --name muggwas --file envs/environment.yaml\n",
    "```\n",
    "\n",
    "Activate envrionment before use each time:\n",
    "\n",
    "```{command line}\n",
    "conda activate muggwas\n",
    "```\n",
    "\n",
    "Deactivate after use:\n",
    "\n",
    "```{command line}\n",
    "conda deactivate\n",
    "```\n",
    "\n",
    "#### **Install MugGWAS**\n",
    "\n",
    "Install MugGWAS in your environment and make sure that all prerequisites are fulfilled.\n",
    "\n",
    "```{command line}\n",
    "pip install -i https://test.pypi.org/simple/ MugGWAS\n",
    "```\n",
    "You might need to restart the kernal to run this tutorial in Jupyter Notebook."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## **Tutaorial**\n",
    "\n",
    "### **Download the Tutorial Dataset**\n",
    "Type this command in your terminal to download the [dataset](https://github.com/lyuchengmarvin/MugGWAS/tree/main/tutorials/data/pyseer_dataset). Remeber to place this under the same directory of this tutorial file.\n",
    "```{comand line}\n",
    "wget https://github.com/lyuchengmarvin/MugGWAS/tree/main/tutorials/data/pyseer_dataset\n",
    "```\n",
    "\n",
    "### **Description of the Dataset:**\n",
    "*Streptococcus pneumoniae* is a common nasopharyngeal commensal bacterium and an important human pathogen. This dataset consists of 616 annotated draft genomes of *S. pneumoniae* isolates collected from children routine primary care visits in Massachusetts in 2001 (Croucher et al. 2015). These genotypic data can be paired with phenotypic information, such as minimum inhibitory concentrations for antibiotics, to study resistance evolution and epidemiology.\n",
    "\n",
    "### **Overview:**\n",
    "1. Annotate variants\n",
    "2. Build gene mutation table\n",
    "3. Estimate population structure effect\n",
    "4. Run GWAS with pyseer\n",
    "\n",
    "![MugGWAS Workflow](MugGWAS_workflow.png)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### **Step 1: Annotate variants with ANNOVAR**\n",
    "\n",
    "We will use the external tool ANNOVAR and annotate the variants from vcf files. Although one vcf file should contain the variant information for multiple samples, ANNOVAR annotates samples iteratively and create an annotation file per sample.\n",
    "\n",
    "This tool also assumed that users will build their customized database for their none-model organisms, so a genome annotation **<ref_prefix>.gff3** and a genome fasta file **<ref_prefix>.fna** are required and placed in the directory **<ref_prefix>/**. The database will utilize the tool `gff3ToGenePred` to build the database, which was also built when users created their conda environments.\n",
    "\n",
    "\n",
    "**Annotation workflow**\n",
    "\n",
    "1. **Make input files**:\n",
    "    - Input: a VCF file `vcf_prefix.vcf` (with multiple samples)\n",
    "    - Convert VCF to ANNOVAR input format `vcf_prefix.<sample_name>.avinput`\n",
    "    - Store in `/path_to_vcf/annovar_files/`\n",
    "2. **Build database**:\n",
    "    - Input: `ref_prefix.gff3` and `ref_prefix.fna` in `db_dir/`\n",
    "    - Build customized database: `ref_prefix_refGene.txt` and `ref_prefix_refGeneMrna.fa`\n",
    "3. **Annotate variants with ANNOVAR**:\n",
    "    - Output1: `vcf_prefix.<sample_name>.avinput.variant_function` infer mutation position on a gene.\n",
    "    - Output2: `vcf_prefix.<sample_name>.avinput.exonic_variant_function` infer mutation effects on translation, namely synonymous, nonsynonymous, stop codon gain, or stop codon loss."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Converting VCF file to annovar input format...\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "NOTICE: output files will be written to pyseer_dataset/original_dataset/annovar_files/snps.<samplename>.avinput\n",
      "NOTICE: Finished reading 198255 lines from VCF file\n",
      "NOTICE: A total of 198248 locus in VCF file passed QC threshold, representing 177962 SNPs (119583 transitions and 41089 transversions) and 0 indels/substitutions\n",
      "NOTICE: Finished writing 6466277 SNP genotypes (4889814 transitions and 1576463 transversions) and 0 indels/substitutions for 603 samples\n",
      "WARNING: 37576 invalid alternative alleles found in input file\n",
      "WARNING: 20286 invalid reference alleles found in input file\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "The VCF file has been converted and saved to pyseer_dataset/original_dataset/annovar_files/.\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "NOTICE: Reading region file pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 regions from 1 chromosomes\n",
      "NOTICE: Finished reading 1 sequences from pyseer_dataset/Spn23F/Spn23F.fna\n",
      "NOTICE: Finished writting FASTA for 2271 genomic regions to pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa\n",
      "WARNING: 3 regions occur more than once with discordant sequence length (for example, cds-WP_000997239.1, cds-WP_087671856.1, cds-WP_010963246.1)\n",
      "WARNING: 111 gene regions do not have complete ORF (for example, cds-SPN23F_RS12435\u001cNC_011900.1:1406982, cds-SPN23F_RS12945\u001cNC_011900.1:989147, cds-SPN23F_RS08965\u001cNC_011900.1:1716895, cds-SPN23F_RS07285\u001cNC_011900.1:1402783, cds-SPN23F_RS13695\u001cNC_011900.1:122237)\n",
      "2025-04-23 21:49:12,454 - INFO - Starting parallel annotation of avinput files.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Database has been built and saved to pyseer_dataset/Spn23F/.\n",
      "\n",
      "Annotating avinput files...\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#22.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#22.avinput.exonic_variant_function\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#17.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#17.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#63.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#63.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#1.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#1.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#19.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#19.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#94.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#94.avinput.exonic_variant_function\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#73.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#73.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#84.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#84.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7868 unique variants in 7868 input lines\n",
      "NOTICE: Processing next batch with 10786 unique variants in 10786 input lines\n",
      "NOTICE: Processing next batch with 10038 unique variants in 10038 input lines\n",
      "NOTICE: Processing next batch with 11592 unique variants in 11592 input lines\n",
      "NOTICE: Processing next batch with 9595 unique variants in 9595 input lines\n",
      "NOTICE: Processing next batch with 11798 unique variants in 11798 input lines\n",
      "NOTICE: Processing next batch with 10754 unique variants in 10754 input lines\n",
      "NOTICE: Processing next batch with 10811 unique variants in 10811 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1344 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3045 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t731818\t731818\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:14,972 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#22.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#89.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#89.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10571 unique variants in 10571 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1447 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1459 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1469 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2023 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t511990\t511990\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1901 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t189572\t189572\tG\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1522 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:49:15,408 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#94.avinput\n",
      "Done with 1534 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:49:15,420 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#17.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2131 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t405131\t405131\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_6#7.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_6#7.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#28.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#28.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:49:15,541 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#84.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1942 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t860468\t860468\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1440 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:49:15,566 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#1.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2186 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t422145\t422145\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:15,614 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#73.avinput\n",
      "NOTICE: Processing next batch with 11944 unique variants in 11944 input lines\n",
      "NOTICE: Processing next batch with 12517 unique variants in 12517 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#38.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#38.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#94.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#94.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1873 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1621668\t1621668\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:49:15,746 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#63.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#84.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#84.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 10986 unique variants in 10986 input lines\n",
      "Done with 1487 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10409 unique variants in 10409 input lines\n",
      "NOTICE: Processing next batch with 10218 unique variants in 10218 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#9.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#9.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2175 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1461333\t1461333\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:15,936 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#19.avinput\n",
      "NOTICE: Processing next batch with 7673 unique variants in 7673 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#5.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#5.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11165 unique variants in 11165 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1498 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2024 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1316404\t1316404\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:17,363 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#89.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1358 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#16.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#16.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2907 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1038349\t1038349\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:17,494 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#9.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10175 unique variants in 10175 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#14.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#14.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1465 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 13054 unique variants in 13054 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1500 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1898 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2003139\t2003139\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1459 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:49:17,892 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#94.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1520 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1485 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2448 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1453103\t1453103\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#52.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#52.avinput.exonic_variant_function\n",
      "2025-04-23 21:49:18,017 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#38.avinput\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1826 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2016840\t2016840\tT\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:18,047 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#84.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2332 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t98814\t98814\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2372 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1623888\t1623888\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1491 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:49:18,122 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_6#7.avinput\n",
      "NOTICE: Processing next batch with 10475 unique variants in 10475 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#7.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#7.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:49:18,148 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#28.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#53.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#53.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10122 unique variants in 10122 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#5.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#5.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7694 unique variants in 7694 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1978 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1070512\t1070512\tA\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#18.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#18.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:49:18,292 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#5.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10122 unique variants in 10122 input lines\n",
      "NOTICE: Processing next batch with 10632 unique variants in 10632 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#70.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#70.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7587 unique variants in 7587 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1469 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2197 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2048161\t2048161\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:19,609 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#16.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1352 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#36.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#36.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2905 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t381850\t381850\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:19,821 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#53.avinput\n",
      "NOTICE: Processing next batch with 11391 unique variants in 11391 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1354 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2707 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t831321\t831321\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#26.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#26.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:49:19,980 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#70.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1452 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1434 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 11093 unique variants in 11093 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1439 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#60.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#60.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 1468 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1948 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t867207\t867207\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:20,214 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#7.avinput\n",
      "NOTICE: Processing next batch with 11108 unique variants in 11108 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2228 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t463155\t463155\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:20,254 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#52.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1479 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2187 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2173210\t2173210\tA\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#21.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#21.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:49:20,321 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#5.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2034 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2171821\t2171821\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:49:20,377 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#14.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#11.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#11.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2127 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t340523\t340523\tA\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10917 unique variants in 10917 input lines\n",
      "2025-04-23 21:49:20,432 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#18.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#3.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#3.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12858 unique variants in 12858 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#14.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#14.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#10.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#10.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 11048 unique variants in 11048 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10060 unique variants in 10060 input lines\n",
      "NOTICE: Processing next batch with 10035 unique variants in 10035 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1480 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1497 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2137 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2207706\t2207706\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:22,129 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#36.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1546 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2082 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t917635\t917635\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:22,239 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#26.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#44.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#44.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10152 unique variants in 10152 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1958 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t791377\t791377\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#7.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#7.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:49:22,390 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#60.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1442 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1457 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1471 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Processing next batch with 10280 unique variants in 10280 input lines\n",
      "Done with 1497 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1770 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t308205\t308205\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#87.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#87.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1859 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1964668\t1964668\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:22,549 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#10.avinput\n",
      "2025-04-23 21:49:22,549 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#14.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2132 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2169458\t2169458\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:22,605 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#21.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1949 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1318089\t1318089\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:22,660 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#3.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#1.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#1.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6930_8#10.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6930_8#10.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 13394 unique variants in 13394 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#17.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#17.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1520 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11798 unique variants in 11798 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#21.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#21.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 12296 unique variants in 12296 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12475 unique variants in 12475 input lines\n",
      "NOTICE: Processing next batch with 12242 unique variants in 12242 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2531 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t518420\t518420\tG\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:22,924 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#11.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#31.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#31.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11233 unique variants in 11233 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1442 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1928 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t665291\t665291\tG\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:24,275 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#44.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1487 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#21.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#21.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2021 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1376626\t1376626\tG\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:24,507 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#7.avinput\n",
      "NOTICE: Processing next batch with 10651 unique variants in 10651 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#17.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#17.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10968 unique variants in 10968 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1533 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1489 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2728 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t184026\t184026\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:49:25,123 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#1.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1510 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1486 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1513 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1489 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3648 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2021634\t2021634\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:25,241 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#21.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#13.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#13.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3579 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1112767\t1112767\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2262 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1138233\t1138233\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3552 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1618518\t1618518\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:25,336 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#17.avinput\n",
      "2025-04-23 21:49:25,336 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#31.avinput\n",
      "2025-04-23 21:49:25,348 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6930_8#10.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2625 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t87829\t87829\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#2.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#2.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 10503 unique variants in 10503 input lines\n",
      "2025-04-23 21:49:25,398 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#87.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#8.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#8.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#17.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#17.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 9816 unique variants in 9816 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_4#12.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_4#12.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#22.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#22.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 7919 unique variants in 7919 input lines\n",
      "NOTICE: Processing next batch with 7632 unique variants in 7632 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12918 unique variants in 12918 input lines\n",
      "NOTICE: Processing next batch with 11587 unique variants in 11587 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1500 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2022 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1312322\t1312322\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:26,626 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#21.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1488 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#10.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#10.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1899 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1558964\t1558964\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:26,850 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#17.avinput\n",
      "NOTICE: Processing next batch with 7690 unique variants in 7690 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1358 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#56.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#56.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 1344 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2730 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t903647\t903647\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:27,071 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#17.avinput\n",
      "NOTICE: Processing next batch with 11194 unique variants in 11194 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3064 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1737361\t1737361\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:27,109 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#8.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_4#6.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_4#6.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#8.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#8.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 1480 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Processing next batch with 10426 unique variants in 10426 input lines\n",
      "Done with 1441 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 12493 unique variants in 12493 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2631 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t601847\t601847\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:27,457 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#13.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1954 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t944182\t944182\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:27,495 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#2.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#6.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#6.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#80.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#80.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7993 unique variants in 7993 input lines\n",
      "NOTICE: Processing next batch with 12351 unique variants in 12351 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1513 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1472 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2533 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1848327\t1848327\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:28,048 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#22.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#74.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#74.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2605 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1354319\t1354319\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:49:28,192 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_4#12.avinput\n",
      "NOTICE: Processing next batch with 10464 unique variants in 10464 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#32.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#32.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1356 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10965 unique variants in 10965 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2924 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2108604\t2108604\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:28,445 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#10.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#22.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#22.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7941 unique variants in 7941 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1354 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1497 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1484 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3187 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1012371\t1012371\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:29,226 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#6.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2124 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1817293\t1817293\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:29,333 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_4#6.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2328 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1897339\t1897339\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#64.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#64.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:49:29,361 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#56.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10468 unique variants in 10468 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#19.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#19.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#65.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#65.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12883 unique variants in 12883 input lines\n",
      "NOTICE: Processing next batch with 10570 unique variants in 10570 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1502 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3132 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2066397\t2066397\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:29,899 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#8.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1439 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#79.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#79.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2000 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t744176\t744176\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:49:30,130 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#80.avinput\n",
      "NOTICE: Processing next batch with 11206 unique variants in 11206 input lines\n",
      "Done with 1354 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1437 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3050 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t461473\t461473\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#20.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#20.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:49:30,280 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#22.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2229 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1535327\t1535327\tC\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:30,330 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#74.avinput\n",
      "NOTICE: Processing next batch with 10466 unique variants in 10466 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#8.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#8.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6930_8#7.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6930_8#7.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 1491 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10723 unique variants in 10723 input lines\n",
      "NOTICE: Processing next batch with 11131 unique variants in 11131 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2318 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t70263\t70263\tT\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:30,703 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#32.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6930_8#19.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6930_8#19.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11192 unique variants in 11192 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1478 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1482 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2288 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1955686\t1955686\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:31,583 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#64.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#55.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#55.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2128 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t85438\t85438\tA\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:31,724 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#65.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10603 unique variants in 10603 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#4.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#4.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9847 unique variants in 9847 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1465 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2600 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2176353\t2176353\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1515 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:49:32,233 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#19.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1471 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2168 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t287134\t287134\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#14.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#14.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:49:32,378 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#79.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1973 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1413033\t1413033\tT\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:32,446 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#20.avinput\n",
      "NOTICE: Processing next batch with 11756 unique variants in 11756 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1484 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#11.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#11.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#79.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#79.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10828 unique variants in 10828 input lines\n",
      "NOTICE: Processing next batch with 11311 unique variants in 11311 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2082 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1977062\t1977062\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:32,742 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#8.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1442 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#21.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#21.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2529 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t618169\t618169\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:32,988 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6930_8#7.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1428 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 21781 unique variants in 21781 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#6.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#6.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1862 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t52947\t52947\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:33,202 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6930_8#19.avinput\n",
      "NOTICE: Processing next batch with 11621 unique variants in 11621 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#18.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#18.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12466 unique variants in 12466 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1513 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1453 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2169 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2187698\t2187698\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:33,884 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#55.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2158 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t713543\t713543\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:33,953 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#4.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#40.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#40.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#3.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#3.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 8978 unique variants in 8978 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10150 unique variants in 10150 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1525 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1547 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2295 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1805521\t1805521\tA\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1489 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:49:34,785 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#11.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2322 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1981\t1981\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#12.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#12.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2622 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1997353\t1997353\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:34,922 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#79.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:49:34,936 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#14.avinput\n",
      "NOTICE: Processing next batch with 9919 unique variants in 9919 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#16.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#16.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#20.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#20.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10308 unique variants in 10308 input lines\n",
      "NOTICE: Processing next batch with 11548 unique variants in 11548 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1498 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2530 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t917635\t917635\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:35,535 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#6.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#30.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#30.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 1567 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1351 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10187 unique variants in 10187 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3117 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1627355\t1627355\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:35,866 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#40.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2126 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1942265\t1942265\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:35,900 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#18.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#1.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#1.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1471 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#9.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#9.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11002 unique variants in 11002 input lines\n",
      "NOTICE: Processing next batch with 10937 unique variants in 10937 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2201 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1857462\t1857462\tA\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:36,172 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#3.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#81.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#81.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12097 unique variants in 12097 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1445 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1951 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t650226\t650226\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:36,957 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#12.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#7.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#7.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1631 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 1490 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 13041 unique variants in 13041 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1442 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2025 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t818964\t818964\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:37,336 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#16.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#91.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#91.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2602 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1929186\t1929186\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:49:37,570 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#21.avinput\n",
      "NOTICE: Processing next batch with 11755 unique variants in 11755 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1861 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2184228\t2184228\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:37,695 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#20.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#42.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#42.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1455 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#23.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#23.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 11082 unique variants in 11082 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1947 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2133671\t2133671\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:37,951 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#30.avinput\n",
      "NOTICE: Processing next batch with 11124 unique variants in 11124 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#16.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#16.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7900 unique variants in 7900 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1513 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1534 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2127 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1292459\t1292459\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:38,420 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#1.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2343 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t713238\t713238\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:38,466 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#9.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_4#13.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_4#13.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#9.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#9.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11174 unique variants in 11174 input lines\n",
      "NOTICE: Processing next batch with 10409 unique variants in 10409 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1509 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3122 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t388865\t388865\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:39,022 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#81.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_4#7.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_4#7.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7958 unique variants in 7958 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1356 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1497 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1546 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2742 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2155092\t2155092\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:39,987 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#16.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1519 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1523 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3095 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t867207\t867207\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#11.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#11.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:49:40,119 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#7.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2318 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t189076\t189076\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:49:40,149 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#91.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2218 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1360494\t1360494\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:40,204 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#42.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2429 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t853902\t853902\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 11616 unique variants in 11616 input lines\n",
      "2025-04-23 21:49:40,229 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#23.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#57.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#57.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6930_8#6.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6930_8#6.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6930_8#18.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6930_8#18.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 10094 unique variants in 10094 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#64.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#64.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9903 unique variants in 9903 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10317 unique variants in 10317 input lines\n",
      "NOTICE: Processing next batch with 10715 unique variants in 10715 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1513 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1527 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1362 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3117 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t379561\t379561\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2187 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t829600\t829600\tG\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2507 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1069199\t1069199\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:40,882 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#9.avinput\n",
      "2025-04-23 21:49:40,882 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_4#7.avinput\n",
      "2025-04-23 21:49:40,891 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_4#13.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#65.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#65.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#18.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#18.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#23.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#23.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10173 unique variants in 10173 input lines\n",
      "NOTICE: Processing next batch with 11596 unique variants in 11596 input lines\n",
      "NOTICE: Processing next batch with 11601 unique variants in 11601 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1444 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1442 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1472 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1948 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t647524\t647524\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:49:42,338 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#57.avinput\n",
      "Done with 1453 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1473 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2365 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t126609\t126609\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1854 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1934051\t1934051\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:42,469 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6930_8#18.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#75.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#75.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1725 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2152929\t2152929\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:42,493 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6930_8#6.avinput\n",
      "2025-04-23 21:49:42,493 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#11.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2128 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t409062\t409062\tA\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:42,535 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#64.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#33.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#33.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 11577 unique variants in 11577 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#78.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#78.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#21.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#21.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#78.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#78.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 7872 unique variants in 7872 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10525 unique variants in 10525 input lines\n",
      "NOTICE: Processing next batch with 11520 unique variants in 11520 input lines\n",
      "NOTICE: Processing next batch with 11479 unique variants in 11479 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1472 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2195 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t920990\t920990\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:43,013 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#18.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1437 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#20.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#20.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1435 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11429 unique variants in 11429 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1854 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2114513\t2114513\tG\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1853 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2166546\t2166546\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:43,396 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#65.avinput\n",
      "2025-04-23 21:49:43,415 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#23.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#68.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#68.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#15.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#15.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10957 unique variants in 10957 input lines\n",
      "NOTICE: Processing next batch with 11078 unique variants in 11078 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1346 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3042 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2131324\t2131324\tG\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:44,363 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#33.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#10.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#10.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9570 unique variants in 9570 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1486 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2614 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t620071\t620071\tT\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1476 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:49:44,934 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#78.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1431 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1440 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#5.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#5.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2000 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t96752\t96752\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:49:45,095 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#78.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1857 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1376981\t1376981\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 11517 unique variants in 11517 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1968 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1937583\t1937583\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:45,168 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#75.avinput\n",
      "2025-04-23 21:49:45,203 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#21.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#19.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#19.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#41.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#41.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#7.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#7.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10380 unique variants in 10380 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Processing next batch with 9941 unique variants in 9941 input lines\n",
      "Done with 1479 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 12287 unique variants in 12287 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2066 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t520351\t520351\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:45,630 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#20.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#54.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#54.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 1474 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1512 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 8504 unique variants in 8504 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2129 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1752393\t1752393\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:45,948 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#68.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2241 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1766172\t1766172\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:46,014 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#15.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#85.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#85.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#95.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#95.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9815 unique variants in 9815 input lines\n",
      "NOTICE: Processing next batch with 10564 unique variants in 10564 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1432 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1947 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t476360\t476360\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:46,467 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#10.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#62.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#62.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 21171 unique variants in 21171 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1515 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1436 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2186 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t739858\t739858\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1432 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:49:47,412 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#19.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1993 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t463155\t463155\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:47,472 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#41.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1488 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1355 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#72.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#72.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1901 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1114418\t1114418\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#88.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#88.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:49:47,605 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#5.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12036 unique variants in 12036 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3582 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1935993\t1935993\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3062 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1335865\t1335865\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 7878 unique variants in 7878 input lines\n",
      "2025-04-23 21:49:47,682 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#7.avinput\n",
      "2025-04-23 21:49:47,686 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#54.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#18.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#18.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#23.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#23.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#17.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#17.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 10934 unique variants in 10934 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1441 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 12216 unique variants in 12216 input lines\n",
      "NOTICE: Processing next batch with 11469 unique variants in 11469 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1887 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t542816\t542816\tA\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:48,071 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#85.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1469 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#4.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#4.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10088 unique variants in 10088 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1997 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1750495\t1750495\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:48,353 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#95.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#52.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#52.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10178 unique variants in 10178 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1345 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3042 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1011734\t1011734\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:49,245 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#88.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#6.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#6.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10369 unique variants in 10369 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1487 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1483 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2325 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t868466\t868466\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:49,980 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#72.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2127 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1403944\t1403944\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1538 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:49:50,023 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#18.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1494 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#15.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#15.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#53.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#53.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2664 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1615724\t1615724\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1447 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:49:50,191 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#17.avinput\n",
      "NOTICE: Processing next batch with 10449 unique variants in 10449 input lines\n",
      "NOTICE: Processing next batch with 11555 unique variants in 11555 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 10445 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1766172\t1766172\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:50,306 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#23.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#39.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#39.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1771 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1945001\t1945001\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:50,349 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#4.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#29.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#29.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 12337 unique variants in 12337 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_6#6.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_6#6.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 1478 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10026 unique variants in 10026 input lines\n",
      "NOTICE: Processing next batch with 10351 unique variants in 10351 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1918 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t636388\t636388\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:50,625 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#52.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#4.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#4.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 14794 unique variants in 14794 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1650 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2337 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t670949\t670949\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:49:51,330 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#62.avinput\n",
      "Done with 1467 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#8.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#8.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2220 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1899953\t1899953\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:51,497 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#6.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12645 unique variants in 12645 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#85.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#85.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10602 unique variants in 10602 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1486 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1449 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1913 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1661635\t1661635\tG\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1765 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t920262\t920262\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:49:52,401 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_6#6.avinput\n",
      "2025-04-23 21:49:52,402 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#15.avinput\n",
      "Done with 1437 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1521 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#13.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#13.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#27.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#27.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1988 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t325183\t325183\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:49:52,578 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#29.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1500 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 11242 unique variants in 11242 input lines\n",
      "NOTICE: Processing next batch with 10560 unique variants in 10560 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2549 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t638603\t638603\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#61.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#61.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:49:52,705 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#53.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3409 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t793946\t793946\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:52,793 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#39.avinput\n",
      "NOTICE: Processing next batch with 11769 unique variants in 11769 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#37.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#37.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#19.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#19.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 11268 unique variants in 11268 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7843 unique variants in 7843 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1537 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2803 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2179612\t2179612\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:53,606 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#4.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1438 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#10.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#10.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10376 unique variants in 10376 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2209 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t744979\t744979\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:49:53,864 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#85.avinput\n",
      "Done with 1527 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#15.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#15.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2030 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1359713\t1359713\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 10580 unique variants in 10580 input lines\n",
      "2025-04-23 21:49:54,105 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#8.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#2.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#2.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7875 unique variants in 7875 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1338 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3046 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t343419\t343419\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:54,542 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#19.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1500 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1507 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#48.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#48.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2020 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1596764\t1596764\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:49:54,743 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#27.avinput\n",
      "NOTICE: Processing next batch with 11778 unique variants in 11778 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2201 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1880176\t1880176\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:54,856 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#13.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#20.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#20.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1501 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10553 unique variants in 10553 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1481 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#4.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#4.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2445 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t135077\t135077\tT\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 11426 unique variants in 11426 input lines\n",
      "2025-04-23 21:49:55,124 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#61.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2126 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t84919\t84919\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:55,151 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#37.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#20.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#20.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#16.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#16.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7787 unique variants in 7787 input lines\n",
      "NOTICE: Processing next batch with 11082 unique variants in 11082 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1343 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1464 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3050 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2179612\t2179612\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:55,923 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#2.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2085 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1863922\t1863922\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:55,961 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#10.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6930_8#11.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6930_8#11.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#11.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#11.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1476 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10086 unique variants in 10086 input lines\n",
      "NOTICE: Processing next batch with 10351 unique variants in 10351 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2408 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t701835\t701835\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:56,249 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#15.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#86.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#86.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10190 unique variants in 10190 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1362 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1468 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3106 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t81806\t81806\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:56,894 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#20.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2376 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t305944\t305944\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#96.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#96.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:49:57,028 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#20.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12644 unique variants in 12644 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1437 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1501 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#6.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#6.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7836 unique variants in 7836 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2036 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2221097\t2221097\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1872 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t810566\t810566\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:57,333 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#48.avinput\n",
      "2025-04-23 21:49:57,362 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#4.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1492 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#40.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#40.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#5.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#5.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10612 unique variants in 10612 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2326 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t253028\t253028\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 11234 unique variants in 11234 input lines\n",
      "2025-04-23 21:49:57,624 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#16.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#93.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#93.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11210 unique variants in 11210 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1486 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1458 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2523 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t93412\t93412\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:58,128 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#11.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1761 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t579911\t579911\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:58,238 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6930_8#11.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#83.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#83.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1453 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 9914 unique variants in 9914 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#69.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#69.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1950 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t300131\t300131\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:58,451 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#86.avinput\n",
      "NOTICE: Processing next batch with 10692 unique variants in 10692 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#79.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#79.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7823 unique variants in 7823 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1367 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3013 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t73379\t73379\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:58,823 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#6.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#55.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#55.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 8101 unique variants in 8101 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1526 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1472 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2025 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1875706\t1875706\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:49:59,556 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#96.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2358 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1618309\t1618309\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#13.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#13.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:49:59,704 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#40.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1432 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10125 unique variants in 10125 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_4#11.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_4#11.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1856 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1471443\t1471443\tA\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1482 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:49:59,903 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#5.avinput\n",
      "NOTICE: Processing next batch with 11981 unique variants in 11981 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2309 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1481852\t1481852\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#14.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#14.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1349 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:00,061 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#93.avinput\n",
      "Done with 1442 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10431 unique variants in 10431 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2951 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2084151\t2084151\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:00,169 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#79.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#21.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#21.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1956 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1946776\t1946776\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:00,214 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#83.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#10.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#10.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 12732 unique variants in 12732 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1498 sequences\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#58.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#58.avinput.exonic_variant_function\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10841 unique variants in 10841 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1361 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10294 unique variants in 10294 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2939 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1069564\t1069564\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2357 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1670793\t1670793\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:00,552 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#55.avinput\n",
      "2025-04-23 21:50:00,553 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#69.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#59.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#59.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#49.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#49.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7570 unique variants in 7570 input lines\n",
      "NOTICE: Processing next batch with 10493 unique variants in 10493 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1451 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1947 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1766172\t1766172\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:01,665 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#13.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#1.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#1.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12217 unique variants in 12217 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1472 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1349 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1505 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1969 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t292381\t292381\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:02,245 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#14.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1444 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2704 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t721128\t721128\tG\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:02,295 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#49.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2334 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1428032\t1428032\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1507 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:02,343 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_4#11.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#18.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#18.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2230 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1338810\t1338810\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:50:02,411 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#58.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#3.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#3.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#32.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#32.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 10344 unique variants in 10344 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2362 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t915768\t915768\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10489 unique variants in 10489 input lines\n",
      "2025-04-23 21:50:02,513 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#10.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#22.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#22.avinput.exonic_variant_function\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 1497 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10348 unique variants in 10348 input lines\n",
      "NOTICE: Processing next batch with 10054 unique variants in 10054 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#14.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#14.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9654 unique variants in 9654 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2293 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t133586\t133586\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1474 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:02,796 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#21.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#5.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#5.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1970 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t896172\t896172\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:50:02,964 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#59.avinput\n",
      "NOTICE: Processing next batch with 10417 unique variants in 10417 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#9.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#9.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7849 unique variants in 7849 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1493 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1491 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1476 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 10447 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1677831\t1677831\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:04,387 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#1.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2023 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1813278\t1813278\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1451 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:04,448 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#18.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1448 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1976 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1394217\t1394217\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#43.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#43.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:50:04,520 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#3.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1457 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#7.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#7.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1976 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2186010\t2186010\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1368 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:04,597 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#22.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2041 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1394068\t1394068\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11525 unique variants in 11525 input lines\n",
      "2025-04-23 21:50:04,629 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#14.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#12.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#12.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1833 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t885726\t885726\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3018 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1112767\t1112767\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 13545 unique variants in 13545 input lines\n",
      "2025-04-23 21:50:04,708 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#32.avinput\n",
      "2025-04-23 21:50:04,713 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#9.avinput\n",
      "NOTICE: Processing next batch with 7865 unique variants in 7865 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#17.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#17.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#22.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#22.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1469 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#56.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#56.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#23.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#23.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 12305 unique variants in 12305 input lines\n",
      "NOTICE: Processing next batch with 12307 unique variants in 12307 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11162 unique variants in 11162 input lines\n",
      "NOTICE: Processing next batch with 11273 unique variants in 11273 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1981 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1774832\t1774832\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:05,066 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#5.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#31.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#31.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12777 unique variants in 12777 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1343 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3045 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1837412\t1837412\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:06,396 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#12.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#67.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#67.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7722 unique variants in 7722 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1454 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1493 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1893 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t730723\t730723\tA\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1496 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:06,984 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#43.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1431 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2225 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t618169\t618169\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:07,035 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#56.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#21.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#21.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1852 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1754148\t1754148\tA\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2878 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2072272\t2072272\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1496 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:07,163 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#17.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#66.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#66.avinput.exonic_variant_function\n",
      "2025-04-23 21:50:07,165 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#23.avinput\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10190 unique variants in 10190 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Processing next batch with 10812 unique variants in 10812 input lines\n",
      "Done with 1528 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#51.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#51.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#17.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#17.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2881 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t711690\t711690\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:50:07,349 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#22.avinput\n",
      "NOTICE: Processing next batch with 10937 unique variants in 10937 input lines\n",
      "NOTICE: Processing next batch with 13835 unique variants in 13835 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#19.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#19.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2866 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1571075\t1571075\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:50:07,495 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#7.avinput\n",
      "NOTICE: Processing next batch with 10815 unique variants in 10815 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#4.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#4.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1564 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 11020 unique variants in 11020 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2108 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t40595\t40595\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:07,915 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#31.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1353 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#50.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#50.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3062 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t608248\t608248\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 8405 unique variants in 8405 input lines\n",
      "2025-04-23 21:50:08,165 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#67.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#6.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#6.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10074 unique variants in 10074 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1459 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1528 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1934 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1066116\t1066116\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:09,449 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#21.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1462 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2297 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t342164\t342164\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#15.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#15.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:50:09,600 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#66.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Processing next batch with 13540 unique variants in 13540 input lines\n",
      "Done with 1531 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1968 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1009080\t1009080\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1473 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:09,766 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#17.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#11.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#11.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1354 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11352 unique variants in 11352 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#87.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#87.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1966 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t46815\t46815\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3225 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1067146\t1067146\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:50:09,902 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#19.avinput\n",
      "2025-04-23 21:50:09,908 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#50.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2122 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1593460\t1593460\tA\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:09,954 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#4.avinput\n",
      "NOTICE: Processing next batch with 7727 unique variants in 7727 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_6#4.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_6#4.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#6.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#6.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#70.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#70.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 7839 unique variants in 7839 input lines\n",
      "NOTICE: Processing next batch with 11596 unique variants in 11596 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11993 unique variants in 11993 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1495 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1464 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3565 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t740860\t740860\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:10,621 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#51.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1917 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1959517\t1959517\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:10,731 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#6.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#2.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#2.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 8076 unique variants in 8076 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#60.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#60.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12610 unique variants in 12610 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1359 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2919 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t32959\t32959\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:11,599 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#87.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1350 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2978 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1155847\t1155847\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:11,734 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#6.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#87.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#87.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11990 unique variants in 11990 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#21.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#21.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7832 unique variants in 7832 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1434 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1895 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1690189\t1690189\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:12,239 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#11.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1435 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1354 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#14.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#14.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2977 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2089789\t2089789\tT\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:12,482 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#2.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1484 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1489 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 11492 unique variants in 11492 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1853 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t802867\t802867\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:12,547 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_6#4.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_4#18.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_4#18.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#2.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#2.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2335 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t917764\t917764\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:12,692 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#70.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3579 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1581470\t1581470\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:12,722 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#15.avinput\n",
      "NOTICE: Processing next batch with 10867 unique variants in 10867 input lines\n",
      "NOTICE: Processing next batch with 11828 unique variants in 11828 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#14.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#14.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#22.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#22.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11201 unique variants in 11201 input lines\n",
      "NOTICE: Processing next batch with 20276 unique variants in 20276 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1527 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2023 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1099035\t1099035\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:13,375 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#60.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1365 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6930_8#13.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6930_8#13.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3013 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t395332\t395332\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:50:13,531 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#21.avinput\n",
      "NOTICE: Processing next batch with 9758 unique variants in 9758 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#73.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#73.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9790 unique variants in 9790 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1442 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2000 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t798704\t798704\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:14,337 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#87.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#47.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#47.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1438 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 15167 unique variants in 15167 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1995 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2185727\t2185727\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:14,713 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#14.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1498 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#94.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#94.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2516 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t208083\t208083\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:14,888 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_4#18.avinput\n",
      "NOTICE: Processing next batch with 12488 unique variants in 12488 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1510 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#4.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#4.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1547 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11792 unique variants in 11792 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3548 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1293735\t1293735\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:15,177 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#2.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2360 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t976669\t976669\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:15,207 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#14.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#84.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#84.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#28.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#28.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1457 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 11188 unique variants in 11188 input lines\n",
      "NOTICE: Processing next batch with 11595 unique variants in 11595 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1435 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2018 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t573278\t573278\tT\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:15,545 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6930_8#13.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1933 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t11439\t11439\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#38.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#38.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:50:15,678 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#73.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10763 unique variants in 10763 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#13.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#13.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10747 unique variants in 10747 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1629 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2332 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1896617\t1896617\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:17,178 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#22.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1549 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1528 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#22.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#22.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1480 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1522 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 8278 unique variants in 8278 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2619 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t263874\t263874\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:17,441 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#4.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2496 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1845476\t1845476\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:17,515 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#94.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2002 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2164924\t2164924\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:50:17,537 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#84.avinput\n",
      "Done with 1443 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#17.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#17.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2166 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t347052\t347052\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:17,639 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#47.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#12.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#12.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#35.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#35.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 12228 unique variants in 12228 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1854 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t693572\t693572\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:17,726 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#28.avinput\n",
      "NOTICE: Processing next batch with 8061 unique variants in 8061 input lines\n",
      "NOTICE: Processing next batch with 10289 unique variants in 10289 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#73.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#73.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1478 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#63.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#63.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 12290 unique variants in 12290 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7835 unique variants in 7835 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2315 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1980451\t1980451\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1517 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:17,994 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#38.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2428 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t152905\t152905\tG\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#25.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#25.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:50:18,154 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#13.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12320 unique variants in 12320 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#6.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#6.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9999 unique variants in 9999 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1357 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3182 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t767373\t767373\tA\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:19,029 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#22.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#86.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#86.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1344 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 11627 unique variants in 11627 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3099 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t178851\t178851\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:19,343 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#35.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1345 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2998 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1089159\t1089159\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#10.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#10.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:50:19,516 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#63.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12988 unique variants in 12988 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#7.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#7.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10568 unique variants in 10568 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1466 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1511 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2355 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1824314\t1824314\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:19,947 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#12.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1487 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2355 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t146953\t146953\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_6#5.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_6#5.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:50:20,064 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#17.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11035 unique variants in 11035 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3587 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1586067\t1586067\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#51.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#51.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:50:20,202 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#73.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#9.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#9.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 20315 unique variants in 20315 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1449 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 7584 unique variants in 7584 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1490 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1836 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1894332\t1894332\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:20,539 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#6.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3581 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2112570\t2112570\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#18.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#18.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:50:20,661 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#25.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 13041 unique variants in 13041 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#5.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#5.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7981 unique variants in 7981 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1535 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2360 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1698028\t1698028\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:21,473 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#86.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1476 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#50.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#50.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Processing next batch with 11703 unique variants in 11703 input lines\n",
      "Done with 1352 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2318 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1104518\t1104518\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:21,736 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#7.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2933 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t629077\t629077\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:21,837 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#9.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#16.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#16.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#14.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#14.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 11630 unique variants in 11630 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1485 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10168 unique variants in 10168 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2625 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t878598\t878598\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:22,212 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#10.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1492 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1361 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#7.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#7.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3096 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1331339\t1331339\tA\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2320 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2060879\t2060879\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:22,408 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#5.avinput\n",
      "2025-04-23 21:50:22,415 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_6#5.avinput\n",
      "NOTICE: Processing next batch with 7859 unique variants in 7859 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#20.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#20.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#15.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#15.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10154 unique variants in 10154 input lines\n",
      "NOTICE: Processing next batch with 9597 unique variants in 9597 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1473 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2616 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1795610\t1795610\tA\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:23,280 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#18.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#3.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#3.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9642 unique variants in 9642 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1335 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1441 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1469 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2969 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t163726\t163726\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:23,957 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#7.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1497 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1860 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t94008\t94008\tT\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:24,028 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#50.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_4#19.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_4#19.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2197 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t267835\t267835\tG\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:50:24,089 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#14.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2531 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1593460\t1593460\tA\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 10179 unique variants in 10179 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#48.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#48.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:50:24,166 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#16.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1634 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#86.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#86.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11052 unique variants in 11052 input lines\n",
      "NOTICE: Processing next batch with 7860 unique variants in 7860 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#61.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#61.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11653 unique variants in 11653 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2324 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t572178\t572178\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1356 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1476 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:24,551 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#51.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3327 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t865620\t865620\tG\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2201 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2050132\t2050132\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:24,670 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#20.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#3.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#3.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:50:24,697 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#15.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9834 unique variants in 9834 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#71.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#71.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#85.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#85.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10553 unique variants in 10553 input lines\n",
      "NOTICE: Processing next batch with 11671 unique variants in 11671 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1449 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2039 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1943182\t1943182\tC\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:25,474 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#3.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#5.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#5.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11794 unique variants in 11794 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1345 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3048 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1687671\t1687671\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:25,888 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#86.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#95.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#95.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1474 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10523 unique variants in 10523 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2196 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2000092\t2000092\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:26,194 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_4#19.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1492 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#12.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#12.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2418 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1548033\t1548033\tT\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:26,446 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#48.avinput\n",
      "NOTICE: Processing next batch with 11111 unique variants in 11111 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1442 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#39.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#39.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 1441 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10043 unique variants in 10043 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1854 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t868466\t868466\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1952 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1425202\t1425202\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:26,759 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#61.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:50:26,781 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#3.avinput\n",
      "Done with 1445 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#29.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#29.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6930_8#12.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6930_8#12.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2243 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1886574\t1886574\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:50:26,948 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#71.avinput\n",
      "NOTICE: Processing next batch with 7786 unique variants in 7786 input lines\n",
      "NOTICE: Processing next batch with 10132 unique variants in 10132 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1457 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#23.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#23.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7766 unique variants in 7766 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1871 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t207243\t207243\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:27,218 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#85.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#15.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#15.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10583 unique variants in 10583 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1547 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1494 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2624 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1748045\t1748045\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:28,073 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#5.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1912 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t121537\t121537\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#3.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#3.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:50:28,218 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#95.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11785 unique variants in 11785 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#7.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#7.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1356 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 13032 unique variants in 13032 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1447 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1357 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3096 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1752393\t1752393\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:28,595 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#29.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1949 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t620759\t620759\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2905 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2018498\t2018498\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:50:28,652 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#39.avinput\n",
      "Done with 1478 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:28,658 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#23.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#16.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#16.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#1.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#1.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#13.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#13.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2199 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t228641\t228641\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:50:28,836 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#12.avinput\n",
      "NOTICE: Processing next batch with 11750 unique variants in 11750 input lines\n",
      "NOTICE: Processing next batch with 10865 unique variants in 10865 input lines\n",
      "NOTICE: Processing next batch with 10345 unique variants in 10345 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1474 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#62.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#62.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2191 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1392741\t1392741\tG\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 11771 unique variants in 11771 input lines\n",
      "2025-04-23 21:50:29,105 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6930_8#12.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#24.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#24.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1488 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10945 unique variants in 10945 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2204 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t218441\t218441\tT\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:29,675 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#15.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#34.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#34.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10528 unique variants in 10528 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1449 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2274 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1801196\t1801196\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1480 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:30,986 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#3.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2296 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1862212\t1862212\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1484 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:31,120 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#1.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#72.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#72.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1502 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10665 unique variants in 10665 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_4#4.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_4#4.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 1493 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2627 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t649707\t649707\tT\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2362 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1882205\t1882205\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:31,362 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#7.avinput\n",
      "NOTICE: Processing next batch with 10128 unique variants in 10128 input lines\n",
      "2025-04-23 21:50:31,378 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#13.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#54.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#54.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3113 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t492257\t492257\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#12.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#12.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:50:31,508 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#16.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 8064 unique variants in 8064 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1516 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 12287 unique variants in 12287 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#20.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#20.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2672 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t533421\t533421\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 11185 unique variants in 11185 input lines\n",
      "2025-04-23 21:50:31,746 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#62.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1520 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_4#10.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_4#10.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7998 unique variants in 7998 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2119 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1004981\t1004981\tA\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:32,105 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#24.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1493 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#15.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#15.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2017 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1528916\t1528916\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:50:32,323 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#34.avinput\n",
      "NOTICE: Processing next batch with 11892 unique variants in 11892 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#82.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#82.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10218 unique variants in 10218 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1370 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1470 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3105 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1404568\t1404568\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:50:33,386 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#54.avinput\n",
      "Done with 1463 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2189 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t752652\t752652\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#92.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#92.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:50:33,533 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_4#4.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1914 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1884802\t1884802\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1356 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:33,580 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#72.avinput\n",
      "NOTICE: Processing next batch with 10056 unique variants in 10056 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#4.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#4.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3126 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1162251\t1162251\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:33,705 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_4#10.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#41.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#41.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10928 unique variants in 10928 input lines\n",
      "NOTICE: Processing next batch with 11708 unique variants in 11708 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#78.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#78.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1434 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 12036 unique variants in 12036 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1849 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1060607\t1060607\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:34,097 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#20.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1487 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#68.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#68.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 8274 unique variants in 8274 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3588 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1671501\t1671501\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:34,337 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#12.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#2.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#2.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1457 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 11529 unique variants in 11529 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1927 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t549796\t549796\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:34,668 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#82.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1543 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#19.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#19.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2642 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t123165\t123165\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 10828 unique variants in 10828 input lines\n",
      "2025-04-23 21:50:34,913 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#15.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#15.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#15.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10194 unique variants in 10194 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1444 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1882 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1737378\t1737378\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:35,684 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#92.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#23.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#23.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1475 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10964 unique variants in 10964 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1513 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2137 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t740193\t740193\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1346 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:36,045 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#4.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1512 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2302 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1639301\t1639301\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#33.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#33.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3091 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1020308\t1020308\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:36,165 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#41.avinput\n",
      "2025-04-23 21:50:36,172 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#68.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10943 unique variants in 10943 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2411 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1845454\t1845454\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#58.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#58.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:50:36,310 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#78.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#59.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#59.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#11.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#11.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 11587 unique variants in 11587 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 20295 unique variants in 20295 input lines\n",
      "NOTICE: Processing next batch with 11116 unique variants in 11116 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1533 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2386 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t147450\t147450\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1483 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:36,853 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#2.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2235 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1450326\t1450326\tG\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#49.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#49.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:50:36,989 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#19.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1474 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 12601 unique variants in 12601 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#57.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#57.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2197 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t45417\t45417\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:37,230 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#15.avinput\n",
      "NOTICE: Processing next batch with 11688 unique variants in 11688 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#42.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#42.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10596 unique variants in 10596 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1480 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1968 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2173575\t2173575\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:37,970 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#23.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#8.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#8.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1474 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10331 unique variants in 10331 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1961 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2163008\t2163008\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:38,372 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#33.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1485 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#4.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#4.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1503 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11621 unique variants in 11621 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2079 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1952061\t1952061\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2540 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1153867\t1153867\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:38,714 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#11.avinput\n",
      "2025-04-23 21:50:38,730 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#59.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#23.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#23.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#13.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#13.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12211 unique variants in 12211 input lines\n",
      "NOTICE: Processing next batch with 10545 unique variants in 10545 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1522 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1441 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1465 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2022 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2155092\t2155092\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:39,539 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#49.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1860 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1586067\t1586067\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:39,587 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#57.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 5710 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t740693\t740693\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:39,660 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#42.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#16.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#16.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#6.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#6.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9629 unique variants in 9629 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#22.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#22.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 11779 unique variants in 11779 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9793 unique variants in 9793 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1633 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1497 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2035 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2218027\t2218027\tA\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:40,311 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#8.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2326 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1739264\t1739264\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#67.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#67.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:50:40,495 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#58.avinput\n",
      "NOTICE: Processing next batch with 7852 unique variants in 7852 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#66.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#66.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1497 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10580 unique variants in 10580 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1496 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2529 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1690189\t1690189\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:40,949 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#4.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2021 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1293480\t1293480\tT\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#20.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#20.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:50:41,066 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#13.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7925 unique variants in 7925 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1490 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#76.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#76.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9993 unique variants in 9993 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3642 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t915768\t915768\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:41,361 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#23.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1448 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6930_8#5.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6930_8#5.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 8196 unique variants in 8196 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2033 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1737378\t1737378\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:41,596 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#16.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1438 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1954 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1048668\t1048668\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#52.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#52.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:50:41,764 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#22.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10166 unique variants in 10166 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#28.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#28.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1546 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10399 unique variants in 10399 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1345 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2627 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t158443\t158443\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:42,143 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#6.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3003 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t623335\t623335\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:42,192 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#67.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#38.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#38.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#3.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#3.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12978 unique variants in 12978 input lines\n",
      "NOTICE: Processing next batch with 10312 unique variants in 10312 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1499 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1351 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2013 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1837745\t1837745\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:42,727 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#66.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3064 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t525030\t525030\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:42,797 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#20.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#13.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#13.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#16.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#16.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12953 unique variants in 12953 input lines\n",
      "NOTICE: Processing next batch with 11061 unique variants in 11061 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1445 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1369 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2004 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2146998\t2146998\tT\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2966 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t509971\t509971\tT\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:43,233 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6930_8#5.avinput\n",
      "2025-04-23 21:50:43,239 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#76.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#1.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#1.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#47.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#47.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 8015 unique variants in 8015 input lines\n",
      "NOTICE: Processing next batch with 10944 unique variants in 10944 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1444 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1924 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1874946\t1874946\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:43,953 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#52.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1468 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#62.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#62.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11272 unique variants in 11272 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1969 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1664294\t1664294\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:44,207 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#28.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1470 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#35.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#35.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9951 unique variants in 9951 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2366 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t125561\t125561\tT\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:44,487 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#3.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#25.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#25.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1502 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 12421 unique variants in 12421 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1347 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2946 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1597070\t1597070\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:44,963 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#38.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2985 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1441594\t1441594\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:45,042 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#1.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#63.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#63.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1502 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#89.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#89.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11888 unique variants in 11888 input lines\n",
      "NOTICE: Processing next batch with 12642 unique variants in 12642 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2235 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1293017\t1293017\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1473 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:45,306 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#16.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1492 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#9.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#9.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2191 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t388134\t388134\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:50:45,465 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#47.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2604 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t932875\t932875\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:45,521 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#13.avinput\n",
      "NOTICE: Processing next batch with 11756 unique variants in 11756 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#23.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#23.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#19.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#19.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10590 unique variants in 10590 input lines\n",
      "NOTICE: Processing next batch with 11944 unique variants in 11944 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1435 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1449 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1854 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t782804\t782804\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:46,462 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#62.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2031 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t268250\t268250\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:46,502 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#35.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#20.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#20.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#10.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#10.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 8925 unique variants in 8925 input lines\n",
      "NOTICE: Processing next batch with 8148 unique variants in 8148 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1529 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2505 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2145694\t2145694\tA\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:47,170 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#25.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_4#15.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_4#15.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1442 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10702 unique variants in 10702 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1876 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t627368\t627368\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:47,562 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#63.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1532 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1498 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#17.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#17.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 1464 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2546 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t262704\t262704\tT\tC\thom\t.\t.>\n",
      "NOTICE: Processing next batch with 10873 unique variants in 10873 input lines\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3131 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2034145\t2034145\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 4029 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1949027\t1949027\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:47,856 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#89.avinput\n",
      "2025-04-23 21:50:47,876 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#9.avinput\n",
      "2025-04-23 21:50:47,877 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#23.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#51.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#51.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#1.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#1.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#19.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#19.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1489 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10366 unique variants in 10366 input lines\n",
      "NOTICE: Processing next batch with 10256 unique variants in 10256 input lines\n",
      "NOTICE: Processing next batch with 11257 unique variants in 11257 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1364 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2307 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t83851\t83851\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1359 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:48,256 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#19.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2946 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1962353\t1962353\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:48,340 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#10.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3165 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1126054\t1126054\tT\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#87.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#87.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:50:48,405 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#20.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#44.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#44.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10861 unique variants in 10861 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#36.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#36.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 10463 unique variants in 10463 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9899 unique variants in 9899 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1473 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2310 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1951977\t1951977\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:49,684 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_4#15.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6930_8#21.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6930_8#21.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1519 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11548 unique variants in 11548 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1465 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1432 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1460 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2150 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1837412\t1837412\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:50,026 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#17.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1964 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1596764\t1596764\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:50,131 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#51.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1864 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t173825\t173825\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#10.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#10.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2167 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1769981\t1769981\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:50,170 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#1.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:50:50,182 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#19.avinput\n",
      "NOTICE: Processing next batch with 10180 unique variants in 10180 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#26.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#26.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#7.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#7.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#18.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#18.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10268 unique variants in 10268 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1457 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1470 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 11604 unique variants in 11604 input lines\n",
      "NOTICE: Processing next batch with 10794 unique variants in 10794 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1512 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3814 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t739858\t739858\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1870 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1412888\t1412888\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:50,533 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#36.avinput\n",
      "2025-04-23 21:50:50,557 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#44.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2123 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t443339\t443339\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:50,657 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#87.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#21.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#21.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#5.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#5.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11134 unique variants in 11134 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#9.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#9.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 13309 unique variants in 13309 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 8155 unique variants in 8155 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1539 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1471 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2483 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t253418\t253418\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1450 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:52,320 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6930_8#21.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2206 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t323867\t323867\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:52,382 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#10.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#14.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#14.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1928 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t677304\t677304\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 1362 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:52,464 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#26.avinput\n",
      "NOTICE: Processing next batch with 7864 unique variants in 7864 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#22.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#22.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2949 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1616485\t1616485\tA\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1500 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:50:52,585 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#9.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#90.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#90.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 7557 unique variants in 7557 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1545 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#80.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#80.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 11540 unique variants in 11540 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2279 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1807111\t1807111\tG\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:52,757 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#18.avinput\n",
      "NOTICE: Processing next batch with 10220 unique variants in 10220 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1436 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2423 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1608650\t1608650\tG\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:52,850 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#7.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#77.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#77.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1859 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1737378\t1737378\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#43.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#43.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:50:52,993 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#21.avinput\n",
      "NOTICE: Processing next batch with 12244 unique variants in 12244 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11813 unique variants in 11813 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6930_8#9.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6930_8#9.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1522 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 20359 unique variants in 20359 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2389 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1770799\t1770799\tA\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:53,460 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#5.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#6.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#6.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10477 unique variants in 10477 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1341 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3048 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t915768\t915768\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:54,074 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#14.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1350 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6930_8#17.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6930_8#17.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2914 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1436093\t1436093\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:54,296 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#22.avinput\n",
      "NOTICE: Processing next batch with 11486 unique variants in 11486 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#10.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#10.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11077 unique variants in 11077 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1446 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1537 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1925 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1894332\t1894332\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:54,889 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#80.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2359 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2146273\t2146273\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:55,012 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#90.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#2.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#2.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12252 unique variants in 12252 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#77.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#77.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7875 unique variants in 7875 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1511 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1519 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2130 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t492320\t492320\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:55,382 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#43.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2476 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1809572\t1809572\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:55,499 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#77.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#31.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#31.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#21.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#21.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 10014 unique variants in 10014 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1461 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 11828 unique variants in 11828 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 5713 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2168305\t2168305\tA\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:55,895 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#6.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#67.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#67.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12038 unique variants in 12038 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1541 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1508 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2655 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1157345\t1157345\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:56,688 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6930_8#17.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#16.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#16.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 1339 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2537 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1826266\t1826266\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:56,849 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#10.avinput\n",
      "NOTICE: Processing next batch with 10961 unique variants in 10961 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2970 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t50647\t50647\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:56,916 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#77.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#4.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#4.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#13.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#13.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10041 unique variants in 10041 input lines\n",
      "NOTICE: Processing next batch with 9938 unique variants in 9938 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1634 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1517 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2333 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1472633\t1472633\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:50:57,517 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6930_8#9.avinput\n",
      "Done with 1431 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2013 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t366693\t366693\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:57,610 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#2.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_4#23.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_4#23.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2249 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t607939\t607939\tC\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 7561 unique variants in 7561 input lines\n",
      "2025-04-23 21:50:57,703 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#31.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#8.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#8.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7779 unique variants in 7779 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#19.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#19.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11129 unique variants in 11129 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1495 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2313 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1952061\t1952061\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:58,312 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#21.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#2.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#2.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 1550 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 11148 unique variants in 11148 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2489 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t670949\t670949\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:58,695 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#67.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#93.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#93.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1447 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Processing next batch with 11371 unique variants in 11371 input lines\n",
      "Done with 1450 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1505 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1761 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t636388\t636388\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:59,077 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#4.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1725 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1425819\t1425819\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:59,103 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#13.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1355 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2369 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t233088\t233088\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#15.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#15.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:50:59,197 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#16.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#83.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#83.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2933 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t869043\t869043\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:50:59,262 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_4#23.avinput\n",
      "NOTICE: Processing next batch with 8020 unique variants in 8020 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1355 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#11.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#11.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 11020 unique variants in 11020 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#2.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#2.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3098 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t36751\t36751\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:50:59,432 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#8.avinput\n",
      "NOTICE: Processing next batch with 10791 unique variants in 10791 input lines\n",
      "NOTICE: Processing next batch with 9607 unique variants in 9607 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#54.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#54.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10170 unique variants in 10170 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1487 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1863 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1894358\t1894358\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:00,142 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#19.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#13.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#13.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10595 unique variants in 10595 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1502 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1371 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2264 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t109403\t109403\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:00,811 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#2.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3154 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1104518\t1104518\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:00,903 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#15.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#55.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#55.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1486 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#19.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#19.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10105 unique variants in 10105 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2397 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1649852\t1649852\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 9798 unique variants in 9798 input lines\n",
      "2025-04-23 21:51:01,148 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#93.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_4#8.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_4#8.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Processing next batch with 11114 unique variants in 11114 input lines\n",
      "Done with 1533 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1489 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1353 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1955 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t3397\t3397\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:01,550 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#11.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2320 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1649852\t1649852\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3322 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t213392\t213392\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1475 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:51:01,630 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#83.avinput\n",
      "2025-04-23 21:51:01,638 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#2.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#10.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#10.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#58.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#58.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 9900 unique variants in 9900 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#20.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#20.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2185 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1018583\t1018583\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:51:01,812 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#54.avinput\n",
      "NOTICE: Processing next batch with 10117 unique variants in 10117 input lines\n",
      "NOTICE: Processing next batch with 10512 unique variants in 10512 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#93.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#93.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10151 unique variants in 10151 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1441 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2208 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t956755\t956755\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:02,641 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#13.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#83.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#83.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10893 unique variants in 10893 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1449 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1439 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1951 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1942265\t1942265\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:03,100 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#55.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1950 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2103098\t2103098\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:03,217 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#19.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#6.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#6.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11284 unique variants in 11284 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#74.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#74.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12115 unique variants in 12115 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1520 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1452 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1443 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2436 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t210298\t210298\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:03,781 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_4#8.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2024 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1042051\t1042051\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:03,859 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#10.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1922 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1528969\t1528969\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:03,905 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#58.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#64.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#64.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#8.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#8.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 8151 unique variants in 8151 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#7.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#7.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1476 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1474 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 13072 unique variants in 13072 input lines\n",
      "NOTICE: Processing next batch with 10769 unique variants in 10769 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2187 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2218027\t2218027\tA\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2408 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t304042\t304042\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:04,261 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#20.avinput\n",
      "2025-04-23 21:51:04,261 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#93.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#11.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#11.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6930_8#8.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6930_8#8.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 8148 unique variants in 8148 input lines\n",
      "NOTICE: Processing next batch with 11901 unique variants in 11901 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1475 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2130 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t340523\t340523\tA\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:05,038 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#83.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6930_8#16.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6930_8#16.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9731 unique variants in 9731 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1433 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1849 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t804470\t804470\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1359 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:51:05,548 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#6.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3122 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1067146\t1067146\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:05,667 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#64.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#23.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#23.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1443 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 7560 unique variants in 7560 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#16.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#16.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1874 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t366600\t366600\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:51:05,912 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#74.avinput\n",
      "Done with 1361 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 13028 unique variants in 13028 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2947 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t647524\t647524\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:06,030 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6930_8#8.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#42.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#42.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1523 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#76.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#76.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 11170 unique variants in 11170 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2294 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t145175\t145175\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:06,246 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#7.avinput\n",
      "NOTICE: Processing next batch with 10144 unique variants in 10144 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#81.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#81.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1494 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 12021 unique variants in 12021 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1490 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3096 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2189433\t2189433\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:06,728 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#8.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2185 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2178264\t2178264\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:06,811 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#11.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#91.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#91.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#1.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#1.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 10324 unique variants in 10324 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1432 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 12616 unique variants in 12616 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1351 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1991 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1560320\t1560320\tT\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:07,184 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6930_8#16.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2917 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1298867\t1298867\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:07,244 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#23.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#20.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#20.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#66.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#66.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11034 unique variants in 11034 input lines\n",
      "NOTICE: Processing next batch with 8139 unique variants in 8139 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1539 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1432 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1485 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2325 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1397418\t1397418\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2224 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t305084\t305084\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:08,441 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#42.avinput\n",
      "2025-04-23 21:51:08,452 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#76.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2627 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1108954\t1108954\tT\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:08,499 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#16.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#30.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#30.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#76.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#76.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#9.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#9.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10737 unique variants in 10737 input lines\n",
      "NOTICE: Processing next batch with 11744 unique variants in 11744 input lines\n",
      "NOTICE: Processing next batch with 7836 unique variants in 7836 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1484 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1457 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1360 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1837 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2173051\t2173051\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2337 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t778275\t778275\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:08,975 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#91.avinput\n",
      "2025-04-23 21:51:08,985 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#81.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2947 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t52947\t52947\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:09,012 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#66.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#17.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#17.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_4#22.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_4#22.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#12.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#12.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11147 unique variants in 11147 input lines\n",
      "NOTICE: Processing next batch with 10704 unique variants in 10704 input lines\n",
      "NOTICE: Processing next batch with 10700 unique variants in 10700 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1505 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2331 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1562092\t1562092\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:09,593 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#1.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1494 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#5.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#5.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2278 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1156234\t1156234\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:09,783 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#20.avinput\n",
      "NOTICE: Processing next batch with 7696 unique variants in 7696 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#3.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#3.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11241 unique variants in 11241 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1371 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3125 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1621668\t1621668\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:10,328 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#9.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#3.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#3.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10688 unique variants in 10688 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1478 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1974 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t255267\t255267\tA\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:10,820 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#30.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1508 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#10.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#10.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2674 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t290369\t290369\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:11,029 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#76.avinput\n",
      "NOTICE: Processing next batch with 10292 unique variants in 10292 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#1.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#1.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7976 unique variants in 7976 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1474 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1354 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1514 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2127 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t940421\t940421\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1480 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:51:11,395 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_4#22.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2910 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t184026\t184026\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:11,434 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#5.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2265 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t363106\t363106\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2325 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t522758\t522758\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#12.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#12.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:51:11,536 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#17.avinput\n",
      "2025-04-23 21:51:11,542 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#12.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#54.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#54.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10552 unique variants in 10552 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#55.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#55.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_6#1.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_6#1.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 12574 unique variants in 12574 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 8468 unique variants in 8468 input lines\n",
      "NOTICE: Processing next batch with 10369 unique variants in 10369 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1431 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1854 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t89414\t89414\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:12,251 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#3.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#3.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#3.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1441 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 8463 unique variants in 8463 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2186 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t463074\t463074\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:12,617 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#3.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1360 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#18.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#18.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3089 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t36751\t36751\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:12,831 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#1.avinput\n",
      "NOTICE: Processing next batch with 7835 unique variants in 7835 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#82.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#82.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1493 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11631 unique variants in 11631 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2018 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t650653\t650653\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:13,150 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#10.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#14.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#14.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1375 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9842 unique variants in 9842 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2768 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1564191\t1564191\tT\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:13,426 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_6#1.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1491 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#92.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#92.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2019 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t810245\t810245\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 11160 unique variants in 11160 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1472 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:51:13,709 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#12.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#65.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#65.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1967 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1015798\t1015798\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:51:13,878 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#55.avinput\n",
      "NOTICE: Processing next batch with 8089 unique variants in 8089 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1349 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#75.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#75.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1524 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3213 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t953453\t953453\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 10173 unique variants in 10173 input lines\n",
      "2025-04-23 21:51:14,123 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#3.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2454 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2173231\t2173231\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:14,207 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#54.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#7.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#7.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1368 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#82.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#82.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10834 unique variants in 10834 input lines\n",
      "NOTICE: Processing next batch with 7604 unique variants in 7604 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3014 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t484866\t484866\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:14,460 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#18.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#92.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#92.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11238 unique variants in 11238 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1502 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1445 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2521 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t149033\t149033\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1886 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2094787\t2094787\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:51:15,365 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#82.avinput\n",
      "Done with 1354 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:51:15,377 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#14.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#9.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#9.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#7.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#7.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2999 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t3236\t3236\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:51:15,514 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#65.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10004 unique variants in 10004 input lines\n",
      "NOTICE: Processing next batch with 9831 unique variants in 9831 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#18.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#18.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Processing next batch with 10119 unique variants in 10119 input lines\n",
      "Done with 1541 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1353 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1963 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1740585\t1740585\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2949 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1929186\t1929186\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:15,888 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#92.avinput\n",
      "2025-04-23 21:51:15,908 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#82.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#21.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#21.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1440 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#49.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#49.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11148 unique variants in 11148 input lines\n",
      "NOTICE: Processing next batch with 10902 unique variants in 10902 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1940 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1793190\t1793190\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:16,208 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#75.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1536 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_4#9.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_4#9.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10093 unique variants in 10093 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1951 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1748060\t1748060\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:16,505 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#7.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#11.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#11.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 8243 unique variants in 8243 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1434 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1854 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t331890\t331890\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:16,956 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#92.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#12.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#12.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10750 unique variants in 10750 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1446 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1443 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1954 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t173825\t173825\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:17,700 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#7.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1719 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t811505\t811505\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:17,722 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#9.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1463 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#2.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#2.avinput.exonic_variant_function\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#17.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#17.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1899 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1435495\t1435495\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:17,923 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#18.avinput\n",
      "NOTICE: Processing next batch with 11472 unique variants in 11472 input lines\n",
      "NOTICE: Processing next batch with 10542 unique variants in 10542 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#39.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#39.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7859 unique variants in 7859 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1474 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1343 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1466 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1487 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3094 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t511117\t511117\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:18,667 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#11.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2002 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1342486\t1342486\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:18,687 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#21.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1904 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t170174\t170174\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#29.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#29.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:51:18,814 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_4#9.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2309 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1824656\t1824656\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#53.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#53.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:51:18,851 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#49.avinput\n",
      "NOTICE: Processing next batch with 7842 unique variants in 7842 input lines\n",
      "NOTICE: Processing next batch with 12226 unique variants in 12226 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#22.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#22.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#18.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#18.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10567 unique variants in 10567 input lines\n",
      "NOTICE: Processing next batch with 7881 unique variants in 7881 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1489 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1971 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t584390\t584390\tG\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:19,571 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#12.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#24.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#24.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11732 unique variants in 11732 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1346 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3042 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t626941\t626941\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:20,032 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#39.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#62.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#62.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11317 unique variants in 11317 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1510 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1530 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1340 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2276 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1136402\t1136402\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:20,523 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#2.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3040 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t903595\t903595\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:20,588 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#53.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1346 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#72.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#72.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2659 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1754635\t1754635\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:51:20,674 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#17.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#34.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#34.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3043 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t343419\t343419\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 10302 unique variants in 10302 input lines\n",
      "2025-04-23 21:51:20,748 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#18.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#63.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#63.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 9996 unique variants in 9996 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#8.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#8.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 12017 unique variants in 12017 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9778 unique variants in 9778 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1478 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2333 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1878218\t1878218\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:21,347 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#22.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#88.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#88.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1509 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10634 unique variants in 10634 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3213 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1878218\t1878218\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:21,769 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#29.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#86.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#86.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11819 unique variants in 11819 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1507 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2661 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1778586\t1778586\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:22,339 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#24.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#96.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#96.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1435 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 7924 unique variants in 7924 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1874 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t606476\t606476\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:22,700 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#62.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1441 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1436 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#11.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#11.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1930 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2129846\t2129846\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:22,924 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#34.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Processing next batch with 10778 unique variants in 10778 input lines\n",
      "Done with 1460 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1929 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1089159\t1089159\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:23,039 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#8.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_4#14.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_4#14.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1511 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10167 unique variants in 10167 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1833 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2010847\t2010847\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#21.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#21.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:51:23,229 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#72.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11949 unique variants in 11949 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#18.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#18.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3549 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t331697\t331697\tA\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:51:23,440 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#63.avinput\n",
      "NOTICE: Processing next batch with 13296 unique variants in 13296 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#16.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#16.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1500 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 11087 unique variants in 11087 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2023 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2191892\t2191892\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:23,807 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#88.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#50.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#50.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7696 unique variants in 7696 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1345 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3049 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t490424\t490424\tT\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:24,207 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#96.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#19.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#19.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 1568 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10537 unique variants in 10537 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2591 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t110761\t110761\tA\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:24,579 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#86.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#8.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#8.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 13329 unique variants in 13329 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1469 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2202 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1633892\t1633892\tG\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1477 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:51:25,251 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_4#14.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#15.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#15.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2159 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t663239\t663239\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:51:25,431 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#11.avinput\n",
      "NOTICE: Processing next batch with 10891 unique variants in 10891 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1343 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#4.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#4.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3008 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1739077\t1739077\tT\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:25,660 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#50.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1506 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10513 unique variants in 10513 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1484 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#20.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#20.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3477 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1460569\t1460569\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:25,877 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#21.avinput\n",
      "NOTICE: Processing next batch with 10815 unique variants in 10815 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2146 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t253418\t253418\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:25,950 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#16.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_6#8.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_6#8.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1544 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#27.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#27.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 12152 unique variants in 12152 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2963 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t476360\t476360\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 12669 unique variants in 12669 input lines\n",
      "2025-04-23 21:51:26,283 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#18.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#11.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#11.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1469 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 13044 unique variants in 13044 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1973 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t713112\t713112\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:26,627 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#19.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6930_8#20.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6930_8#20.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12625 unique variants in 12625 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1484 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1502 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2599 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t620176\t620176\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:27,671 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#8.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2375 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t477646\t477646\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:51:27,783 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#15.avinput\n",
      "Done with 1505 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#37.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#37.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#6.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#6.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 11120 unique variants in 11120 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2253 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1608650\t1608650\tG\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:27,974 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#4.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1535 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 13904 unique variants in 13904 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#7.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#7.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1962 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1837565\t1837565\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:28,183 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#20.avinput\n",
      "NOTICE: Processing next batch with 10862 unique variants in 10862 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#10.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#10.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1508 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 11259 unique variants in 11259 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3518 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t924767\t924767\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:28,621 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_6#8.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#15.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#15.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1525 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10166 unique variants in 10166 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1486 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2023 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1813653\t1813653\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:29,027 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#27.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2634 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1024636\t1024636\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1524 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:51:29,158 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#11.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_4#20.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_4#20.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7889 unique variants in 7889 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#32.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#32.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2835 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t610528\t610528\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:51:29,341 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6930_8#20.avinput\n",
      "NOTICE: Processing next batch with 10988 unique variants in 10988 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#74.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#74.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11044 unique variants in 11044 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1524 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2201 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t754979\t754979\tG\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:30,128 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#37.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1537 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#64.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#64.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1965 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t557630\t557630\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 10727 unique variants in 10727 input lines\n",
      "2025-04-23 21:51:30,417 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#7.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#22.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#22.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1495 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1432 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 7827 unique variants in 7827 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1849 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t50188\t50188\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:30,750 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#10.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3563 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1993768\t1993768\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:30,809 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#6.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#19.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#19.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 1476 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#9.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#9.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1340 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 12272 unique variants in 12272 input lines\n",
      "NOTICE: Processing next batch with 10492 unique variants in 10492 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2194 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t339738\t339738\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:31,056 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#15.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2990 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t286582\t286582\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:31,117 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_4#20.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#58.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#58.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#1.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#1.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11152 unique variants in 11152 input lines\n",
      "NOTICE: Processing next batch with 7836 unique variants in 7836 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1478 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2126 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t255433\t255433\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:31,714 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#32.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1489 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#13.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#13.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2257 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t697930\t697930\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:31,907 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#74.avinput\n",
      "NOTICE: Processing next batch with 7726 unique variants in 7726 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6930_8#14.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6930_8#14.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1370 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 9756 unique variants in 9756 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3127 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t726102\t726102\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:32,228 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#22.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#5.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#5.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12015 unique variants in 12015 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1516 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2153 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1980451\t1980451\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:32,681 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#64.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#93.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#93.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 1479 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1369 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10807 unique variants in 10807 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2042 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t915768\t915768\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3016 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t206127\t206127\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:32,968 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#1.avinput\n",
      "2025-04-23 21:51:32,973 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#9.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#83.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#83.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#3.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#3.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10583 unique variants in 10583 input lines\n",
      "NOTICE: Processing next batch with 11588 unique variants in 11588 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1498 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1358 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1487 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2927 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1083761\t1083761\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:33,431 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#13.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2881 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2142592\t2142592\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2218 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t544856\t544856\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:33,474 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#19.avinput\n",
      "2025-04-23 21:51:33,483 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#58.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#74.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#74.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#40.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#40.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#68.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#68.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9829 unique variants in 9829 input lines\n",
      "NOTICE: Processing next batch with 9954 unique variants in 9954 input lines\n",
      "NOTICE: Processing next batch with 10907 unique variants in 10907 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1456 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2026 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t952924\t952924\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:34,144 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6930_8#14.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#14.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#14.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 13614 unique variants in 13614 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1502 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2608 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t660546\t660546\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1522 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:51:34,907 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#5.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#69.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#69.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 1478 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2190 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1776687\t1776687\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:51:35,067 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#93.avinput\n",
      "NOTICE: Processing next batch with 10174 unique variants in 10174 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2127 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1808486\t1808486\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#21.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#21.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:51:35,208 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#3.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11205 unique variants in 11205 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#79.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#79.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1448 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 11654 unique variants in 11654 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1888 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t854734\t854734\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1446 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:51:35,556 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#74.avinput\n",
      "Done with 1440 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#90.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#90.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1504 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1960 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t753834\t753834\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:35,729 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#68.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1878 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t454774\t454774\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 10619 unique variants in 10619 input lines\n",
      "2025-04-23 21:51:35,789 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#83.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2044 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1162557\t1162557\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#80.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#80.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:51:35,864 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#40.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#77.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#77.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 7985 unique variants in 7985 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#5.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#5.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11564 unique variants in 11564 input lines\n",
      "NOTICE: Processing next batch with 9942 unique variants in 9942 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1450 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1547 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1954 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1593460\t1593460\tA\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:37,083 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#69.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#67.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#67.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2833 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1156234\t1156234\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:51:37,246 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#14.avinput\n",
      "NOTICE: Processing next batch with 10870 unique variants in 10870 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#13.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#13.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1439 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7561 unique variants in 7561 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1349 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1882 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1025144\t1025144\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1503 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:51:37,573 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#21.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2956 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1147295\t1147295\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:37,610 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#80.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#23.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#23.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#5.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#5.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2529 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1947315\t1947315\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1478 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:51:37,777 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#79.avinput\n",
      "NOTICE: Processing next batch with 11148 unique variants in 11148 input lines\n",
      "NOTICE: Processing next batch with 9633 unique variants in 9633 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1450 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2319 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t500320\t500320\tT\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:37,900 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#90.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#57.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#57.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1721 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t290369\t290369\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:38,000 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#5.avinput\n",
      "NOTICE: Processing next batch with 10653 unique variants in 10653 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#56.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#56.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10032 unique variants in 10032 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#10.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#10.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1439 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 21082 unique variants in 21082 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1880 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1688823\t1688823\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:38,386 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#77.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#3.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#3.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7853 unique variants in 7853 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1351 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2933 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1448661\t1448661\tG\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:39,306 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#13.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#12.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#12.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1479 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 7861 unique variants in 7861 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1970 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t82401\t82401\tG\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:39,708 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#67.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1445 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#1.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#1.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1995 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1690189\t1690189\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:51:39,886 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#5.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Processing next batch with 10721 unique variants in 10721 input lines\n",
      "Done with 1473 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1450 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#16.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#16.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2007 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2173575\t2173575\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1500 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:51:40,139 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#23.avinput\n",
      "NOTICE: Processing next batch with 12391 unique variants in 12391 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1946 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1579868\t1579868\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:40,178 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#56.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1341 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2354 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1750903\t1750903\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#90.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#90.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:51:40,275 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#57.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#80.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#80.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3040 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2050132\t2050132\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:40,376 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#3.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_6#3.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_6#3.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 10204 unique variants in 10204 input lines\n",
      "NOTICE: Processing next batch with 11457 unique variants in 11457 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 8411 unique variants in 8411 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#1.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#1.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10798 unique variants in 10798 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1341 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3047 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1982583\t1982583\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:41,176 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#12.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#20.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#20.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10875 unique variants in 10875 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1471 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1344 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2137 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t323867\t323867\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:42,066 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#1.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3074 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t465720\t465720\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:42,139 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_6#3.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#61.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#61.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#71.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#71.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 10763 unique variants in 10763 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1457 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 11281 unique variants in 11281 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1497 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1951 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1754148\t1754148\tA\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:51:42,507 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#90.avinput\n",
      "Done with 1644 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1532 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1439 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2630 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1579951\t1579951\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:42,613 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#16.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#36.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#36.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1949 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1311974\t1311974\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 10318 unique variants in 10318 input lines\n",
      "2025-04-23 21:51:42,719 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#1.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#70.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#70.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1876 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1886574\t1886574\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2337 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t743677\t743677\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:42,770 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#80.avinput\n",
      "2025-04-23 21:51:42,807 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#10.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#60.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#60.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 11604 unique variants in 11604 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#26.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#26.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#44.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#44.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 11300 unique variants in 11300 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10630 unique variants in 10630 input lines\n",
      "NOTICE: Processing next batch with 10772 unique variants in 10772 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1521 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2455 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1677831\t1677831\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:43,753 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#20.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#19.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#19.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11905 unique variants in 11905 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1483 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1516 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2116 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1934051\t1934051\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:44,409 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#61.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#2.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#2.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2170 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t225384\t225384\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:51:44,565 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#71.avinput\n",
      "NOTICE: Processing next batch with 11646 unique variants in 11646 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1457 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#15.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#15.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10328 unique variants in 10328 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1835 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1399638\t1399638\tG\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:44,844 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#36.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#10.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#10.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1535 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1494 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 11553 unique variants in 11553 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1433 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1512 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2360 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t82105\t82105\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:45,194 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#70.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2340 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t213392\t213392\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:45,235 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#44.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2206 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t815100\t815100\tG\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2322 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2193246\t2193246\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:45,277 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#26.avinput\n",
      "2025-04-23 21:51:45,298 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#60.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#51.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#51.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#8.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#8.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#22.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#22.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 8159 unique variants in 8159 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#6.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#6.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 9575 unique variants in 9575 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10798 unique variants in 10798 input lines\n",
      "NOTICE: Processing next batch with 10074 unique variants in 10074 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1546 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2759 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t517522\t517522\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:46,585 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#19.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1485 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#17.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#17.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2019 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t715280\t715280\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:51:46,768 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#15.avinput\n",
      "NOTICE: Processing next batch with 12196 unique variants in 12196 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1488 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#19.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#19.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 1366 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2016 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t849149\t849149\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 10096 unique variants in 10096 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2947 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1008734\t1008734\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:47,037 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#2.avinput\n",
      "2025-04-23 21:51:47,046 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#51.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1541 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#8.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#8.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#4.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#4.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7624 unique variants in 7624 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2478 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t554104\t554104\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:47,277 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#10.avinput\n",
      "NOTICE: Processing next batch with 13977 unique variants in 13977 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1462 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1433 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#89.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#89.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1916 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1530599\t1530599\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1527 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 7863 unique variants in 7863 input lines\n",
      "2025-04-23 21:51:47,488 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#6.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1944 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1530859\t1530859\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:47,526 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#8.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1948 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t742873\t742873\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:47,612 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#22.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#35.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#35.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#13.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#13.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7651 unique variants in 7651 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6930_8#22.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6930_8#22.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 20308 unique variants in 20308 input lines\n",
      "NOTICE: Processing next batch with 10969 unique variants in 10969 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1355 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2714 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1616485\t1616485\tA\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:48,748 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#4.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#8.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#8.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 1344 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1447 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1501 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10308 unique variants in 10308 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3041 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1595299\t1595299\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:49,039 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#89.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1947 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2076507\t2076507\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:49,064 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#19.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1363 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2352 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1614905\t1614905\tC\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:49,152 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#17.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#2.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#2.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#94.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#94.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2965 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t958465\t958465\tT\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:51:49,235 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#35.avinput\n",
      "NOTICE: Processing next batch with 11632 unique variants in 11632 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#84.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#84.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 9506 unique variants in 9506 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#47.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#47.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 8924 unique variants in 8924 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 8607 unique variants in 8607 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1510 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1519 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2397 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t388134\t388134\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:50,051 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6930_8#22.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2914 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1660495\t1660495\tT\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:50,126 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#8.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#89.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#89.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#52.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#52.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10070 unique variants in 10070 input lines\n",
      "NOTICE: Processing next batch with 11430 unique variants in 11430 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1490 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2021 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2139598\t2139598\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:51,001 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#8.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1359 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1354 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1375 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#14.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#14.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2895 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1547861\t1547861\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:51,144 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#47.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3347 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t503095\t503095\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:51:51,182 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#84.avinput\n",
      "NOTICE: Processing next batch with 11154 unique variants in 11154 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 4340 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t133640\t133640\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_4#2.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_4#2.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:51:51,275 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#94.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#23.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#23.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1479 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 8242 unique variants in 8242 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_4#16.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_4#16.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 10925 unique variants in 10925 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1628 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 9997 unique variants in 9997 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2245 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t818964\t818964\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:51,549 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#2.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#13.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#13.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2335 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1564173\t1564173\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 10336 unique variants in 10336 input lines\n",
      "2025-04-23 21:51:51,815 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#13.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#48.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#48.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7961 unique variants in 7961 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1468 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1921 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1749294\t1749294\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:52,348 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#89.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#27.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#27.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1512 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 9967 unique variants in 9967 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1942 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1687671\t1687671\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:52,731 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#52.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1338 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#37.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#37.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3096 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1792421\t1792421\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:52,949 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_4#2.avinput\n",
      "NOTICE: Processing next batch with 11240 unique variants in 11240 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#71.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#71.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11680 unique variants in 11680 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1498 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1452 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1836 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t916437\t916437\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2294 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t752600\t752600\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:51:53,469 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_4#16.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1543 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:51:53,485 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#23.avinput\n",
      "Done with 1350 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#70.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#70.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3039 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t7296\t7296\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#60.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#60.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:51:53,618 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#48.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2327 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t860258\t860258\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1445 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:51:53,681 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#14.avinput\n",
      "NOTICE: Processing next batch with 10060 unique variants in 10060 input lines\n",
      "NOTICE: Processing next batch with 10582 unique variants in 10582 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#21.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#21.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2009 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t506646\t506646\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#50.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#50.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:51:53,818 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#13.avinput\n",
      "NOTICE: Processing next batch with 9845 unique variants in 9845 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12493 unique variants in 12493 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#9.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#9.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10089 unique variants in 10089 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1454 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2034 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1481852\t1481852\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:54,631 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#27.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#3.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#3.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11813 unique variants in 11813 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1486 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2327 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1889707\t1889707\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:55,193 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#37.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1537 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#18.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#18.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2361 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1886574\t1886574\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:55,403 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#71.avinput\n",
      "NOTICE: Processing next batch with 10386 unique variants in 10386 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1463 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#14.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#14.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1902 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1741050\t1741050\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:55,601 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#70.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1458 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 11471 unique variants in 11471 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#11.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#11.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 1476 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1931 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t262704\t262704\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:55,797 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#21.avinput\n",
      "NOTICE: Processing next batch with 12191 unique variants in 12191 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2406 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1571075\t1571075\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1463 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:51:55,914 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#60.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#1.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#1.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10509 unique variants in 10509 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1904 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1608176\t1608176\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#88.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#88.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:51:56,053 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#9.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9967 unique variants in 9967 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#5.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#5.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 1531 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 11009 unique variants in 11009 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2496 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1837745\t1837745\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:56,435 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#50.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#9.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#9.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11300 unique variants in 11300 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1544 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2286 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1344105\t1344105\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:57,353 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#3.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1464 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6930_8#23.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6930_8#23.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2182 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t713112\t713112\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:57,585 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#18.avinput\n",
      "NOTICE: Processing next batch with 9741 unique variants in 9741 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1431 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#24.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#24.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10186 unique variants in 10186 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1938 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1693093\t1693093\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:57,886 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#14.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1432 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#12.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#12.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1509 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1998 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1880585\t1880585\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1493 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:51:58,083 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#88.avinput\n",
      "NOTICE: Processing next batch with 9896 unique variants in 9896 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1467 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2272 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t665689\t665689\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#34.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#34.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 10381 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1135780\t1135780\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:58,247 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#1.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:51:58,273 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#11.avinput\n",
      "NOTICE: Processing next batch with 12366 unique variants in 12366 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1954 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1609419\t1609419\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#18.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#18.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:51:58,387 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#5.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#16.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#16.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11633 unique variants in 11633 input lines\n",
      "NOTICE: Processing next batch with 10582 unique variants in 10582 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#7.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#7.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 1526 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 11782 unique variants in 11782 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2337 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1356136\t1356136\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:58,743 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#9.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#23.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#23.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 7840 unique variants in 7840 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1466 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2036 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1040551\t1040551\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:51:59,576 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6930_8#23.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1446 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#9.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#9.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1922 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2114513\t2114513\tG\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 9988 unique variants in 9988 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:51:59,834 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#24.avinput\n",
      "Done with 1441 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1957 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t216653\t216653\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#53.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#53.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:52:00,008 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#12.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 13423 unique variants in 13423 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#15.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#15.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10964 unique variants in 10964 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1367 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3013 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t661794\t661794\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1466 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:52:00,548 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#23.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1444 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1968 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1383301\t1383301\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:52:00,675 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#16.avinput\n",
      "Done with 1499 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_4#17.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_4#17.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1946 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1076889\t1076889\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1545 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:52:00,773 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#34.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#12.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#12.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 10309 unique variants in 10309 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2535 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1649005\t1649005\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:00,880 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#18.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2624 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t887602\t887602\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#22.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#22.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 10122 unique variants in 10122 input lines\n",
      "2025-04-23 21:52:00,913 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#7.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#95.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#95.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 10718 unique variants in 10718 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_3#3.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_3#3.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12363 unique variants in 12363 input lines\n",
      "NOTICE: Processing next batch with 10935 unique variants in 10935 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1450 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1833 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t920262\t920262\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:01,831 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#9.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#88.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#88.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10751 unique variants in 10751 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1491 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2292 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t982615\t982615\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:02,443 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#15.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1443 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#59.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#59.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12448 unique variants in 12448 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1896 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t94721\t94721\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:02,748 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#53.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1490 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1460 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#49.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#49.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1504 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1895 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1617721\t1617721\tC\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2022 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t301005\t301005\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:02,977 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_4#17.avinput\n",
      "2025-04-23 21:52:02,978 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#12.avinput\n",
      "NOTICE: Processing next batch with 10511 unique variants in 10511 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2363 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t886441\t886441\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_4#21.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_4#21.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:52:03,097 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#22.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#6.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#6.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 1504 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10402 unique variants in 10402 input lines\n",
      "NOTICE: Processing next batch with 9995 unique variants in 9995 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#14.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#14.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1449 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2401 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1857023\t1857023\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 10618 unique variants in 10618 input lines\n",
      "2025-04-23 21:52:03,312 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_3#3.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_5#18.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_5#18.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2009 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1291126\t1291126\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:52:03,458 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#95.avinput\n",
      "NOTICE: Processing next batch with 10872 unique variants in 10872 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#65.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#65.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11625 unique variants in 11625 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1488 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2094 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t767373\t767373\tA\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:04,286 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#88.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#23.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#23.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12356 unique variants in 12356 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1443 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1503 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1443 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1748 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t283354\t283354\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:05,132 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#6.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2004 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1737378\t1737378\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2060 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t191795\t191795\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:52:05,164 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#49.avinput\n",
      "2025-04-23 21:52:05,166 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#59.avinput\n",
      "Done with 1515 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1477 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#33.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#33.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#75.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#75.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_2#41.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_2#41.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2191 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1009584\t1009584\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:05,335 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_4#21.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2318 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t233088\t233088\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 10512 unique variants in 10512 input lines\n",
      "2025-04-23 21:52:05,369 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#14.avinput\n",
      "NOTICE: Processing next batch with 10186 unique variants in 10186 input lines\n",
      "NOTICE: Processing next batch with 12046 unique variants in 12046 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#75.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#75.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 1502 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#82.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#82.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 11140 unique variants in 11140 input lines\n",
      "NOTICE: Processing next batch with 10328 unique variants in 10328 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2366 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1114081\t1114081\tG\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:05,678 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_5#18.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#2.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#2.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1510 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9797 unique variants in 9797 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2306 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2017325\t2017325\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:05,996 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#65.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#92.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#92.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10985 unique variants in 10985 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1554 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2463 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t389030\t389030\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:06,938 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#23.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_1#20.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_1#20.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10407 unique variants in 10407 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1452 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1489 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1945 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1925987\t1925987\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:07,374 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#41.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1900 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t454774\t454774\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:07,403 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_2#33.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#78.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#78.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 1460 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#68.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#68.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 1506 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 15123 unique variants in 15123 input lines\n",
      "NOTICE: Processing next batch with 12183 unique variants in 12183 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1981 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1626946\t1626946\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:07,695 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#82.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3130 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t782804\t782804\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:52:07,743 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#75.avinput\n",
      "Done with 1527 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1443 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_3#15.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_3#15.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#69.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#69.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1948 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t793946\t793946\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 10153 unique variants in 10153 input lines\n",
      "2025-04-23 21:52:07,910 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#2.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2388 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t133776\t133776\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Processing next batch with 10166 unique variants in 10166 input lines\n",
      "2025-04-23 21:52:07,972 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#75.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6930_8#15.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6930_8#15.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_4#12.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_4#12.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10315 unique variants in 10315 input lines\n",
      "NOTICE: Processing next batch with 12592 unique variants in 12592 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1484 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2129 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t429693\t429693\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:08,382 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#92.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_4#4.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_4#4.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10492 unique variants in 10492 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1466 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2053 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t589205\t589205\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:09,300 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_1#20.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_2#22.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_2#22.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 9910 unique variants in 9910 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1461 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1896 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1613503\t1613503\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 1432 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1556 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "2025-04-23 21:52:09,888 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_3#15.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1472 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2223 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t404216\t404216\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_7#12.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_7#12.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:52:10,019 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#69.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2636 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1067146\t1067146\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:52:10,057 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#68.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2365 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1343258\t1343258\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:10,116 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6930_8#15.avinput\n",
      "NOTICE: Processing next batch with 10830 unique variants in 10830 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_6#4.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_6#4.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#66.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#66.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7001_1#4.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7001_1#4.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 9766 unique variants in 9766 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10479 unique variants in 10479 input lines\n",
      "NOTICE: Processing next batch with 10183 unique variants in 10183 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1519 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1526 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1500 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2017 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1942254\t1942254\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2162 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t574712\t574712\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:10,653 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_4#12.avinput\n",
      "2025-04-23 21:52:10,663 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#78.avinput\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2263 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t392021\t392021\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:10,712 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_4#4.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_5#76.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_5#76.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#81.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#81.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_2#91.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_2#91.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12601 unique variants in 12601 input lines\n",
      "NOTICE: Processing next batch with 10378 unique variants in 10378 input lines\n",
      "NOTICE: Processing next batch with 7922 unique variants in 7922 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1449 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1878 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1774317\t1774317\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:11,462 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_2#22.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#81.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#81.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 10575 unique variants in 10575 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1444 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1537 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1943 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1793190\t1793190\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1968 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2001498\t2001498\tT\tC\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... 2025-04-23 21:52:12,190 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_7#12.avinput\n",
      "2025-04-23 21:52:12,191 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_6#4.avinput\n",
      "Done with 1499 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_5#91.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_5#91.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6952_7#17.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6952_7#17.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... \n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2022 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t254544\t254544\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "2025-04-23 21:52:12,365 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#66.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1346 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1462 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10163 unique variants in 10163 input lines\n",
      "NOTICE: Processing next batch with 10378 unique variants in 10378 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3062 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t2218710\t2218710\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7553_6#2.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7553_6#2.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... 2025-04-23 21:52:12,499 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#91.avinput\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1912 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1444964\t1444964\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:12,569 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7001_1#4.avinput\n",
      "NOTICE: Processing next batch with 13625 unique variants in 13625 input lines\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#2.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#2.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_3#57.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_3#57.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... NOTICE: Processing next batch with 10026 unique variants in 10026 input lines\n",
      "Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Processing next batch with 7778 unique variants in 7778 input lines\n",
      "Done with 1505 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2169 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t744176\t744176\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:12,980 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_2#81.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.7622_4#11.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.7622_4#11.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1534 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 10189 unique variants in 10189 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2047 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t823106\t823106\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:13,377 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_5#76.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6925_1#56.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6925_1#56.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Processing next batch with 12811 unique variants in 12811 input lines\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1465 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2359 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t45417\t45417\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:13,911 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#81.avinput\n",
      "NOTICE: Output files are written to pyseer_dataset/original_dataset/annovar_files/snps.6999_2#13.avinput.variant_function, pyseer_dataset/original_dataset/annovar_files/snps.6999_2#13.avinput.exonic_variant_function\n",
      "NOTICE: Reading gene annotation from pyseer_dataset/Spn23F/Spn23F_refGene.txt ... Done with 2271 transcripts (including 70 without coding sequence annotation) for 2268 unique genes\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1363 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "NOTICE: Processing next batch with 12212 unique variants in 12212 input lines\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 3026 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1888312\t1888312\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:14,278 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_3#57.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1436 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "Done with 1459 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2206 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1735571\t1735571\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1976 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1815339\t1815339\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:14,511 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_5#91.avinput\n",
      "2025-04-23 21:52:14,511 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6952_7#17.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1454 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1972 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1104706\t1104706\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:14,664 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#2.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1445 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 1947 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1387037\t1387037\tC\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:15,121 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7622_4#11.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1493 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2629 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1156767\t1156767\tA\tG\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:15,341 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.7553_6#2.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1520 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2814 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t486190\t486190\tC\tT\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:15,800 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6925_1#56.avinput\n",
      "NOTICE: Reading FASTA sequences from pyseer_dataset/Spn23F/Spn23F_refGeneMrna.fa ... Done with 1520 sequences\n",
      "WARNING: A total of 111 sequences will be ignored due to lack of correct ORF annotation\n",
      "\n",
      "---------------------------------------------------------------------------------------\n",
      "WARNING: 2015 exonic SNPs have WRONG reference alleles specified in your input file!\n",
      "WARNING: An example input line is <NC_011900.1\t1670793\t1670793\tG\tA\thom\t.\t.>\n",
      "WARNING: ANNOVAR can still annotate exonic_variant_function for the mutation correctly!\n",
      "WARNING: you may have used wrong -buildver, or specified incorrect reference allele, or used outdated mRNA FASTA file!\n",
      "---------------------------------------------------------------------------------------\n",
      "\n",
      "2025-04-23 21:52:16,113 - INFO - Successfully annotated pyseer_dataset/original_dataset/annovar_files/snps.6999_2#13.avinput\n",
      "2025-04-23 21:52:16,231 - INFO - Completed parallel annotation of avinput files.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Annotated 603 avinput files in 183.78 seconds.\n",
      "Annotated avinput files are saved in pyseer_dataset/original_dataset/annovar_files/.\n"
     ]
    }
   ],
   "source": [
    "from muggwas import run_annovar\n",
    "\n",
    "## User inputs\n",
    "# the path to the ANNOVAR installation\n",
    "annovar_dir = '../annovar/'\n",
    "# the directory where the reference files are located\n",
    "db_dir = 'pyseer_dataset/Spn23F/' #ref is susceptible to penicillin\n",
    "# path to the vcf file\n",
    "input_dir = 'pyseer_dataset/original_dataset/snps.newname.vcf.gz'\n",
    "# output prefix or the converted annovar input file\n",
    "vcf_prefix = 'snps'\n",
    "# output prefix for the annovar database\n",
    "ref_prefix = 'Spn23F' #ref is susceptible to penicillin\n",
    "\n",
    "## Run ANNOVAR\n",
    "run_annovar(\n",
    "    annovar_dir=annovar_dir,\n",
    "    db_dir=db_dir,\n",
    "    input_dir=input_dir,\n",
    "    vcf_prefix=vcf_prefix,\n",
    "    ref_prefix=ref_prefix\n",
    ")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### **Step 2: Build Gene Mutation Table:**\n",
    "\n",
    "For each sample:\n",
    "1. **Build gene maps** from the gff3 file.\n",
    "2. Compile mutation info **gene-by-gene**.\n",
    "3. **Determine mutation types**.\n",
    "    - binary: mutation or wildtype\n",
    "    - multiple: nonsense, nonstop, missense, silent or wildtype\n",
    "4. **Output mutation types per gene** across all samples:\n",
    "\n",
    "    |Gene|Sample1|Sample2|Sample3|\n",
    "    |:-:|:-:|:-:|:-:|\n",
    "    |g1|m|w|w|m|\n",
    "    |g2|m|m|m|w|\n",
    "    |g3|w|w|w|m|\n",
    "    |...|...|...|...|\n",
    "\n",
    "**Import functions and annotation data**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "from muggwas import compile_gene_mutations, write_gene_mutation_summary, write_gene_annotation_summary\n",
    "\n",
    "# Directory containing the ANNOVAR output files\n",
    "annovar_output_dir = 'pyseer_dataset/original_dataset/annovar_files'\n",
    "# Read gff3 to build gene maps\n",
    "gff_file = 'pyseer_dataset/Spn23F/Spn23F.gff3'\n",
    "# Write the mutation types to a tab delimited file\n",
    "output_mutation_table = 'pyseer_dataset/original_dataset/gene_mutation_summary.txt'\n",
    "# Write the annotations to a tab delimited file\n",
    "output_annotation_table = 'pyseer_dataset/original_dataset/gene_annotation_summary.txt'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Compile mutation and output gene summary table**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Time elapsed for compiling gene mutations: 37.84 seconds\n",
      "Total number of samples processed: 603\n",
      "Gene mutation summary has been written to pyseer_dataset/original_dataset/gene_mutation_summary.txt.\n",
      "\n",
      "Gene annotation summary has been written to pyseer_dataset/original_dataset/gene_annotation_summary.txt.\n"
     ]
    }
   ],
   "source": [
    "## Compile the gene mutations for each sample\n",
    "# if you want to output binary mutation types (mutated or not), set model = 'binary'\n",
    "gene_mutation_summary = compile_gene_mutations(annovar_output_dir, gff_file, model = 'binary')\n",
    "## Write the gene mutation summary\n",
    "write_gene_mutation_summary(gene_mutation_summary, output_mutation_table)\n",
    "## Write the gene annotations\n",
    "write_gene_annotation_summary(gff_file, output_annotation_table)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### **Step 3: Estimate Population Structure Effect:**\n",
    "\n",
    "In gene-trait association analyses, false positives could arise from shared genetic lineage, especially in clonal organisms. There are two methods to estimate the effect of population structure:\n",
    "- Phylogenetic-based: Infer population structure based on phylogenetic distances. Since MugGWAS will use a linear mixed model, the distance will be estimated from the shared branch length between the MRCA and the root.\n",
    "- Kinship: Variants on core gene sequences represent the result of vertical evolution. To make inferences for identical by descent, the script will calculate the kinship matrix from the genotype matrix of the presence and absence of variants. --> (not supported yet as of 2025.04.15)\n",
    "\n",
    "Inputs: \n",
    "- Phylogeny-based: Output from IQ-tree `core_gene_tree.nwk`. A high-quality phylogeny based on single-copy core genes from a pangenome.\n",
    "- Kinship: A VCF file documenting variants on core genes `core_gene_snp.vcf.gz`. --> (not supported yet as of 2025.04.15)\n",
    "\n",
    "Output: \n",
    "- `phylogeny_distances.tsv`: a distance matrix to account for population structure effect in pyseer.\n",
    "\n",
    "    | |Sample1|Sample2|Sample3|Sample4|\n",
    "    |:-:|:-:|:-:|:-:|:-:|\n",
    "    |Sample1|0|4|0|0|\n",
    "    |Sample2|4|0|0|0|\n",
    "    |Sample3|0|0|0|3|\n",
    "    |Sample4|0|0|3|0|\n",
    "    |...|...|...|...|\n",
    "\n",
    "**Phylogeny-based method**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "from muggwas import phylogeny2distmatrix\n",
    "\n",
    "# Read the phylogenetic tree\n",
    "phylogeny = 'pyseer_dataset/core_genome_aln.tree'\n",
    "output_file = 'pyseer_dataset/phylogeny_distances.tsv'\n",
    "\n",
    "# Convert the phylogenetic tree to a distance matrix\n",
    "phylogeny2distmatrix(phylogeny, output_file)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### **Step 4: Run GWAS on pyseer**\n",
    "\n",
    "This function filter the genes that have no mutations in the population and run the [linear mixed effect model (FaST-LMM)](https://pyseer.readthedocs.io/en/master/usage.html#mixed-model-fast-lmm) from `pyseer` to infer gene-trait association. Read the GWAS [tutorial](https://pyseer.readthedocs.io/en/master/tutorial.html) for more details.\n",
    "\n",
    "\n",
    "\n",
    "- Input:\n",
    "    1. The **phenotype file**: a tab-delimited file with the first column as sample names and the second column as the phenotypic value (support binary or numeric).\n",
    "    2. The **gene mutation summary table**: the output from step 2.\n",
    "    3. The **distance matrix**: the output from step 3.\n",
    "\n",
    "- Output: A summary table of **GWAS result** that looks like this:\n",
    "\n",
    "    |Genes|Allele Freq.|filter-value|lrt-pvalue|beta|beta-std-err|variant_h2|notes|\n",
    "    |:---|:--|:------|:----------|:------|:-----------|:------|:-----------|\n",
    "    |g1|3.00E-01|6.96E-02|1.59E-01|2.09E-01|1.35E-01|4.81E-01|\n",
    "    |g2|6.00E-01|2.89E-01|2.50E-01|-8.73E-02|7.03E-02|4.02E-01|\t\n",
    "    |g3|6.00E-01|7.02E-01|8.27E-01|-2.92E-02|1.30E-01|7.94E-02|\n",
    "    |...|...|...|...|...|...|...|...|"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2025-04-16 11:13:18,980 - INFO - Start running GWAS with pyseer\n",
      "2025-04-16 11:13:18,989 - INFO - Gene gene-SPN23F_RS00005 has a mutation rate of 0.27.\n",
      "2025-04-16 11:13:18,990 - INFO - Gene cds-WP_000660615.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:18,990 - INFO - Gene gene-SPN23F_RS00010 has a mutation rate of 0.27.\n",
      "2025-04-16 11:13:18,990 - INFO - Gene cds-WP_000581157.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:18,991 - INFO - Gene gene-SPN23F_RS00015 has a mutation rate of 0.11333333333333333.\n",
      "2025-04-16 11:13:18,991 - INFO - Gene cds-WP_000285194.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:18,992 - INFO - Gene gene-SPN23F_RS00020 has a mutation rate of 0.2966666666666667.\n",
      "2025-04-16 11:13:18,992 - INFO - Gene cds-WP_001218707.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:18,992 - INFO - Gene gene-SPN23F_RS00025 has a mutation rate of 0.7133333333333334.\n",
      "2025-04-16 11:13:18,993 - INFO - Gene cds-WP_000163930.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:18,993 - INFO - Gene gene-SPN23F_RS00030 has a mutation rate of 0.5933333333333334.\n",
      "2025-04-16 11:13:18,994 - INFO - Gene cds-WP_000258117.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:18,994 - INFO - Gene gene-SPN23F_RS00035 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:18,994 - INFO - Gene cds-WP_001234978.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:18,995 - INFO - Gene gene-SPN23F_RS00040 has a mutation rate of 0.06.\n",
      "2025-04-16 11:13:18,995 - INFO - Gene cds-WP_000041909.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:18,995 - INFO - Gene gene-SPN23F_RS14045 has a mutation rate of 0.28.\n",
      "2025-04-16 11:13:18,995 - INFO - Gene cds-WP_000429334.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:18,996 - INFO - Gene gene-SPN23F_RS00050 has a mutation rate of 0.3933333333333333.\n",
      "2025-04-16 11:13:18,996 - INFO - Gene cds-WP_001224768.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:18,996 - INFO - Gene gene-SPN23F_RS00055 has a mutation rate of 0.94.\n",
      "2025-04-16 11:13:18,997 - INFO - Gene cds-WP_001208992.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:18,997 - INFO - Gene gene-SPN23F_RS00060 has a mutation rate of 0.12333333333333334.\n",
      "2025-04-16 11:13:18,997 - INFO - Gene cds-WP_000892185.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:18,998 - INFO - Gene gene-SPN23F_RS00065 has a mutation rate of 0.5666666666666667.\n",
      "2025-04-16 11:13:18,998 - INFO - Gene cds-WP_000744545.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:18,998 - INFO - Gene gene-SPN23F_RS00070 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:18,998 - INFO - Gene cds-WP_000436644.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:18,999 - INFO - Gene gene-SPN23F_RS00075 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:18,999 - INFO - Gene cds-WP_000588897.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:18,999 - INFO - Gene gene-SPN23F_RS00080 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,000 - INFO - Gene gene-SPN23F_RS00085 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,000 - INFO - Gene gene-SPN23F_RS00090 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,001 - INFO - Gene gene-SPN23F_RS00095 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,001 - INFO - Gene gene-SPN23F_RS00100 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,001 - INFO - Gene gene-SPN23F_RS00105 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,001 - INFO - Gene cds-SPN23F_RS11705 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,002 - INFO - Gene cds-SPN23F_RS11710 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,002 - INFO - Gene gene-SPN23F_RS00140 has a mutation rate of 0.4.\n",
      "2025-04-16 11:13:19,003 - INFO - Gene cds-WP_000939545.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,003 - INFO - Gene gene-SPN23F_RS00145 has a mutation rate of 0.11.\n",
      "2025-04-16 11:13:19,003 - INFO - Gene cds-WP_000205044.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,004 - INFO - Gene gene-SPN23F_RS00150 has a mutation rate of 0.46.\n",
      "2025-04-16 11:13:19,004 - INFO - Gene cds-WP_000291870.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,004 - INFO - Gene gene-SPN23F_RS11715 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,005 - INFO - Gene gene-SPN23F_RS13665 has a mutation rate of 0.01.\n",
      "2025-04-16 11:13:19,005 - INFO - Gene cds-WP_001106362.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,005 - INFO - Gene gene-SPN23F_RS13670 has a mutation rate of 0.05.\n",
      "2025-04-16 11:13:19,006 - INFO - Gene cds-WP_001814135.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,006 - INFO - Gene gene-SPN23F_RS00165 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,006 - INFO - Gene cds-WP_000401841.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,006 - INFO - Gene gene-SPN23F_RS00170 has a mutation rate of 0.11333333333333333.\n",
      "2025-04-16 11:13:19,007 - INFO - Gene cds-WP_000170931.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,007 - INFO - Gene gene-SPN23F_RS00175 has a mutation rate of 0.03.\n",
      "2025-04-16 11:13:19,007 - INFO - Gene cds-WP_000285962.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,008 - INFO - Gene gene-SPN23F_RS00180 has a mutation rate of 0.03666666666666667.\n",
      "2025-04-16 11:13:19,008 - INFO - Gene cds-WP_000156419.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,009 - INFO - Gene gene-SPN23F_RS00185 has a mutation rate of 0.06333333333333334.\n",
      "2025-04-16 11:13:19,009 - INFO - Gene cds-WP_000464160.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,009 - INFO - Gene gene-SPN23F_RS00190 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,009 - INFO - Gene cds-WP_001112859.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,010 - INFO - Gene gene-SPN23F_RS00195 has a mutation rate of 0.07.\n",
      "2025-04-16 11:13:19,010 - INFO - Gene cds-WP_001247549.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,010 - INFO - Gene gene-SPN23F_RS11720 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:19,011 - INFO - Gene cds-WP_000024181.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,011 - INFO - Gene cds-SPN23F_RS00200 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,011 - INFO - Gene gene-SPN23F_RS00205 has a mutation rate of 0.04.\n",
      "2025-04-16 11:13:19,012 - INFO - Gene cds-WP_001209433.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,012 - INFO - Gene gene-SPN23F_RS13455 has a mutation rate of 0.023333333333333334.\n",
      "2025-04-16 11:13:19,012 - INFO - Gene cds-WP_001030863.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,013 - INFO - Gene gene-SPN23F_RS00210 has a mutation rate of 0.26666666666666666.\n",
      "2025-04-16 11:13:19,013 - INFO - Gene cds-WP_000701992.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,014 - INFO - Gene gene-SPN23F_RS00215 has a mutation rate of 0.2966666666666667.\n",
      "2025-04-16 11:13:19,014 - INFO - Gene cds-WP_000691236.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,014 - INFO - Gene gene-SPN23F_RS00220 has a mutation rate of 0.26666666666666666.\n",
      "2025-04-16 11:13:19,014 - INFO - Gene cds-WP_075213698.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,015 - INFO - Gene gene-SPN23F_RS00225 has a mutation rate of 0.34.\n",
      "2025-04-16 11:13:19,015 - INFO - Gene cds-WP_001809263.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,015 - INFO - Gene cds-SPN23F_RS00230 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,016 - INFO - Gene gene-SPN23F_RS00240 has a mutation rate of 0.22333333333333333.\n",
      "2025-04-16 11:13:19,016 - INFO - Gene cds-WP_000010163.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,016 - INFO - Gene cds-SPN23F_RS13675 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,017 - INFO - Gene gene-SPN23F_RS00260 has a mutation rate of 0.7433333333333333.\n",
      "2025-04-16 11:13:19,017 - INFO - Gene cds-WP_001812055.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,018 - INFO - Gene gene-SPN23F_RS00265 has a mutation rate of 0.2866666666666667.\n",
      "2025-04-16 11:13:19,018 - INFO - Gene cds-WP_000076479.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,018 - INFO - Gene gene-SPN23F_RS13680 has a mutation rate of 0.41333333333333333.\n",
      "2025-04-16 11:13:19,018 - INFO - Gene cds-WP_001814139.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,019 - INFO - Gene gene-SPN23F_RS00275 has a mutation rate of 0.8533333333333334.\n",
      "2025-04-16 11:13:19,019 - INFO - Gene cds-WP_000009180.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,019 - INFO - Gene gene-SPN23F_RS00280 has a mutation rate of 0.7166666666666667.\n",
      "2025-04-16 11:13:19,020 - INFO - Gene cds-WP_000366348.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,020 - INFO - Gene gene-SPN23F_RS00285 has a mutation rate of 0.5866666666666667.\n",
      "2025-04-16 11:13:19,020 - INFO - Gene cds-WP_000616164.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,020 - INFO - Gene gene-SPN23F_RS00290 has a mutation rate of 0.69.\n",
      "2025-04-16 11:13:19,021 - INFO - Gene cds-WP_000717451.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,021 - INFO - Gene gene-SPN23F_RS00295 has a mutation rate of 0.14.\n",
      "2025-04-16 11:13:19,021 - INFO - Gene cds-WP_000136449.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,022 - INFO - Gene cds-SPN23F_RS11730 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,022 - INFO - Gene gene-SPN23F_RS00300 has a mutation rate of 0.7233333333333334.\n",
      "2025-04-16 11:13:19,022 - INFO - Gene cds-WP_001093075.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,023 - INFO - Gene gene-SPN23F_RS14050 has a mutation rate of 0.023333333333333334.\n",
      "2025-04-16 11:13:19,023 - INFO - Gene cds-WP_000346297.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,023 - INFO - Gene gene-SPN23F_RS00315 has a mutation rate of 0.5666666666666667.\n",
      "2025-04-16 11:13:19,024 - INFO - Gene cds-WP_000668272.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,024 - INFO - Gene gene-SPN23F_RS00320 has a mutation rate of 0.87.\n",
      "2025-04-16 11:13:19,024 - INFO - Gene cds-WP_000801611.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,025 - INFO - Gene gene-SPN23F_RS00325 has a mutation rate of 0.5433333333333333.\n",
      "2025-04-16 11:13:19,025 - INFO - Gene cds-WP_000043310.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,025 - INFO - Gene gene-SPN23F_RS00330 has a mutation rate of 0.94.\n",
      "2025-04-16 11:13:19,026 - INFO - Gene cds-WP_000361217.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,026 - INFO - Gene gene-SPN23F_RS00335 has a mutation rate of 0.6733333333333333.\n",
      "2025-04-16 11:13:19,026 - INFO - Gene cds-WP_000220632.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,026 - INFO - Gene gene-SPN23F_RS00340 has a mutation rate of 0.44666666666666666.\n",
      "2025-04-16 11:13:19,027 - INFO - Gene cds-WP_000182575.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,027 - INFO - Gene gene-SPN23F_RS00345 has a mutation rate of 0.4766666666666667.\n",
      "2025-04-16 11:13:19,027 - INFO - Gene cds-WP_000717506.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,028 - INFO - Gene gene-SPN23F_RS00350 has a mutation rate of 0.2633333333333333.\n",
      "2025-04-16 11:13:19,028 - INFO - Gene cds-WP_000894018.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,028 - INFO - Gene gene-SPN23F_RS00355 has a mutation rate of 0.66.\n",
      "2025-04-16 11:13:19,028 - INFO - Gene cds-WP_000167083.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,029 - INFO - Gene gene-SPN23F_RS00360 has a mutation rate of 0.82.\n",
      "2025-04-16 11:13:19,029 - INFO - Gene cds-WP_000772228.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,029 - INFO - Gene gene-SPN23F_RS13685 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:19,029 - INFO - Gene cds-WP_001809271.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,030 - INFO - Gene gene-SPN23F_RS00365 has a mutation rate of 0.18666666666666668.\n",
      "2025-04-16 11:13:19,030 - INFO - Gene cds-WP_000805601.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,030 - INFO - Gene gene-SPN23F_RS00370 has a mutation rate of 0.6833333333333333.\n",
      "2025-04-16 11:13:19,031 - INFO - Gene cds-WP_000099019.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,031 - INFO - Gene gene-SPN23F_RS00375 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,031 - INFO - Gene cds-WP_001813327.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,032 - INFO - Gene gene-SPN23F_RS00380 has a mutation rate of 0.29333333333333333.\n",
      "2025-04-16 11:13:19,032 - INFO - Gene cds-WP_000610746.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,032 - INFO - Gene gene-SPN23F_RS00385 has a mutation rate of 0.8333333333333334.\n",
      "2025-04-16 11:13:19,033 - INFO - Gene cds-WP_000679942.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,033 - INFO - Gene gene-SPN23F_RS00390 has a mutation rate of 0.26.\n",
      "2025-04-16 11:13:19,033 - INFO - Gene cds-WP_001007198.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,034 - INFO - Gene gene-SPN23F_RS00395 has a mutation rate of 0.6266666666666667.\n",
      "2025-04-16 11:13:19,034 - INFO - Gene cds-WP_000196572.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,034 - INFO - Gene gene-SPN23F_RS00400 has a mutation rate of 0.20333333333333334.\n",
      "2025-04-16 11:13:19,035 - INFO - Gene cds-WP_000156967.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,035 - INFO - Gene gene-SPN23F_RS00405 has a mutation rate of 0.18333333333333332.\n",
      "2025-04-16 11:13:19,035 - INFO - Gene cds-WP_000616999.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,035 - INFO - Gene gene-SPN23F_RS00410 has a mutation rate of 0.13333333333333333.\n",
      "2025-04-16 11:13:19,036 - INFO - Gene cds-WP_000185298.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,036 - INFO - Gene gene-SPN23F_RS00415 has a mutation rate of 0.17.\n",
      "2025-04-16 11:13:19,036 - INFO - Gene cds-WP_000167454.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,037 - INFO - Gene gene-SPN23F_RS00420 has a mutation rate of 0.5566666666666666.\n",
      "2025-04-16 11:13:19,037 - INFO - Gene cds-WP_000903552.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,037 - INFO - Gene gene-SPN23F_RS00425 has a mutation rate of 0.4266666666666667.\n",
      "2025-04-16 11:13:19,037 - INFO - Gene cds-WP_000649425.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,038 - INFO - Gene gene-SPN23F_RS00430 has a mutation rate of 0.75.\n",
      "2025-04-16 11:13:19,038 - INFO - Gene cds-WP_000220701.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,038 - INFO - Gene gene-SPN23F_RS00435 has a mutation rate of 0.48333333333333334.\n",
      "2025-04-16 11:13:19,039 - INFO - Gene cds-WP_000185151.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,039 - INFO - Gene gene-SPN23F_RS13355 has a mutation rate of 0.5233333333333333.\n",
      "2025-04-16 11:13:19,039 - INFO - Gene cds-WP_000695849.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,039 - INFO - Gene gene-SPN23F_RS00445 has a mutation rate of 0.36333333333333334.\n",
      "2025-04-16 11:13:19,040 - INFO - Gene cds-WP_000450181.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,040 - INFO - Gene cds-SPN23F_RS11750 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,040 - INFO - Gene gene-SPN23F_RS00450 has a mutation rate of 0.79.\n",
      "2025-04-16 11:13:19,041 - INFO - Gene cds-WP_000614999.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,041 - INFO - Gene gene-SPN23F_RS13180 has a mutation rate of 0.5266666666666666.\n",
      "2025-04-16 11:13:19,041 - INFO - Gene cds-WP_000849912.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,042 - INFO - Gene gene-SPN23F_RS00460 has a mutation rate of 0.2733333333333333.\n",
      "2025-04-16 11:13:19,042 - INFO - Gene cds-WP_000897726.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,042 - INFO - Gene gene-SPN23F_RS00465 has a mutation rate of 0.04666666666666667.\n",
      "2025-04-16 11:13:19,042 - INFO - Gene cds-WP_001283896.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,043 - INFO - Gene gene-SPN23F_RS13690 has a mutation rate of 0.5266666666666666.\n",
      "2025-04-16 11:13:19,043 - INFO - Gene cds-WP_000692796.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,044 - INFO - Gene gene-SPN23F_RS00475 has a mutation rate of 0.87.\n",
      "2025-04-16 11:13:19,044 - INFO - Gene cds-WP_000771901.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,044 - INFO - Gene gene-SPN23F_RS00480 has a mutation rate of 0.75.\n",
      "2025-04-16 11:13:19,045 - INFO - Gene cds-WP_000291603.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,045 - INFO - Gene cds-SPN23F_RS00485 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,045 - INFO - Gene gene-SPN23F_RS00490 has a mutation rate of 0.043333333333333335.\n",
      "2025-04-16 11:13:19,045 - INFO - Gene cds-WP_000520642.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,046 - INFO - Gene gene-SPN23F_RS00495 has a mutation rate of 0.21.\n",
      "2025-04-16 11:13:19,046 - INFO - Gene cds-WP_000769768.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,046 - INFO - Gene gene-SPN23F_RS00500 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,047 - INFO - Gene cds-WP_000092756.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,047 - INFO - Gene cds-SPN23F_RS11760 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,047 - INFO - Gene cds-SPN23F_RS12785 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,047 - INFO - Gene gene-SPN23F_RS00510 has a mutation rate of 0.49333333333333335.\n",
      "2025-04-16 11:13:19,048 - INFO - Gene cds-WP_000148253.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,048 - INFO - Gene gene-SPN23F_RS00515 has a mutation rate of 0.53.\n",
      "2025-04-16 11:13:19,048 - INFO - Gene cds-WP_001809441.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,048 - INFO - Gene gene-SPN23F_RS00520 has a mutation rate of 0.06666666666666667.\n",
      "2025-04-16 11:13:19,049 - INFO - Gene cds-WP_000714582.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,049 - INFO - Gene gene-SPN23F_RS00525 has a mutation rate of 0.03333333333333333.\n",
      "2025-04-16 11:13:19,050 - INFO - Gene cds-WP_001075102.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,050 - INFO - Gene gene-SPN23F_RS00530 has a mutation rate of 0.17666666666666667.\n",
      "2025-04-16 11:13:19,050 - INFO - Gene cds-WP_000800432.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,050 - INFO - Gene gene-SPN23F_RS00535 has a mutation rate of 0.06333333333333334.\n",
      "2025-04-16 11:13:19,051 - INFO - Gene cds-WP_001030031.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,051 - INFO - Gene gene-SPN23F_RS14055 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,051 - INFO - Gene cds-WP_001813665.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,052 - INFO - Gene gene-SPN23F_RS00540 has a mutation rate of 0.6966666666666667.\n",
      "2025-04-16 11:13:19,052 - INFO - Gene cds-WP_001140340.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,052 - INFO - Gene gene-SPN23F_RS00545 has a mutation rate of 0.6533333333333333.\n",
      "2025-04-16 11:13:19,053 - INFO - Gene cds-WP_000808442.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,053 - INFO - Gene gene-SPN23F_RS00550 has a mutation rate of 0.8666666666666667.\n",
      "2025-04-16 11:13:19,053 - INFO - Gene cds-WP_001233692.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,054 - INFO - Gene gene-SPN23F_RS00555 has a mutation rate of 0.5966666666666667.\n",
      "2025-04-16 11:13:19,054 - INFO - Gene cds-WP_000198687.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,054 - INFO - Gene gene-SPN23F_RS00560 has a mutation rate of 0.4866666666666667.\n",
      "2025-04-16 11:13:19,055 - INFO - Gene cds-WP_000273863.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,055 - INFO - Gene cds-SPN23F_RS00565 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,055 - INFO - Gene gene-SPN23F_RS00570 has a mutation rate of 0.8333333333333334.\n",
      "2025-04-16 11:13:19,055 - INFO - Gene cds-WP_000096197.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,056 - INFO - Gene gene-SPN23F_RS00575 has a mutation rate of 0.43666666666666665.\n",
      "2025-04-16 11:13:19,056 - INFO - Gene cds-WP_001035700.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,056 - INFO - Gene gene-SPN23F_RS00580 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,057 - INFO - Gene cds-WP_000436237.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,057 - INFO - Gene gene-SPN23F_RS13190 has a mutation rate of 0.6433333333333333.\n",
      "2025-04-16 11:13:19,057 - INFO - Gene cds-WP_000693348.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,058 - INFO - Gene gene-SPN23F_RS00590 has a mutation rate of 0.42333333333333334.\n",
      "2025-04-16 11:13:19,058 - INFO - Gene cds-WP_000200892.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,058 - INFO - Gene gene-SPN23F_RS00595 has a mutation rate of 0.5533333333333333.\n",
      "2025-04-16 11:13:19,059 - INFO - Gene cds-WP_000500038.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,059 - INFO - Gene gene-SPN23F_RS00600 has a mutation rate of 0.06333333333333334.\n",
      "2025-04-16 11:13:19,059 - INFO - Gene cds-WP_000838912.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,059 - INFO - Gene gene-SPN23F_RS00605 has a mutation rate of 0.33.\n",
      "2025-04-16 11:13:19,060 - INFO - Gene cds-WP_000836442.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,060 - INFO - Gene gene-SPN23F_RS00610 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,060 - INFO - Gene cds-WP_001861316.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,060 - INFO - Gene gene-SPN23F_RS00615 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,061 - INFO - Gene cds-WP_001230165.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,061 - INFO - Gene gene-SPN23F_RS00620 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,061 - INFO - Gene cds-WP_000724257.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,062 - INFO - Gene gene-SPN23F_RS00625 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,062 - INFO - Gene cds-WP_000571330.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,062 - INFO - Gene gene-SPN23F_RS00630 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,063 - INFO - Gene cds-WP_001038484.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,063 - INFO - Gene cds-SPN23F_RS00635 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,063 - INFO - Gene gene-SPN23F_RS00640 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,064 - INFO - Gene cds-WP_001861316.1-2 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,064 - INFO - Gene gene-SPN23F_RS00650 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,064 - INFO - Gene cds-WP_000436237.1-2 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,065 - INFO - Gene gene-SPN23F_RS13360 has a mutation rate of 0.27666666666666667.\n",
      "2025-04-16 11:13:19,065 - INFO - Gene cds-WP_001814148.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,065 - INFO - Gene gene-SPN23F_RS00665 has a mutation rate of 1.0.\n",
      "2025-04-16 11:13:19,066 - INFO - Gene cds-WP_001814150.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,066 - INFO - Gene cds-SPN23F_RS13695 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,066 - INFO - Gene gene-SPN23F_RS00670 has a mutation rate of 0.51.\n",
      "2025-04-16 11:13:19,067 - INFO - Gene cds-WP_000770312.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,067 - INFO - Gene gene-SPN23F_RS00675 has a mutation rate of 0.8566666666666667.\n",
      "2025-04-16 11:13:19,067 - INFO - Gene cds-WP_020901307.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,068 - INFO - Gene gene-SPN23F_RS13700 has a mutation rate of 0.8666666666666667.\n",
      "2025-04-16 11:13:19,068 - INFO - Gene cds-WP_231844674.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,068 - INFO - Gene cds-SPN23F_RS14150 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,068 - INFO - Gene gene-SPN23F_RS00685 has a mutation rate of 0.8166666666666667.\n",
      "2025-04-16 11:13:19,069 - INFO - Gene cds-WP_001282967.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,069 - INFO - Gene gene-SPN23F_RS00690 has a mutation rate of 0.6833333333333333.\n",
      "2025-04-16 11:13:19,069 - INFO - Gene cds-WP_000193610.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,070 - INFO - Gene gene-SPN23F_RS00695 has a mutation rate of 0.59.\n",
      "2025-04-16 11:13:19,070 - INFO - Gene cds-WP_000220946.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,070 - INFO - Gene gene-SPN23F_RS13710 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,070 - INFO - Gene cds-WP_001838083.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,071 - INFO - Gene gene-SPN23F_RS00705 has a mutation rate of 0.29.\n",
      "2025-04-16 11:13:19,071 - INFO - Gene cds-WP_000331980.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,071 - INFO - Gene gene-SPN23F_RS00710 has a mutation rate of 0.31.\n",
      "2025-04-16 11:13:19,071 - INFO - Gene cds-WP_000639577.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,072 - INFO - Gene cds-SPN23F_RS13715 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,072 - INFO - Gene gene-SPN23F_RS00715 has a mutation rate of 0.15.\n",
      "2025-04-16 11:13:19,072 - INFO - Gene cds-WP_000977347.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,073 - INFO - Gene gene-SPN23F_RS00720 has a mutation rate of 0.20666666666666667.\n",
      "2025-04-16 11:13:19,073 - INFO - Gene cds-WP_000180803.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,074 - INFO - Gene gene-SPN23F_RS00725 has a mutation rate of 1.0.\n",
      "2025-04-16 11:13:19,074 - INFO - Gene cds-WP_000865706.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,074 - INFO - Gene gene-SPN23F_RS00730 has a mutation rate of 0.6366666666666667.\n",
      "2025-04-16 11:13:19,075 - INFO - Gene cds-WP_000569831.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,075 - INFO - Gene gene-SPN23F_RS00735 has a mutation rate of 0.77.\n",
      "2025-04-16 11:13:19,075 - INFO - Gene cds-WP_000655033.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,076 - INFO - Gene cds-SPN23F_RS00745 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,076 - INFO - Gene cds-SPN23F_RS13720 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,076 - INFO - Gene gene-SPN23F_RS13725 has a mutation rate of 0.22333333333333333.\n",
      "2025-04-16 11:13:19,077 - INFO - Gene cds-WP_001810713.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,077 - INFO - Gene gene-SPN23F_RS13730 has a mutation rate of 0.6066666666666667.\n",
      "2025-04-16 11:13:19,077 - INFO - Gene cds-WP_000371189.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,078 - INFO - Gene gene-SPN23F_RS00770 has a mutation rate of 0.43.\n",
      "2025-04-16 11:13:19,078 - INFO - Gene cds-WP_000375722.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,078 - INFO - Gene gene-SPN23F_RS00775 has a mutation rate of 0.3333333333333333.\n",
      "2025-04-16 11:13:19,078 - INFO - Gene cds-WP_000288029.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,079 - INFO - Gene cds-SPN23F_RS14240 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,079 - INFO - Gene gene-SPN23F_RS00785 has a mutation rate of 0.37333333333333335.\n",
      "2025-04-16 11:13:19,079 - INFO - Gene cds-WP_001290270.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,079 - INFO - Gene gene-SPN23F_RS00790 has a mutation rate of 0.5766666666666667.\n",
      "2025-04-16 11:13:19,080 - INFO - Gene cds-WP_000069860.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,080 - INFO - Gene gene-SPN23F_RS00795 has a mutation rate of 0.5666666666666667.\n",
      "2025-04-16 11:13:19,080 - INFO - Gene cds-WP_001017600.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,081 - INFO - Gene gene-SPN23F_RS00800 has a mutation rate of 0.7733333333333333.\n",
      "2025-04-16 11:13:19,081 - INFO - Gene cds-WP_000570720.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,081 - INFO - Gene gene-SPN23F_RS00805 has a mutation rate of 0.36666666666666664.\n",
      "2025-04-16 11:13:19,081 - INFO - Gene cds-WP_000933464.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,082 - INFO - Gene gene-SPN23F_RS00810 has a mutation rate of 0.7066666666666667.\n",
      "2025-04-16 11:13:19,082 - INFO - Gene cds-WP_000720790.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,082 - INFO - Gene gene-SPN23F_RS00815 has a mutation rate of 0.97.\n",
      "2025-04-16 11:13:19,083 - INFO - Gene cds-WP_000862430.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,083 - INFO - Gene gene-SPN23F_RS00825 has a mutation rate of 0.75.\n",
      "2025-04-16 11:13:19,084 - INFO - Gene cds-WP_001141646.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,084 - INFO - Gene cds-SPN23F_RS13735 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,085 - INFO - Gene gene-SPN23F_RS00835 has a mutation rate of 0.23333333333333334.\n",
      "2025-04-16 11:13:19,085 - INFO - Gene cds-WP_000724936.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,085 - INFO - Gene gene-SPN23F_RS00840 has a mutation rate of 0.43333333333333335.\n",
      "2025-04-16 11:13:19,085 - INFO - Gene cds-WP_000694512.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,086 - INFO - Gene gene-SPN23F_RS00845 has a mutation rate of 0.6433333333333333.\n",
      "2025-04-16 11:13:19,086 - INFO - Gene cds-WP_000231785.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,086 - INFO - Gene gene-SPN23F_RS00850 has a mutation rate of 0.5933333333333334.\n",
      "2025-04-16 11:13:19,087 - INFO - Gene cds-WP_000085677.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,087 - INFO - Gene gene-SPN23F_RS00855 has a mutation rate of 0.45.\n",
      "2025-04-16 11:13:19,087 - INFO - Gene cds-WP_000444642.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,087 - INFO - Gene gene-SPN23F_RS00860 has a mutation rate of 0.67.\n",
      "2025-04-16 11:13:19,088 - INFO - Gene cds-WP_000748391.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,088 - INFO - Gene gene-SPN23F_RS00865 has a mutation rate of 0.46.\n",
      "2025-04-16 11:13:19,088 - INFO - Gene cds-WP_000532353.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,089 - INFO - Gene gene-SPN23F_RS00870 has a mutation rate of 0.76.\n",
      "2025-04-16 11:13:19,089 - INFO - Gene cds-WP_000800582.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,089 - INFO - Gene gene-SPN23F_RS00875 has a mutation rate of 0.54.\n",
      "2025-04-16 11:13:19,090 - INFO - Gene cds-WP_000278896.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,090 - INFO - Gene gene-SPN23F_RS00880 has a mutation rate of 0.04.\n",
      "2025-04-16 11:13:19,090 - INFO - Gene cds-WP_000729035.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,091 - INFO - Gene gene-SPN23F_RS00885 has a mutation rate of 0.1.\n",
      "2025-04-16 11:13:19,091 - INFO - Gene cds-WP_000849621.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,091 - INFO - Gene gene-SPN23F_RS00890 has a mutation rate of 0.17.\n",
      "2025-04-16 11:13:19,092 - INFO - Gene cds-WP_000075789.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,092 - INFO - Gene gene-SPN23F_RS00895 has a mutation rate of 0.5166666666666667.\n",
      "2025-04-16 11:13:19,092 - INFO - Gene cds-WP_000745054.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,093 - INFO - Gene gene-SPN23F_RS00900 has a mutation rate of 0.36333333333333334.\n",
      "2025-04-16 11:13:19,093 - INFO - Gene cds-WP_000861955.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,093 - INFO - Gene gene-SPN23F_RS14060 has a mutation rate of 0.4.\n",
      "2025-04-16 11:13:19,094 - INFO - Gene cds-WP_001813544.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,094 - INFO - Gene gene-SPN23F_RS00905 has a mutation rate of 0.32.\n",
      "2025-04-16 11:13:19,094 - INFO - Gene cds-WP_000018169.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,095 - INFO - Gene gene-SPN23F_RS00910 has a mutation rate of 0.15666666666666668.\n",
      "2025-04-16 11:13:19,095 - INFO - Gene cds-WP_001099502.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,095 - INFO - Gene gene-SPN23F_RS00915 has a mutation rate of 0.5933333333333334.\n",
      "2025-04-16 11:13:19,096 - INFO - Gene cds-WP_000456639.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,096 - INFO - Gene gene-SPN23F_RS00920 has a mutation rate of 0.6866666666666666.\n",
      "2025-04-16 11:13:19,096 - INFO - Gene cds-WP_000493841.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,097 - INFO - Gene gene-SPN23F_RS00925 has a mutation rate of 0.5833333333333334.\n",
      "2025-04-16 11:13:19,097 - INFO - Gene cds-WP_001284100.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,097 - INFO - Gene gene-SPN23F_RS00930 has a mutation rate of 0.56.\n",
      "2025-04-16 11:13:19,097 - INFO - Gene cds-WP_000271489.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,098 - INFO - Gene gene-SPN23F_RS00935 has a mutation rate of 0.9366666666666666.\n",
      "2025-04-16 11:13:19,098 - INFO - Gene cds-WP_000166600.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,098 - INFO - Gene gene-SPN23F_RS00940 has a mutation rate of 0.75.\n",
      "2025-04-16 11:13:19,099 - INFO - Gene cds-WP_000987429.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,099 - INFO - Gene gene-SPN23F_RS00945 has a mutation rate of 0.76.\n",
      "2025-04-16 11:13:19,099 - INFO - Gene cds-WP_000255102.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,100 - INFO - Gene gene-SPN23F_RS00950 has a mutation rate of 0.25666666666666665.\n",
      "2025-04-16 11:13:19,100 - INFO - Gene cds-WP_000078027.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,100 - INFO - Gene gene-SPN23F_RS00955 has a mutation rate of 0.02666666666666667.\n",
      "2025-04-16 11:13:19,101 - INFO - Gene cds-WP_000815629.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,101 - INFO - Gene gene-SPN23F_RS00960 has a mutation rate of 0.6533333333333333.\n",
      "2025-04-16 11:13:19,101 - INFO - Gene cds-WP_001152894.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,101 - INFO - Gene gene-SPN23F_RS00965 has a mutation rate of 0.4666666666666667.\n",
      "2025-04-16 11:13:19,102 - INFO - Gene cds-WP_001042860.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,102 - INFO - Gene gene-SPN23F_RS00970 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,102 - INFO - Gene cds-WP_000591165.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,103 - INFO - Gene gene-SPN23F_RS00975 has a mutation rate of 0.47333333333333333.\n",
      "2025-04-16 11:13:19,103 - INFO - Gene cds-WP_000725133.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,103 - INFO - Gene gene-SPN23F_RS00980 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,103 - INFO - Gene cds-WP_000507059.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,104 - INFO - Gene gene-SPN23F_RS00985 has a mutation rate of 0.08.\n",
      "2025-04-16 11:13:19,104 - INFO - Gene cds-WP_001808830.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,104 - INFO - Gene gene-SPN23F_RS00990 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,104 - INFO - Gene cds-WP_000017620.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,105 - INFO - Gene gene-SPN23F_RS00995 has a mutation rate of 0.8366666666666667.\n",
      "2025-04-16 11:13:19,105 - INFO - Gene cds-WP_000464336.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,105 - INFO - Gene gene-SPN23F_RS01000 has a mutation rate of 0.15.\n",
      "2025-04-16 11:13:19,106 - INFO - Gene cds-WP_000769490.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,106 - INFO - Gene gene-SPN23F_RS01005 has a mutation rate of 0.15.\n",
      "2025-04-16 11:13:19,106 - INFO - Gene cds-WP_000877334.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,107 - INFO - Gene gene-SPN23F_RS13460 has a mutation rate of 0.02666666666666667.\n",
      "2025-04-16 11:13:19,107 - INFO - Gene cds-WP_000956126.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,107 - INFO - Gene gene-SPN23F_RS01015 has a mutation rate of 0.5766666666666667.\n",
      "2025-04-16 11:13:19,108 - INFO - Gene cds-WP_000118100.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,108 - INFO - Gene gene-SPN23F_RS01020 has a mutation rate of 0.56.\n",
      "2025-04-16 11:13:19,108 - INFO - Gene cds-WP_000560899.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,109 - INFO - Gene gene-SPN23F_RS13465 has a mutation rate of 0.14333333333333334.\n",
      "2025-04-16 11:13:19,109 - INFO - Gene cds-WP_000521619.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,109 - INFO - Gene gene-SPN23F_RS01030 has a mutation rate of 0.6366666666666667.\n",
      "2025-04-16 11:13:19,110 - INFO - Gene cds-WP_000423414.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,110 - INFO - Gene gene-SPN23F_RS01035 has a mutation rate of 0.4866666666666667.\n",
      "2025-04-16 11:13:19,110 - INFO - Gene cds-WP_001064208.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,110 - INFO - Gene gene-SPN23F_RS01040 has a mutation rate of 0.33666666666666667.\n",
      "2025-04-16 11:13:19,111 - INFO - Gene cds-WP_000725993.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,111 - INFO - Gene gene-SPN23F_RS01045 has a mutation rate of 0.013333333333333334.\n",
      "2025-04-16 11:13:19,111 - INFO - Gene cds-WP_001284513.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,112 - INFO - Gene gene-SPN23F_RS01050 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:19,112 - INFO - Gene cds-WP_000160197.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,112 - INFO - Gene gene-SPN23F_RS01055 has a mutation rate of 0.04666666666666667.\n",
      "2025-04-16 11:13:19,113 - INFO - Gene cds-WP_000024537.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,113 - INFO - Gene gene-SPN23F_RS01060 has a mutation rate of 0.023333333333333334.\n",
      "2025-04-16 11:13:19,113 - INFO - Gene cds-WP_001055347.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,114 - INFO - Gene gene-SPN23F_RS01065 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,114 - INFO - Gene cds-WP_000512911.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,115 - INFO - Gene gene-SPN23F_RS01070 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,115 - INFO - Gene cds-WP_000533766.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,115 - INFO - Gene gene-SPN23F_RS01075 has a mutation rate of 0.023333333333333334.\n",
      "2025-04-16 11:13:19,115 - INFO - Gene cds-WP_000818137.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,116 - INFO - Gene gene-SPN23F_RS01080 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,116 - INFO - Gene cds-WP_000529936.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,116 - INFO - Gene gene-SPN23F_RS01085 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,117 - INFO - Gene cds-WP_000960946.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,117 - INFO - Gene gene-SPN23F_RS01090 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,117 - INFO - Gene cds-WP_000772918.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,118 - INFO - Gene gene-SPN23F_RS01095 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,118 - INFO - Gene cds-WP_000440801.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,118 - INFO - Gene gene-SPN23F_RS01100 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,118 - INFO - Gene cds-WP_000616545.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,119 - INFO - Gene gene-SPN23F_RS01105 has a mutation rate of 0.013333333333333334.\n",
      "2025-04-16 11:13:19,119 - INFO - Gene cds-WP_000497691.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,119 - INFO - Gene gene-SPN23F_RS01110 has a mutation rate of 0.1.\n",
      "2025-04-16 11:13:19,120 - INFO - Gene cds-WP_000013542.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,120 - INFO - Gene gene-SPN23F_RS01115 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,120 - INFO - Gene cds-WP_001085703.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,120 - INFO - Gene gene-SPN23F_RS01120 has a mutation rate of 0.03.\n",
      "2025-04-16 11:13:19,121 - INFO - Gene cds-WP_000245505.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,121 - INFO - Gene gene-SPN23F_RS01125 has a mutation rate of 0.42.\n",
      "2025-04-16 11:13:19,121 - INFO - Gene cds-WP_000086630.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,121 - INFO - Gene gene-SPN23F_RS01130 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,122 - INFO - Gene cds-WP_000624044.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,122 - INFO - Gene gene-SPN23F_RS01135 has a mutation rate of 0.013333333333333334.\n",
      "2025-04-16 11:13:19,122 - INFO - Gene cds-WP_000874201.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,123 - INFO - Gene gene-SPN23F_RS01140 has a mutation rate of 0.13.\n",
      "2025-04-16 11:13:19,123 - INFO - Gene cds-WP_000057241.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,123 - INFO - Gene gene-SPN23F_RS01145 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,124 - INFO - Gene cds-WP_000766087.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,124 - INFO - Gene gene-SPN23F_RS01150 has a mutation rate of 0.02666666666666667.\n",
      "2025-04-16 11:13:19,124 - INFO - Gene cds-WP_000465397.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,125 - INFO - Gene gene-SPN23F_RS01155 has a mutation rate of 0.35333333333333333.\n",
      "2025-04-16 11:13:19,125 - INFO - Gene cds-WP_001050431.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,126 - INFO - Gene gene-SPN23F_RS01160 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,126 - INFO - Gene cds-WP_001029883.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,126 - INFO - Gene gene-SPN23F_RS01165 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,126 - INFO - Gene cds-WP_001808836.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,127 - INFO - Gene gene-SPN23F_RS01170 has a mutation rate of 0.013333333333333334.\n",
      "2025-04-16 11:13:19,127 - INFO - Gene cds-WP_000090781.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,127 - INFO - Gene gene-SPN23F_RS01175 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,127 - INFO - Gene cds-WP_001118385.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,128 - INFO - Gene gene-SPN23F_RS01180 has a mutation rate of 0.03.\n",
      "2025-04-16 11:13:19,128 - INFO - Gene cds-WP_000568988.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,128 - INFO - Gene gene-SPN23F_RS01185 has a mutation rate of 0.023333333333333334.\n",
      "2025-04-16 11:13:19,128 - INFO - Gene cds-WP_000331493.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,129 - INFO - Gene gene-SPN23F_RS01190 has a mutation rate of 0.19666666666666666.\n",
      "2025-04-16 11:13:19,129 - INFO - Gene cds-WP_000644124.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,129 - INFO - Gene gene-SPN23F_RS01195 has a mutation rate of 0.58.\n",
      "2025-04-16 11:13:19,130 - INFO - Gene cds-WP_000354918.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,130 - INFO - Gene gene-SPN23F_RS01200 has a mutation rate of 0.16.\n",
      "2025-04-16 11:13:19,130 - INFO - Gene cds-WP_000241536.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,130 - INFO - Gene cds-SPN23F_RS13205 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,131 - INFO - Gene gene-SPN23F_RS01215 has a mutation rate of 0.9933333333333333.\n",
      "2025-04-16 11:13:19,131 - INFO - Gene cds-WP_001289362.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,131 - INFO - Gene cds-SPN23F_RS11835 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,131 - INFO - Gene cds-SPN23F_RS01230 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,132 - INFO - Gene gene-SPN23F_RS01235 has a mutation rate of 0.5133333333333333.\n",
      "2025-04-16 11:13:19,132 - INFO - Gene cds-WP_000406171.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,132 - INFO - Gene gene-SPN23F_RS01240 has a mutation rate of 0.4866666666666667.\n",
      "2025-04-16 11:13:19,133 - INFO - Gene cds-WP_000824966.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,133 - INFO - Gene gene-SPN23F_RS01245 has a mutation rate of 0.31333333333333335.\n",
      "2025-04-16 11:13:19,133 - INFO - Gene cds-WP_001070840.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,134 - INFO - Gene gene-SPN23F_RS01250 has a mutation rate of 0.09333333333333334.\n",
      "2025-04-16 11:13:19,134 - INFO - Gene cds-WP_000402908.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,134 - INFO - Gene gene-SPN23F_RS01255 has a mutation rate of 0.3433333333333333.\n",
      "2025-04-16 11:13:19,134 - INFO - Gene cds-WP_000238716.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,135 - INFO - Gene gene-SPN23F_RS01260 has a mutation rate of 0.9133333333333333.\n",
      "2025-04-16 11:13:19,135 - INFO - Gene cds-WP_001096822.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,135 - INFO - Gene gene-SPN23F_RS01265 has a mutation rate of 0.58.\n",
      "2025-04-16 11:13:19,136 - INFO - Gene cds-WP_000248305.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,136 - INFO - Gene gene-SPN23F_RS01270 has a mutation rate of 0.28.\n",
      "2025-04-16 11:13:19,136 - INFO - Gene cds-WP_000395437.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,137 - INFO - Gene gene-SPN23F_RS01275 has a mutation rate of 0.4166666666666667.\n",
      "2025-04-16 11:13:19,137 - INFO - Gene cds-WP_001217828.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,137 - INFO - Gene gene-SPN23F_RS01280 has a mutation rate of 0.9366666666666666.\n",
      "2025-04-16 11:13:19,137 - INFO - Gene cds-WP_000011766.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,138 - INFO - Gene gene-SPN23F_RS01285 has a mutation rate of 0.9766666666666667.\n",
      "2025-04-16 11:13:19,138 - INFO - Gene cds-WP_001144922.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,138 - INFO - Gene gene-SPN23F_RS01290 has a mutation rate of 0.7166666666666667.\n",
      "2025-04-16 11:13:19,138 - INFO - Gene cds-WP_000628392.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,139 - INFO - Gene gene-SPN23F_RS01295 has a mutation rate of 0.19.\n",
      "2025-04-16 11:13:19,139 - INFO - Gene cds-WP_000970381.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,139 - INFO - Gene gene-SPN23F_RS01300 has a mutation rate of 0.6866666666666666.\n",
      "2025-04-16 11:13:19,140 - INFO - Gene cds-WP_001860761.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,140 - INFO - Gene gene-SPN23F_RS01305 has a mutation rate of 0.8933333333333333.\n",
      "2025-04-16 11:13:19,140 - INFO - Gene cds-WP_001098895.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,141 - INFO - Gene gene-SPN23F_RS01310 has a mutation rate of 0.23333333333333334.\n",
      "2025-04-16 11:13:19,141 - INFO - Gene cds-WP_000466719.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,142 - INFO - Gene gene-SPN23F_RS01315 has a mutation rate of 0.24666666666666667.\n",
      "2025-04-16 11:13:19,142 - INFO - Gene cds-WP_000189848.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,142 - INFO - Gene gene-SPN23F_RS01320 has a mutation rate of 0.36.\n",
      "2025-04-16 11:13:19,143 - INFO - Gene cds-WP_000900659.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,143 - INFO - Gene gene-SPN23F_RS01325 has a mutation rate of 0.3466666666666667.\n",
      "2025-04-16 11:13:19,143 - INFO - Gene cds-WP_000814080.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,144 - INFO - Gene gene-SPN23F_RS01330 has a mutation rate of 0.6766666666666666.\n",
      "2025-04-16 11:13:19,145 - INFO - Gene cds-WP_000943525.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,172 - INFO - Gene gene-SPN23F_RS01335 has a mutation rate of 0.7933333333333333.\n",
      "2025-04-16 11:13:19,172 - INFO - Gene cds-WP_000334246.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,173 - INFO - Gene gene-SPN23F_RS01340 has a mutation rate of 0.7233333333333334.\n",
      "2025-04-16 11:13:19,174 - INFO - Gene cds-WP_000229772.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,174 - INFO - Gene gene-SPN23F_RS01345 has a mutation rate of 0.88.\n",
      "2025-04-16 11:13:19,175 - INFO - Gene cds-WP_001232813.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,176 - INFO - Gene gene-SPN23F_RS01350 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,176 - INFO - Gene cds-WP_001142334.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,180 - INFO - Gene gene-SPN23F_RS01355 has a mutation rate of 0.07.\n",
      "2025-04-16 11:13:19,183 - INFO - Gene cds-WP_000087873.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,184 - INFO - Gene gene-SPN23F_RS01360 has a mutation rate of 0.4766666666666667.\n",
      "2025-04-16 11:13:19,184 - INFO - Gene cds-WP_000090357.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,185 - INFO - Gene gene-SPN23F_RS01365 has a mutation rate of 0.8866666666666667.\n",
      "2025-04-16 11:13:19,185 - INFO - Gene cds-WP_000071374.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,185 - INFO - Gene gene-SPN23F_RS01370 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,186 - INFO - Gene cds-WP_000436644.1-2 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,186 - INFO - Gene gene-SPN23F_RS01375 has a mutation rate of 0.14333333333333334.\n",
      "2025-04-16 11:13:19,186 - INFO - Gene cds-WP_000047864.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,187 - INFO - Gene cds-SPN23F_RS01380 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,187 - INFO - Gene gene-SPN23F_RS13470 has a mutation rate of 0.14333333333333334.\n",
      "2025-04-16 11:13:19,187 - INFO - Gene cds-WP_000776626.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,187 - INFO - Gene gene-SPN23F_RS01390 has a mutation rate of 0.5633333333333334.\n",
      "2025-04-16 11:13:19,188 - INFO - Gene cds-WP_000243914.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,188 - INFO - Gene gene-SPN23F_RS01395 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,188 - INFO - Gene cds-WP_000901569.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,188 - INFO - Gene gene-SPN23F_RS13740 has a mutation rate of 0.03666666666666667.\n",
      "2025-04-16 11:13:19,189 - INFO - Gene cds-WP_001815638.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,189 - INFO - Gene gene-SPN23F_RS01400 has a mutation rate of 0.6333333333333333.\n",
      "2025-04-16 11:13:19,189 - INFO - Gene cds-WP_001815639.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,190 - INFO - Gene gene-SPN23F_RS01410 has a mutation rate of 0.53.\n",
      "2025-04-16 11:13:19,190 - INFO - Gene cds-WP_000991679.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,190 - INFO - Gene gene-SPN23F_RS01415 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,191 - INFO - Gene cds-WP_000136861.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,191 - INFO - Gene gene-SPN23F_RS01420 has a mutation rate of 0.07666666666666666.\n",
      "2025-04-16 11:13:19,191 - INFO - Gene cds-WP_000022715.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,192 - INFO - Gene gene-SPN23F_RS01425 has a mutation rate of 0.13666666666666666.\n",
      "2025-04-16 11:13:19,192 - INFO - Gene cds-WP_000021975.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,193 - INFO - Gene gene-SPN23F_RS01430 has a mutation rate of 0.57.\n",
      "2025-04-16 11:13:19,193 - INFO - Gene cds-WP_000649170.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,194 - INFO - Gene gene-SPN23F_RS01435 has a mutation rate of 0.6466666666666666.\n",
      "2025-04-16 11:13:19,194 - INFO - Gene cds-WP_000162352.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,194 - INFO - Gene gene-SPN23F_RS01440 has a mutation rate of 0.6166666666666667.\n",
      "2025-04-16 11:13:19,195 - INFO - Gene cds-WP_000359298.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,195 - INFO - Gene gene-SPN23F_RS01445 has a mutation rate of 0.7666666666666667.\n",
      "2025-04-16 11:13:19,195 - INFO - Gene cds-WP_000660466.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,196 - INFO - Gene gene-SPN23F_RS01450 has a mutation rate of 0.7066666666666667.\n",
      "2025-04-16 11:13:19,196 - INFO - Gene cds-WP_000240956.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,196 - INFO - Gene gene-SPN23F_RS01455 has a mutation rate of 0.98.\n",
      "2025-04-16 11:13:19,197 - INFO - Gene cds-WP_000659086.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,197 - INFO - Gene gene-SPN23F_RS01460 has a mutation rate of 0.22666666666666666.\n",
      "2025-04-16 11:13:19,197 - INFO - Gene cds-WP_000380917.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,198 - INFO - Gene gene-SPN23F_RS01465 has a mutation rate of 0.22.\n",
      "2025-04-16 11:13:19,198 - INFO - Gene cds-WP_000372516.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,198 - INFO - Gene gene-SPN23F_RS01470 has a mutation rate of 0.03.\n",
      "2025-04-16 11:13:19,198 - INFO - Gene cds-WP_000886031.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,199 - INFO - Gene gene-SPN23F_RS01475 has a mutation rate of 0.1.\n",
      "2025-04-16 11:13:19,199 - INFO - Gene cds-WP_001044624.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,200 - INFO - Gene gene-SPN23F_RS01480 has a mutation rate of 0.04.\n",
      "2025-04-16 11:13:19,200 - INFO - Gene cds-WP_000075966.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,200 - INFO - Gene gene-SPN23F_RS01485 has a mutation rate of 0.2966666666666667.\n",
      "2025-04-16 11:13:19,200 - INFO - Gene cds-WP_001211068.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,201 - INFO - Gene gene-SPN23F_RS13475 has a mutation rate of 0.05333333333333334.\n",
      "2025-04-16 11:13:19,201 - INFO - Gene cds-WP_000050748.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,201 - INFO - Gene gene-SPN23F_RS01495 has a mutation rate of 0.8833333333333333.\n",
      "2025-04-16 11:13:19,202 - INFO - Gene cds-WP_000688748.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,202 - INFO - Gene gene-SPN23F_RS01500 has a mutation rate of 0.61.\n",
      "2025-04-16 11:13:19,202 - INFO - Gene cds-WP_000203063.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,203 - INFO - Gene gene-SPN23F_RS01505 has a mutation rate of 0.8833333333333333.\n",
      "2025-04-16 11:13:19,203 - INFO - Gene cds-WP_001193713.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,203 - INFO - Gene cds-SPN23F_RS01510 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,203 - INFO - Gene gene-SPN23F_RS01515 has a mutation rate of 0.89.\n",
      "2025-04-16 11:13:19,204 - INFO - Gene cds-WP_000167295.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,204 - INFO - Gene gene-SPN23F_RS01520 has a mutation rate of 0.4666666666666667.\n",
      "2025-04-16 11:13:19,204 - INFO - Gene cds-WP_000161480.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,205 - INFO - Gene gene-SPN23F_RS01525 has a mutation rate of 0.5666666666666667.\n",
      "2025-04-16 11:13:19,205 - INFO - Gene cds-WP_000684786.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,245 - INFO - Gene gene-SPN23F_RS01530 has a mutation rate of 0.6033333333333334.\n",
      "2025-04-16 11:13:19,245 - INFO - Gene cds-WP_000185874.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,246 - INFO - Gene gene-SPN23F_RS01535 has a mutation rate of 0.28.\n",
      "2025-04-16 11:13:19,246 - INFO - Gene cds-WP_000706826.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,246 - INFO - Gene gene-SPN23F_RS01540 has a mutation rate of 0.5066666666666667.\n",
      "2025-04-16 11:13:19,247 - INFO - Gene cds-WP_000592943.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,247 - INFO - Gene gene-SPN23F_RS01545 has a mutation rate of 0.21333333333333335.\n",
      "2025-04-16 11:13:19,247 - INFO - Gene cds-WP_000178622.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,247 - INFO - Gene gene-SPN23F_RS01550 has a mutation rate of 0.12.\n",
      "2025-04-16 11:13:19,248 - INFO - Gene cds-WP_000026610.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,248 - INFO - Gene gene-SPN23F_RS01555 has a mutation rate of 0.5066666666666667.\n",
      "2025-04-16 11:13:19,248 - INFO - Gene cds-WP_000148012.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,249 - INFO - Gene gene-SPN23F_RS01560 has a mutation rate of 0.31333333333333335.\n",
      "2025-04-16 11:13:19,249 - INFO - Gene cds-WP_000381739.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,249 - INFO - Gene gene-SPN23F_RS01565 has a mutation rate of 0.88.\n",
      "2025-04-16 11:13:19,250 - INFO - Gene cds-WP_000657078.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,250 - INFO - Gene gene-SPN23F_RS01570 has a mutation rate of 0.14.\n",
      "2025-04-16 11:13:19,250 - INFO - Gene cds-WP_000412453.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,251 - INFO - Gene gene-SPN23F_RS01575 has a mutation rate of 0.37333333333333335.\n",
      "2025-04-16 11:13:19,251 - INFO - Gene cds-WP_000838117.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,251 - INFO - Gene gene-SPN23F_RS01580 has a mutation rate of 0.43666666666666665.\n",
      "2025-04-16 11:13:19,252 - INFO - Gene cds-WP_000711739.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,252 - INFO - Gene gene-SPN23F_RS01585 has a mutation rate of 0.17666666666666667.\n",
      "2025-04-16 11:13:19,252 - INFO - Gene cds-WP_001082474.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,253 - INFO - Gene gene-SPN23F_RS01590 has a mutation rate of 0.44.\n",
      "2025-04-16 11:13:19,253 - INFO - Gene cds-WP_000159418.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,253 - INFO - Gene gene-SPN23F_RS01595 has a mutation rate of 0.18333333333333332.\n",
      "2025-04-16 11:13:19,253 - INFO - Gene cds-WP_000818547.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,254 - INFO - Gene gene-SPN23F_RS01600 has a mutation rate of 0.44333333333333336.\n",
      "2025-04-16 11:13:19,254 - INFO - Gene cds-WP_000872289.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,254 - INFO - Gene gene-SPN23F_RS01605 has a mutation rate of 0.8833333333333333.\n",
      "2025-04-16 11:13:19,255 - INFO - Gene cds-WP_000470846.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,255 - INFO - Gene gene-SPN23F_RS01610 has a mutation rate of 0.93.\n",
      "2025-04-16 11:13:19,255 - INFO - Gene cds-WP_001063398.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,255 - INFO - Gene cds-SPN23F_RS01615 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,256 - INFO - Gene gene-SPN23F_RS01620 has a mutation rate of 0.12.\n",
      "2025-04-16 11:13:19,256 - INFO - Gene cds-WP_000032551.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,257 - INFO - Gene gene-SPN23F_RS01625 has a mutation rate of 0.8266666666666667.\n",
      "2025-04-16 11:13:19,257 - INFO - Gene cds-WP_000743617.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,257 - INFO - Gene gene-SPN23F_RS01630 has a mutation rate of 0.9766666666666667.\n",
      "2025-04-16 11:13:19,258 - INFO - Gene cds-WP_001156826.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,258 - INFO - Gene gene-SPN23F_RS01635 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,259 - INFO - Gene cds-WP_001022228.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,259 - INFO - Gene cds-SPN23F_RS11870 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,259 - INFO - Gene gene-SPN23F_RS01645 has a mutation rate of 0.9533333333333334.\n",
      "2025-04-16 11:13:19,260 - INFO - Gene cds-WP_000091067.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,260 - INFO - Gene gene-SPN23F_RS01650 has a mutation rate of 0.98.\n",
      "2025-04-16 11:13:19,260 - INFO - Gene cds-WP_000565369.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,260 - INFO - Gene gene-SPN23F_RS01655 has a mutation rate of 0.72.\n",
      "2025-04-16 11:13:19,261 - INFO - Gene cds-WP_000664144.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,261 - INFO - Gene gene-SPN23F_RS01660 has a mutation rate of 0.69.\n",
      "2025-04-16 11:13:19,261 - INFO - Gene cds-WP_001142541.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,262 - INFO - Gene gene-SPN23F_RS01665 has a mutation rate of 0.8166666666666667.\n",
      "2025-04-16 11:13:19,262 - INFO - Gene cds-WP_001003129.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,262 - INFO - Gene gene-SPN23F_RS01670 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,262 - INFO - Gene cds-WP_000750326.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,263 - INFO - Gene gene-SPN23F_RS01675 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,263 - INFO - Gene cds-WP_001814163.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,263 - INFO - Gene gene-SPN23F_RS01680 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,264 - INFO - Gene cds-WP_000413826.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,264 - INFO - Gene gene-SPN23F_RS01685 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,264 - INFO - Gene cds-WP_001045980.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,265 - INFO - Gene gene-SPN23F_RS01690 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,265 - INFO - Gene cds-WP_000049683.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,265 - INFO - Gene gene-SPN23F_RS01695 has a mutation rate of 0.09.\n",
      "2025-04-16 11:13:19,265 - INFO - Gene cds-WP_000636508.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,266 - INFO - Gene gene-SPN23F_RS01700 has a mutation rate of 0.04666666666666667.\n",
      "2025-04-16 11:13:19,266 - INFO - Gene cds-WP_000794266.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,267 - INFO - Gene gene-SPN23F_RS01705 has a mutation rate of 0.11666666666666667.\n",
      "2025-04-16 11:13:19,267 - INFO - Gene cds-WP_000645462.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,268 - INFO - Gene gene-SPN23F_RS01710 has a mutation rate of 0.03.\n",
      "2025-04-16 11:13:19,268 - INFO - Gene cds-WP_000779205.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,268 - INFO - Gene gene-SPN23F_RS01715 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,269 - INFO - Gene cds-WP_000676145.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,269 - INFO - Gene gene-SPN23F_RS01720 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,269 - INFO - Gene cds-WP_000131451.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,270 - INFO - Gene gene-SPN23F_RS01725 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,270 - INFO - Gene cds-WP_000141499.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,270 - INFO - Gene gene-SPN23F_RS01730 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,271 - INFO - Gene cds-WP_000600912.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,271 - INFO - Gene cds-SPN23F_RS13750 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,271 - INFO - Gene gene-SPN23F_RS01745 has a mutation rate of 0.8066666666666666.\n",
      "2025-04-16 11:13:19,272 - INFO - Gene cds-WP_000842557.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,272 - INFO - Gene gene-SPN23F_RS01750 has a mutation rate of 0.9933333333333333.\n",
      "2025-04-16 11:13:19,272 - INFO - Gene cds-WP_001032454.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,272 - INFO - Gene gene-SPN23F_RS01755 has a mutation rate of 0.39.\n",
      "2025-04-16 11:13:19,273 - INFO - Gene cds-WP_001039987.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,273 - INFO - Gene gene-SPN23F_RS01760 has a mutation rate of 0.2733333333333333.\n",
      "2025-04-16 11:13:19,273 - INFO - Gene cds-WP_000248774.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,274 - INFO - Gene gene-SPN23F_RS01765 has a mutation rate of 0.44333333333333336.\n",
      "2025-04-16 11:13:19,274 - INFO - Gene cds-WP_000179549.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,274 - INFO - Gene gene-SPN23F_RS01770 has a mutation rate of 0.23333333333333334.\n",
      "2025-04-16 11:13:19,275 - INFO - Gene cds-WP_000200645.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,275 - INFO - Gene gene-SPN23F_RS11890 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,275 - INFO - Gene gene-SPN23F_RS01780 has a mutation rate of 0.6133333333333333.\n",
      "2025-04-16 11:13:19,276 - INFO - Gene cds-WP_000711414.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,276 - INFO - Gene gene-SPN23F_RS01785 has a mutation rate of 0.9233333333333333.\n",
      "2025-04-16 11:13:19,276 - INFO - Gene cds-WP_000039255.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,276 - INFO - Gene gene-SPN23F_RS01790 has a mutation rate of 0.30666666666666664.\n",
      "2025-04-16 11:13:19,277 - INFO - Gene cds-WP_000158793.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,277 - INFO - Gene gene-SPN23F_RS01795 has a mutation rate of 0.5433333333333333.\n",
      "2025-04-16 11:13:19,277 - INFO - Gene cds-WP_000518008.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,278 - INFO - Gene gene-SPN23F_RS01800 has a mutation rate of 0.6766666666666666.\n",
      "2025-04-16 11:13:19,278 - INFO - Gene cds-WP_000771053.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,278 - INFO - Gene gene-SPN23F_RS01805 has a mutation rate of 0.9033333333333333.\n",
      "2025-04-16 11:13:19,278 - INFO - Gene cds-WP_000765684.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,279 - INFO - Gene gene-SPN23F_RS12810 has a mutation rate of 0.54.\n",
      "2025-04-16 11:13:19,279 - INFO - Gene cds-WP_001809487.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,279 - INFO - Gene cds-SPN23F_RS12815 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,280 - INFO - Gene gene-SPN23F_RS14065 has a mutation rate of 0.7666666666666667.\n",
      "2025-04-16 11:13:19,280 - INFO - Gene cds-WP_000619581.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,281 - INFO - Gene gene-SPN23F_RS01820 has a mutation rate of 0.46.\n",
      "2025-04-16 11:13:19,281 - INFO - Gene cds-WP_000163315.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,281 - INFO - Gene gene-SPN23F_RS01825 has a mutation rate of 0.2866666666666667.\n",
      "2025-04-16 11:13:19,281 - INFO - Gene cds-WP_000373457.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,282 - INFO - Gene gene-SPN23F_RS01830 has a mutation rate of 0.95.\n",
      "2025-04-16 11:13:19,282 - INFO - Gene cds-WP_000562413.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,282 - INFO - Gene gene-SPN23F_RS01835 has a mutation rate of 0.8166666666666667.\n",
      "2025-04-16 11:13:19,283 - INFO - Gene cds-WP_000210618.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,283 - INFO - Gene gene-SPN23F_RS01840 has a mutation rate of 0.55.\n",
      "2025-04-16 11:13:19,283 - INFO - Gene cds-WP_001224637.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,284 - INFO - Gene gene-SPN23F_RS01845 has a mutation rate of 0.89.\n",
      "2025-04-16 11:13:19,284 - INFO - Gene cds-WP_000743674.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,284 - INFO - Gene gene-SPN23F_RS01850 has a mutation rate of 0.17666666666666667.\n",
      "2025-04-16 11:13:19,285 - INFO - Gene cds-WP_000698434.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,285 - INFO - Gene cds-SPN23F_RS13765 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,285 - INFO - Gene gene-SPN23F_RS01870 has a mutation rate of 0.2733333333333333.\n",
      "2025-04-16 11:13:19,286 - INFO - Gene cds-WP_000754501.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,286 - INFO - Gene gene-SPN23F_RS01875 has a mutation rate of 0.82.\n",
      "2025-04-16 11:13:19,286 - INFO - Gene cds-WP_001100142.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,295 - INFO - Gene gene-SPN23F_RS01880 has a mutation rate of 0.05.\n",
      "2025-04-16 11:13:19,315 - INFO - Gene cds-WP_000765212.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,316 - INFO - Gene cds-SPN23F_RS11910 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,317 - INFO - Gene gene-SPN23F_RS01900 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,317 - INFO - Gene cds-WP_000391688.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,317 - INFO - Gene gene-SPN23F_RS01905 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,318 - INFO - Gene cds-WP_000927663.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,318 - INFO - Gene gene-SPN23F_RS01910 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,319 - INFO - Gene cds-WP_000764147.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,319 - INFO - Gene gene-SPN23F_RS01915 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,322 - INFO - Gene cds-WP_000682970.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,323 - INFO - Gene gene-SPN23F_RS01930 has a mutation rate of 0.5833333333333334.\n",
      "2025-04-16 11:13:19,324 - INFO - Gene cds-WP_000116466.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,324 - INFO - Gene gene-SPN23F_RS01935 has a mutation rate of 0.9833333333333333.\n",
      "2025-04-16 11:13:19,325 - INFO - Gene cds-WP_000454503.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,325 - INFO - Gene gene-SPN23F_RS13220 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,326 - INFO - Gene cds-WP_001815645.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,326 - INFO - Gene gene-SPN23F_RS01940 has a mutation rate of 0.30666666666666664.\n",
      "2025-04-16 11:13:19,327 - INFO - Gene cds-WP_001048155.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,327 - INFO - Gene gene-SPN23F_RS01945 has a mutation rate of 0.8066666666666666.\n",
      "2025-04-16 11:13:19,327 - INFO - Gene cds-WP_000146861.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,328 - INFO - Gene gene-SPN23F_RS01950 has a mutation rate of 0.03666666666666667.\n",
      "2025-04-16 11:13:19,328 - INFO - Gene cds-WP_000002019.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,328 - INFO - Gene gene-SPN23F_RS01955 has a mutation rate of 0.3933333333333333.\n",
      "2025-04-16 11:13:19,329 - INFO - Gene cds-WP_000622953.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,329 - INFO - Gene gene-SPN23F_RS01960 has a mutation rate of 0.8233333333333334.\n",
      "2025-04-16 11:13:19,329 - INFO - Gene cds-WP_001035036.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,330 - INFO - Gene gene-SPN23F_RS01965 has a mutation rate of 0.76.\n",
      "2025-04-16 11:13:19,330 - INFO - Gene cds-WP_000891487.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,330 - INFO - Gene gene-SPN23F_RS01970 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:19,330 - INFO - Gene cds-WP_000206761.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,331 - INFO - Gene gene-SPN23F_RS01975 has a mutation rate of 0.6733333333333333.\n",
      "2025-04-16 11:13:19,331 - INFO - Gene cds-WP_000735961.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,331 - INFO - Gene gene-SPN23F_RS01980 has a mutation rate of 0.64.\n",
      "2025-04-16 11:13:19,332 - INFO - Gene cds-WP_000884265.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,332 - INFO - Gene gene-SPN23F_RS01985 has a mutation rate of 0.22333333333333333.\n",
      "2025-04-16 11:13:19,332 - INFO - Gene cds-WP_000079361.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,333 - INFO - Gene gene-SPN23F_RS01990 has a mutation rate of 0.5466666666666666.\n",
      "2025-04-16 11:13:19,333 - INFO - Gene cds-WP_000869657.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,333 - INFO - Gene gene-SPN23F_RS01995 has a mutation rate of 0.03666666666666667.\n",
      "2025-04-16 11:13:19,334 - INFO - Gene cds-WP_000400073.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,334 - INFO - Gene gene-SPN23F_RS02000 has a mutation rate of 0.03333333333333333.\n",
      "2025-04-16 11:13:19,334 - INFO - Gene cds-WP_000386347.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,335 - INFO - Gene gene-SPN23F_RS02005 has a mutation rate of 0.13666666666666666.\n",
      "2025-04-16 11:13:19,335 - INFO - Gene cds-WP_000852948.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,336 - INFO - Gene gene-SPN23F_RS02010 has a mutation rate of 0.01.\n",
      "2025-04-16 11:13:19,336 - INFO - Gene cds-WP_000257841.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,337 - INFO - Gene gene-SPN23F_RS02015 has a mutation rate of 0.49.\n",
      "2025-04-16 11:13:19,337 - INFO - Gene cds-WP_000857430.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,337 - INFO - Gene gene-SPN23F_RS02020 has a mutation rate of 0.29333333333333333.\n",
      "2025-04-16 11:13:19,337 - INFO - Gene cds-WP_000167623.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,338 - INFO - Gene gene-SPN23F_RS02025 has a mutation rate of 0.6466666666666666.\n",
      "2025-04-16 11:13:19,338 - INFO - Gene cds-WP_000763052.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,339 - INFO - Gene gene-SPN23F_RS02030 has a mutation rate of 0.05.\n",
      "2025-04-16 11:13:19,339 - INFO - Gene cds-WP_000774063.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,339 - INFO - Gene gene-SPN23F_RS02035 has a mutation rate of 0.33666666666666667.\n",
      "2025-04-16 11:13:19,340 - INFO - Gene cds-WP_001052244.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,340 - INFO - Gene gene-SPN23F_RS02040 has a mutation rate of 0.08666666666666667.\n",
      "2025-04-16 11:13:19,340 - INFO - Gene cds-WP_000565515.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,340 - INFO - Gene gene-SPN23F_RS02045 has a mutation rate of 0.08.\n",
      "2025-04-16 11:13:19,341 - INFO - Gene cds-WP_000488675.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,341 - INFO - Gene gene-SPN23F_RS02050 has a mutation rate of 0.08.\n",
      "2025-04-16 11:13:19,342 - INFO - Gene cds-WP_001173353.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,342 - INFO - Gene gene-SPN23F_RS02055 has a mutation rate of 0.59.\n",
      "2025-04-16 11:13:19,342 - INFO - Gene cds-WP_001017399.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,342 - INFO - Gene gene-SPN23F_RS02060 has a mutation rate of 0.08666666666666667.\n",
      "2025-04-16 11:13:19,343 - INFO - Gene cds-WP_000148169.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,343 - INFO - Gene cds-SPN23F_RS14155 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,343 - INFO - Gene gene-SPN23F_RS02075 has a mutation rate of 0.18.\n",
      "2025-04-16 11:13:19,344 - INFO - Gene cds-WP_000203654.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,344 - INFO - Gene gene-SPN23F_RS02080 has a mutation rate of 0.3333333333333333.\n",
      "2025-04-16 11:13:19,344 - INFO - Gene cds-WP_000510591.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,345 - INFO - Gene gene-SPN23F_RS02085 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,345 - INFO - Gene cds-WP_000568640.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,345 - INFO - Gene gene-SPN23F_RS02090 has a mutation rate of 0.87.\n",
      "2025-04-16 11:13:19,346 - INFO - Gene cds-WP_001008663.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,346 - INFO - Gene gene-SPN23F_RS02095 has a mutation rate of 0.52.\n",
      "2025-04-16 11:13:19,346 - INFO - Gene cds-WP_000143735.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,347 - INFO - Gene gene-SPN23F_RS02100 has a mutation rate of 0.84.\n",
      "2025-04-16 11:13:19,347 - INFO - Gene cds-WP_000705421.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,347 - INFO - Gene gene-SPN23F_RS02105 has a mutation rate of 0.48.\n",
      "2025-04-16 11:13:19,348 - INFO - Gene cds-WP_001025419.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,348 - INFO - Gene cds-SPN23F_RS02110 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,349 - INFO - Gene gene-SPN23F_RS02115 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,349 - INFO - Gene cds-WP_001140948.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,349 - INFO - Gene gene-SPN23F_RS02120 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,350 - INFO - Gene cds-WP_000216436.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,350 - INFO - Gene gene-SPN23F_RS02125 has a mutation rate of 0.6033333333333334.\n",
      "2025-04-16 11:13:19,350 - INFO - Gene cds-WP_000036668.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,351 - INFO - Gene gene-SPN23F_RS02130 has a mutation rate of 0.27666666666666667.\n",
      "2025-04-16 11:13:19,351 - INFO - Gene cds-WP_000411764.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,351 - INFO - Gene gene-SPN23F_RS02135 has a mutation rate of 0.023333333333333334.\n",
      "2025-04-16 11:13:19,351 - INFO - Gene cds-WP_001253803.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,352 - INFO - Gene gene-SPN23F_RS02140 has a mutation rate of 0.3933333333333333.\n",
      "2025-04-16 11:13:19,352 - INFO - Gene cds-WP_000218054.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,352 - INFO - Gene gene-SPN23F_RS02145 has a mutation rate of 0.30333333333333334.\n",
      "2025-04-16 11:13:19,353 - INFO - Gene cds-WP_001810356.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,353 - INFO - Gene gene-SPN23F_RS02150 has a mutation rate of 0.03.\n",
      "2025-04-16 11:13:19,353 - INFO - Gene cds-WP_001027900.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,353 - INFO - Gene gene-SPN23F_RS02155 has a mutation rate of 0.4.\n",
      "2025-04-16 11:13:19,354 - INFO - Gene cds-WP_000952973.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,354 - INFO - Gene gene-SPN23F_RS02160 has a mutation rate of 0.23333333333333334.\n",
      "2025-04-16 11:13:19,354 - INFO - Gene cds-WP_001130038.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,355 - INFO - Gene gene-SPN23F_RS02165 has a mutation rate of 0.26666666666666666.\n",
      "2025-04-16 11:13:19,355 - INFO - Gene cds-WP_000189477.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,355 - INFO - Gene gene-SPN23F_RS02170 has a mutation rate of 0.45666666666666667.\n",
      "2025-04-16 11:13:19,356 - INFO - Gene cds-WP_001227307.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,356 - INFO - Gene gene-SPN23F_RS02175 has a mutation rate of 1.0.\n",
      "2025-04-16 11:13:19,356 - INFO - Gene cds-WP_000750651.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,357 - INFO - Gene gene-SPN23F_RS02185 has a mutation rate of 0.5866666666666667.\n",
      "2025-04-16 11:13:19,357 - INFO - Gene cds-WP_001815847.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,358 - INFO - Gene gene-SPN23F_RS02190 has a mutation rate of 0.2733333333333333.\n",
      "2025-04-16 11:13:19,358 - INFO - Gene cds-WP_000280773.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,359 - INFO - Gene gene-SPN23F_RS02195 has a mutation rate of 0.69.\n",
      "2025-04-16 11:13:19,359 - INFO - Gene cds-WP_000904591.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,359 - INFO - Gene gene-SPN23F_RS02200 has a mutation rate of 0.12666666666666668.\n",
      "2025-04-16 11:13:19,359 - INFO - Gene cds-WP_000260644.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,360 - INFO - Gene gene-SPN23F_RS02205 has a mutation rate of 0.11.\n",
      "2025-04-16 11:13:19,360 - INFO - Gene cds-WP_001815848.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,360 - INFO - Gene gene-SPN23F_RS02215 has a mutation rate of 0.19666666666666666.\n",
      "2025-04-16 11:13:19,361 - INFO - Gene cds-WP_000384133.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,361 - INFO - Gene cds-SPN23F_RS13770 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,361 - INFO - Gene gene-SPN23F_RS02240 has a mutation rate of 0.6266666666666667.\n",
      "2025-04-16 11:13:19,362 - INFO - Gene cds-WP_001043417.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,362 - INFO - Gene gene-SPN23F_RS02245 has a mutation rate of 0.36.\n",
      "2025-04-16 11:13:19,362 - INFO - Gene cds-WP_000691685.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,362 - INFO - Gene gene-SPN23F_RS02250 has a mutation rate of 0.24666666666666667.\n",
      "2025-04-16 11:13:19,363 - INFO - Gene cds-WP_001063601.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,363 - INFO - Gene gene-SPN23F_RS02255 has a mutation rate of 0.4533333333333333.\n",
      "2025-04-16 11:13:19,363 - INFO - Gene cds-WP_000403162.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,363 - INFO - Gene gene-SPN23F_RS13225 has a mutation rate of 0.3333333333333333.\n",
      "2025-04-16 11:13:19,364 - INFO - Gene cds-WP_000693286.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,364 - INFO - Gene gene-SPN23F_RS02265 has a mutation rate of 0.5466666666666666.\n",
      "2025-04-16 11:13:19,364 - INFO - Gene cds-WP_000656555.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,365 - INFO - Gene gene-SPN23F_RS02270 has a mutation rate of 0.22333333333333333.\n",
      "2025-04-16 11:13:19,365 - INFO - Gene cds-WP_001148103.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,365 - INFO - Gene gene-SPN23F_RS02275 has a mutation rate of 0.61.\n",
      "2025-04-16 11:13:19,366 - INFO - Gene cds-WP_000181373.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,366 - INFO - Gene gene-SPN23F_RS02280 has a mutation rate of 0.11333333333333333.\n",
      "2025-04-16 11:13:19,366 - INFO - Gene cds-WP_000185828.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,367 - INFO - Gene gene-SPN23F_RS02285 has a mutation rate of 0.71.\n",
      "2025-04-16 11:13:19,367 - INFO - Gene cds-WP_001826724.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,368 - INFO - Gene gene-SPN23F_RS02290 has a mutation rate of 0.49333333333333335.\n",
      "2025-04-16 11:13:19,368 - INFO - Gene cds-WP_000867297.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,368 - INFO - Gene gene-SPN23F_RS02295 has a mutation rate of 0.7533333333333333.\n",
      "2025-04-16 11:13:19,369 - INFO - Gene cds-WP_000333647.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,369 - INFO - Gene gene-SPN23F_RS02300 has a mutation rate of 0.16.\n",
      "2025-04-16 11:13:19,369 - INFO - Gene cds-WP_000418402.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,370 - INFO - Gene gene-SPN23F_RS14075 has a mutation rate of 0.3466666666666667.\n",
      "2025-04-16 11:13:19,370 - INFO - Gene cds-WP_000692891.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,370 - INFO - Gene gene-SPN23F_RS02305 has a mutation rate of 0.12.\n",
      "2025-04-16 11:13:19,371 - INFO - Gene cds-WP_000105234.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,371 - INFO - Gene gene-SPN23F_RS02310 has a mutation rate of 0.3466666666666667.\n",
      "2025-04-16 11:13:19,371 - INFO - Gene cds-WP_000026645.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,371 - INFO - Gene gene-SPN23F_RS02315 has a mutation rate of 0.9933333333333333.\n",
      "2025-04-16 11:13:19,372 - INFO - Gene cds-WP_000795165.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,372 - INFO - Gene gene-SPN23F_RS02320 has a mutation rate of 0.10666666666666667.\n",
      "2025-04-16 11:13:19,372 - INFO - Gene cds-WP_001096743.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,373 - INFO - Gene gene-SPN23F_RS02325 has a mutation rate of 0.16333333333333333.\n",
      "2025-04-16 11:13:19,373 - INFO - Gene cds-WP_000119900.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,373 - INFO - Gene gene-SPN23F_RS02330 has a mutation rate of 0.043333333333333335.\n",
      "2025-04-16 11:13:19,373 - INFO - Gene cds-WP_000659553.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,374 - INFO - Gene gene-SPN23F_RS02335 has a mutation rate of 0.4166666666666667.\n",
      "2025-04-16 11:13:19,374 - INFO - Gene cds-WP_001122904.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,374 - INFO - Gene gene-SPN23F_RS11955 has a mutation rate of 0.03666666666666667.\n",
      "2025-04-16 11:13:19,375 - INFO - Gene cds-WP_001808898.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,375 - INFO - Gene gene-SPN23F_RS13775 has a mutation rate of 0.4.\n",
      "2025-04-16 11:13:19,375 - INFO - Gene cds-WP_000410533.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,376 - INFO - Gene cds-SPN23F_RS02345 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,376 - INFO - Gene gene-SPN23F_RS02350 has a mutation rate of 0.03666666666666667.\n",
      "2025-04-16 11:13:19,377 - INFO - Gene cds-WP_000240090.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,377 - INFO - Gene gene-SPN23F_RS02355 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,377 - INFO - Gene cds-WP_000651177.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,378 - INFO - Gene gene-SPN23F_RS02360 has a mutation rate of 0.13666666666666666.\n",
      "2025-04-16 11:13:19,378 - INFO - Gene cds-WP_000187269.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,378 - INFO - Gene gene-SPN23F_RS02365 has a mutation rate of 0.4.\n",
      "2025-04-16 11:13:19,379 - INFO - Gene cds-WP_000029036.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,379 - INFO - Gene gene-SPN23F_RS02370 has a mutation rate of 0.7866666666666666.\n",
      "2025-04-16 11:13:19,379 - INFO - Gene cds-WP_000229084.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,380 - INFO - Gene gene-SPN23F_RS02380 has a mutation rate of 0.7766666666666666.\n",
      "2025-04-16 11:13:19,380 - INFO - Gene cds-WP_001088694.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,381 - INFO - Gene gene-SPN23F_RS02385 has a mutation rate of 0.95.\n",
      "2025-04-16 11:13:19,381 - INFO - Gene cds-WP_000255770.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,381 - INFO - Gene gene-SPN23F_RS02390 has a mutation rate of 0.28.\n",
      "2025-04-16 11:13:19,381 - INFO - Gene cds-WP_000046031.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,382 - INFO - Gene gene-SPN23F_RS02395 has a mutation rate of 0.19333333333333333.\n",
      "2025-04-16 11:13:19,382 - INFO - Gene cds-WP_000034665.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,382 - INFO - Gene gene-SPN23F_RS02400 has a mutation rate of 0.8.\n",
      "2025-04-16 11:13:19,383 - INFO - Gene cds-WP_000114427.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,383 - INFO - Gene gene-SPN23F_RS02405 has a mutation rate of 0.49666666666666665.\n",
      "2025-04-16 11:13:19,384 - INFO - Gene cds-WP_001066295.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,384 - INFO - Gene gene-SPN23F_RS13365 has a mutation rate of 0.73.\n",
      "2025-04-16 11:13:19,384 - INFO - Gene cds-WP_025169639.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,384 - INFO - Gene gene-SPN23F_RS02415 has a mutation rate of 0.03.\n",
      "2025-04-16 11:13:19,385 - INFO - Gene cds-WP_000777753.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,385 - INFO - Gene gene-SPN23F_RS02420 has a mutation rate of 0.22333333333333333.\n",
      "2025-04-16 11:13:19,385 - INFO - Gene cds-WP_001278301.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,386 - INFO - Gene gene-SPN23F_RS02425 has a mutation rate of 0.5566666666666666.\n",
      "2025-04-16 11:13:19,386 - INFO - Gene cds-WP_000889923.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,386 - INFO - Gene gene-SPN23F_RS02430 has a mutation rate of 0.73.\n",
      "2025-04-16 11:13:19,387 - INFO - Gene cds-WP_000653774.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,387 - INFO - Gene gene-SPN23F_RS02435 has a mutation rate of 0.4633333333333333.\n",
      "2025-04-16 11:13:19,387 - INFO - Gene cds-WP_000132572.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,473 - INFO - Gene gene-SPN23F_RS02440 has a mutation rate of 0.6.\n",
      "2025-04-16 11:13:19,473 - INFO - Gene cds-WP_000876675.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,473 - INFO - Gene gene-SPN23F_RS02445 has a mutation rate of 0.73.\n",
      "2025-04-16 11:13:19,474 - INFO - Gene cds-WP_001219120.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,474 - INFO - Gene gene-SPN23F_RS02450 has a mutation rate of 0.81.\n",
      "2025-04-16 11:13:19,475 - INFO - Gene cds-WP_001017642.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,475 - INFO - Gene gene-SPN23F_RS02455 has a mutation rate of 0.56.\n",
      "2025-04-16 11:13:19,475 - INFO - Gene cds-WP_000358815.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,476 - INFO - Gene gene-SPN23F_RS02460 has a mutation rate of 0.74.\n",
      "2025-04-16 11:13:19,476 - INFO - Gene cds-WP_001069060.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,476 - INFO - Gene gene-SPN23F_RS02465 has a mutation rate of 0.78.\n",
      "2025-04-16 11:13:19,477 - INFO - Gene cds-WP_196300924.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,477 - INFO - Gene gene-SPN23F_RS14160 has a mutation rate of 0.5066666666666667.\n",
      "2025-04-16 11:13:19,477 - INFO - Gene cds-WP_000205166.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,478 - INFO - Gene gene-SPN23F_RS02475 has a mutation rate of 0.13666666666666666.\n",
      "2025-04-16 11:13:19,478 - INFO - Gene cds-WP_001093259.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,478 - INFO - Gene gene-SPN23F_RS02480 has a mutation rate of 0.09.\n",
      "2025-04-16 11:13:19,478 - INFO - Gene cds-WP_001093248.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,479 - INFO - Gene gene-SPN23F_RS02485 has a mutation rate of 0.4866666666666667.\n",
      "2025-04-16 11:13:19,479 - INFO - Gene cds-WP_000379965.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,479 - INFO - Gene gene-SPN23F_RS02490 has a mutation rate of 0.43666666666666665.\n",
      "2025-04-16 11:13:19,480 - INFO - Gene cds-WP_001809846.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,480 - INFO - Gene cds-SPN23F_RS14245 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,480 - INFO - Gene gene-SPN23F_RS02495 has a mutation rate of 0.10333333333333333.\n",
      "2025-04-16 11:13:19,480 - INFO - Gene cds-WP_000346296.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,481 - INFO - Gene cds-SPN23F_RS02510 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,481 - INFO - Gene gene-SPN23F_RS02520 has a mutation rate of 0.24.\n",
      "2025-04-16 11:13:19,481 - INFO - Gene cds-WP_000877381.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,482 - INFO - Gene gene-SPN23F_RS02525 has a mutation rate of 0.7966666666666666.\n",
      "2025-04-16 11:13:19,482 - INFO - Gene cds-WP_000760532.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,482 - INFO - Gene gene-SPN23F_RS02530 has a mutation rate of 0.15333333333333332.\n",
      "2025-04-16 11:13:19,483 - INFO - Gene cds-WP_000276498.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,483 - INFO - Gene gene-SPN23F_RS02535 has a mutation rate of 0.56.\n",
      "2025-04-16 11:13:19,483 - INFO - Gene cds-WP_000394044.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,484 - INFO - Gene gene-SPN23F_RS02540 has a mutation rate of 0.06666666666666667.\n",
      "2025-04-16 11:13:19,484 - INFO - Gene cds-WP_000363002.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,484 - INFO - Gene gene-SPN23F_RS02545 has a mutation rate of 0.4666666666666667.\n",
      "2025-04-16 11:13:19,485 - INFO - Gene cds-WP_001266083.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,485 - INFO - Gene gene-SPN23F_RS02550 has a mutation rate of 0.3466666666666667.\n",
      "2025-04-16 11:13:19,485 - INFO - Gene cds-WP_001808691.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,486 - INFO - Gene gene-SPN23F_RS02555 has a mutation rate of 0.03.\n",
      "2025-04-16 11:13:19,486 - INFO - Gene cds-WP_000032273.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,486 - INFO - Gene gene-SPN23F_RS02560 has a mutation rate of 0.016666666666666666.\n",
      "2025-04-16 11:13:19,487 - INFO - Gene cds-WP_000857562.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,487 - INFO - Gene gene-SPN23F_RS02565 has a mutation rate of 0.32.\n",
      "2025-04-16 11:13:19,487 - INFO - Gene cds-WP_001041390.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,488 - INFO - Gene gene-SPN23F_RS02570 has a mutation rate of 0.5833333333333334.\n",
      "2025-04-16 11:13:19,488 - INFO - Gene cds-WP_000039212.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,489 - INFO - Gene gene-SPN23F_RS02575 has a mutation rate of 0.1.\n",
      "2025-04-16 11:13:19,489 - INFO - Gene cds-WP_001273601.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,489 - INFO - Gene gene-SPN23F_RS02580 has a mutation rate of 0.6733333333333333.\n",
      "2025-04-16 11:13:19,490 - INFO - Gene cds-WP_000840388.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,490 - INFO - Gene gene-SPN23F_RS02585 has a mutation rate of 0.55.\n",
      "2025-04-16 11:13:19,490 - INFO - Gene cds-WP_001068163.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,490 - INFO - Gene gene-SPN23F_RS13780 has a mutation rate of 0.24666666666666667.\n",
      "2025-04-16 11:13:19,491 - INFO - Gene cds-WP_000693323.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,491 - INFO - Gene gene-SPN23F_RS02590 has a mutation rate of 0.06.\n",
      "2025-04-16 11:13:19,492 - INFO - Gene cds-WP_000368737.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,492 - INFO - Gene gene-SPN23F_RS02595 has a mutation rate of 0.7933333333333333.\n",
      "2025-04-16 11:13:19,492 - INFO - Gene cds-WP_000752893.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,493 - INFO - Gene gene-SPN23F_RS02600 has a mutation rate of 0.41.\n",
      "2025-04-16 11:13:19,493 - INFO - Gene cds-WP_000119707.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,493 - INFO - Gene gene-SPN23F_RS02605 has a mutation rate of 0.47333333333333333.\n",
      "2025-04-16 11:13:19,494 - INFO - Gene cds-WP_000038619.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,494 - INFO - Gene gene-SPN23F_RS02610 has a mutation rate of 0.5033333333333333.\n",
      "2025-04-16 11:13:19,494 - INFO - Gene cds-WP_000580663.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,495 - INFO - Gene gene-SPN23F_RS02615 has a mutation rate of 0.8.\n",
      "2025-04-16 11:13:19,495 - INFO - Gene cds-WP_000124566.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,496 - INFO - Gene gene-SPN23F_RS02620 has a mutation rate of 0.2866666666666667.\n",
      "2025-04-16 11:13:19,496 - INFO - Gene cds-WP_001286128.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,496 - INFO - Gene gene-SPN23F_RS02625 has a mutation rate of 0.8233333333333334.\n",
      "2025-04-16 11:13:19,497 - INFO - Gene cds-WP_000032241.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,497 - INFO - Gene gene-SPN23F_RS02630 has a mutation rate of 0.47.\n",
      "2025-04-16 11:13:19,497 - INFO - Gene cds-WP_000033775.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,498 - INFO - Gene gene-SPN23F_RS02635 has a mutation rate of 0.56.\n",
      "2025-04-16 11:13:19,498 - INFO - Gene cds-WP_001183267.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,499 - INFO - Gene cds-SPN23F_RS12005 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,499 - INFO - Gene gene-SPN23F_RS02660 has a mutation rate of 0.78.\n",
      "2025-04-16 11:13:19,499 - INFO - Gene cds-WP_000442363.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,500 - INFO - Gene gene-SPN23F_RS02665 has a mutation rate of 0.52.\n",
      "2025-04-16 11:13:19,500 - INFO - Gene cds-WP_000584535.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,500 - INFO - Gene gene-SPN23F_RS02670 has a mutation rate of 0.82.\n",
      "2025-04-16 11:13:19,501 - INFO - Gene cds-WP_000120595.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,501 - INFO - Gene gene-SPN23F_RS02675 has a mutation rate of 0.6633333333333333.\n",
      "2025-04-16 11:13:19,501 - INFO - Gene cds-WP_000151832.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,502 - INFO - Gene gene-SPN23F_RS02680 has a mutation rate of 0.53.\n",
      "2025-04-16 11:13:19,502 - INFO - Gene cds-WP_001813261.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,503 - INFO - Gene gene-SPN23F_RS02685 has a mutation rate of 0.63.\n",
      "2025-04-16 11:13:19,503 - INFO - Gene cds-WP_000619941.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,503 - INFO - Gene gene-SPN23F_RS02690 has a mutation rate of 0.7566666666666667.\n",
      "2025-04-16 11:13:19,503 - INFO - Gene cds-WP_000909140.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,504 - INFO - Gene gene-SPN23F_RS02695 has a mutation rate of 0.7933333333333333.\n",
      "2025-04-16 11:13:19,504 - INFO - Gene cds-WP_000597976.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,504 - INFO - Gene cds-SPN23F_RS02700 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,505 - INFO - Gene gene-SPN23F_RS02705 has a mutation rate of 0.6166666666666667.\n",
      "2025-04-16 11:13:19,505 - INFO - Gene cds-WP_001814186.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,505 - INFO - Gene gene-SPN23F_RS02710 has a mutation rate of 0.77.\n",
      "2025-04-16 11:13:19,506 - INFO - Gene cds-WP_000108220.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,506 - INFO - Gene gene-SPN23F_RS02715 has a mutation rate of 0.7133333333333334.\n",
      "2025-04-16 11:13:19,506 - INFO - Gene cds-WP_000089976.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,506 - INFO - Gene gene-SPN23F_RS02720 has a mutation rate of 0.72.\n",
      "2025-04-16 11:13:19,507 - INFO - Gene cds-WP_001118979.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,507 - INFO - Gene gene-SPN23F_RS02725 has a mutation rate of 0.4633333333333333.\n",
      "2025-04-16 11:13:19,508 - INFO - Gene cds-WP_000539953.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,508 - INFO - Gene gene-SPN23F_RS02730 has a mutation rate of 0.7.\n",
      "2025-04-16 11:13:19,508 - INFO - Gene cds-WP_000613675.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,509 - INFO - Gene gene-SPN23F_RS02735 has a mutation rate of 0.7033333333333334.\n",
      "2025-04-16 11:13:19,509 - INFO - Gene cds-WP_000591096.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,510 - INFO - Gene gene-SPN23F_RS02740 has a mutation rate of 0.016666666666666666.\n",
      "2025-04-16 11:13:19,510 - INFO - Gene cds-WP_000567908.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,510 - INFO - Gene gene-SPN23F_RS02745 has a mutation rate of 0.64.\n",
      "2025-04-16 11:13:19,511 - INFO - Gene cds-WP_001201737.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,511 - INFO - Gene gene-SPN23F_RS02750 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,512 - INFO - Gene cds-WP_000436237.1-3 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,512 - INFO - Gene gene-SPN23F_RS02755 has a mutation rate of 0.84.\n",
      "2025-04-16 11:13:19,512 - INFO - Gene cds-WP_001166179.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,513 - INFO - Gene cds-SPN23F_RS13235 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,513 - INFO - Gene gene-SPN23F_RS02765 has a mutation rate of 0.5966666666666667.\n",
      "2025-04-16 11:13:19,513 - INFO - Gene cds-WP_001068292.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,514 - INFO - Gene gene-SPN23F_RS02770 has a mutation rate of 0.5.\n",
      "2025-04-16 11:13:19,514 - INFO - Gene cds-WP_000173731.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,515 - INFO - Gene gene-SPN23F_RS02775 has a mutation rate of 0.06.\n",
      "2025-04-16 11:13:19,515 - INFO - Gene cds-WP_000902984.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,516 - INFO - Gene gene-SPN23F_RS13785 has a mutation rate of 0.4866666666666667.\n",
      "2025-04-16 11:13:19,516 - INFO - Gene cds-WP_198523890.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,517 - INFO - Gene gene-SPN23F_RS02780 has a mutation rate of 0.3466666666666667.\n",
      "2025-04-16 11:13:19,517 - INFO - Gene cds-WP_000697960.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,518 - INFO - Gene gene-SPN23F_RS02785 has a mutation rate of 0.7466666666666667.\n",
      "2025-04-16 11:13:19,518 - INFO - Gene cds-WP_000831369.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,519 - INFO - Gene gene-SPN23F_RS02790 has a mutation rate of 0.043333333333333335.\n",
      "2025-04-16 11:13:19,519 - INFO - Gene cds-WP_001019003.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,519 - INFO - Gene gene-SPN23F_RS02795 has a mutation rate of 0.8166666666666667.\n",
      "2025-04-16 11:13:19,520 - INFO - Gene cds-WP_000843701.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,520 - INFO - Gene gene-SPN23F_RS02800 has a mutation rate of 0.9033333333333333.\n",
      "2025-04-16 11:13:19,521 - INFO - Gene cds-WP_000443837.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,521 - INFO - Gene gene-SPN23F_RS02805 has a mutation rate of 0.8233333333333334.\n",
      "2025-04-16 11:13:19,521 - INFO - Gene cds-WP_000131117.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,521 - INFO - Gene gene-SPN23F_RS02810 has a mutation rate of 0.91.\n",
      "2025-04-16 11:13:19,522 - INFO - Gene cds-WP_000624342.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,522 - INFO - Gene gene-SPN23F_RS02815 has a mutation rate of 0.8966666666666666.\n",
      "2025-04-16 11:13:19,522 - INFO - Gene cds-WP_001229604.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,523 - INFO - Gene gene-SPN23F_RS02820 has a mutation rate of 0.9233333333333333.\n",
      "2025-04-16 11:13:19,523 - INFO - Gene cds-WP_000632597.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,523 - INFO - Gene gene-SPN23F_RS02825 has a mutation rate of 0.09.\n",
      "2025-04-16 11:13:19,523 - INFO - Gene cds-WP_000065637.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,524 - INFO - Gene gene-SPN23F_RS02830 has a mutation rate of 0.56.\n",
      "2025-04-16 11:13:19,524 - INFO - Gene cds-WP_000290013.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,525 - INFO - Gene gene-SPN23F_RS02835 has a mutation rate of 0.41.\n",
      "2025-04-16 11:13:19,525 - INFO - Gene cds-WP_000286563.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,525 - INFO - Gene gene-SPN23F_RS02840 has a mutation rate of 0.56.\n",
      "2025-04-16 11:13:19,526 - INFO - Gene cds-WP_001229509.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,526 - INFO - Gene gene-SPN23F_RS02845 has a mutation rate of 0.4166666666666667.\n",
      "2025-04-16 11:13:19,527 - INFO - Gene cds-WP_001267410.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,527 - INFO - Gene gene-SPN23F_RS02850 has a mutation rate of 0.7766666666666666.\n",
      "2025-04-16 11:13:19,527 - INFO - Gene cds-WP_001061144.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,528 - INFO - Gene cds-SPN23F_RS02855 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,528 - INFO - Gene gene-SPN23F_RS02860 has a mutation rate of 0.5933333333333334.\n",
      "2025-04-16 11:13:19,529 - INFO - Gene cds-WP_000714633.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,529 - INFO - Gene gene-SPN23F_RS02865 has a mutation rate of 0.6633333333333333.\n",
      "2025-04-16 11:13:19,530 - INFO - Gene cds-WP_000670163.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,530 - INFO - Gene gene-SPN23F_RS02870 has a mutation rate of 0.6666666666666666.\n",
      "2025-04-16 11:13:19,531 - INFO - Gene cds-WP_000125044.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,531 - INFO - Gene gene-SPN23F_RS02875 has a mutation rate of 0.22333333333333333.\n",
      "2025-04-16 11:13:19,531 - INFO - Gene cds-WP_000078149.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,532 - INFO - Gene cds-SPN23F_RS13790 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,532 - INFO - Gene gene-SPN23F_RS02890 has a mutation rate of 0.12666666666666668.\n",
      "2025-04-16 11:13:19,532 - INFO - Gene cds-WP_001074663.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,533 - INFO - Gene gene-SPN23F_RS02895 has a mutation rate of 0.72.\n",
      "2025-04-16 11:13:19,534 - INFO - Gene cds-WP_001077246.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,534 - INFO - Gene gene-SPN23F_RS02900 has a mutation rate of 0.6366666666666667.\n",
      "2025-04-16 11:13:19,538 - INFO - Gene cds-WP_000335612.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,539 - INFO - Gene gene-SPN23F_RS02905 has a mutation rate of 0.5866666666666667.\n",
      "2025-04-16 11:13:19,544 - INFO - Gene cds-WP_000747819.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,570 - INFO - Gene gene-SPN23F_RS02910 has a mutation rate of 0.14666666666666667.\n",
      "2025-04-16 11:13:19,570 - INFO - Gene cds-WP_001085809.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,571 - INFO - Gene gene-SPN23F_RS02915 has a mutation rate of 0.03666666666666667.\n",
      "2025-04-16 11:13:19,571 - INFO - Gene cds-WP_001085677.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,572 - INFO - Gene gene-SPN23F_RS13485 has a mutation rate of 0.48333333333333334.\n",
      "2025-04-16 11:13:19,572 - INFO - Gene cds-WP_000849975.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,573 - INFO - Gene gene-SPN23F_RS13795 has a mutation rate of 0.5266666666666666.\n",
      "2025-04-16 11:13:19,573 - INFO - Gene cds-WP_000895062.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,574 - INFO - Gene gene-SPN23F_RS14165 has a mutation rate of 0.13333333333333333.\n",
      "2025-04-16 11:13:19,578 - INFO - Gene cds-WP_000643406.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,579 - INFO - Gene gene-SPN23F_RS02935 has a mutation rate of 0.49333333333333335.\n",
      "2025-04-16 11:13:19,581 - INFO - Gene cds-WP_001240779.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,581 - INFO - Gene gene-SPN23F_RS02940 has a mutation rate of 0.25666666666666665.\n",
      "2025-04-16 11:13:19,582 - INFO - Gene cds-WP_000594189.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,582 - INFO - Gene gene-SPN23F_RS02945 has a mutation rate of 0.11333333333333333.\n",
      "2025-04-16 11:13:19,583 - INFO - Gene cds-WP_000760712.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,583 - INFO - Gene gene-SPN23F_RS14170 has a mutation rate of 0.49.\n",
      "2025-04-16 11:13:19,584 - INFO - Gene cds-WP_001814192.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,584 - INFO - Gene gene-SPN23F_RS02950 has a mutation rate of 0.5166666666666667.\n",
      "2025-04-16 11:13:19,584 - INFO - Gene cds-WP_001814193.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,584 - INFO - Gene gene-SPN23F_RS13800 has a mutation rate of 0.5633333333333334.\n",
      "2025-04-16 11:13:19,585 - INFO - Gene cds-WP_020901271.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,585 - INFO - Gene gene-SPN23F_RS02965 has a mutation rate of 0.83.\n",
      "2025-04-16 11:13:19,585 - INFO - Gene cds-WP_000750048.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,586 - INFO - Gene gene-SPN23F_RS02970 has a mutation rate of 0.11.\n",
      "2025-04-16 11:13:19,586 - INFO - Gene cds-WP_000242176.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,586 - INFO - Gene gene-SPN23F_RS02975 has a mutation rate of 0.10333333333333333.\n",
      "2025-04-16 11:13:19,587 - INFO - Gene cds-WP_000590563.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,587 - INFO - Gene gene-SPN23F_RS02980 has a mutation rate of 0.47.\n",
      "2025-04-16 11:13:19,588 - INFO - Gene cds-WP_000338720.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,588 - INFO - Gene gene-SPN23F_RS02985 has a mutation rate of 1.0.\n",
      "2025-04-16 11:13:19,589 - INFO - Gene cds-WP_000515195.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,589 - INFO - Gene gene-SPN23F_RS14175 has a mutation rate of 0.79.\n",
      "2025-04-16 11:13:19,589 - INFO - Gene cds-WP_223200702.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,590 - INFO - Gene gene-SPN23F_RS13810 has a mutation rate of 0.81.\n",
      "2025-04-16 11:13:19,590 - INFO - Gene cds-WP_001810421.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,590 - INFO - Gene gene-SPN23F_RS03000 has a mutation rate of 0.5533333333333333.\n",
      "2025-04-16 11:13:19,590 - INFO - Gene cds-WP_000525415.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,591 - INFO - Gene cds-SPN23F_RS12855 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,591 - INFO - Gene gene-SPN23F_RS03015 has a mutation rate of 0.31.\n",
      "2025-04-16 11:13:19,592 - INFO - Gene cds-WP_000828435.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,592 - INFO - Gene gene-SPN23F_RS03020 has a mutation rate of 0.5333333333333333.\n",
      "2025-04-16 11:13:19,593 - INFO - Gene cds-WP_000361078.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,593 - INFO - Gene gene-SPN23F_RS03025 has a mutation rate of 0.5433333333333333.\n",
      "2025-04-16 11:13:19,593 - INFO - Gene cds-WP_000421654.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,594 - INFO - Gene gene-SPN23F_RS03030 has a mutation rate of 0.15333333333333332.\n",
      "2025-04-16 11:13:19,594 - INFO - Gene cds-WP_000759490.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,594 - INFO - Gene gene-SPN23F_RS03035 has a mutation rate of 0.2866666666666667.\n",
      "2025-04-16 11:13:19,594 - INFO - Gene cds-WP_000509848.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,595 - INFO - Gene gene-SPN23F_RS03040 has a mutation rate of 0.043333333333333335.\n",
      "2025-04-16 11:13:19,595 - INFO - Gene cds-WP_000759278.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,595 - INFO - Gene gene-SPN23F_RS03045 has a mutation rate of 0.49666666666666665.\n",
      "2025-04-16 11:13:19,596 - INFO - Gene cds-WP_000998305.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,596 - INFO - Gene gene-SPN23F_RS03050 has a mutation rate of 0.11.\n",
      "2025-04-16 11:13:19,596 - INFO - Gene cds-WP_000222616.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,597 - INFO - Gene gene-SPN23F_RS03055 has a mutation rate of 0.2.\n",
      "2025-04-16 11:13:19,597 - INFO - Gene cds-WP_000831072.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,597 - INFO - Gene gene-SPN23F_RS03060 has a mutation rate of 0.7333333333333333.\n",
      "2025-04-16 11:13:19,597 - INFO - Gene cds-WP_001006363.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,598 - INFO - Gene gene-SPN23F_RS03065 has a mutation rate of 0.83.\n",
      "2025-04-16 11:13:19,598 - INFO - Gene cds-WP_000473015.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,598 - INFO - Gene gene-SPN23F_RS03070 has a mutation rate of 0.82.\n",
      "2025-04-16 11:13:19,599 - INFO - Gene cds-WP_000554861.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,599 - INFO - Gene gene-SPN23F_RS03075 has a mutation rate of 0.05.\n",
      "2025-04-16 11:13:19,599 - INFO - Gene cds-WP_000033354.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,600 - INFO - Gene cds-SPN23F_RS03080 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,600 - INFO - Gene gene-SPN23F_RS03085 has a mutation rate of 0.6666666666666666.\n",
      "2025-04-16 11:13:19,600 - INFO - Gene cds-WP_000078572.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,601 - INFO - Gene gene-SPN23F_RS03090 has a mutation rate of 0.30666666666666664.\n",
      "2025-04-16 11:13:19,601 - INFO - Gene cds-WP_000158602.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,602 - INFO - Gene gene-SPN23F_RS03095 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,602 - INFO - Gene cds-WP_001051780.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,602 - INFO - Gene gene-SPN23F_RS03100 has a mutation rate of 0.7.\n",
      "2025-04-16 11:13:19,603 - INFO - Gene cds-WP_000850189.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,603 - INFO - Gene gene-SPN23F_RS03105 has a mutation rate of 0.4033333333333333.\n",
      "2025-04-16 11:13:19,604 - INFO - Gene cds-WP_000573197.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,604 - INFO - Gene gene-SPN23F_RS03110 has a mutation rate of 0.7133333333333334.\n",
      "2025-04-16 11:13:19,605 - INFO - Gene cds-WP_000129690.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,605 - INFO - Gene gene-SPN23F_RS03115 has a mutation rate of 0.3433333333333333.\n",
      "2025-04-16 11:13:19,605 - INFO - Gene cds-WP_000354338.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,606 - INFO - Gene gene-SPN23F_RS03120 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:19,606 - INFO - Gene cds-WP_001142146.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,606 - INFO - Gene gene-SPN23F_RS03125 has a mutation rate of 0.22.\n",
      "2025-04-16 11:13:19,607 - INFO - Gene cds-WP_001222592.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,607 - INFO - Gene gene-SPN23F_RS03130 has a mutation rate of 0.24333333333333335.\n",
      "2025-04-16 11:13:19,607 - INFO - Gene cds-WP_000051787.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,608 - INFO - Gene gene-SPN23F_RS03135 has a mutation rate of 0.22333333333333333.\n",
      "2025-04-16 11:13:19,609 - INFO - Gene cds-WP_000259256.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,609 - INFO - Gene cds-SPN23F_RS03140 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,610 - INFO - Gene gene-SPN23F_RS03145 has a mutation rate of 0.9233333333333333.\n",
      "2025-04-16 11:13:19,610 - INFO - Gene cds-WP_001235022.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,610 - INFO - Gene gene-SPN23F_RS03150 has a mutation rate of 0.22.\n",
      "2025-04-16 11:13:19,610 - INFO - Gene cds-WP_000164111.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,611 - INFO - Gene gene-SPN23F_RS03155 has a mutation rate of 0.023333333333333334.\n",
      "2025-04-16 11:13:19,611 - INFO - Gene cds-WP_000262802.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,612 - INFO - Gene cds-SPN23F_RS03160 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,612 - INFO - Gene cds-SPN23F_RS03165 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,612 - INFO - Gene gene-SPN23F_RS03170 has a mutation rate of 0.49666666666666665.\n",
      "2025-04-16 11:13:19,612 - INFO - Gene cds-WP_000571281.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,613 - INFO - Gene gene-SPN23F_RS03175 has a mutation rate of 0.7433333333333333.\n",
      "2025-04-16 11:13:19,613 - INFO - Gene cds-WP_000863025.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,613 - INFO - Gene gene-SPN23F_RS03180 has a mutation rate of 0.81.\n",
      "2025-04-16 11:13:19,613 - INFO - Gene cds-WP_000724826.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,614 - INFO - Gene gene-SPN23F_RS03185 has a mutation rate of 0.8766666666666667.\n",
      "2025-04-16 11:13:19,614 - INFO - Gene cds-WP_000031148.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,614 - INFO - Gene gene-SPN23F_RS03190 has a mutation rate of 0.6266666666666667.\n",
      "2025-04-16 11:13:19,615 - INFO - Gene cds-WP_001206731.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,615 - INFO - Gene gene-SPN23F_RS03195 has a mutation rate of 0.5.\n",
      "2025-04-16 11:13:19,615 - INFO - Gene cds-WP_000170915.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,615 - INFO - Gene gene-SPN23F_RS14080 has a mutation rate of 0.1.\n",
      "2025-04-16 11:13:19,616 - INFO - Gene cds-WP_001814198.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,616 - INFO - Gene gene-SPN23F_RS03205 has a mutation rate of 0.6233333333333333.\n",
      "2025-04-16 11:13:19,617 - INFO - Gene cds-WP_000394069.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,617 - INFO - Gene gene-SPN23F_RS03210 has a mutation rate of 0.36666666666666664.\n",
      "2025-04-16 11:13:19,618 - INFO - Gene cds-WP_000808917.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,618 - INFO - Gene gene-SPN23F_RS03215 has a mutation rate of 0.8133333333333334.\n",
      "2025-04-16 11:13:19,618 - INFO - Gene cds-WP_000517360.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,618 - INFO - Gene gene-SPN23F_RS03220 has a mutation rate of 0.5033333333333333.\n",
      "2025-04-16 11:13:19,619 - INFO - Gene cds-WP_001002414.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,619 - INFO - Gene cds-SPN23F_RS03225 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,619 - INFO - Gene gene-SPN23F_RS03235 has a mutation rate of 0.7866666666666666.\n",
      "2025-04-16 11:13:19,619 - INFO - Gene cds-WP_000005006.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,620 - INFO - Gene gene-SPN23F_RS03240 has a mutation rate of 0.81.\n",
      "2025-04-16 11:13:19,620 - INFO - Gene cds-WP_001154147.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,621 - INFO - Gene gene-SPN23F_RS03245 has a mutation rate of 0.7066666666666667.\n",
      "2025-04-16 11:13:19,621 - INFO - Gene cds-WP_029645453.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,621 - INFO - Gene cds-SPN23F_RS03250 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,622 - INFO - Gene gene-SPN23F_RS03255 has a mutation rate of 0.68.\n",
      "2025-04-16 11:13:19,622 - INFO - Gene cds-WP_000102469.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,623 - INFO - Gene gene-SPN23F_RS03260 has a mutation rate of 0.23.\n",
      "2025-04-16 11:13:19,623 - INFO - Gene cds-WP_000120707.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,623 - INFO - Gene gene-SPN23F_RS03265 has a mutation rate of 0.24.\n",
      "2025-04-16 11:13:19,624 - INFO - Gene cds-WP_000449824.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,624 - INFO - Gene gene-SPN23F_RS13820 has a mutation rate of 0.31.\n",
      "2025-04-16 11:13:19,625 - INFO - Gene cds-WP_000693513.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,625 - INFO - Gene gene-SPN23F_RS03275 has a mutation rate of 0.66.\n",
      "2025-04-16 11:13:19,626 - INFO - Gene cds-WP_000202464.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,626 - INFO - Gene gene-SPN23F_RS03280 has a mutation rate of 0.22333333333333333.\n",
      "2025-04-16 11:13:19,626 - INFO - Gene cds-WP_000468587.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,627 - INFO - Gene gene-SPN23F_RS03285 has a mutation rate of 0.24666666666666667.\n",
      "2025-04-16 11:13:19,627 - INFO - Gene cds-WP_000915286.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,627 - INFO - Gene gene-SPN23F_RS03290 has a mutation rate of 0.8433333333333334.\n",
      "2025-04-16 11:13:19,628 - INFO - Gene cds-WP_000522118.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,628 - INFO - Gene gene-SPN23F_RS03295 has a mutation rate of 0.29.\n",
      "2025-04-16 11:13:19,629 - INFO - Gene cds-WP_000241364.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,638 - INFO - Gene gene-SPN23F_RS03300 has a mutation rate of 0.47333333333333333.\n",
      "2025-04-16 11:13:19,639 - INFO - Gene cds-WP_000397220.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,662 - INFO - Gene gene-SPN23F_RS03305 has a mutation rate of 0.43333333333333335.\n",
      "2025-04-16 11:13:19,663 - INFO - Gene cds-WP_001225556.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,663 - INFO - Gene gene-SPN23F_RS03310 has a mutation rate of 0.8866666666666667.\n",
      "2025-04-16 11:13:19,664 - INFO - Gene cds-WP_001812917.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,664 - INFO - Gene gene-SPN23F_RS03315 has a mutation rate of 0.42333333333333334.\n",
      "2025-04-16 11:13:19,665 - INFO - Gene cds-WP_001078222.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,665 - INFO - Gene gene-SPN23F_RS03320 has a mutation rate of 0.63.\n",
      "2025-04-16 11:13:19,666 - INFO - Gene cds-WP_000221742.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,666 - INFO - Gene gene-SPN23F_RS03325 has a mutation rate of 0.28.\n",
      "2025-04-16 11:13:19,673 - INFO - Gene cds-WP_001167216.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,674 - INFO - Gene gene-SPN23F_RS03330 has a mutation rate of 0.7066666666666667.\n",
      "2025-04-16 11:13:19,675 - INFO - Gene cds-WP_000935064.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,675 - INFO - Gene gene-SPN23F_RS03335 has a mutation rate of 0.85.\n",
      "2025-04-16 11:13:19,737 - INFO - Gene cds-WP_000136301.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,737 - INFO - Gene gene-SPN23F_RS03340 has a mutation rate of 0.013333333333333334.\n",
      "2025-04-16 11:13:19,738 - INFO - Gene cds-WP_000191795.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,738 - INFO - Gene gene-SPN23F_RS03345 has a mutation rate of 0.16.\n",
      "2025-04-16 11:13:19,738 - INFO - Gene cds-WP_001052655.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,739 - INFO - Gene cds-SPN23F_RS12060 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,739 - INFO - Gene cds-SPN23F_RS03365 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,739 - INFO - Gene gene-SPN23F_RS03370 has a mutation rate of 0.9133333333333333.\n",
      "2025-04-16 11:13:19,740 - INFO - Gene cds-WP_001294281.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,740 - INFO - Gene cds-SPN23F_RS03375 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,740 - INFO - Gene gene-SPN23F_RS14180 has a mutation rate of 0.4533333333333333.\n",
      "2025-04-16 11:13:19,741 - INFO - Gene cds-WP_000231555.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,741 - INFO - Gene cds-SPN23F_RS03385 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,742 - INFO - Gene gene-SPN23F_RS03390 has a mutation rate of 0.10666666666666667.\n",
      "2025-04-16 11:13:19,742 - INFO - Gene cds-WP_001861299.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,742 - INFO - Gene gene-SPN23F_RS03395 has a mutation rate of 0.27.\n",
      "2025-04-16 11:13:19,742 - INFO - Gene cds-WP_000680056.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,743 - INFO - Gene gene-SPN23F_RS03400 has a mutation rate of 0.20333333333333334.\n",
      "2025-04-16 11:13:19,743 - INFO - Gene cds-WP_000219939.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,743 - INFO - Gene gene-SPN23F_RS03405 has a mutation rate of 0.09.\n",
      "2025-04-16 11:13:19,744 - INFO - Gene cds-WP_000004139.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,744 - INFO - Gene gene-SPN23F_RS03410 has a mutation rate of 0.016666666666666666.\n",
      "2025-04-16 11:13:19,744 - INFO - Gene cds-WP_000136976.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,745 - INFO - Gene gene-SPN23F_RS03415 has a mutation rate of 0.38.\n",
      "2025-04-16 11:13:19,745 - INFO - Gene cds-WP_000515974.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,745 - INFO - Gene gene-SPN23F_RS03420 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,745 - INFO - Gene cds-WP_000613477.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,746 - INFO - Gene gene-SPN23F_RS03425 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:19,746 - INFO - Gene cds-WP_000462126.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,746 - INFO - Gene gene-SPN23F_RS03430 has a mutation rate of 0.33.\n",
      "2025-04-16 11:13:19,746 - INFO - Gene cds-WP_000726149.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,747 - INFO - Gene gene-SPN23F_RS03435 has a mutation rate of 0.44666666666666666.\n",
      "2025-04-16 11:13:19,747 - INFO - Gene cds-WP_000941421.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,747 - INFO - Gene gene-SPN23F_RS03440 has a mutation rate of 0.44.\n",
      "2025-04-16 11:13:19,747 - INFO - Gene cds-WP_000662293.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,748 - INFO - Gene gene-SPN23F_RS03445 has a mutation rate of 0.2.\n",
      "2025-04-16 11:13:19,748 - INFO - Gene cds-WP_001186003.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,749 - INFO - Gene gene-SPN23F_RS03450 has a mutation rate of 0.41333333333333333.\n",
      "2025-04-16 11:13:19,749 - INFO - Gene cds-WP_000062214.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,750 - INFO - Gene gene-SPN23F_RS03455 has a mutation rate of 0.44333333333333336.\n",
      "2025-04-16 11:13:19,750 - INFO - Gene cds-WP_000268676.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,751 - INFO - Gene gene-SPN23F_RS03460 has a mutation rate of 0.19666666666666666.\n",
      "2025-04-16 11:13:19,751 - INFO - Gene cds-WP_010963246.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,751 - INFO - Gene cds-WP_010963246.1_1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,752 - INFO - Gene gene-SPN23F_RS03465 has a mutation rate of 0.5966666666666667.\n",
      "2025-04-16 11:13:19,752 - INFO - Gene cds-WP_000022268.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,752 - INFO - Gene gene-SPN23F_RS03470 has a mutation rate of 0.5766666666666667.\n",
      "2025-04-16 11:13:19,753 - INFO - Gene cds-WP_000625536.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,753 - INFO - Gene gene-SPN23F_RS03475 has a mutation rate of 0.7766666666666666.\n",
      "2025-04-16 11:13:19,754 - INFO - Gene cds-WP_000974062.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,754 - INFO - Gene gene-SPN23F_RS03480 has a mutation rate of 0.46.\n",
      "2025-04-16 11:13:19,754 - INFO - Gene cds-WP_000670830.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,755 - INFO - Gene gene-SPN23F_RS03485 has a mutation rate of 0.38333333333333336.\n",
      "2025-04-16 11:13:19,755 - INFO - Gene cds-WP_000011015.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,755 - INFO - Gene gene-SPN23F_RS03490 has a mutation rate of 0.42333333333333334.\n",
      "2025-04-16 11:13:19,755 - INFO - Gene cds-WP_000003936.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,756 - INFO - Gene gene-SPN23F_RS03495 has a mutation rate of 0.16.\n",
      "2025-04-16 11:13:19,756 - INFO - Gene cds-WP_000255160.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,757 - INFO - Gene gene-SPN23F_RS03500 has a mutation rate of 0.5866666666666667.\n",
      "2025-04-16 11:13:19,757 - INFO - Gene cds-WP_000879650.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,757 - INFO - Gene gene-SPN23F_RS03505 has a mutation rate of 0.11333333333333333.\n",
      "2025-04-16 11:13:19,758 - INFO - Gene cds-WP_000974746.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,758 - INFO - Gene gene-SPN23F_RS03510 has a mutation rate of 0.14.\n",
      "2025-04-16 11:13:19,759 - INFO - Gene cds-WP_001224351.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,759 - INFO - Gene gene-SPN23F_RS03515 has a mutation rate of 0.6533333333333333.\n",
      "2025-04-16 11:13:19,759 - INFO - Gene cds-WP_000804658.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,760 - INFO - Gene gene-SPN23F_RS03520 has a mutation rate of 0.45.\n",
      "2025-04-16 11:13:19,760 - INFO - Gene cds-WP_000976708.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,760 - INFO - Gene gene-SPN23F_RS03525 has a mutation rate of 0.46.\n",
      "2025-04-16 11:13:19,760 - INFO - Gene cds-WP_000025400.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,761 - INFO - Gene gene-SPN23F_RS03530 has a mutation rate of 0.16666666666666666.\n",
      "2025-04-16 11:13:19,761 - INFO - Gene cds-WP_000436233.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,761 - INFO - Gene gene-SPN23F_RS03535 has a mutation rate of 0.6066666666666667.\n",
      "2025-04-16 11:13:19,762 - INFO - Gene cds-WP_000731918.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,762 - INFO - Gene gene-SPN23F_RS03540 has a mutation rate of 0.17.\n",
      "2025-04-16 11:13:19,762 - INFO - Gene cds-WP_000011699.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,763 - INFO - Gene gene-SPN23F_RS03545 has a mutation rate of 0.6166666666666667.\n",
      "2025-04-16 11:13:19,773 - INFO - Gene cds-WP_000953865.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,791 - INFO - Gene gene-SPN23F_RS03550 has a mutation rate of 0.12333333333333334.\n",
      "2025-04-16 11:13:19,792 - INFO - Gene cds-WP_000268761.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,792 - INFO - Gene gene-SPN23F_RS03555 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,793 - INFO - Gene cds-WP_000379621.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,793 - INFO - Gene cds-SPN23F_RS13825 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,794 - INFO - Gene gene-SPN23F_RS03560 has a mutation rate of 0.7366666666666667.\n",
      "2025-04-16 11:13:19,794 - INFO - Gene cds-WP_000819199.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,799 - INFO - Gene gene-SPN23F_RS03565 has a mutation rate of 0.12.\n",
      "2025-04-16 11:13:19,801 - INFO - Gene cds-WP_001105899.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,802 - INFO - Gene gene-SPN23F_RS03570 has a mutation rate of 0.42.\n",
      "2025-04-16 11:13:19,802 - INFO - Gene cds-WP_000686921.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,803 - INFO - Gene gene-SPN23F_RS03575 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,803 - INFO - Gene cds-WP_001196049.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,803 - INFO - Gene gene-SPN23F_RS03580 has a mutation rate of 0.043333333333333335.\n",
      "2025-04-16 11:13:19,804 - INFO - Gene cds-WP_001216099.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,804 - INFO - Gene gene-SPN23F_RS03585 has a mutation rate of 0.04.\n",
      "2025-04-16 11:13:19,804 - INFO - Gene cds-WP_000807422.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,805 - INFO - Gene gene-SPN23F_RS13830 has a mutation rate of 0.39.\n",
      "2025-04-16 11:13:19,805 - INFO - Gene cds-WP_001809980.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,805 - INFO - Gene gene-SPN23F_RS03595 has a mutation rate of 0.5966666666666667.\n",
      "2025-04-16 11:13:19,805 - INFO - Gene cds-WP_000709036.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,806 - INFO - Gene gene-SPN23F_RS03600 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,806 - INFO - Gene cds-WP_000746105.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,806 - INFO - Gene gene-SPN23F_RS03605 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,806 - INFO - Gene cds-WP_000768912.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,807 - INFO - Gene gene-SPN23F_RS03610 has a mutation rate of 0.03666666666666667.\n",
      "2025-04-16 11:13:19,807 - INFO - Gene cds-WP_000151029.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,807 - INFO - Gene gene-SPN23F_RS03615 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,808 - INFO - Gene cds-WP_000727307.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,808 - INFO - Gene gene-SPN23F_RS03620 has a mutation rate of 0.73.\n",
      "2025-04-16 11:13:19,808 - INFO - Gene cds-WP_001209571.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,809 - INFO - Gene gene-SPN23F_RS03625 has a mutation rate of 0.79.\n",
      "2025-04-16 11:13:19,809 - INFO - Gene cds-WP_000728634.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,810 - INFO - Gene gene-SPN23F_RS03630 has a mutation rate of 0.10666666666666667.\n",
      "2025-04-16 11:13:19,810 - INFO - Gene cds-WP_000811482.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,810 - INFO - Gene gene-SPN23F_RS03635 has a mutation rate of 0.6033333333333334.\n",
      "2025-04-16 11:13:19,810 - INFO - Gene cds-WP_001180430.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,811 - INFO - Gene gene-SPN23F_RS03640 has a mutation rate of 0.47.\n",
      "2025-04-16 11:13:19,811 - INFO - Gene cds-WP_001291369.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,811 - INFO - Gene gene-SPN23F_RS13835 has a mutation rate of 0.16.\n",
      "2025-04-16 11:13:19,812 - INFO - Gene cds-WP_001814210.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,812 - INFO - Gene gene-SPN23F_RS03650 has a mutation rate of 0.13333333333333333.\n",
      "2025-04-16 11:13:19,813 - INFO - Gene cds-WP_000665420.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,813 - INFO - Gene gene-SPN23F_RS03655 has a mutation rate of 0.3433333333333333.\n",
      "2025-04-16 11:13:19,814 - INFO - Gene cds-WP_000738408.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,814 - INFO - Gene gene-SPN23F_RS03660 has a mutation rate of 0.77.\n",
      "2025-04-16 11:13:19,814 - INFO - Gene cds-WP_001269471.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,815 - INFO - Gene gene-SPN23F_RS03665 has a mutation rate of 0.38.\n",
      "2025-04-16 11:13:19,815 - INFO - Gene cds-WP_000141921.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,815 - INFO - Gene gene-SPN23F_RS03670 has a mutation rate of 0.51.\n",
      "2025-04-16 11:13:19,816 - INFO - Gene cds-WP_000167105.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,816 - INFO - Gene gene-SPN23F_RS03675 has a mutation rate of 0.7766666666666666.\n",
      "2025-04-16 11:13:19,817 - INFO - Gene cds-WP_000403346.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,817 - INFO - Gene gene-SPN23F_RS03680 has a mutation rate of 0.11333333333333333.\n",
      "2025-04-16 11:13:19,817 - INFO - Gene cds-WP_000418692.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,818 - INFO - Gene gene-SPN23F_RS03685 has a mutation rate of 0.9466666666666667.\n",
      "2025-04-16 11:13:19,818 - INFO - Gene cds-WP_001245829.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,818 - INFO - Gene gene-SPN23F_RS03690 has a mutation rate of 0.8166666666666667.\n",
      "2025-04-16 11:13:19,819 - INFO - Gene cds-WP_000612881.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,819 - INFO - Gene gene-SPN23F_RS03695 has a mutation rate of 0.46.\n",
      "2025-04-16 11:13:19,819 - INFO - Gene cds-WP_001149115.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,820 - INFO - Gene gene-SPN23F_RS03700 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,820 - INFO - Gene cds-WP_000436228.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,820 - INFO - Gene gene-SPN23F_RS03705 has a mutation rate of 0.02666666666666667.\n",
      "2025-04-16 11:13:19,820 - INFO - Gene cds-WP_000590619.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,821 - INFO - Gene gene-SPN23F_RS03710 has a mutation rate of 0.09666666666666666.\n",
      "2025-04-16 11:13:19,821 - INFO - Gene cds-WP_000491790.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,821 - INFO - Gene gene-SPN23F_RS03715 has a mutation rate of 0.09666666666666666.\n",
      "2025-04-16 11:13:19,822 - INFO - Gene cds-WP_001145225.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,822 - INFO - Gene gene-SPN23F_RS03720 has a mutation rate of 0.8666666666666667.\n",
      "2025-04-16 11:13:19,823 - INFO - Gene cds-WP_000411854.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,823 - INFO - Gene gene-SPN23F_RS03725 has a mutation rate of 0.93.\n",
      "2025-04-16 11:13:19,823 - INFO - Gene cds-WP_000848695.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,824 - INFO - Gene gene-SPN23F_RS03730 has a mutation rate of 0.25.\n",
      "2025-04-16 11:13:19,824 - INFO - Gene cds-WP_000830659.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,824 - INFO - Gene gene-SPN23F_RS03735 has a mutation rate of 0.5566666666666666.\n",
      "2025-04-16 11:13:19,825 - INFO - Gene cds-WP_000241398.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,825 - INFO - Gene gene-SPN23F_RS03740 has a mutation rate of 0.6133333333333333.\n",
      "2025-04-16 11:13:19,826 - INFO - Gene cds-WP_001812999.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,826 - INFO - Gene gene-SPN23F_RS03745 has a mutation rate of 0.7866666666666666.\n",
      "2025-04-16 11:13:19,826 - INFO - Gene cds-WP_000134047.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,827 - INFO - Gene gene-SPN23F_RS03750 has a mutation rate of 0.33666666666666667.\n",
      "2025-04-16 11:13:19,827 - INFO - Gene cds-WP_000064817.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,828 - INFO - Gene gene-SPN23F_RS03755 has a mutation rate of 0.6033333333333334.\n",
      "2025-04-16 11:13:19,828 - INFO - Gene cds-WP_000973421.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,828 - INFO - Gene gene-SPN23F_RS03760 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,829 - INFO - Gene cds-WP_000767193.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,829 - INFO - Gene gene-SPN23F_RS03765 has a mutation rate of 0.94.\n",
      "2025-04-16 11:13:19,829 - INFO - Gene cds-WP_001814211.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,830 - INFO - Gene cds-SPN23F_RS12100 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,830 - INFO - Gene gene-SPN23F_RS03775 has a mutation rate of 0.52.\n",
      "2025-04-16 11:13:19,830 - INFO - Gene cds-WP_000882517.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,830 - INFO - Gene gene-SPN23F_RS03780 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:19,831 - INFO - Gene cds-WP_000443573.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,831 - INFO - Gene gene-SPN23F_RS03785 has a mutation rate of 0.32.\n",
      "2025-04-16 11:13:19,832 - INFO - Gene cds-WP_001011633.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,832 - INFO - Gene gene-SPN23F_RS03790 has a mutation rate of 0.49333333333333335.\n",
      "2025-04-16 11:13:19,832 - INFO - Gene cds-WP_000140941.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,833 - INFO - Gene gene-SPN23F_RS03795 has a mutation rate of 0.7033333333333334.\n",
      "2025-04-16 11:13:19,833 - INFO - Gene cds-WP_000192081.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,833 - INFO - Gene gene-SPN23F_RS03800 has a mutation rate of 0.06333333333333334.\n",
      "2025-04-16 11:13:19,834 - INFO - Gene cds-WP_000601063.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,834 - INFO - Gene gene-SPN23F_RS03805 has a mutation rate of 0.43.\n",
      "2025-04-16 11:13:19,835 - INFO - Gene cds-WP_000863006.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,835 - INFO - Gene gene-SPN23F_RS03810 has a mutation rate of 0.056666666666666664.\n",
      "2025-04-16 11:13:19,835 - INFO - Gene cds-WP_000429260.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,836 - INFO - Gene gene-SPN23F_RS03815 has a mutation rate of 0.4633333333333333.\n",
      "2025-04-16 11:13:19,836 - INFO - Gene cds-WP_000033102.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,837 - INFO - Gene gene-SPN23F_RS03820 has a mutation rate of 0.32.\n",
      "2025-04-16 11:13:19,837 - INFO - Gene cds-WP_001814214.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,837 - INFO - Gene gene-SPN23F_RS03825 has a mutation rate of 0.043333333333333335.\n",
      "2025-04-16 11:13:19,838 - INFO - Gene cds-WP_000143447.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,838 - INFO - Gene gene-SPN23F_RS03830 has a mutation rate of 0.6033333333333334.\n",
      "2025-04-16 11:13:19,838 - INFO - Gene cds-WP_000887338.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,839 - INFO - Gene cds-SPN23F_RS03835 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,839 - INFO - Gene gene-SPN23F_RS03840 has a mutation rate of 0.58.\n",
      "2025-04-16 11:13:19,839 - INFO - Gene cds-WP_001275919.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,839 - INFO - Gene gene-SPN23F_RS03850 has a mutation rate of 0.69.\n",
      "2025-04-16 11:13:19,840 - INFO - Gene cds-WP_000022100.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,840 - INFO - Gene gene-SPN23F_RS03855 has a mutation rate of 0.5133333333333333.\n",
      "2025-04-16 11:13:19,840 - INFO - Gene cds-WP_000704759.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,841 - INFO - Gene gene-SPN23F_RS03860 has a mutation rate of 0.11666666666666667.\n",
      "2025-04-16 11:13:19,841 - INFO - Gene cds-WP_001274003.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,841 - INFO - Gene gene-SPN23F_RS03865 has a mutation rate of 0.5566666666666666.\n",
      "2025-04-16 11:13:19,845 - INFO - Gene cds-WP_000180492.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,846 - INFO - Gene cds-SPN23F_RS14185 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,871 - INFO - Gene gene-SPN23F_RS03875 has a mutation rate of 0.4766666666666667.\n",
      "2025-04-16 11:13:19,871 - INFO - Gene cds-WP_001809884.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,872 - INFO - Gene gene-SPN23F_RS03880 has a mutation rate of 0.7.\n",
      "2025-04-16 11:13:19,872 - INFO - Gene cds-WP_001202950.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,873 - INFO - Gene gene-SPN23F_RS03885 has a mutation rate of 0.8366666666666667.\n",
      "2025-04-16 11:13:19,873 - INFO - Gene cds-WP_000773688.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,874 - INFO - Gene gene-SPN23F_RS03890 has a mutation rate of 0.8466666666666667.\n",
      "2025-04-16 11:13:19,874 - INFO - Gene cds-WP_000246087.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,875 - INFO - Gene gene-SPN23F_RS03895 has a mutation rate of 0.25333333333333335.\n",
      "2025-04-16 11:13:19,876 - INFO - Gene cds-WP_001036142.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,877 - INFO - Gene gene-SPN23F_RS03900 has a mutation rate of 0.14.\n",
      "2025-04-16 11:13:19,879 - INFO - Gene cds-WP_000929854.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,880 - INFO - Gene gene-SPN23F_RS03905 has a mutation rate of 0.18.\n",
      "2025-04-16 11:13:19,880 - INFO - Gene cds-WP_000038659.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,881 - INFO - Gene gene-SPN23F_RS03910 has a mutation rate of 0.36666666666666664.\n",
      "2025-04-16 11:13:19,881 - INFO - Gene cds-WP_000022879.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,882 - INFO - Gene cds-SPN23F_RS13500 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,882 - INFO - Gene gene-SPN23F_RS03930 has a mutation rate of 0.08333333333333333.\n",
      "2025-04-16 11:13:19,883 - INFO - Gene cds-WP_000628789.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,883 - INFO - Gene gene-SPN23F_RS03935 has a mutation rate of 0.47333333333333333.\n",
      "2025-04-16 11:13:19,884 - INFO - Gene cds-WP_000037258.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,884 - INFO - Gene cds-SPN23F_RS14085 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,884 - INFO - Gene gene-SPN23F_RS03945 has a mutation rate of 0.7233333333333334.\n",
      "2025-04-16 11:13:19,885 - INFO - Gene cds-WP_001821666.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,885 - INFO - Gene gene-SPN23F_RS03950 has a mutation rate of 0.09666666666666666.\n",
      "2025-04-16 11:13:19,885 - INFO - Gene cds-WP_000219018.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,886 - INFO - Gene gene-SPN23F_RS03955 has a mutation rate of 0.6566666666666666.\n",
      "2025-04-16 11:13:19,886 - INFO - Gene cds-WP_001809072.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,886 - INFO - Gene gene-SPN23F_RS03960 has a mutation rate of 0.013333333333333334.\n",
      "2025-04-16 11:13:19,886 - INFO - Gene cds-WP_000454809.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,887 - INFO - Gene gene-SPN23F_RS03965 has a mutation rate of 0.04.\n",
      "2025-04-16 11:13:19,887 - INFO - Gene cds-WP_000137265.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,887 - INFO - Gene gene-SPN23F_RS03970 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:19,888 - INFO - Gene cds-WP_000699369.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,888 - INFO - Gene gene-SPN23F_RS03975 has a mutation rate of 0.07666666666666666.\n",
      "2025-04-16 11:13:19,888 - INFO - Gene cds-WP_000037113.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,889 - INFO - Gene gene-SPN23F_RS03980 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,889 - INFO - Gene gene-SPN23F_RS03985 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,889 - INFO - Gene gene-SPN23F_RS03990 has a mutation rate of 0.34.\n",
      "2025-04-16 11:13:19,890 - INFO - Gene cds-WP_001001623.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,890 - INFO - Gene cds-SPN23F_RS12120 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,891 - INFO - Gene gene-SPN23F_RS04010 has a mutation rate of 0.3433333333333333.\n",
      "2025-04-16 11:13:19,891 - INFO - Gene cds-WP_000920158.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,891 - INFO - Gene gene-SPN23F_RS04015 has a mutation rate of 0.7533333333333333.\n",
      "2025-04-16 11:13:19,892 - INFO - Gene cds-WP_000283791.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,892 - INFO - Gene gene-SPN23F_RS04020 has a mutation rate of 0.03666666666666667.\n",
      "2025-04-16 11:13:19,893 - INFO - Gene cds-WP_001041547.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,893 - INFO - Gene gene-SPN23F_RS04025 has a mutation rate of 0.043333333333333335.\n",
      "2025-04-16 11:13:19,893 - INFO - Gene cds-WP_000114495.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,894 - INFO - Gene gene-SPN23F_RS04030 has a mutation rate of 0.89.\n",
      "2025-04-16 11:13:19,894 - INFO - Gene cds-WP_000196409.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,895 - INFO - Gene gene-SPN23F_RS04035 has a mutation rate of 0.97.\n",
      "2025-04-16 11:13:19,895 - INFO - Gene cds-WP_000887731.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,896 - INFO - Gene gene-SPN23F_RS04040 has a mutation rate of 0.5066666666666667.\n",
      "2025-04-16 11:13:19,897 - INFO - Gene cds-WP_001218480.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,897 - INFO - Gene gene-SPN23F_RS04045 has a mutation rate of 0.30333333333333334.\n",
      "2025-04-16 11:13:19,898 - INFO - Gene cds-WP_000797060.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,898 - INFO - Gene gene-SPN23F_RS04055 has a mutation rate of 0.38.\n",
      "2025-04-16 11:13:19,899 - INFO - Gene cds-WP_000719163.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,899 - INFO - Gene gene-SPN23F_RS04060 has a mutation rate of 0.29.\n",
      "2025-04-16 11:13:19,899 - INFO - Gene cds-WP_000489841.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,900 - INFO - Gene gene-SPN23F_RS12130 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,900 - INFO - Gene gene-SPN23F_RS04065 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,900 - INFO - Gene gene-SPN23F_RS04070 has a mutation rate of 0.10666666666666667.\n",
      "2025-04-16 11:13:19,901 - INFO - Gene cds-WP_000920643.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,901 - INFO - Gene gene-SPN23F_RS04075 has a mutation rate of 0.63.\n",
      "2025-04-16 11:13:19,902 - INFO - Gene cds-WP_000640779.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,902 - INFO - Gene gene-SPN23F_RS04080 has a mutation rate of 0.45.\n",
      "2025-04-16 11:13:19,903 - INFO - Gene cds-WP_000701447.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,903 - INFO - Gene gene-SPN23F_RS04085 has a mutation rate of 0.8933333333333333.\n",
      "2025-04-16 11:13:19,903 - INFO - Gene cds-WP_277618925.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,904 - INFO - Gene gene-SPN23F_RS04090 has a mutation rate of 0.6766666666666666.\n",
      "2025-04-16 11:13:19,904 - INFO - Gene cds-WP_000073931.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,904 - INFO - Gene gene-SPN23F_RS04095 has a mutation rate of 0.83.\n",
      "2025-04-16 11:13:19,905 - INFO - Gene cds-WP_000638997.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,905 - INFO - Gene gene-SPN23F_RS04100 has a mutation rate of 0.15333333333333332.\n",
      "2025-04-16 11:13:19,905 - INFO - Gene cds-WP_001200062.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,905 - INFO - Gene gene-SPN23F_RS12135 has a mutation rate of 0.96.\n",
      "2025-04-16 11:13:19,906 - INFO - Gene cds-WP_001076703.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,906 - INFO - Gene gene-SPN23F_RS04115 has a mutation rate of 0.63.\n",
      "2025-04-16 11:13:19,907 - INFO - Gene cds-WP_001108634.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,907 - INFO - Gene gene-SPN23F_RS14090 has a mutation rate of 0.35.\n",
      "2025-04-16 11:13:19,907 - INFO - Gene cds-WP_001818232.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,908 - INFO - Gene gene-SPN23F_RS04120 has a mutation rate of 0.20666666666666667.\n",
      "2025-04-16 11:13:19,908 - INFO - Gene cds-WP_000627748.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,908 - INFO - Gene gene-SPN23F_RS04125 has a mutation rate of 0.42.\n",
      "2025-04-16 11:13:19,909 - INFO - Gene cds-WP_001814217.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,909 - INFO - Gene gene-SPN23F_RS04130 has a mutation rate of 0.17666666666666667.\n",
      "2025-04-16 11:13:19,909 - INFO - Gene cds-WP_000756111.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,910 - INFO - Gene gene-SPN23F_RS04135 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,910 - INFO - Gene cds-WP_000260605.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,911 - INFO - Gene gene-SPN23F_RS04140 has a mutation rate of 0.07.\n",
      "2025-04-16 11:13:19,911 - INFO - Gene cds-WP_000158366.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,911 - INFO - Gene gene-SPN23F_RS04145 has a mutation rate of 0.23666666666666666.\n",
      "2025-04-16 11:13:19,940 - INFO - Gene cds-WP_000444455.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,941 - INFO - Gene gene-SPN23F_RS04150 has a mutation rate of 0.22333333333333333.\n",
      "2025-04-16 11:13:19,941 - INFO - Gene cds-WP_001819123.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,942 - INFO - Gene gene-SPN23F_RS04155 has a mutation rate of 0.7333333333333333.\n",
      "2025-04-16 11:13:19,943 - INFO - Gene cds-WP_000526339.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,950 - INFO - Gene gene-SPN23F_RS04160 has a mutation rate of 0.04666666666666667.\n",
      "2025-04-16 11:13:19,951 - INFO - Gene cds-WP_001042999.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,952 - INFO - Gene gene-SPN23F_RS04165 has a mutation rate of 0.72.\n",
      "2025-04-16 11:13:19,952 - INFO - Gene cds-WP_001212055.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,953 - INFO - Gene gene-SPN23F_RS04170 has a mutation rate of 0.9966666666666667.\n",
      "2025-04-16 11:13:19,953 - INFO - Gene cds-WP_000561506.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,954 - INFO - Gene gene-SPN23F_RS04175 has a mutation rate of 0.33.\n",
      "2025-04-16 11:13:19,954 - INFO - Gene cds-WP_000820852.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,955 - INFO - Gene gene-SPN23F_RS04180 has a mutation rate of 0.34.\n",
      "2025-04-16 11:13:19,955 - INFO - Gene cds-WP_001042814.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,956 - INFO - Gene cds-SPN23F_RS14190 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,956 - INFO - Gene gene-SPN23F_RS04190 has a mutation rate of 0.39666666666666667.\n",
      "2025-04-16 11:13:19,956 - INFO - Gene cds-WP_000749599.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,957 - INFO - Gene cds-SPN23F_RS04200 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,958 - INFO - Gene gene-SPN23F_RS04205 has a mutation rate of 0.6166666666666667.\n",
      "2025-04-16 11:13:19,958 - INFO - Gene cds-WP_000420561.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,958 - INFO - Gene gene-SPN23F_RS04210 has a mutation rate of 0.95.\n",
      "2025-04-16 11:13:19,959 - INFO - Gene cds-WP_000759790.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,959 - INFO - Gene gene-SPN23F_RS04215 has a mutation rate of 0.49333333333333335.\n",
      "2025-04-16 11:13:19,959 - INFO - Gene cds-WP_000617849.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,960 - INFO - Gene gene-SPN23F_RS04220 has a mutation rate of 0.7833333333333333.\n",
      "2025-04-16 11:13:19,960 - INFO - Gene cds-WP_000123613.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,961 - INFO - Gene gene-SPN23F_RS14095 has a mutation rate of 0.44.\n",
      "2025-04-16 11:13:19,961 - INFO - Gene cds-WP_000268689.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,961 - INFO - Gene gene-SPN23F_RS13625 has a mutation rate of 0.23.\n",
      "2025-04-16 11:13:19,962 - INFO - Gene cds-WP_001815868.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,962 - INFO - Gene gene-SPN23F_RS04230 has a mutation rate of 0.5866666666666667.\n",
      "2025-04-16 11:13:19,962 - INFO - Gene cds-WP_000359142.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,963 - INFO - Gene gene-SPN23F_RS04235 has a mutation rate of 0.5866666666666667.\n",
      "2025-04-16 11:13:19,963 - INFO - Gene cds-WP_000778576.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:19,964 - INFO - Gene gene-SPN23F_RS04240 has a mutation rate of 0.12666666666666668.\n",
      "2025-04-16 11:13:20,019 - INFO - Gene cds-WP_001085044.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,019 - INFO - Gene gene-SPN23F_RS04250 has a mutation rate of 0.5033333333333333.\n",
      "2025-04-16 11:13:20,020 - INFO - Gene cds-WP_000680766.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,020 - INFO - Gene gene-SPN23F_RS04255 has a mutation rate of 0.43.\n",
      "2025-04-16 11:13:20,020 - INFO - Gene cds-WP_000173358.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,021 - INFO - Gene gene-SPN23F_RS04260 has a mutation rate of 0.8033333333333333.\n",
      "2025-04-16 11:13:20,021 - INFO - Gene cds-WP_000489409.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,021 - INFO - Gene gene-SPN23F_RS04265 has a mutation rate of 0.81.\n",
      "2025-04-16 11:13:20,022 - INFO - Gene cds-WP_255262024.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,022 - INFO - Gene cds-SPN23F_RS04270 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,022 - INFO - Gene gene-SPN23F_RS04275 has a mutation rate of 0.8233333333333334.\n",
      "2025-04-16 11:13:20,023 - INFO - Gene cds-WP_000661015.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,023 - INFO - Gene cds-SPN23F_RS14195 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,023 - INFO - Gene cds-SPN23F_RS04285 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,023 - INFO - Gene gene-SPN23F_RS04290 has a mutation rate of 0.16666666666666666.\n",
      "2025-04-16 11:13:20,024 - INFO - Gene cds-WP_000366717.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,024 - INFO - Gene gene-SPN23F_RS04295 has a mutation rate of 0.7266666666666667.\n",
      "2025-04-16 11:13:20,024 - INFO - Gene cds-WP_000088773.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,025 - INFO - Gene gene-SPN23F_RS04300 has a mutation rate of 0.59.\n",
      "2025-04-16 11:13:20,025 - INFO - Gene cds-WP_000764364.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,026 - INFO - Gene gene-SPN23F_RS04305 has a mutation rate of 0.7466666666666667.\n",
      "2025-04-16 11:13:20,026 - INFO - Gene cds-WP_000969454.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,026 - INFO - Gene gene-SPN23F_RS04310 has a mutation rate of 0.20333333333333334.\n",
      "2025-04-16 11:13:20,027 - INFO - Gene cds-WP_001246757.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,027 - INFO - Gene gene-SPN23F_RS04315 has a mutation rate of 0.51.\n",
      "2025-04-16 11:13:20,027 - INFO - Gene cds-WP_000593571.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,027 - INFO - Gene gene-SPN23F_RS04320 has a mutation rate of 0.26666666666666666.\n",
      "2025-04-16 11:13:20,028 - INFO - Gene cds-WP_000602454.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,028 - INFO - Gene gene-SPN23F_RS04325 has a mutation rate of 0.6966666666666667.\n",
      "2025-04-16 11:13:20,028 - INFO - Gene cds-WP_000745369.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,028 - INFO - Gene gene-SPN23F_RS04330 has a mutation rate of 0.23666666666666666.\n",
      "2025-04-16 11:13:20,029 - INFO - Gene cds-WP_001809535.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,029 - INFO - Gene gene-SPN23F_RS04335 has a mutation rate of 0.4166666666666667.\n",
      "2025-04-16 11:13:20,029 - INFO - Gene cds-WP_001025704.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,030 - INFO - Gene gene-SPN23F_RS04340 has a mutation rate of 0.16666666666666666.\n",
      "2025-04-16 11:13:20,030 - INFO - Gene cds-WP_000745389.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,030 - INFO - Gene gene-SPN23F_RS04345 has a mutation rate of 0.13666666666666666.\n",
      "2025-04-16 11:13:20,031 - INFO - Gene cds-WP_000403199.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,031 - INFO - Gene gene-SPN23F_RS04350 has a mutation rate of 0.8.\n",
      "2025-04-16 11:13:20,031 - INFO - Gene cds-WP_000728276.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,032 - INFO - Gene gene-SPN23F_RS04355 has a mutation rate of 0.6.\n",
      "2025-04-16 11:13:20,032 - INFO - Gene cds-WP_000875739.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,033 - INFO - Gene gene-SPN23F_RS04360 has a mutation rate of 0.8.\n",
      "2025-04-16 11:13:20,033 - INFO - Gene cds-WP_000254684.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,034 - INFO - Gene gene-SPN23F_RS04365 has a mutation rate of 0.7533333333333333.\n",
      "2025-04-16 11:13:20,034 - INFO - Gene cds-WP_000689706.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,034 - INFO - Gene gene-SPN23F_RS04370 has a mutation rate of 0.43.\n",
      "2025-04-16 11:13:20,035 - INFO - Gene cds-WP_000033397.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,035 - INFO - Gene gene-SPN23F_RS04375 has a mutation rate of 0.5966666666666667.\n",
      "2025-04-16 11:13:20,035 - INFO - Gene cds-WP_000806714.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,036 - INFO - Gene gene-SPN23F_RS04380 has a mutation rate of 0.23.\n",
      "2025-04-16 11:13:20,036 - INFO - Gene cds-WP_000358228.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,036 - INFO - Gene gene-SPN23F_RS04385 has a mutation rate of 0.7233333333333334.\n",
      "2025-04-16 11:13:20,037 - INFO - Gene cds-WP_001166870.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,037 - INFO - Gene cds-SPN23F_RS13850 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,037 - INFO - Gene gene-SPN23F_RS04390 has a mutation rate of 0.53.\n",
      "2025-04-16 11:13:20,038 - INFO - Gene cds-WP_001261449.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,038 - INFO - Gene cds-SPN23F_RS04400 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,038 - INFO - Gene gene-SPN23F_RS12175 has a mutation rate of 0.3433333333333333.\n",
      "2025-04-16 11:13:20,039 - INFO - Gene cds-WP_000997239.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,039 - INFO - Gene cds-WP_000997239.1_1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,039 - INFO - Gene gene-SPN23F_RS04415 has a mutation rate of 0.03333333333333333.\n",
      "2025-04-16 11:13:20,040 - INFO - Gene cds-WP_000493529.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,040 - INFO - Gene gene-SPN23F_RS04420 has a mutation rate of 0.5566666666666666.\n",
      "2025-04-16 11:13:20,040 - INFO - Gene cds-WP_000083731.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,041 - INFO - Gene gene-SPN23F_RS04425 has a mutation rate of 0.13.\n",
      "2025-04-16 11:13:20,041 - INFO - Gene cds-WP_000002997.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,041 - INFO - Gene gene-SPN23F_RS04430 has a mutation rate of 0.37.\n",
      "2025-04-16 11:13:20,042 - INFO - Gene cds-WP_000024409.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,042 - INFO - Gene gene-SPN23F_RS04435 has a mutation rate of 0.12666666666666668.\n",
      "2025-04-16 11:13:20,043 - INFO - Gene cds-WP_001095450.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,043 - INFO - Gene gene-SPN23F_RS04440 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,043 - INFO - Gene cds-WP_001232085.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,044 - INFO - Gene gene-SPN23F_RS04445 has a mutation rate of 0.6433333333333333.\n",
      "2025-04-16 11:13:20,044 - INFO - Gene cds-WP_000658183.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,044 - INFO - Gene gene-SPN23F_RS12185 has a mutation rate of 0.43.\n",
      "2025-04-16 11:13:20,045 - INFO - Gene cds-WP_001813350.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,045 - INFO - Gene cds-SPN23F_RS04450 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,045 - INFO - Gene cds-SPN23F_RS04455 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,046 - INFO - Gene cds-SPN23F_RS13855 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,046 - INFO - Gene gene-SPN23F_RS04475 has a mutation rate of 0.7.\n",
      "2025-04-16 11:13:20,048 - INFO - Gene cds-WP_000443777.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,067 - INFO - Gene gene-SPN23F_RS04480 has a mutation rate of 0.72.\n",
      "2025-04-16 11:13:20,075 - INFO - Gene cds-WP_000387328.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,076 - INFO - Gene gene-SPN23F_RS04485 has a mutation rate of 0.96.\n",
      "2025-04-16 11:13:20,077 - INFO - Gene cds-WP_000942392.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,077 - INFO - Gene gene-SPN23F_RS13860 has a mutation rate of 0.7133333333333334.\n",
      "2025-04-16 11:13:20,078 - INFO - Gene cds-WP_000583432.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,078 - INFO - Gene gene-SPN23F_RS04495 has a mutation rate of 0.93.\n",
      "2025-04-16 11:13:20,079 - INFO - Gene cds-WP_000939884.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,079 - INFO - Gene gene-SPN23F_RS04500 has a mutation rate of 0.9233333333333333.\n",
      "2025-04-16 11:13:20,083 - INFO - Gene cds-WP_000655951.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,084 - INFO - Gene gene-SPN23F_RS04505 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,085 - INFO - Gene cds-WP_000848180.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,085 - INFO - Gene gene-SPN23F_RS04510 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,086 - INFO - Gene cds-WP_001125943.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,086 - INFO - Gene gene-SPN23F_RS04515 has a mutation rate of 0.013333333333333334.\n",
      "2025-04-16 11:13:20,086 - INFO - Gene cds-WP_000124836.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,087 - INFO - Gene gene-SPN23F_RS04520 has a mutation rate of 0.11333333333333333.\n",
      "2025-04-16 11:13:20,087 - INFO - Gene cds-WP_000157154.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,088 - INFO - Gene gene-SPN23F_RS04525 has a mutation rate of 0.5333333333333333.\n",
      "2025-04-16 11:13:20,088 - INFO - Gene cds-WP_001054489.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,088 - INFO - Gene gene-SPN23F_RS04530 has a mutation rate of 0.6033333333333334.\n",
      "2025-04-16 11:13:20,089 - INFO - Gene cds-WP_000210693.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,089 - INFO - Gene gene-SPN23F_RS04535 has a mutation rate of 0.72.\n",
      "2025-04-16 11:13:20,089 - INFO - Gene cds-WP_074192727.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,090 - INFO - Gene gene-SPN23F_RS04540 has a mutation rate of 0.7233333333333334.\n",
      "2025-04-16 11:13:20,090 - INFO - Gene cds-WP_000006688.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,090 - INFO - Gene gene-SPN23F_RS04545 has a mutation rate of 0.33.\n",
      "2025-04-16 11:13:20,091 - INFO - Gene cds-WP_000275156.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,091 - INFO - Gene gene-SPN23F_RS04550 has a mutation rate of 0.5233333333333333.\n",
      "2025-04-16 11:13:20,092 - INFO - Gene cds-WP_000378155.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,092 - INFO - Gene gene-SPN23F_RS04555 has a mutation rate of 0.07666666666666666.\n",
      "2025-04-16 11:13:20,092 - INFO - Gene cds-WP_000143264.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,093 - INFO - Gene gene-SPN23F_RS04560 has a mutation rate of 0.9.\n",
      "2025-04-16 11:13:20,093 - INFO - Gene cds-WP_001114585.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,093 - INFO - Gene gene-SPN23F_RS04565 has a mutation rate of 0.75.\n",
      "2025-04-16 11:13:20,093 - INFO - Gene cds-WP_001226118.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,094 - INFO - Gene gene-SPN23F_RS04570 has a mutation rate of 0.3566666666666667.\n",
      "2025-04-16 11:13:20,094 - INFO - Gene cds-WP_000136127.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,094 - INFO - Gene gene-SPN23F_RS04575 has a mutation rate of 0.47.\n",
      "2025-04-16 11:13:20,095 - INFO - Gene cds-WP_001810029.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,095 - INFO - Gene gene-SPN23F_RS04580 has a mutation rate of 0.01.\n",
      "2025-04-16 11:13:20,095 - INFO - Gene cds-WP_000282517.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,096 - INFO - Gene gene-SPN23F_RS04585 has a mutation rate of 0.8466666666666667.\n",
      "2025-04-16 11:13:20,096 - INFO - Gene cds-WP_000654898.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,097 - INFO - Gene gene-SPN23F_RS04590 has a mutation rate of 0.17666666666666667.\n",
      "2025-04-16 11:13:20,097 - INFO - Gene cds-WP_001051750.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,097 - INFO - Gene gene-SPN23F_RS04595 has a mutation rate of 0.4166666666666667.\n",
      "2025-04-16 11:13:20,098 - INFO - Gene cds-WP_000413762.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,098 - INFO - Gene gene-SPN23F_RS04600 has a mutation rate of 0.49333333333333335.\n",
      "2025-04-16 11:13:20,098 - INFO - Gene cds-WP_000495909.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,099 - INFO - Gene gene-SPN23F_RS04605 has a mutation rate of 0.8033333333333333.\n",
      "2025-04-16 11:13:20,099 - INFO - Gene cds-WP_000244125.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,100 - INFO - Gene gene-SPN23F_RS04610 has a mutation rate of 0.58.\n",
      "2025-04-16 11:13:20,100 - INFO - Gene cds-WP_000230399.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,100 - INFO - Gene gene-SPN23F_RS04615 has a mutation rate of 0.22.\n",
      "2025-04-16 11:13:20,101 - INFO - Gene cds-WP_000727935.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,101 - INFO - Gene cds-SPN23F_RS04620 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,101 - INFO - Gene cds-SPN23F_RS12930 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,102 - INFO - Gene gene-SPN23F_RS04630 has a mutation rate of 0.25666666666666665.\n",
      "2025-04-16 11:13:20,102 - INFO - Gene cds-WP_000780815.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,103 - INFO - Gene gene-SPN23F_RS04635 has a mutation rate of 0.5633333333333334.\n",
      "2025-04-16 11:13:20,103 - INFO - Gene cds-WP_001085886.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,104 - INFO - Gene gene-SPN23F_RS04640 has a mutation rate of 0.6766666666666666.\n",
      "2025-04-16 11:13:20,104 - INFO - Gene cds-WP_000954117.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,104 - INFO - Gene gene-SPN23F_RS04645 has a mutation rate of 0.6733333333333333.\n",
      "2025-04-16 11:13:20,104 - INFO - Gene cds-WP_001231241.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,105 - INFO - Gene gene-SPN23F_RS04650 has a mutation rate of 0.43666666666666665.\n",
      "2025-04-16 11:13:20,105 - INFO - Gene cds-WP_000064394.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,105 - INFO - Gene gene-SPN23F_RS04655 has a mutation rate of 0.44666666666666666.\n",
      "2025-04-16 11:13:20,106 - INFO - Gene cds-WP_000393812.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,106 - INFO - Gene gene-SPN23F_RS04660 has a mutation rate of 0.38333333333333336.\n",
      "2025-04-16 11:13:20,106 - INFO - Gene cds-WP_000520411.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,107 - INFO - Gene gene-SPN23F_RS04665 has a mutation rate of 0.59.\n",
      "2025-04-16 11:13:20,107 - INFO - Gene cds-WP_000689789.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,108 - INFO - Gene gene-SPN23F_RS04670 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:20,108 - INFO - Gene cds-WP_000021242.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,108 - INFO - Gene gene-SPN23F_RS04675 has a mutation rate of 0.35333333333333333.\n",
      "2025-04-16 11:13:20,109 - INFO - Gene cds-WP_000440255.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,109 - INFO - Gene gene-SPN23F_RS04680 has a mutation rate of 0.49333333333333335.\n",
      "2025-04-16 11:13:20,109 - INFO - Gene cds-WP_000801724.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,110 - INFO - Gene cds-SPN23F_RS12200 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,110 - INFO - Gene gene-SPN23F_RS04695 has a mutation rate of 0.5466666666666666.\n",
      "2025-04-16 11:13:20,111 - INFO - Gene cds-WP_000226583.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,111 - INFO - Gene cds-SPN23F_RS04700 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,112 - INFO - Gene gene-SPN23F_RS04705 has a mutation rate of 0.23666666666666666.\n",
      "2025-04-16 11:13:20,112 - INFO - Gene cds-WP_000447990.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,113 - INFO - Gene gene-SPN23F_RS04710 has a mutation rate of 0.45.\n",
      "2025-04-16 11:13:20,113 - INFO - Gene cds-WP_000757373.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,114 - INFO - Gene gene-SPN23F_RS04715 has a mutation rate of 0.6333333333333333.\n",
      "2025-04-16 11:13:20,114 - INFO - Gene cds-WP_001020311.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,114 - INFO - Gene gene-SPN23F_RS04720 has a mutation rate of 0.5666666666666667.\n",
      "2025-04-16 11:13:20,115 - INFO - Gene cds-WP_000744528.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,115 - INFO - Gene gene-SPN23F_RS04725 has a mutation rate of 0.7866666666666666.\n",
      "2025-04-16 11:13:20,115 - INFO - Gene cds-WP_000700316.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,116 - INFO - Gene gene-SPN23F_RS04730 has a mutation rate of 0.7966666666666666.\n",
      "2025-04-16 11:13:20,116 - INFO - Gene cds-WP_000672557.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,117 - INFO - Gene cds-SPN23F_RS14100 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,117 - INFO - Gene cds-SPN23F_RS14105 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,118 - INFO - Gene cds-SPN23F_RS04745 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,118 - INFO - Gene gene-SPN23F_RS04750 has a mutation rate of 0.6866666666666666.\n",
      "2025-04-16 11:13:20,119 - INFO - Gene cds-WP_000222032.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,119 - INFO - Gene gene-SPN23F_RS04755 has a mutation rate of 0.6133333333333333.\n",
      "2025-04-16 11:13:20,119 - INFO - Gene cds-WP_000709247.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,120 - INFO - Gene gene-SPN23F_RS04760 has a mutation rate of 0.29.\n",
      "2025-04-16 11:13:20,120 - INFO - Gene cds-WP_000916795.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,120 - INFO - Gene gene-SPN23F_RS04765 has a mutation rate of 0.4.\n",
      "2025-04-16 11:13:20,120 - INFO - Gene cds-WP_000742997.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,121 - INFO - Gene gene-SPN23F_RS04770 has a mutation rate of 0.08333333333333333.\n",
      "2025-04-16 11:13:20,121 - INFO - Gene cds-WP_000737071.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,122 - INFO - Gene gene-SPN23F_RS04775 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,122 - INFO - Gene gene-SPN23F_RS04780 has a mutation rate of 0.6266666666666667.\n",
      "2025-04-16 11:13:20,132 - INFO - Gene cds-WP_000542473.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,132 - INFO - Gene gene-SPN23F_RS04785 has a mutation rate of 0.73.\n",
      "2025-04-16 11:13:20,159 - INFO - Gene cds-WP_000121618.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,160 - INFO - Gene gene-SPN23F_RS04790 has a mutation rate of 0.6233333333333333.\n",
      "2025-04-16 11:13:20,161 - INFO - Gene cds-WP_000632758.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,161 - INFO - Gene gene-SPN23F_RS04795 has a mutation rate of 0.11.\n",
      "2025-04-16 11:13:20,162 - INFO - Gene cds-WP_001117401.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,162 - INFO - Gene gene-SPN23F_RS04800 has a mutation rate of 0.36333333333333334.\n",
      "2025-04-16 11:13:20,163 - INFO - Gene cds-WP_000068143.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,168 - INFO - Gene gene-SPN23F_RS04805 has a mutation rate of 0.38666666666666666.\n",
      "2025-04-16 11:13:20,171 - INFO - Gene cds-WP_001028801.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,171 - INFO - Gene gene-SPN23F_RS04810 has a mutation rate of 0.6633333333333333.\n",
      "2025-04-16 11:13:20,172 - INFO - Gene cds-WP_000761903.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,172 - INFO - Gene gene-SPN23F_RS04815 has a mutation rate of 0.5966666666666667.\n",
      "2025-04-16 11:13:20,173 - INFO - Gene cds-WP_000969297.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,173 - INFO - Gene gene-SPN23F_RS04820 has a mutation rate of 0.42333333333333334.\n",
      "2025-04-16 11:13:20,174 - INFO - Gene cds-WP_000942935.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,174 - INFO - Gene gene-SPN23F_RS04825 has a mutation rate of 0.48.\n",
      "2025-04-16 11:13:20,175 - INFO - Gene cds-WP_000575519.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,175 - INFO - Gene gene-SPN23F_RS04830 has a mutation rate of 0.16666666666666666.\n",
      "2025-04-16 11:13:20,176 - INFO - Gene cds-WP_000356095.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,176 - INFO - Gene gene-SPN23F_RS04835 has a mutation rate of 0.49.\n",
      "2025-04-16 11:13:20,176 - INFO - Gene cds-WP_000476374.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,176 - INFO - Gene gene-SPN23F_RS04840 has a mutation rate of 0.85.\n",
      "2025-04-16 11:13:20,177 - INFO - Gene cds-WP_001227759.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,177 - INFO - Gene gene-SPN23F_RS04845 has a mutation rate of 0.7533333333333333.\n",
      "2025-04-16 11:13:20,177 - INFO - Gene cds-WP_000914566.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,178 - INFO - Gene gene-SPN23F_RS04850 has a mutation rate of 0.6933333333333334.\n",
      "2025-04-16 11:13:20,179 - INFO - Gene cds-WP_000710803.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,179 - INFO - Gene gene-SPN23F_RS04855 has a mutation rate of 0.09.\n",
      "2025-04-16 11:13:20,180 - INFO - Gene cds-WP_001207345.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,180 - INFO - Gene gene-SPN23F_RS04860 has a mutation rate of 0.2633333333333333.\n",
      "2025-04-16 11:13:20,181 - INFO - Gene cds-WP_000790743.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,181 - INFO - Gene gene-SPN23F_RS04865 has a mutation rate of 0.13666666666666666.\n",
      "2025-04-16 11:13:20,182 - INFO - Gene cds-WP_001244693.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,182 - INFO - Gene gene-SPN23F_RS04870 has a mutation rate of 0.21.\n",
      "2025-04-16 11:13:20,183 - INFO - Gene cds-WP_001180357.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,183 - INFO - Gene gene-SPN23F_RS04875 has a mutation rate of 0.34.\n",
      "2025-04-16 11:13:20,183 - INFO - Gene cds-WP_000677519.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,184 - INFO - Gene cds-SPN23F_RS13875 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,184 - INFO - Gene gene-SPN23F_RS12220 has a mutation rate of 0.03333333333333333.\n",
      "2025-04-16 11:13:20,185 - INFO - Gene cds-WP_229294054.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,185 - INFO - Gene cds-SPN23F_RS13400 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,186 - INFO - Gene gene-SPN23F_RS13505 has a mutation rate of 0.07666666666666666.\n",
      "2025-04-16 11:13:20,186 - INFO - Gene cds-WP_001058283.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,186 - INFO - Gene cds-SPN23F_RS13880 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,187 - INFO - Gene gene-SPN23F_RS04905 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:20,187 - INFO - Gene cds-WP_000234989.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,188 - INFO - Gene gene-SPN23F_RS04910 has a mutation rate of 0.023333333333333334.\n",
      "2025-04-16 11:13:20,188 - INFO - Gene cds-WP_000179769.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,189 - INFO - Gene gene-SPN23F_RS04915 has a mutation rate of 0.76.\n",
      "2025-04-16 11:13:20,189 - INFO - Gene cds-WP_000262020.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,189 - INFO - Gene gene-SPN23F_RS04920 has a mutation rate of 0.6566666666666666.\n",
      "2025-04-16 11:13:20,190 - INFO - Gene cds-WP_000900020.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,190 - INFO - Gene gene-SPN23F_RS04925 has a mutation rate of 0.29333333333333333.\n",
      "2025-04-16 11:13:20,191 - INFO - Gene cds-WP_000710119.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,191 - INFO - Gene gene-SPN23F_RS04930 has a mutation rate of 0.9466666666666667.\n",
      "2025-04-16 11:13:20,192 - INFO - Gene cds-WP_000423450.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,192 - INFO - Gene cds-SPN23F_RS13885 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,192 - INFO - Gene gene-SPN23F_RS04945 has a mutation rate of 0.42333333333333334.\n",
      "2025-04-16 11:13:20,193 - INFO - Gene cds-WP_000579605.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,193 - INFO - Gene gene-SPN23F_RS04950 has a mutation rate of 0.6866666666666666.\n",
      "2025-04-16 11:13:20,193 - INFO - Gene cds-WP_000405356.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,194 - INFO - Gene gene-SPN23F_RS04955 has a mutation rate of 0.11.\n",
      "2025-04-16 11:13:20,194 - INFO - Gene cds-WP_001814230.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,194 - INFO - Gene gene-SPN23F_RS14250 has a mutation rate of 0.17666666666666667.\n",
      "2025-04-16 11:13:20,194 - INFO - Gene cds-WP_000530001.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,195 - INFO - Gene gene-SPN23F_RS04965 has a mutation rate of 0.056666666666666664.\n",
      "2025-04-16 11:13:20,195 - INFO - Gene cds-WP_000529543.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,195 - INFO - Gene gene-SPN23F_RS04970 has a mutation rate of 0.17333333333333334.\n",
      "2025-04-16 11:13:20,196 - INFO - Gene cds-WP_000263037.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,196 - INFO - Gene cds-SPN23F_RS13630 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,197 - INFO - Gene gene-SPN23F_RS04990 has a mutation rate of 0.36.\n",
      "2025-04-16 11:13:20,198 - INFO - Gene cds-WP_000837490.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,198 - INFO - Gene gene-SPN23F_RS12940 has a mutation rate of 0.05333333333333334.\n",
      "2025-04-16 11:13:20,199 - INFO - Gene cds-WP_133457516.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,199 - INFO - Gene gene-SPN23F_RS04995 has a mutation rate of 0.16333333333333333.\n",
      "2025-04-16 11:13:20,199 - INFO - Gene cds-WP_000059559.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,200 - INFO - Gene gene-SPN23F_RS12235 has a mutation rate of 0.30666666666666664.\n",
      "2025-04-16 11:13:20,201 - INFO - Gene cds-WP_029645654.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,201 - INFO - Gene gene-SPN23F_RS13520 has a mutation rate of 0.08.\n",
      "2025-04-16 11:13:20,201 - INFO - Gene cds-WP_000757483.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,202 - INFO - Gene gene-SPN23F_RS05015 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,202 - INFO - Gene cds-WP_001018880.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,203 - INFO - Gene gene-SPN23F_RS05020 has a mutation rate of 0.15666666666666668.\n",
      "2025-04-16 11:13:20,203 - INFO - Gene cds-WP_000593060.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,203 - INFO - Gene gene-SPN23F_RS05025 has a mutation rate of 0.15666666666666668.\n",
      "2025-04-16 11:13:20,204 - INFO - Gene cds-WP_000771190.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,204 - INFO - Gene gene-SPN23F_RS05030 has a mutation rate of 0.30666666666666664.\n",
      "2025-04-16 11:13:20,204 - INFO - Gene cds-WP_000286251.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,204 - INFO - Gene cds-SPN23F_RS12245 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,205 - INFO - Gene gene-SPN23F_RS05035 has a mutation rate of 0.5233333333333333.\n",
      "2025-04-16 11:13:20,205 - INFO - Gene cds-WP_000703357.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,205 - INFO - Gene gene-SPN23F_RS05040 has a mutation rate of 0.5166666666666667.\n",
      "2025-04-16 11:13:20,205 - INFO - Gene cds-WP_000058193.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,206 - INFO - Gene gene-SPN23F_RS05045 has a mutation rate of 0.54.\n",
      "2025-04-16 11:13:20,206 - INFO - Gene cds-WP_000792155.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,206 - INFO - Gene gene-SPN23F_RS05050 has a mutation rate of 0.33.\n",
      "2025-04-16 11:13:20,207 - INFO - Gene cds-WP_000637238.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,207 - INFO - Gene gene-SPN23F_RS05055 has a mutation rate of 0.2733333333333333.\n",
      "2025-04-16 11:13:20,207 - INFO - Gene cds-WP_000125412.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,207 - INFO - Gene gene-SPN23F_RS05060 has a mutation rate of 0.81.\n",
      "2025-04-16 11:13:20,208 - INFO - Gene cds-WP_000227855.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,212 - INFO - Gene gene-SPN23F_RS05065 has a mutation rate of 0.07666666666666666.\n",
      "2025-04-16 11:13:20,215 - INFO - Gene cds-WP_000201898.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,217 - INFO - Gene gene-SPN23F_RS05070 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,233 - INFO - Gene cds-WP_000331928.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,242 - INFO - Gene gene-SPN23F_RS05075 has a mutation rate of 0.22.\n",
      "2025-04-16 11:13:20,242 - INFO - Gene cds-WP_000412279.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,243 - INFO - Gene gene-SPN23F_RS05080 has a mutation rate of 0.71.\n",
      "2025-04-16 11:13:20,243 - INFO - Gene cds-WP_001219427.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,244 - INFO - Gene gene-SPN23F_RS05085 has a mutation rate of 0.21333333333333335.\n",
      "2025-04-16 11:13:20,244 - INFO - Gene cds-WP_001256250.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,245 - INFO - Gene gene-SPN23F_RS05090 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:20,245 - INFO - Gene cds-WP_000863859.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,249 - INFO - Gene gene-SPN23F_RS05095 has a mutation rate of 0.63.\n",
      "2025-04-16 11:13:20,251 - INFO - Gene cds-WP_000061646.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,251 - INFO - Gene gene-SPN23F_RS13525 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,251 - INFO - Gene cds-WP_001808548.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,252 - INFO - Gene gene-SPN23F_RS05105 has a mutation rate of 0.11666666666666667.\n",
      "2025-04-16 11:13:20,252 - INFO - Gene cds-WP_001227083.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,253 - INFO - Gene gene-SPN23F_RS05110 has a mutation rate of 0.36.\n",
      "2025-04-16 11:13:20,253 - INFO - Gene cds-WP_001028731.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,253 - INFO - Gene gene-SPN23F_RS05115 has a mutation rate of 0.29333333333333333.\n",
      "2025-04-16 11:13:20,253 - INFO - Gene cds-WP_000033831.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,254 - INFO - Gene gene-SPN23F_RS05120 has a mutation rate of 0.23.\n",
      "2025-04-16 11:13:20,254 - INFO - Gene cds-WP_000631543.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,255 - INFO - Gene cds-SPN23F_RS12260 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,255 - INFO - Gene gene-SPN23F_RS05140 has a mutation rate of 0.51.\n",
      "2025-04-16 11:13:20,255 - INFO - Gene cds-WP_000992887.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,256 - INFO - Gene gene-SPN23F_RS05145 has a mutation rate of 0.3466666666666667.\n",
      "2025-04-16 11:13:20,256 - INFO - Gene cds-WP_000286922.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,257 - INFO - Gene gene-SPN23F_RS05150 has a mutation rate of 0.5233333333333333.\n",
      "2025-04-16 11:13:20,257 - INFO - Gene cds-WP_000743330.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,257 - INFO - Gene gene-SPN23F_RS05155 has a mutation rate of 0.06666666666666667.\n",
      "2025-04-16 11:13:20,258 - INFO - Gene cds-WP_000653403.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,258 - INFO - Gene gene-SPN23F_RS05160 has a mutation rate of 0.04.\n",
      "2025-04-16 11:13:20,258 - INFO - Gene cds-WP_000462487.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,259 - INFO - Gene gene-SPN23F_RS05165 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,259 - INFO - Gene cds-WP_000863509.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,320 - INFO - Gene gene-SPN23F_RS05170 has a mutation rate of 0.65.\n",
      "2025-04-16 11:13:20,320 - INFO - Gene cds-WP_000639667.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,321 - INFO - Gene gene-SPN23F_RS05175 has a mutation rate of 0.03.\n",
      "2025-04-16 11:13:20,321 - INFO - Gene cds-WP_001283813.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,321 - INFO - Gene gene-SPN23F_RS05180 has a mutation rate of 0.46.\n",
      "2025-04-16 11:13:20,322 - INFO - Gene cds-WP_000681292.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,322 - INFO - Gene gene-SPN23F_RS05185 has a mutation rate of 0.22333333333333333.\n",
      "2025-04-16 11:13:20,322 - INFO - Gene cds-WP_000171678.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,322 - INFO - Gene gene-SPN23F_RS05190 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,323 - INFO - Gene cds-WP_000799059.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,323 - INFO - Gene gene-SPN23F_RS05195 has a mutation rate of 0.6333333333333333.\n",
      "2025-04-16 11:13:20,323 - INFO - Gene cds-WP_001210007.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,324 - INFO - Gene gene-SPN23F_RS05200 has a mutation rate of 0.11666666666666667.\n",
      "2025-04-16 11:13:20,324 - INFO - Gene cds-WP_000451584.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,324 - INFO - Gene cds-SPN23F_RS12265 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,325 - INFO - Gene gene-SPN23F_RS05220 has a mutation rate of 0.2.\n",
      "2025-04-16 11:13:20,325 - INFO - Gene cds-WP_000981526.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,326 - INFO - Gene gene-SPN23F_RS05225 has a mutation rate of 0.5566666666666666.\n",
      "2025-04-16 11:13:20,326 - INFO - Gene cds-WP_000509042.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,326 - INFO - Gene gene-SPN23F_RS05230 has a mutation rate of 0.25.\n",
      "2025-04-16 11:13:20,327 - INFO - Gene cds-WP_000109141.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,327 - INFO - Gene gene-SPN23F_RS05235 has a mutation rate of 0.41333333333333333.\n",
      "2025-04-16 11:13:20,327 - INFO - Gene cds-WP_000613700.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,328 - INFO - Gene gene-SPN23F_RS05240 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,328 - INFO - Gene cds-WP_000916509.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,328 - INFO - Gene cds-SPN23F_RS12945 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,329 - INFO - Gene gene-SPN23F_RS05250 has a mutation rate of 0.20666666666666667.\n",
      "2025-04-16 11:13:20,329 - INFO - Gene cds-WP_000587444.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,330 - INFO - Gene gene-SPN23F_RS05255 has a mutation rate of 0.4033333333333333.\n",
      "2025-04-16 11:13:20,330 - INFO - Gene cds-WP_000592056.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,330 - INFO - Gene gene-SPN23F_RS05260 has a mutation rate of 0.62.\n",
      "2025-04-16 11:13:20,330 - INFO - Gene cds-WP_000161400.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,331 - INFO - Gene gene-SPN23F_RS05265 has a mutation rate of 0.023333333333333334.\n",
      "2025-04-16 11:13:20,331 - INFO - Gene cds-WP_001284639.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,331 - INFO - Gene gene-SPN23F_RS05270 has a mutation rate of 0.7266666666666667.\n",
      "2025-04-16 11:13:20,332 - INFO - Gene cds-WP_000584856.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,332 - INFO - Gene gene-SPN23F_RS05275 has a mutation rate of 0.5433333333333333.\n",
      "2025-04-16 11:13:20,333 - INFO - Gene cds-WP_000400565.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,333 - INFO - Gene gene-SPN23F_RS05280 has a mutation rate of 0.38.\n",
      "2025-04-16 11:13:20,334 - INFO - Gene cds-WP_001083032.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,334 - INFO - Gene gene-SPN23F_RS05285 has a mutation rate of 0.58.\n",
      "2025-04-16 11:13:20,334 - INFO - Gene cds-WP_001042568.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,335 - INFO - Gene gene-SPN23F_RS05290 has a mutation rate of 0.5466666666666666.\n",
      "2025-04-16 11:13:20,335 - INFO - Gene cds-WP_000283064.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,335 - INFO - Gene gene-SPN23F_RS05300 has a mutation rate of 0.51.\n",
      "2025-04-16 11:13:20,336 - INFO - Gene cds-WP_000199381.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,336 - INFO - Gene gene-SPN23F_RS05305 has a mutation rate of 0.9866666666666667.\n",
      "2025-04-16 11:13:20,337 - INFO - Gene cds-WP_000370252.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,337 - INFO - Gene gene-SPN23F_RS05310 has a mutation rate of 0.48.\n",
      "2025-04-16 11:13:20,337 - INFO - Gene cds-WP_000787276.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,337 - INFO - Gene gene-SPN23F_RS05315 has a mutation rate of 0.43.\n",
      "2025-04-16 11:13:20,338 - INFO - Gene cds-WP_000687033.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,338 - INFO - Gene gene-SPN23F_RS05320 has a mutation rate of 0.24666666666666667.\n",
      "2025-04-16 11:13:20,338 - INFO - Gene cds-WP_000697292.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,339 - INFO - Gene gene-SPN23F_RS13405 has a mutation rate of 0.21333333333333335.\n",
      "2025-04-16 11:13:20,339 - INFO - Gene cds-WP_001809343.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,340 - INFO - Gene gene-SPN23F_RS13410 has a mutation rate of 0.6833333333333333.\n",
      "2025-04-16 11:13:20,340 - INFO - Gene cds-WP_001809344.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,340 - INFO - Gene gene-SPN23F_RS05335 has a mutation rate of 0.43.\n",
      "2025-04-16 11:13:20,341 - INFO - Gene cds-WP_000706416.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,341 - INFO - Gene gene-SPN23F_RS05340 has a mutation rate of 0.19.\n",
      "2025-04-16 11:13:20,346 - INFO - Gene cds-WP_000131080.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,363 - INFO - Gene gene-SPN23F_RS05345 has a mutation rate of 0.12.\n",
      "2025-04-16 11:13:20,375 - INFO - Gene cds-WP_000022813.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,376 - INFO - Gene gene-SPN23F_RS05350 has a mutation rate of 0.4066666666666667.\n",
      "2025-04-16 11:13:20,377 - INFO - Gene cds-WP_044811956.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,377 - INFO - Gene cds-SPN23F_RS12955 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,377 - INFO - Gene cds-SPN23F_RS12965 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,378 - INFO - Gene gene-SPN23F_RS05360 has a mutation rate of 0.99.\n",
      "2025-04-16 11:13:20,378 - INFO - Gene cds-WP_000772339.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,379 - INFO - Gene gene-SPN23F_RS05365 has a mutation rate of 1.0.\n",
      "2025-04-16 11:13:20,382 - INFO - Gene cds-WP_000767248.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,384 - INFO - Gene gene-SPN23F_RS05370 has a mutation rate of 0.7666666666666667.\n",
      "2025-04-16 11:13:20,384 - INFO - Gene cds-WP_000174775.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,385 - INFO - Gene gene-SPN23F_RS05375 has a mutation rate of 0.9533333333333334.\n",
      "2025-04-16 11:13:20,385 - INFO - Gene cds-WP_000417160.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,386 - INFO - Gene gene-SPN23F_RS05380 has a mutation rate of 0.7833333333333333.\n",
      "2025-04-16 11:13:20,386 - INFO - Gene cds-WP_000054793.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,387 - INFO - Gene gene-SPN23F_RS05385 has a mutation rate of 0.61.\n",
      "2025-04-16 11:13:20,387 - INFO - Gene cds-WP_000201315.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,387 - INFO - Gene gene-SPN23F_RS05390 has a mutation rate of 0.55.\n",
      "2025-04-16 11:13:20,388 - INFO - Gene cds-WP_000201099.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,388 - INFO - Gene gene-SPN23F_RS05395 has a mutation rate of 0.6.\n",
      "2025-04-16 11:13:20,388 - INFO - Gene cds-WP_000392544.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,389 - INFO - Gene gene-SPN23F_RS05400 has a mutation rate of 0.41333333333333333.\n",
      "2025-04-16 11:13:20,389 - INFO - Gene cds-WP_000817884.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,390 - INFO - Gene gene-SPN23F_RS05405 has a mutation rate of 0.5133333333333333.\n",
      "2025-04-16 11:13:20,390 - INFO - Gene cds-WP_000873991.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,390 - INFO - Gene gene-SPN23F_RS05410 has a mutation rate of 0.7666666666666667.\n",
      "2025-04-16 11:13:20,391 - INFO - Gene cds-WP_001162928.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,391 - INFO - Gene gene-SPN23F_RS05415 has a mutation rate of 0.5133333333333333.\n",
      "2025-04-16 11:13:20,391 - INFO - Gene cds-WP_000752697.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,392 - INFO - Gene gene-SPN23F_RS05420 has a mutation rate of 0.3233333333333333.\n",
      "2025-04-16 11:13:20,392 - INFO - Gene cds-WP_000448719.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,392 - INFO - Gene gene-SPN23F_RS05425 has a mutation rate of 0.49333333333333335.\n",
      "2025-04-16 11:13:20,393 - INFO - Gene cds-WP_000105362.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,393 - INFO - Gene gene-SPN23F_RS05430 has a mutation rate of 0.63.\n",
      "2025-04-16 11:13:20,393 - INFO - Gene cds-WP_000278538.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,394 - INFO - Gene gene-SPN23F_RS05435 has a mutation rate of 0.11666666666666667.\n",
      "2025-04-16 11:13:20,394 - INFO - Gene cds-WP_001105925.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,394 - INFO - Gene gene-SPN23F_RS05440 has a mutation rate of 0.8366666666666667.\n",
      "2025-04-16 11:13:20,394 - INFO - Gene cds-WP_000924492.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,395 - INFO - Gene gene-SPN23F_RS05445 has a mutation rate of 0.32.\n",
      "2025-04-16 11:13:20,395 - INFO - Gene cds-WP_001135781.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,396 - INFO - Gene gene-SPN23F_RS05450 has a mutation rate of 0.37333333333333335.\n",
      "2025-04-16 11:13:20,396 - INFO - Gene cds-WP_000401326.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,396 - INFO - Gene gene-SPN23F_RS05455 has a mutation rate of 0.7.\n",
      "2025-04-16 11:13:20,396 - INFO - Gene cds-WP_001810198.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,397 - INFO - Gene gene-SPN23F_RS05460 has a mutation rate of 0.20333333333333334.\n",
      "2025-04-16 11:13:20,397 - INFO - Gene cds-WP_000499435.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,398 - INFO - Gene gene-SPN23F_RS05465 has a mutation rate of 0.4866666666666667.\n",
      "2025-04-16 11:13:20,398 - INFO - Gene cds-WP_000443793.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,398 - INFO - Gene gene-SPN23F_RS05470 has a mutation rate of 0.6333333333333333.\n",
      "2025-04-16 11:13:20,399 - INFO - Gene cds-WP_000700472.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,399 - INFO - Gene gene-SPN23F_RS05475 has a mutation rate of 0.11666666666666667.\n",
      "2025-04-16 11:13:20,399 - INFO - Gene cds-WP_000138142.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,400 - INFO - Gene gene-SPN23F_RS05480 has a mutation rate of 0.013333333333333334.\n",
      "2025-04-16 11:13:20,400 - INFO - Gene cds-WP_000146947.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,401 - INFO - Gene gene-SPN23F_RS05485 has a mutation rate of 0.17666666666666667.\n",
      "2025-04-16 11:13:20,401 - INFO - Gene cds-WP_000259249.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,401 - INFO - Gene gene-SPN23F_RS05490 has a mutation rate of 0.3233333333333333.\n",
      "2025-04-16 11:13:20,402 - INFO - Gene cds-WP_000523249.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,402 - INFO - Gene gene-SPN23F_RS05495 has a mutation rate of 0.08333333333333333.\n",
      "2025-04-16 11:13:20,402 - INFO - Gene cds-WP_000451376.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,403 - INFO - Gene gene-SPN23F_RS05500 has a mutation rate of 0.4066666666666667.\n",
      "2025-04-16 11:13:20,403 - INFO - Gene cds-WP_000917600.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,403 - INFO - Gene gene-SPN23F_RS14110 has a mutation rate of 0.18333333333333332.\n",
      "2025-04-16 11:13:20,403 - INFO - Gene cds-WP_000471586.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,404 - INFO - Gene gene-SPN23F_RS05510 has a mutation rate of 0.8.\n",
      "2025-04-16 11:13:20,404 - INFO - Gene cds-WP_000169288.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,404 - INFO - Gene gene-SPN23F_RS05515 has a mutation rate of 0.8133333333333334.\n",
      "2025-04-16 11:13:20,405 - INFO - Gene cds-WP_000359339.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,405 - INFO - Gene gene-SPN23F_RS05520 has a mutation rate of 0.03.\n",
      "2025-04-16 11:13:20,405 - INFO - Gene cds-WP_001078331.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,406 - INFO - Gene gene-SPN23F_RS05525 has a mutation rate of 0.31666666666666665.\n",
      "2025-04-16 11:13:20,406 - INFO - Gene cds-WP_000285718.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,406 - INFO - Gene gene-SPN23F_RS05530 has a mutation rate of 0.31333333333333335.\n",
      "2025-04-16 11:13:20,407 - INFO - Gene cds-WP_001229140.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,407 - INFO - Gene gene-SPN23F_RS05535 has a mutation rate of 0.5633333333333334.\n",
      "2025-04-16 11:13:20,407 - INFO - Gene cds-WP_000604274.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,408 - INFO - Gene gene-SPN23F_RS05540 has a mutation rate of 0.78.\n",
      "2025-04-16 11:13:20,408 - INFO - Gene cds-WP_001216910.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,409 - INFO - Gene gene-SPN23F_RS05545 has a mutation rate of 0.05.\n",
      "2025-04-16 11:13:20,409 - INFO - Gene cds-WP_000029274.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,409 - INFO - Gene cds-SPN23F_RS05550 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,410 - INFO - Gene cds-SPN23F_RS12290 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,410 - INFO - Gene cds-SPN23F_RS05565 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,410 - INFO - Gene gene-SPN23F_RS05570 has a mutation rate of 0.6366666666666667.\n",
      "2025-04-16 11:13:20,411 - INFO - Gene cds-WP_000242183.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,411 - INFO - Gene gene-SPN23F_RS13900 has a mutation rate of 0.2.\n",
      "2025-04-16 11:13:20,411 - INFO - Gene cds-WP_001814238.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,412 - INFO - Gene gene-SPN23F_RS05580 has a mutation rate of 0.49.\n",
      "2025-04-16 11:13:20,412 - INFO - Gene cds-WP_001047216.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,412 - INFO - Gene gene-SPN23F_RS05585 has a mutation rate of 0.45666666666666667.\n",
      "2025-04-16 11:13:20,413 - INFO - Gene cds-WP_000133341.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,413 - INFO - Gene gene-SPN23F_RS05590 has a mutation rate of 0.8466666666666667.\n",
      "2025-04-16 11:13:20,413 - INFO - Gene cds-WP_000923593.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,414 - INFO - Gene gene-SPN23F_RS05595 has a mutation rate of 0.03.\n",
      "2025-04-16 11:13:20,414 - INFO - Gene cds-WP_001034415.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,415 - INFO - Gene gene-SPN23F_RS05600 has a mutation rate of 0.43333333333333335.\n",
      "2025-04-16 11:13:20,415 - INFO - Gene cds-WP_001041072.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,415 - INFO - Gene gene-SPN23F_RS05605 has a mutation rate of 0.76.\n",
      "2025-04-16 11:13:20,416 - INFO - Gene cds-WP_000743813.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,416 - INFO - Gene gene-SPN23F_RS05610 has a mutation rate of 0.07666666666666666.\n",
      "2025-04-16 11:13:20,418 - INFO - Gene cds-WP_000043230.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,436 - INFO - Gene gene-SPN23F_RS05615 has a mutation rate of 0.6933333333333334.\n",
      "2025-04-16 11:13:20,446 - INFO - Gene cds-WP_000417468.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,447 - INFO - Gene gene-SPN23F_RS05620 has a mutation rate of 0.05.\n",
      "2025-04-16 11:13:20,447 - INFO - Gene cds-WP_001181378.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,447 - INFO - Gene gene-SPN23F_RS05625 has a mutation rate of 0.21333333333333335.\n",
      "2025-04-16 11:13:20,448 - INFO - Gene cds-WP_000985093.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,448 - INFO - Gene gene-SPN23F_RS05635 has a mutation rate of 0.6766666666666666.\n",
      "2025-04-16 11:13:20,448 - INFO - Gene cds-WP_001013238.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,449 - INFO - Gene gene-SPN23F_RS05640 has a mutation rate of 0.8466666666666667.\n",
      "2025-04-16 11:13:20,449 - INFO - Gene cds-WP_001002634.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,450 - INFO - Gene cds-SPN23F_RS05645 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,453 - INFO - Gene gene-SPN23F_RS05650 has a mutation rate of 0.5.\n",
      "2025-04-16 11:13:20,454 - INFO - Gene cds-WP_000254802.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,455 - INFO - Gene gene-SPN23F_RS05655 has a mutation rate of 0.35333333333333333.\n",
      "2025-04-16 11:13:20,455 - INFO - Gene cds-WP_000078846.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,456 - INFO - Gene gene-SPN23F_RS05660 has a mutation rate of 0.5533333333333333.\n",
      "2025-04-16 11:13:20,456 - INFO - Gene cds-WP_001152992.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,457 - INFO - Gene gene-SPN23F_RS05665 has a mutation rate of 0.043333333333333335.\n",
      "2025-04-16 11:13:20,457 - INFO - Gene cds-WP_000204727.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,457 - INFO - Gene gene-SPN23F_RS05670 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,458 - INFO - Gene cds-WP_000731139.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,458 - INFO - Gene gene-SPN23F_RS05675 has a mutation rate of 0.23333333333333334.\n",
      "2025-04-16 11:13:20,458 - INFO - Gene cds-WP_001809704.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,459 - INFO - Gene gene-SPN23F_RS05680 has a mutation rate of 0.016666666666666666.\n",
      "2025-04-16 11:13:20,459 - INFO - Gene cds-WP_000208080.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,460 - INFO - Gene gene-SPN23F_RS05685 has a mutation rate of 0.013333333333333334.\n",
      "2025-04-16 11:13:20,460 - INFO - Gene cds-WP_001289493.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,460 - INFO - Gene gene-SPN23F_RS05690 has a mutation rate of 0.11.\n",
      "2025-04-16 11:13:20,461 - INFO - Gene cds-WP_000886210.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,461 - INFO - Gene gene-SPN23F_RS05695 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,462 - INFO - Gene cds-WP_000722076.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,462 - INFO - Gene gene-SPN23F_RS05700 has a mutation rate of 0.41.\n",
      "2025-04-16 11:13:20,462 - INFO - Gene cds-WP_000886152.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,463 - INFO - Gene gene-SPN23F_RS14115 has a mutation rate of 0.09.\n",
      "2025-04-16 11:13:20,463 - INFO - Gene cds-WP_001818232.1-2 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,464 - INFO - Gene gene-SPN23F_RS05705 has a mutation rate of 0.8033333333333333.\n",
      "2025-04-16 11:13:20,464 - INFO - Gene cds-WP_000845292.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,464 - INFO - Gene gene-SPN23F_RS05710 has a mutation rate of 0.6633333333333333.\n",
      "2025-04-16 11:13:20,464 - INFO - Gene cds-WP_000699492.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,465 - INFO - Gene gene-SPN23F_RS05715 has a mutation rate of 0.17.\n",
      "2025-04-16 11:13:20,465 - INFO - Gene cds-WP_001284128.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,465 - INFO - Gene gene-SPN23F_RS05720 has a mutation rate of 0.18333333333333332.\n",
      "2025-04-16 11:13:20,466 - INFO - Gene cds-WP_000747408.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,466 - INFO - Gene gene-SPN23F_RS05725 has a mutation rate of 0.2733333333333333.\n",
      "2025-04-16 11:13:20,466 - INFO - Gene cds-WP_001097402.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,467 - INFO - Gene gene-SPN23F_RS05730 has a mutation rate of 0.45666666666666667.\n",
      "2025-04-16 11:13:20,467 - INFO - Gene cds-WP_000159176.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,468 - INFO - Gene gene-SPN23F_RS05735 has a mutation rate of 0.4666666666666667.\n",
      "2025-04-16 11:13:20,468 - INFO - Gene cds-WP_000091534.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,468 - INFO - Gene gene-SPN23F_RS05740 has a mutation rate of 0.6366666666666667.\n",
      "2025-04-16 11:13:20,469 - INFO - Gene cds-WP_000216026.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,469 - INFO - Gene gene-SPN23F_RS05745 has a mutation rate of 0.5733333333333334.\n",
      "2025-04-16 11:13:20,470 - INFO - Gene cds-WP_000639840.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,470 - INFO - Gene gene-SPN23F_RS05750 has a mutation rate of 0.5.\n",
      "2025-04-16 11:13:20,470 - INFO - Gene cds-WP_000607027.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,470 - INFO - Gene gene-SPN23F_RS05755 has a mutation rate of 0.7233333333333334.\n",
      "2025-04-16 11:13:20,471 - INFO - Gene cds-WP_000792105.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,471 - INFO - Gene gene-SPN23F_RS05760 has a mutation rate of 0.6066666666666667.\n",
      "2025-04-16 11:13:20,472 - INFO - Gene cds-WP_000726258.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,472 - INFO - Gene gene-SPN23F_RS05765 has a mutation rate of 0.4166666666666667.\n",
      "2025-04-16 11:13:20,472 - INFO - Gene cds-WP_001096313.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,473 - INFO - Gene gene-SPN23F_RS05770 has a mutation rate of 0.8266666666666667.\n",
      "2025-04-16 11:13:20,473 - INFO - Gene cds-WP_000096133.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,474 - INFO - Gene gene-SPN23F_RS05775 has a mutation rate of 0.8066666666666666.\n",
      "2025-04-16 11:13:20,474 - INFO - Gene cds-WP_000522315.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,475 - INFO - Gene gene-SPN23F_RS05780 has a mutation rate of 0.58.\n",
      "2025-04-16 11:13:20,475 - INFO - Gene cds-WP_000763460.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,476 - INFO - Gene gene-SPN23F_RS05785 has a mutation rate of 1.0.\n",
      "2025-04-16 11:13:20,476 - INFO - Gene cds-WP_000153026.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,477 - INFO - Gene gene-SPN23F_RS05790 has a mutation rate of 1.0.\n",
      "2025-04-16 11:13:20,477 - INFO - Gene cds-WP_000280916.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,478 - INFO - Gene gene-SPN23F_RS05795 has a mutation rate of 0.9166666666666666.\n",
      "2025-04-16 11:13:20,478 - INFO - Gene cds-WP_000661487.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,478 - INFO - Gene gene-SPN23F_RS05800 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,479 - INFO - Gene gene-SPN23F_RS05805 has a mutation rate of 0.57.\n",
      "2025-04-16 11:13:20,479 - INFO - Gene cds-WP_000931178.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,479 - INFO - Gene cds-SPN23F_RS05810 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,480 - INFO - Gene gene-SPN23F_RS05815 has a mutation rate of 0.8933333333333333.\n",
      "2025-04-16 11:13:20,480 - INFO - Gene cds-WP_001042410.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,480 - INFO - Gene cds-SPN23F_RS12980 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,481 - INFO - Gene gene-SPN23F_RS05830 has a mutation rate of 0.7466666666666667.\n",
      "2025-04-16 11:13:20,481 - INFO - Gene cds-WP_000929151.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,482 - INFO - Gene gene-SPN23F_RS05835 has a mutation rate of 0.4666666666666667.\n",
      "2025-04-16 11:13:20,482 - INFO - Gene cds-WP_000762432.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,482 - INFO - Gene gene-SPN23F_RS05840 has a mutation rate of 0.8166666666666667.\n",
      "2025-04-16 11:13:20,483 - INFO - Gene cds-WP_001074723.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,483 - INFO - Gene cds-SPN23F_RS14200 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,483 - INFO - Gene gene-SPN23F_RS05855 has a mutation rate of 0.4766666666666667.\n",
      "2025-04-16 11:13:20,484 - INFO - Gene cds-WP_000461585.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,484 - INFO - Gene gene-SPN23F_RS05860 has a mutation rate of 0.34.\n",
      "2025-04-16 11:13:20,484 - INFO - Gene cds-WP_000638779.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,485 - INFO - Gene gene-SPN23F_RS05865 has a mutation rate of 0.18.\n",
      "2025-04-16 11:13:20,486 - INFO - Gene cds-WP_001237218.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,486 - INFO - Gene gene-SPN23F_RS05870 has a mutation rate of 0.5866666666666667.\n",
      "2025-04-16 11:13:20,487 - INFO - Gene cds-WP_000179648.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,487 - INFO - Gene gene-SPN23F_RS05875 has a mutation rate of 0.38333333333333336.\n",
      "2025-04-16 11:13:20,490 - INFO - Gene cds-WP_001812957.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,491 - INFO - Gene gene-SPN23F_RS14120 has a mutation rate of 0.3933333333333333.\n",
      "2025-04-16 11:13:20,493 - INFO - Gene cds-WP_001814242.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,524 - INFO - Gene gene-SPN23F_RS05880 has a mutation rate of 0.49666666666666665.\n",
      "2025-04-16 11:13:20,526 - INFO - Gene cds-WP_000705306.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,526 - INFO - Gene gene-SPN23F_RS05885 has a mutation rate of 0.22333333333333333.\n",
      "2025-04-16 11:13:20,527 - INFO - Gene cds-WP_000643971.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,527 - INFO - Gene gene-SPN23F_RS05890 has a mutation rate of 0.4066666666666667.\n",
      "2025-04-16 11:13:20,528 - INFO - Gene cds-WP_000837609.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,528 - INFO - Gene gene-SPN23F_RS05895 has a mutation rate of 0.82.\n",
      "2025-04-16 11:13:20,536 - INFO - Gene cds-WP_000411210.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,537 - INFO - Gene gene-SPN23F_RS05900 has a mutation rate of 0.44666666666666666.\n",
      "2025-04-16 11:13:20,537 - INFO - Gene cds-WP_000609894.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,537 - INFO - Gene gene-SPN23F_RS05905 has a mutation rate of 0.46.\n",
      "2025-04-16 11:13:20,538 - INFO - Gene cds-WP_000638527.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,539 - INFO - Gene gene-SPN23F_RS05910 has a mutation rate of 0.7766666666666666.\n",
      "2025-04-16 11:13:20,539 - INFO - Gene cds-WP_000835899.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,539 - INFO - Gene gene-SPN23F_RS05915 has a mutation rate of 0.6833333333333333.\n",
      "2025-04-16 11:13:20,540 - INFO - Gene cds-WP_000811874.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,540 - INFO - Gene gene-SPN23F_RS05920 has a mutation rate of 0.6966666666666667.\n",
      "2025-04-16 11:13:20,541 - INFO - Gene cds-WP_001199652.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,541 - INFO - Gene gene-SPN23F_RS05925 has a mutation rate of 0.6266666666666667.\n",
      "2025-04-16 11:13:20,542 - INFO - Gene cds-WP_001126409.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,543 - INFO - Gene gene-SPN23F_RS05930 has a mutation rate of 0.013333333333333334.\n",
      "2025-04-16 11:13:20,543 - INFO - Gene cds-WP_000166701.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,544 - INFO - Gene gene-SPN23F_RS05935 has a mutation rate of 0.8133333333333334.\n",
      "2025-04-16 11:13:20,544 - INFO - Gene cds-WP_001293838.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,545 - INFO - Gene gene-SPN23F_RS05940 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,545 - INFO - Gene cds-WP_000850024.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,545 - INFO - Gene gene-SPN23F_RS05945 has a mutation rate of 0.02666666666666667.\n",
      "2025-04-16 11:13:20,545 - INFO - Gene cds-WP_000244450.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,546 - INFO - Gene gene-SPN23F_RS05950 has a mutation rate of 0.81.\n",
      "2025-04-16 11:13:20,546 - INFO - Gene cds-WP_000773231.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,547 - INFO - Gene cds-SPN23F_RS14205 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,547 - INFO - Gene gene-SPN23F_RS05960 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,548 - INFO - Gene cds-WP_001155337.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,548 - INFO - Gene gene-SPN23F_RS05965 has a mutation rate of 0.87.\n",
      "2025-04-16 11:13:20,548 - INFO - Gene cds-WP_001861318.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,549 - INFO - Gene gene-SPN23F_RS05970 has a mutation rate of 0.5766666666666667.\n",
      "2025-04-16 11:13:20,549 - INFO - Gene cds-WP_000895733.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,550 - INFO - Gene gene-SPN23F_RS05975 has a mutation rate of 0.32666666666666666.\n",
      "2025-04-16 11:13:20,550 - INFO - Gene cds-WP_000219823.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,551 - INFO - Gene gene-SPN23F_RS05980 has a mutation rate of 0.29.\n",
      "2025-04-16 11:13:20,551 - INFO - Gene cds-WP_000801931.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,552 - INFO - Gene gene-SPN23F_RS05985 has a mutation rate of 0.66.\n",
      "2025-04-16 11:13:20,552 - INFO - Gene cds-WP_001809228.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,552 - INFO - Gene gene-SPN23F_RS05990 has a mutation rate of 0.4066666666666667.\n",
      "2025-04-16 11:13:20,553 - INFO - Gene cds-WP_000863628.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,553 - INFO - Gene gene-SPN23F_RS05995 has a mutation rate of 0.03333333333333333.\n",
      "2025-04-16 11:13:20,553 - INFO - Gene cds-WP_000402071.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,554 - INFO - Gene gene-SPN23F_RS06000 has a mutation rate of 0.6433333333333333.\n",
      "2025-04-16 11:13:20,554 - INFO - Gene cds-WP_000702355.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,554 - INFO - Gene gene-SPN23F_RS06005 has a mutation rate of 0.7433333333333333.\n",
      "2025-04-16 11:13:20,555 - INFO - Gene cds-WP_001003498.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,555 - INFO - Gene gene-SPN23F_RS06010 has a mutation rate of 0.8933333333333333.\n",
      "2025-04-16 11:13:20,556 - INFO - Gene cds-WP_000023518.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,556 - INFO - Gene gene-SPN23F_RS06015 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,557 - INFO - Gene gene-SPN23F_RS06020 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,557 - INFO - Gene cds-WP_001068669.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,558 - INFO - Gene gene-SPN23F_RS06025 has a mutation rate of 0.4066666666666667.\n",
      "2025-04-16 11:13:20,558 - INFO - Gene cds-WP_000237416.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,558 - INFO - Gene gene-SPN23F_RS06030 has a mutation rate of 0.5333333333333333.\n",
      "2025-04-16 11:13:20,559 - INFO - Gene cds-WP_000712098.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,560 - INFO - Gene gene-SPN23F_RS06035 has a mutation rate of 0.29333333333333333.\n",
      "2025-04-16 11:13:20,560 - INFO - Gene cds-WP_000362435.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,560 - INFO - Gene gene-SPN23F_RS06040 has a mutation rate of 0.48333333333333334.\n",
      "2025-04-16 11:13:20,561 - INFO - Gene cds-WP_001162128.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,561 - INFO - Gene gene-SPN23F_RS06045 has a mutation rate of 0.21333333333333335.\n",
      "2025-04-16 11:13:20,561 - INFO - Gene cds-WP_000401779.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,562 - INFO - Gene gene-SPN23F_RS06050 has a mutation rate of 0.01.\n",
      "2025-04-16 11:13:20,562 - INFO - Gene cds-WP_000710764.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,562 - INFO - Gene gene-SPN23F_RS06055 has a mutation rate of 0.25.\n",
      "2025-04-16 11:13:20,672 - INFO - Gene cds-WP_000028150.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,673 - INFO - Gene cds-SPN23F_RS12315 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,673 - INFO - Gene gene-SPN23F_RS06075 has a mutation rate of 0.8066666666666666.\n",
      "2025-04-16 11:13:20,674 - INFO - Gene cds-WP_000199560.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,674 - INFO - Gene gene-SPN23F_RS13545 has a mutation rate of 0.36666666666666664.\n",
      "2025-04-16 11:13:20,675 - INFO - Gene cds-WP_000711939.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,676 - INFO - Gene cds-SPN23F_RS06085 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,676 - INFO - Gene cds-SPN23F_RS12320 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,677 - INFO - Gene cds-SPN23F_RS12990 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,677 - INFO - Gene cds-SPN23F_RS06110 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,677 - INFO - Gene cds-SPN23F_RS06115 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,678 - INFO - Gene gene-SPN23F_RS06120 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,678 - INFO - Gene cds-WP_001248801.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,678 - INFO - Gene gene-SPN23F_RS06125 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,679 - INFO - Gene cds-WP_001136124.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,679 - INFO - Gene gene-SPN23F_RS06130 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,680 - INFO - Gene cds-WP_000251935.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,680 - INFO - Gene gene-SPN23F_RS06135 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,680 - INFO - Gene cds-WP_000111248.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,681 - INFO - Gene gene-SPN23F_RS06140 has a mutation rate of 0.07333333333333333.\n",
      "2025-04-16 11:13:20,682 - INFO - Gene cds-WP_000191781.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,682 - INFO - Gene gene-SPN23F_RS06145 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,683 - INFO - Gene cds-WP_000387940.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,683 - INFO - Gene gene-SPN23F_RS06150 has a mutation rate of 0.01.\n",
      "2025-04-16 11:13:20,684 - INFO - Gene cds-WP_000380542.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,685 - INFO - Gene gene-SPN23F_RS06155 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,685 - INFO - Gene cds-WP_000067171.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,685 - INFO - Gene gene-SPN23F_RS13000 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,686 - INFO - Gene cds-WP_000400484.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,686 - INFO - Gene gene-SPN23F_RS06165 has a mutation rate of 0.01.\n",
      "2025-04-16 11:13:20,687 - INFO - Gene cds-WP_001018350.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,687 - INFO - Gene gene-SPN23F_RS06170 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,687 - INFO - Gene cds-WP_000185365.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,688 - INFO - Gene gene-SPN23F_RS06175 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,688 - INFO - Gene cds-WP_000691259.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,689 - INFO - Gene gene-SPN23F_RS06180 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,689 - INFO - Gene cds-WP_000248712.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,689 - INFO - Gene gene-SPN23F_RS06185 has a mutation rate of 0.18333333333333332.\n",
      "2025-04-16 11:13:20,690 - INFO - Gene cds-WP_000739357.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,690 - INFO - Gene gene-SPN23F_RS06190 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,691 - INFO - Gene cds-WP_000587140.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,692 - INFO - Gene gene-SPN23F_RS06195 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,693 - INFO - Gene cds-WP_000537363.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,693 - INFO - Gene gene-SPN23F_RS06200 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,694 - INFO - Gene cds-WP_000654135.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,694 - INFO - Gene gene-SPN23F_RS06205 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,694 - INFO - Gene cds-WP_000448316.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,695 - INFO - Gene gene-SPN23F_RS06210 has a mutation rate of 0.06666666666666667.\n",
      "2025-04-16 11:13:20,695 - INFO - Gene cds-WP_000436478.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,696 - INFO - Gene gene-SPN23F_RS06215 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,696 - INFO - Gene cds-WP_223200707.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,697 - INFO - Gene gene-SPN23F_RS06220 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,697 - INFO - Gene cds-WP_001165928.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,698 - INFO - Gene gene-SPN23F_RS12335 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,698 - INFO - Gene cds-WP_087671856.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,698 - INFO - Gene cds-WP_087671856.1_1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,699 - INFO - Gene gene-SPN23F_RS13005 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,699 - INFO - Gene cds-WP_000807141.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,700 - INFO - Gene gene-SPN23F_RS06235 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,701 - INFO - Gene cds-WP_000945302.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,701 - INFO - Gene gene-SPN23F_RS06240 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,702 - INFO - Gene cds-WP_000356590.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,702 - INFO - Gene gene-SPN23F_RS12340 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,702 - INFO - Gene cds-WP_001108416.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,703 - INFO - Gene gene-SPN23F_RS06250 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,736 - INFO - Gene cds-WP_001076599.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,737 - INFO - Gene gene-SPN23F_RS06255 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,738 - INFO - Gene cds-WP_000386151.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,739 - INFO - Gene gene-SPN23F_RS13010 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,739 - INFO - Gene cds-WP_000807141.1-2 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,740 - INFO - Gene gene-SPN23F_RS06260 has a mutation rate of 0.7566666666666667.\n",
      "2025-04-16 11:13:20,740 - INFO - Gene cds-WP_000821287.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,740 - INFO - Gene cds-SPN23F_RS13915 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,741 - INFO - Gene gene-SPN23F_RS06265 has a mutation rate of 0.37.\n",
      "2025-04-16 11:13:20,746 - INFO - Gene cds-WP_000566556.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,748 - INFO - Gene gene-SPN23F_RS06270 has a mutation rate of 0.38.\n",
      "2025-04-16 11:13:20,749 - INFO - Gene cds-WP_001814249.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,750 - INFO - Gene cds-SPN23F_RS06275 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,751 - INFO - Gene gene-SPN23F_RS13555 has a mutation rate of 0.6833333333333333.\n",
      "2025-04-16 11:13:20,751 - INFO - Gene cds-WP_001168348.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,752 - INFO - Gene gene-SPN23F_RS06280 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,753 - INFO - Gene cds-WP_000237019.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,753 - INFO - Gene gene-SPN23F_RS06285 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,754 - INFO - Gene cds-WP_000450956.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,754 - INFO - Gene gene-SPN23F_RS06290 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,754 - INFO - Gene cds-WP_000919629.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,755 - INFO - Gene gene-SPN23F_RS06295 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,755 - INFO - Gene cds-WP_000237138.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,755 - INFO - Gene gene-SPN23F_RS06300 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,756 - INFO - Gene cds-WP_000401235.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,757 - INFO - Gene gene-SPN23F_RS06305 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,757 - INFO - Gene cds-WP_000817505.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,757 - INFO - Gene gene-SPN23F_RS06310 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,758 - INFO - Gene cds-WP_000456893.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,758 - INFO - Gene gene-SPN23F_RS06315 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,759 - INFO - Gene cds-WP_000209103.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,760 - INFO - Gene gene-SPN23F_RS06320 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,760 - INFO - Gene cds-WP_000276173.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,761 - INFO - Gene gene-SPN23F_RS06325 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,761 - INFO - Gene cds-WP_001814252.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,761 - INFO - Gene gene-SPN23F_RS06330 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,762 - INFO - Gene cds-WP_000116042.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,762 - INFO - Gene gene-SPN23F_RS06335 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,763 - INFO - Gene cds-WP_000677375.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,763 - INFO - Gene gene-SPN23F_RS06340 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,764 - INFO - Gene cds-WP_000394413.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,764 - INFO - Gene gene-SPN23F_RS06345 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,765 - INFO - Gene cds-WP_001039645.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,765 - INFO - Gene gene-SPN23F_RS06350 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,765 - INFO - Gene cds-WP_000180153.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,766 - INFO - Gene gene-SPN23F_RS06355 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,766 - INFO - Gene cds-WP_000644601.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,767 - INFO - Gene gene-SPN23F_RS06360 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,768 - INFO - Gene cds-WP_000013903.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,768 - INFO - Gene gene-SPN23F_RS06365 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,769 - INFO - Gene cds-WP_001814253.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,769 - INFO - Gene gene-SPN23F_RS06370 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,769 - INFO - Gene cds-WP_000417606.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,770 - INFO - Gene cds-SPN23F_RS06375 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,770 - INFO - Gene gene-SPN23F_RS06380 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,771 - INFO - Gene cds-WP_001814254.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,771 - INFO - Gene gene-SPN23F_RS06385 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,772 - INFO - Gene cds-WP_000214003.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,772 - INFO - Gene gene-SPN23F_RS06390 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,772 - INFO - Gene cds-WP_000512672.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,773 - INFO - Gene gene-SPN23F_RS06395 has a mutation rate of 0.39666666666666667.\n",
      "2025-04-16 11:13:20,773 - INFO - Gene cds-WP_000579607.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,774 - INFO - Gene gene-SPN23F_RS06400 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,774 - INFO - Gene cds-WP_000494445.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,774 - INFO - Gene gene-SPN23F_RS06405 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,775 - INFO - Gene cds-WP_000159218.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,776 - INFO - Gene gene-SPN23F_RS06410 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,776 - INFO - Gene cds-WP_000152419.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,777 - INFO - Gene gene-SPN23F_RS06415 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,777 - INFO - Gene cds-WP_000401892.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,777 - INFO - Gene gene-SPN23F_RS06420 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,778 - INFO - Gene cds-WP_000933090.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,779 - INFO - Gene gene-SPN23F_RS06425 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,779 - INFO - Gene cds-WP_000756060.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,780 - INFO - Gene gene-SPN23F_RS06430 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,780 - INFO - Gene cds-WP_001080431.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,781 - INFO - Gene gene-SPN23F_RS06435 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,781 - INFO - Gene cds-WP_001286517.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,781 - INFO - Gene gene-SPN23F_RS06440 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,782 - INFO - Gene cds-WP_001814255.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,782 - INFO - Gene gene-SPN23F_RS06445 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,783 - INFO - Gene cds-WP_224781690.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,783 - INFO - Gene gene-SPN23F_RS06450 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,784 - INFO - Gene cds-WP_001245731.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,785 - INFO - Gene gene-SPN23F_RS13560 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,785 - INFO - Gene cds-WP_000025229.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,788 - INFO - Gene gene-SPN23F_RS06460 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,815 - INFO - Gene cds-WP_000154111.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,821 - INFO - Gene gene-SPN23F_RS06465 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,822 - INFO - Gene cds-WP_000998897.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,822 - INFO - Gene gene-SPN23F_RS06470 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,823 - INFO - Gene cds-WP_000969642.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,823 - INFO - Gene cds-SPN23F_RS06475 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,824 - INFO - Gene gene-SPN23F_RS06480 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,828 - INFO - Gene cds-WP_001245667.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,831 - INFO - Gene cds-SPN23F_RS06485 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,832 - INFO - Gene gene-SPN23F_RS06490 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,832 - INFO - Gene cds-WP_000398567.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,833 - INFO - Gene gene-SPN23F_RS06495 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,834 - INFO - Gene cds-WP_000013900.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,834 - INFO - Gene gene-SPN23F_RS13565 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,835 - INFO - Gene cds-WP_001860875.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,835 - INFO - Gene gene-SPN23F_RS06500 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,836 - INFO - Gene cds-WP_000653448.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,836 - INFO - Gene gene-SPN23F_RS06505 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,836 - INFO - Gene cds-WP_029645481.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,837 - INFO - Gene gene-SPN23F_RS06510 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,837 - INFO - Gene cds-WP_001097588.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,838 - INFO - Gene gene-SPN23F_RS06515 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,838 - INFO - Gene cds-WP_001054261.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,839 - INFO - Gene gene-SPN23F_RS06520 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,839 - INFO - Gene cds-WP_000627865.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,839 - INFO - Gene gene-SPN23F_RS06525 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,840 - INFO - Gene cds-WP_001814258.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,840 - INFO - Gene gene-SPN23F_RS06530 has a mutation rate of 0.08.\n",
      "2025-04-16 11:13:20,841 - INFO - Gene cds-WP_000093679.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,841 - INFO - Gene gene-SPN23F_RS06535 has a mutation rate of 0.10666666666666667.\n",
      "2025-04-16 11:13:20,841 - INFO - Gene cds-WP_000869762.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,842 - INFO - Gene gene-SPN23F_RS06540 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,843 - INFO - Gene cds-WP_000492399.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,843 - INFO - Gene gene-SPN23F_RS06545 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,844 - INFO - Gene cds-WP_000227919.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,844 - INFO - Gene gene-SPN23F_RS06550 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,845 - INFO - Gene cds-WP_000420682.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,845 - INFO - Gene gene-SPN23F_RS06555 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,846 - INFO - Gene cds-WP_000985015.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,846 - INFO - Gene gene-SPN23F_RS06560 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,847 - INFO - Gene cds-WP_000813488.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,847 - INFO - Gene gene-SPN23F_RS12360 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,848 - INFO - Gene cds-WP_000879507.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,848 - INFO - Gene gene-SPN23F_RS06565 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,848 - INFO - Gene cds-WP_000398284.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,849 - INFO - Gene gene-SPN23F_RS06570 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,849 - INFO - Gene cds-WP_001009056.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,850 - INFO - Gene gene-SPN23F_RS06575 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,851 - INFO - Gene cds-WP_000342539.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,851 - INFO - Gene gene-SPN23F_RS06580 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,852 - INFO - Gene cds-WP_000723888.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,852 - INFO - Gene gene-SPN23F_RS06585 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,852 - INFO - Gene cds-WP_000331160.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,853 - INFO - Gene cds-SPN23F_RS06590 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,853 - INFO - Gene gene-SPN23F_RS06595 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,853 - INFO - Gene cds-WP_000769868.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,854 - INFO - Gene gene-SPN23F_RS06600 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,854 - INFO - Gene cds-WP_001224318.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,854 - INFO - Gene cds-SPN23F_RS14255 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,855 - INFO - Gene gene-SPN23F_RS06605 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,855 - INFO - Gene cds-WP_000691741.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,856 - INFO - Gene gene-SPN23F_RS12370 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,856 - INFO - Gene cds-WP_000336323.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,857 - INFO - Gene gene-SPN23F_RS06610 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,857 - INFO - Gene cds-WP_001227347.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,858 - INFO - Gene gene-SPN23F_RS06615 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,858 - INFO - Gene cds-WP_000804885.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,859 - INFO - Gene gene-SPN23F_RS06620 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,860 - INFO - Gene cds-WP_000857133.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,861 - INFO - Gene gene-SPN23F_RS13920 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,861 - INFO - Gene cds-WP_001845478.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,861 - INFO - Gene gene-SPN23F_RS06625 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,866 - INFO - Gene cds-WP_000814511.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,869 - INFO - Gene cds-SPN23F_RS06630 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,881 - INFO - Gene gene-SPN23F_RS06640 has a mutation rate of 0.07333333333333333.\n",
      "2025-04-16 11:13:20,894 - INFO - Gene cds-WP_000821309.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,895 - INFO - Gene gene-SPN23F_RS06645 has a mutation rate of 0.08.\n",
      "2025-04-16 11:13:20,895 - INFO - Gene cds-WP_000840776.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,896 - INFO - Gene gene-SPN23F_RS06650 has a mutation rate of 0.03.\n",
      "2025-04-16 11:13:20,896 - INFO - Gene cds-WP_000566524.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,896 - INFO - Gene gene-SPN23F_RS06655 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,897 - INFO - Gene cds-WP_000645887.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,897 - INFO - Gene gene-SPN23F_RS06660 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,900 - INFO - Gene cds-WP_001210966.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,902 - INFO - Gene gene-SPN23F_RS06665 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,903 - INFO - Gene cds-WP_000822766.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,903 - INFO - Gene gene-SPN23F_RS13570 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,904 - INFO - Gene cds-WP_000159607.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,904 - INFO - Gene gene-SPN23F_RS06675 has a mutation rate of 0.03333333333333333.\n",
      "2025-04-16 11:13:20,905 - INFO - Gene cds-WP_001196960.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,905 - INFO - Gene gene-SPN23F_RS06680 has a mutation rate of 0.23333333333333334.\n",
      "2025-04-16 11:13:20,905 - INFO - Gene cds-WP_001287278.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,906 - INFO - Gene gene-SPN23F_RS06685 has a mutation rate of 0.2733333333333333.\n",
      "2025-04-16 11:13:20,906 - INFO - Gene cds-WP_001815711.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,907 - INFO - Gene gene-SPN23F_RS06690 has a mutation rate of 0.9566666666666667.\n",
      "2025-04-16 11:13:20,908 - INFO - Gene cds-WP_000828838.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,908 - INFO - Gene gene-SPN23F_RS06695 has a mutation rate of 0.9.\n",
      "2025-04-16 11:13:20,909 - INFO - Gene cds-WP_000876328.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,909 - INFO - Gene gene-SPN23F_RS06700 has a mutation rate of 0.06333333333333334.\n",
      "2025-04-16 11:13:20,910 - INFO - Gene cds-WP_000818207.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,910 - INFO - Gene gene-SPN23F_RS06705 has a mutation rate of 0.10333333333333333.\n",
      "2025-04-16 11:13:20,910 - INFO - Gene cds-WP_000692438.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,911 - INFO - Gene gene-SPN23F_RS06710 has a mutation rate of 0.37.\n",
      "2025-04-16 11:13:20,911 - INFO - Gene cds-WP_000216377.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,912 - INFO - Gene gene-SPN23F_RS06715 has a mutation rate of 0.19666666666666666.\n",
      "2025-04-16 11:13:20,912 - INFO - Gene cds-WP_000782680.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,913 - INFO - Gene gene-SPN23F_RS06720 has a mutation rate of 0.06.\n",
      "2025-04-16 11:13:20,913 - INFO - Gene cds-WP_000461502.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,914 - INFO - Gene gene-SPN23F_RS06725 has a mutation rate of 0.5966666666666667.\n",
      "2025-04-16 11:13:20,914 - INFO - Gene cds-WP_000565082.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,914 - INFO - Gene gene-SPN23F_RS06730 has a mutation rate of 0.5833333333333334.\n",
      "2025-04-16 11:13:20,915 - INFO - Gene cds-WP_000446975.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,915 - INFO - Gene gene-SPN23F_RS06735 has a mutation rate of 0.5066666666666667.\n",
      "2025-04-16 11:13:20,916 - INFO - Gene cds-WP_000225898.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,916 - INFO - Gene gene-SPN23F_RS06740 has a mutation rate of 0.7266666666666667.\n",
      "2025-04-16 11:13:20,917 - INFO - Gene cds-WP_001061973.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,918 - INFO - Gene gene-SPN23F_RS06745 has a mutation rate of 0.55.\n",
      "2025-04-16 11:13:20,918 - INFO - Gene cds-WP_000032347.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,918 - INFO - Gene gene-SPN23F_RS06750 has a mutation rate of 0.8566666666666667.\n",
      "2025-04-16 11:13:20,919 - INFO - Gene cds-WP_000686390.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,919 - INFO - Gene gene-SPN23F_RS06755 has a mutation rate of 0.39666666666666667.\n",
      "2025-04-16 11:13:20,920 - INFO - Gene cds-WP_001151380.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,920 - INFO - Gene gene-SPN23F_RS06760 has a mutation rate of 0.8733333333333333.\n",
      "2025-04-16 11:13:20,921 - INFO - Gene cds-WP_000769886.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,921 - INFO - Gene gene-SPN23F_RS06765 has a mutation rate of 0.27666666666666667.\n",
      "2025-04-16 11:13:20,922 - INFO - Gene cds-WP_000065988.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,922 - INFO - Gene gene-SPN23F_RS06770 has a mutation rate of 0.5233333333333333.\n",
      "2025-04-16 11:13:20,923 - INFO - Gene cds-WP_001140486.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,923 - INFO - Gene gene-SPN23F_RS06775 has a mutation rate of 0.51.\n",
      "2025-04-16 11:13:20,924 - INFO - Gene cds-WP_001269887.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,924 - INFO - Gene gene-SPN23F_RS06780 has a mutation rate of 0.9066666666666666.\n",
      "2025-04-16 11:13:20,925 - INFO - Gene cds-WP_000702137.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,926 - INFO - Gene gene-SPN23F_RS06785 has a mutation rate of 0.8333333333333334.\n",
      "2025-04-16 11:13:20,926 - INFO - Gene cds-WP_000762476.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,927 - INFO - Gene gene-SPN23F_RS06790 has a mutation rate of 0.37666666666666665.\n",
      "2025-04-16 11:13:20,927 - INFO - Gene cds-WP_000767775.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,927 - INFO - Gene gene-SPN23F_RS06795 has a mutation rate of 0.8233333333333334.\n",
      "2025-04-16 11:13:20,928 - INFO - Gene cds-WP_001079973.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,928 - INFO - Gene gene-SPN23F_RS06800 has a mutation rate of 0.9466666666666667.\n",
      "2025-04-16 11:13:20,929 - INFO - Gene cds-WP_001237588.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,929 - INFO - Gene gene-SPN23F_RS06805 has a mutation rate of 0.8466666666666667.\n",
      "2025-04-16 11:13:20,929 - INFO - Gene cds-WP_000569202.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,930 - INFO - Gene gene-SPN23F_RS06810 has a mutation rate of 0.6966666666666667.\n",
      "2025-04-16 11:13:20,930 - INFO - Gene cds-WP_001181074.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,931 - INFO - Gene gene-SPN23F_RS06815 has a mutation rate of 0.8.\n",
      "2025-04-16 11:13:20,932 - INFO - Gene cds-WP_000811723.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,933 - INFO - Gene gene-SPN23F_RS06820 has a mutation rate of 0.44666666666666666.\n",
      "2025-04-16 11:13:20,933 - INFO - Gene cds-WP_000847075.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,934 - INFO - Gene gene-SPN23F_RS06825 has a mutation rate of 0.23.\n",
      "2025-04-16 11:13:20,934 - INFO - Gene cds-WP_000784184.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,935 - INFO - Gene gene-SPN23F_RS06830 has a mutation rate of 0.6566666666666666.\n",
      "2025-04-16 11:13:20,935 - INFO - Gene cds-WP_000725682.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,936 - INFO - Gene gene-SPN23F_RS06835 has a mutation rate of 0.5266666666666666.\n",
      "2025-04-16 11:13:20,936 - INFO - Gene cds-WP_000712734.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,937 - INFO - Gene gene-SPN23F_RS06840 has a mutation rate of 0.19666666666666666.\n",
      "2025-04-16 11:13:20,938 - INFO - Gene cds-WP_000753792.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,940 - INFO - Gene gene-SPN23F_RS06845 has a mutation rate of 0.21666666666666667.\n",
      "2025-04-16 11:13:20,943 - INFO - Gene cds-WP_000742913.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,969 - INFO - Gene gene-SPN23F_RS06850 has a mutation rate of 0.41333333333333333.\n",
      "2025-04-16 11:13:20,970 - INFO - Gene cds-WP_000116181.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,970 - INFO - Gene gene-SPN23F_RS06855 has a mutation rate of 0.8933333333333333.\n",
      "2025-04-16 11:13:20,971 - INFO - Gene cds-WP_000152223.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,971 - INFO - Gene gene-SPN23F_RS06860 has a mutation rate of 0.35.\n",
      "2025-04-16 11:13:20,972 - INFO - Gene cds-WP_000360300.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,972 - INFO - Gene gene-SPN23F_RS06865 has a mutation rate of 0.7633333333333333.\n",
      "2025-04-16 11:13:20,977 - INFO - Gene cds-WP_000744823.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,979 - INFO - Gene gene-SPN23F_RS06870 has a mutation rate of 0.6066666666666667.\n",
      "2025-04-16 11:13:20,979 - INFO - Gene cds-WP_000837347.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,980 - INFO - Gene gene-SPN23F_RS06875 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,980 - INFO - Gene cds-WP_000946463.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,982 - INFO - Gene gene-SPN23F_RS06880 has a mutation rate of 0.17333333333333334.\n",
      "2025-04-16 11:13:20,982 - INFO - Gene cds-WP_000128639.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,983 - INFO - Gene gene-SPN23F_RS06885 has a mutation rate of 0.2733333333333333.\n",
      "2025-04-16 11:13:20,984 - INFO - Gene cds-WP_000109217.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,985 - INFO - Gene gene-SPN23F_RS06890 has a mutation rate of 0.21333333333333335.\n",
      "2025-04-16 11:13:20,986 - INFO - Gene cds-WP_001201789.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,986 - INFO - Gene cds-SPN23F_RS06895 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,987 - INFO - Gene gene-SPN23F_RS06900 has a mutation rate of 0.016666666666666666.\n",
      "2025-04-16 11:13:20,987 - INFO - Gene cds-WP_000746335.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,989 - INFO - Gene gene-SPN23F_RS06905 has a mutation rate of 0.8166666666666667.\n",
      "2025-04-16 11:13:20,990 - INFO - Gene cds-WP_001160272.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:20,990 - INFO - Gene gene-SPN23F_RS06910 has a mutation rate of 0.65.\n",
      "2025-04-16 11:13:21,021 - INFO - Gene cds-WP_000444038.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,022 - INFO - Gene gene-SPN23F_RS06915 has a mutation rate of 0.03333333333333333.\n",
      "2025-04-16 11:13:21,022 - INFO - Gene cds-WP_001041974.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,023 - INFO - Gene gene-SPN23F_RS06920 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,023 - INFO - Gene cds-WP_000631259.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,024 - INFO - Gene gene-SPN23F_RS06925 has a mutation rate of 0.7833333333333333.\n",
      "2025-04-16 11:13:21,025 - INFO - Gene cds-WP_000850629.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,025 - INFO - Gene gene-SPN23F_RS06930 has a mutation rate of 0.31.\n",
      "2025-04-16 11:13:21,026 - INFO - Gene cds-WP_000875454.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,027 - INFO - Gene gene-SPN23F_RS06935 has a mutation rate of 0.7066666666666667.\n",
      "2025-04-16 11:13:21,028 - INFO - Gene cds-WP_000524782.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,028 - INFO - Gene gene-SPN23F_RS06940 has a mutation rate of 0.13333333333333333.\n",
      "2025-04-16 11:13:21,028 - INFO - Gene cds-WP_001170072.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,029 - INFO - Gene gene-SPN23F_RS06945 has a mutation rate of 0.26666666666666666.\n",
      "2025-04-16 11:13:21,029 - INFO - Gene cds-WP_000517372.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,030 - INFO - Gene gene-SPN23F_RS06950 has a mutation rate of 0.13.\n",
      "2025-04-16 11:13:21,031 - INFO - Gene cds-WP_000895040.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,032 - INFO - Gene gene-SPN23F_RS06955 has a mutation rate of 0.31666666666666665.\n",
      "2025-04-16 11:13:21,032 - INFO - Gene cds-WP_000886661.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,033 - INFO - Gene gene-SPN23F_RS06960 has a mutation rate of 0.33.\n",
      "2025-04-16 11:13:21,033 - INFO - Gene cds-WP_000115143.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,035 - INFO - Gene gene-SPN23F_RS06965 has a mutation rate of 0.01.\n",
      "2025-04-16 11:13:21,035 - INFO - Gene cds-WP_000048055.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,036 - INFO - Gene gene-SPN23F_RS06970 has a mutation rate of 0.3466666666666667.\n",
      "2025-04-16 11:13:21,036 - INFO - Gene cds-WP_000864617.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,037 - INFO - Gene gene-SPN23F_RS06975 has a mutation rate of 0.5833333333333334.\n",
      "2025-04-16 11:13:21,039 - INFO - Gene cds-WP_001090149.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,039 - INFO - Gene gene-SPN23F_RS14125 has a mutation rate of 0.55.\n",
      "2025-04-16 11:13:21,040 - INFO - Gene cds-WP_000240828.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,071 - INFO - Gene gene-SPN23F_RS14210 has a mutation rate of 0.8733333333333333.\n",
      "2025-04-16 11:13:21,073 - INFO - Gene cds-WP_306423773.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,074 - INFO - Gene gene-SPN23F_RS06990 has a mutation rate of 0.6133333333333333.\n",
      "2025-04-16 11:13:21,076 - INFO - Gene cds-WP_000835661.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,081 - INFO - Gene gene-SPN23F_RS06995 has a mutation rate of 0.24.\n",
      "2025-04-16 11:13:21,081 - INFO - Gene cds-WP_000058033.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,083 - INFO - Gene gene-SPN23F_RS07000 has a mutation rate of 0.33666666666666667.\n",
      "2025-04-16 11:13:21,083 - INFO - Gene cds-WP_000283118.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,084 - INFO - Gene gene-SPN23F_RS07005 has a mutation rate of 0.19.\n",
      "2025-04-16 11:13:21,085 - INFO - Gene cds-WP_000797183.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,086 - INFO - Gene gene-SPN23F_RS07010 has a mutation rate of 0.22.\n",
      "2025-04-16 11:13:21,087 - INFO - Gene cds-WP_001107505.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,087 - INFO - Gene cds-SPN23F_RS07015 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,089 - INFO - Gene cds-SPN23F_RS07020 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,089 - INFO - Gene cds-SPN23F_RS07025 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,090 - INFO - Gene gene-SPN23F_RS12405 has a mutation rate of 0.15333333333333332.\n",
      "2025-04-16 11:13:21,090 - INFO - Gene cds-WP_001810463.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,091 - INFO - Gene gene-SPN23F_RS07035 has a mutation rate of 0.64.\n",
      "2025-04-16 11:13:21,092 - INFO - Gene cds-WP_000169077.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,093 - INFO - Gene cds-SPN23F_RS13930 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,094 - INFO - Gene gene-SPN23F_RS07050 has a mutation rate of 0.5866666666666667.\n",
      "2025-04-16 11:13:21,094 - INFO - Gene cds-WP_000217307.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,095 - INFO - Gene gene-SPN23F_RS07055 has a mutation rate of 0.15666666666666668.\n",
      "2025-04-16 11:13:21,096 - INFO - Gene cds-WP_000122904.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,097 - INFO - Gene gene-SPN23F_RS07060 has a mutation rate of 0.5933333333333334.\n",
      "2025-04-16 11:13:21,098 - INFO - Gene cds-WP_000708386.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,099 - INFO - Gene gene-SPN23F_RS07065 has a mutation rate of 0.45.\n",
      "2025-04-16 11:13:21,100 - INFO - Gene cds-WP_000657825.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,101 - INFO - Gene gene-SPN23F_RS07070 has a mutation rate of 0.2733333333333333.\n",
      "2025-04-16 11:13:21,101 - INFO - Gene cds-WP_000368971.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,102 - INFO - Gene gene-SPN23F_RS07075 has a mutation rate of 0.43333333333333335.\n",
      "2025-04-16 11:13:21,104 - INFO - Gene cds-WP_000102999.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,104 - INFO - Gene gene-SPN23F_RS07080 has a mutation rate of 0.43666666666666665.\n",
      "2025-04-16 11:13:21,104 - INFO - Gene cds-WP_000229353.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,105 - INFO - Gene cds-SPN23F_RS07085 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,107 - INFO - Gene cds-SPN23F_RS13935 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,108 - INFO - Gene gene-SPN23F_RS07100 has a mutation rate of 0.29.\n",
      "2025-04-16 11:13:21,108 - INFO - Gene cds-WP_000691850.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,109 - INFO - Gene gene-SPN23F_RS07105 has a mutation rate of 0.25.\n",
      "2025-04-16 11:13:21,111 - INFO - Gene cds-WP_001248797.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,112 - INFO - Gene gene-SPN23F_RS07110 has a mutation rate of 0.15333333333333332.\n",
      "2025-04-16 11:13:21,114 - INFO - Gene cds-WP_001136122.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,115 - INFO - Gene gene-SPN23F_RS07115 has a mutation rate of 0.26666666666666666.\n",
      "2025-04-16 11:13:21,118 - INFO - Gene cds-WP_000065720.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,118 - INFO - Gene gene-SPN23F_RS07120 has a mutation rate of 0.8.\n",
      "2025-04-16 11:13:21,119 - INFO - Gene cds-WP_000936188.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,120 - INFO - Gene gene-SPN23F_RS07125 has a mutation rate of 0.86.\n",
      "2025-04-16 11:13:21,121 - INFO - Gene cds-WP_000071674.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,122 - INFO - Gene gene-SPN23F_RS07130 has a mutation rate of 0.6633333333333333.\n",
      "2025-04-16 11:13:21,123 - INFO - Gene cds-WP_000872215.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,123 - INFO - Gene gene-SPN23F_RS07135 has a mutation rate of 0.25666666666666665.\n",
      "2025-04-16 11:13:21,124 - INFO - Gene cds-WP_001102210.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,125 - INFO - Gene gene-SPN23F_RS07140 has a mutation rate of 0.35.\n",
      "2025-04-16 11:13:21,128 - INFO - Gene cds-WP_000290657.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,129 - INFO - Gene gene-SPN23F_RS07145 has a mutation rate of 0.6566666666666666.\n",
      "2025-04-16 11:13:21,129 - INFO - Gene cds-WP_000972931.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,131 - INFO - Gene gene-SPN23F_RS07150 has a mutation rate of 0.57.\n",
      "2025-04-16 11:13:21,131 - INFO - Gene cds-WP_000526284.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,132 - INFO - Gene gene-SPN23F_RS07155 has a mutation rate of 0.83.\n",
      "2025-04-16 11:13:21,135 - INFO - Gene cds-WP_000930000.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,135 - INFO - Gene gene-SPN23F_RS07170 has a mutation rate of 0.06.\n",
      "2025-04-16 11:13:21,136 - INFO - Gene cds-WP_001272961.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,138 - INFO - Gene gene-SPN23F_RS07175 has a mutation rate of 0.30333333333333334.\n",
      "2025-04-16 11:13:21,139 - INFO - Gene cds-WP_000855736.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,139 - INFO - Gene gene-SPN23F_RS07180 has a mutation rate of 0.8033333333333333.\n",
      "2025-04-16 11:13:21,140 - INFO - Gene cds-WP_000272304.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,161 - INFO - Gene gene-SPN23F_RS07185 has a mutation rate of 0.02666666666666667.\n",
      "2025-04-16 11:13:21,171 - INFO - Gene cds-WP_000926600.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,180 - INFO - Gene gene-SPN23F_RS07190 has a mutation rate of 0.18.\n",
      "2025-04-16 11:13:21,183 - INFO - Gene cds-WP_000590986.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,184 - INFO - Gene gene-SPN23F_RS07195 has a mutation rate of 0.14.\n",
      "2025-04-16 11:13:21,186 - INFO - Gene cds-WP_001103449.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,189 - INFO - Gene gene-SPN23F_RS07200 has a mutation rate of 0.18333333333333332.\n",
      "2025-04-16 11:13:21,189 - INFO - Gene cds-WP_000889093.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,191 - INFO - Gene gene-SPN23F_RS07205 has a mutation rate of 0.4866666666666667.\n",
      "2025-04-16 11:13:21,191 - INFO - Gene cds-WP_001170346.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,191 - INFO - Gene gene-SPN23F_RS07210 has a mutation rate of 0.22333333333333333.\n",
      "2025-04-16 11:13:21,194 - INFO - Gene cds-WP_000405832.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,195 - INFO - Gene gene-SPN23F_RS07215 has a mutation rate of 0.32.\n",
      "2025-04-16 11:13:21,196 - INFO - Gene cds-WP_000858730.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,197 - INFO - Gene gene-SPN23F_RS07220 has a mutation rate of 0.5633333333333334.\n",
      "2025-04-16 11:13:21,198 - INFO - Gene cds-WP_001097986.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,198 - INFO - Gene gene-SPN23F_RS07225 has a mutation rate of 0.24666666666666667.\n",
      "2025-04-16 11:13:21,200 - INFO - Gene cds-WP_000689945.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,200 - INFO - Gene gene-SPN23F_RS07230 has a mutation rate of 0.09.\n",
      "2025-04-16 11:13:21,202 - INFO - Gene cds-WP_000138517.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,203 - INFO - Gene gene-SPN23F_RS07235 has a mutation rate of 0.39.\n",
      "2025-04-16 11:13:21,203 - INFO - Gene cds-WP_000036789.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,204 - INFO - Gene gene-SPN23F_RS07240 has a mutation rate of 0.2833333333333333.\n",
      "2025-04-16 11:13:21,205 - INFO - Gene cds-WP_000175413.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,207 - INFO - Gene gene-SPN23F_RS07245 has a mutation rate of 0.5933333333333334.\n",
      "2025-04-16 11:13:21,208 - INFO - Gene cds-WP_000915925.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,209 - INFO - Gene gene-SPN23F_RS07250 has a mutation rate of 0.5233333333333333.\n",
      "2025-04-16 11:13:21,209 - INFO - Gene cds-WP_000673677.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,210 - INFO - Gene gene-SPN23F_RS07255 has a mutation rate of 0.3233333333333333.\n",
      "2025-04-16 11:13:21,210 - INFO - Gene cds-WP_000371287.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,211 - INFO - Gene gene-SPN23F_RS07260 has a mutation rate of 0.74.\n",
      "2025-04-16 11:13:21,211 - INFO - Gene cds-WP_000164779.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,211 - INFO - Gene gene-SPN23F_RS07265 has a mutation rate of 0.43666666666666665.\n",
      "2025-04-16 11:13:21,212 - INFO - Gene cds-WP_000038750.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,212 - INFO - Gene gene-SPN23F_RS07270 has a mutation rate of 0.3933333333333333.\n",
      "2025-04-16 11:13:21,212 - INFO - Gene cds-WP_000369575.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,213 - INFO - Gene gene-SPN23F_RS07275 has a mutation rate of 0.7766666666666666.\n",
      "2025-04-16 11:13:21,213 - INFO - Gene cds-WP_001099689.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,213 - INFO - Gene gene-SPN23F_RS07280 has a mutation rate of 0.7533333333333333.\n",
      "2025-04-16 11:13:21,213 - INFO - Gene cds-WP_048814490.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,214 - INFO - Gene cds-SPN23F_RS07285 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,214 - INFO - Gene gene-SPN23F_RS07290 has a mutation rate of 0.45.\n",
      "2025-04-16 11:13:21,214 - INFO - Gene cds-WP_000915899.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,215 - INFO - Gene gene-SPN23F_RS07295 has a mutation rate of 0.49333333333333335.\n",
      "2025-04-16 11:13:21,215 - INFO - Gene cds-WP_000850582.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,215 - INFO - Gene cds-SPN23F_RS12425 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,215 - INFO - Gene cds-SPN23F_RS12435 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,216 - INFO - Gene gene-SPN23F_RS07335 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,216 - INFO - Gene cds-WP_001040724.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,216 - INFO - Gene gene-SPN23F_RS07340 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,217 - INFO - Gene cds-WP_001155339.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,217 - INFO - Gene gene-SPN23F_RS07345 has a mutation rate of 0.6766666666666666.\n",
      "2025-04-16 11:13:21,217 - INFO - Gene cds-WP_000348121.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,218 - INFO - Gene gene-SPN23F_RS07350 has a mutation rate of 0.6266666666666667.\n",
      "2025-04-16 11:13:21,218 - INFO - Gene cds-WP_000248992.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,218 - INFO - Gene gene-SPN23F_RS07355 has a mutation rate of 0.7333333333333333.\n",
      "2025-04-16 11:13:21,218 - INFO - Gene cds-WP_001021423.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,219 - INFO - Gene gene-SPN23F_RS07360 has a mutation rate of 0.66.\n",
      "2025-04-16 11:13:21,219 - INFO - Gene cds-WP_000935666.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,219 - INFO - Gene gene-SPN23F_RS14215 has a mutation rate of 0.04.\n",
      "2025-04-16 11:13:21,219 - INFO - Gene cds-WP_024266129.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,220 - INFO - Gene cds-SPN23F_RS13940 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,220 - INFO - Gene gene-SPN23F_RS07385 has a mutation rate of 0.4533333333333333.\n",
      "2025-04-16 11:13:21,220 - INFO - Gene cds-WP_000121714.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,221 - INFO - Gene gene-SPN23F_RS07390 has a mutation rate of 0.58.\n",
      "2025-04-16 11:13:21,221 - INFO - Gene cds-WP_000434650.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,221 - INFO - Gene gene-SPN23F_RS07395 has a mutation rate of 0.25333333333333335.\n",
      "2025-04-16 11:13:21,222 - INFO - Gene cds-WP_000759191.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,222 - INFO - Gene gene-SPN23F_RS07400 has a mutation rate of 0.38.\n",
      "2025-04-16 11:13:21,222 - INFO - Gene cds-WP_000891765.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,222 - INFO - Gene gene-SPN23F_RS07405 has a mutation rate of 0.07666666666666666.\n",
      "2025-04-16 11:13:21,223 - INFO - Gene cds-WP_000120830.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,223 - INFO - Gene gene-SPN23F_RS07410 has a mutation rate of 0.5033333333333333.\n",
      "2025-04-16 11:13:21,223 - INFO - Gene cds-WP_001065962.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,224 - INFO - Gene gene-SPN23F_RS07415 has a mutation rate of 0.14333333333333334.\n",
      "2025-04-16 11:13:21,224 - INFO - Gene cds-WP_000435455.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,224 - INFO - Gene gene-SPN23F_RS07420 has a mutation rate of 0.43.\n",
      "2025-04-16 11:13:21,224 - INFO - Gene cds-WP_000665619.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,225 - INFO - Gene gene-SPN23F_RS07425 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:21,225 - INFO - Gene cds-WP_000068050.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,226 - INFO - Gene gene-SPN23F_RS07430 has a mutation rate of 0.03333333333333333.\n",
      "2025-04-16 11:13:21,226 - INFO - Gene cds-WP_000094354.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,226 - INFO - Gene gene-SPN23F_RS07435 has a mutation rate of 0.15.\n",
      "2025-04-16 11:13:21,227 - INFO - Gene cds-WP_000301210.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,227 - INFO - Gene gene-SPN23F_RS07440 has a mutation rate of 0.42333333333333334.\n",
      "2025-04-16 11:13:21,228 - INFO - Gene cds-WP_000996644.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,228 - INFO - Gene gene-SPN23F_RS07445 has a mutation rate of 0.02666666666666667.\n",
      "2025-04-16 11:13:21,228 - INFO - Gene cds-WP_000359036.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,229 - INFO - Gene gene-SPN23F_RS07450 has a mutation rate of 0.17.\n",
      "2025-04-16 11:13:21,229 - INFO - Gene cds-WP_000558554.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,230 - INFO - Gene gene-SPN23F_RS07455 has a mutation rate of 0.36.\n",
      "2025-04-16 11:13:21,230 - INFO - Gene cds-WP_000392851.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,230 - INFO - Gene gene-SPN23F_RS07460 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,231 - INFO - Gene cds-WP_001054562.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,231 - INFO - Gene gene-SPN23F_RS07465 has a mutation rate of 0.43.\n",
      "2025-04-16 11:13:21,262 - INFO - Gene cds-WP_001153871.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,262 - INFO - Gene gene-SPN23F_RS07470 has a mutation rate of 0.25666666666666665.\n",
      "2025-04-16 11:13:21,263 - INFO - Gene cds-WP_000428147.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,264 - INFO - Gene gene-SPN23F_RS07480 has a mutation rate of 0.07333333333333333.\n",
      "2025-04-16 11:13:21,264 - INFO - Gene cds-WP_000818760.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,265 - INFO - Gene gene-SPN23F_RS07485 has a mutation rate of 0.44666666666666666.\n",
      "2025-04-16 11:13:21,265 - INFO - Gene cds-WP_001291665.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,266 - INFO - Gene gene-SPN23F_RS07490 has a mutation rate of 0.10666666666666667.\n",
      "2025-04-16 11:13:21,273 - INFO - Gene cds-WP_000702436.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,273 - INFO - Gene gene-SPN23F_RS07495 has a mutation rate of 0.6166666666666667.\n",
      "2025-04-16 11:13:21,274 - INFO - Gene cds-WP_000405297.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,274 - INFO - Gene gene-SPN23F_RS07500 has a mutation rate of 0.33666666666666667.\n",
      "2025-04-16 11:13:21,275 - INFO - Gene cds-WP_000048117.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,275 - INFO - Gene gene-SPN23F_RS07505 has a mutation rate of 0.2733333333333333.\n",
      "2025-04-16 11:13:21,275 - INFO - Gene cds-WP_000808221.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,276 - INFO - Gene gene-SPN23F_RS07510 has a mutation rate of 0.9933333333333333.\n",
      "2025-04-16 11:13:21,277 - INFO - Gene cds-WP_001096217.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,277 - INFO - Gene gene-SPN23F_RS07515 has a mutation rate of 0.5766666666666667.\n",
      "2025-04-16 11:13:21,277 - INFO - Gene cds-WP_000521362.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,278 - INFO - Gene gene-SPN23F_RS07520 has a mutation rate of 0.73.\n",
      "2025-04-16 11:13:21,278 - INFO - Gene cds-WP_000031964.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,278 - INFO - Gene gene-SPN23F_RS13945 has a mutation rate of 0.8833333333333333.\n",
      "2025-04-16 11:13:21,279 - INFO - Gene cds-WP_001833285.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,279 - INFO - Gene cds-SPN23F_RS13950 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,280 - INFO - Gene gene-SPN23F_RS07530 has a mutation rate of 0.6866666666666666.\n",
      "2025-04-16 11:13:21,280 - INFO - Gene cds-WP_000748875.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,280 - INFO - Gene gene-SPN23F_RS07535 has a mutation rate of 0.16666666666666666.\n",
      "2025-04-16 11:13:21,281 - INFO - Gene cds-WP_000017793.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,281 - INFO - Gene gene-SPN23F_RS07540 has a mutation rate of 0.92.\n",
      "2025-04-16 11:13:21,282 - INFO - Gene cds-WP_000590317.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,282 - INFO - Gene gene-SPN23F_RS07545 has a mutation rate of 0.15333333333333332.\n",
      "2025-04-16 11:13:21,283 - INFO - Gene cds-WP_000051196.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,283 - INFO - Gene cds-SPN23F_RS07550 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,284 - INFO - Gene gene-SPN23F_RS07555 has a mutation rate of 0.7166666666666667.\n",
      "2025-04-16 11:13:21,284 - INFO - Gene cds-WP_000892254.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,285 - INFO - Gene gene-SPN23F_RS07560 has a mutation rate of 0.14.\n",
      "2025-04-16 11:13:21,286 - INFO - Gene cds-WP_000036043.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,286 - INFO - Gene gene-SPN23F_RS07565 has a mutation rate of 0.23.\n",
      "2025-04-16 11:13:21,286 - INFO - Gene cds-WP_000349601.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,287 - INFO - Gene gene-SPN23F_RS07570 has a mutation rate of 0.53.\n",
      "2025-04-16 11:13:21,287 - INFO - Gene cds-WP_000390785.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,288 - INFO - Gene gene-SPN23F_RS07575 has a mutation rate of 0.21666666666666667.\n",
      "2025-04-16 11:13:21,288 - INFO - Gene cds-WP_000689596.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,289 - INFO - Gene gene-SPN23F_RS07580 has a mutation rate of 0.8233333333333334.\n",
      "2025-04-16 11:13:21,289 - INFO - Gene cds-WP_000339024.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,290 - INFO - Gene gene-SPN23F_RS07585 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,290 - INFO - Gene cds-WP_000068664.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,290 - INFO - Gene gene-SPN23F_RS07590 has a mutation rate of 0.26.\n",
      "2025-04-16 11:13:21,291 - INFO - Gene cds-WP_000609607.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,291 - INFO - Gene gene-SPN23F_RS07595 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:21,292 - INFO - Gene cds-WP_001151785.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,292 - INFO - Gene gene-SPN23F_RS07600 has a mutation rate of 0.3566666666666667.\n",
      "2025-04-16 11:13:21,293 - INFO - Gene cds-WP_000167151.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,293 - INFO - Gene gene-SPN23F_RS13035 has a mutation rate of 0.8833333333333333.\n",
      "2025-04-16 11:13:21,294 - INFO - Gene cds-WP_000565218.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,294 - INFO - Gene gene-SPN23F_RS07610 has a mutation rate of 0.79.\n",
      "2025-04-16 11:13:21,295 - INFO - Gene cds-WP_000777782.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,295 - INFO - Gene gene-SPN23F_RS07615 has a mutation rate of 0.6966666666666667.\n",
      "2025-04-16 11:13:21,296 - INFO - Gene cds-WP_000747267.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,296 - INFO - Gene gene-SPN23F_RS07625 has a mutation rate of 0.44.\n",
      "2025-04-16 11:13:21,297 - INFO - Gene cds-WP_000039618.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,297 - INFO - Gene gene-SPN23F_RS07630 has a mutation rate of 0.36.\n",
      "2025-04-16 11:13:21,298 - INFO - Gene cds-WP_000682799.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,298 - INFO - Gene gene-SPN23F_RS07635 has a mutation rate of 0.36333333333333334.\n",
      "2025-04-16 11:13:21,298 - INFO - Gene cds-WP_001245198.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,299 - INFO - Gene gene-SPN23F_RS07640 has a mutation rate of 0.16666666666666666.\n",
      "2025-04-16 11:13:21,299 - INFO - Gene cds-WP_000697794.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,300 - INFO - Gene gene-SPN23F_RS07645 has a mutation rate of 0.13.\n",
      "2025-04-16 11:13:21,301 - INFO - Gene cds-WP_000508271.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,301 - INFO - Gene gene-SPN23F_RS07650 has a mutation rate of 0.6066666666666667.\n",
      "2025-04-16 11:13:21,301 - INFO - Gene cds-WP_000822856.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,302 - INFO - Gene gene-SPN23F_RS07655 has a mutation rate of 0.7133333333333334.\n",
      "2025-04-16 11:13:21,302 - INFO - Gene cds-WP_000412220.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,303 - INFO - Gene gene-SPN23F_RS07660 has a mutation rate of 0.3233333333333333.\n",
      "2025-04-16 11:13:21,303 - INFO - Gene cds-WP_001277408.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,303 - INFO - Gene gene-SPN23F_RS07665 has a mutation rate of 0.6166666666666667.\n",
      "2025-04-16 11:13:21,304 - INFO - Gene cds-WP_000032458.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,304 - INFO - Gene gene-SPN23F_RS07670 has a mutation rate of 0.6933333333333334.\n",
      "2025-04-16 11:13:21,305 - INFO - Gene cds-WP_000813909.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,305 - INFO - Gene gene-SPN23F_RS07675 has a mutation rate of 0.8933333333333333.\n",
      "2025-04-16 11:13:21,306 - INFO - Gene cds-WP_001279155.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,334 - INFO - Gene gene-SPN23F_RS07680 has a mutation rate of 0.7533333333333333.\n",
      "2025-04-16 11:13:21,335 - INFO - Gene cds-WP_078134417.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,336 - INFO - Gene gene-SPN23F_RS07685 has a mutation rate of 0.73.\n",
      "2025-04-16 11:13:21,337 - INFO - Gene cds-WP_000027902.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,337 - INFO - Gene gene-SPN23F_RS07690 has a mutation rate of 0.58.\n",
      "2025-04-16 11:13:21,337 - INFO - Gene cds-WP_000762070.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,343 - INFO - Gene gene-SPN23F_RS07695 has a mutation rate of 0.06666666666666667.\n",
      "2025-04-16 11:13:21,344 - INFO - Gene cds-WP_001050086.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,345 - INFO - Gene gene-SPN23F_RS07700 has a mutation rate of 0.47333333333333333.\n",
      "2025-04-16 11:13:21,345 - INFO - Gene cds-WP_000521404.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,346 - INFO - Gene gene-SPN23F_RS07705 has a mutation rate of 0.03666666666666667.\n",
      "2025-04-16 11:13:21,346 - INFO - Gene cds-WP_000742290.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,346 - INFO - Gene gene-SPN23F_RS07710 has a mutation rate of 0.2866666666666667.\n",
      "2025-04-16 11:13:21,347 - INFO - Gene cds-WP_001011066.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,348 - INFO - Gene gene-SPN23F_RS07720 has a mutation rate of 0.5166666666666667.\n",
      "2025-04-16 11:13:21,348 - INFO - Gene cds-WP_000080998.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,348 - INFO - Gene gene-SPN23F_RS07725 has a mutation rate of 0.45.\n",
      "2025-04-16 11:13:21,349 - INFO - Gene cds-WP_000350453.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,349 - INFO - Gene gene-SPN23F_RS07730 has a mutation rate of 0.08666666666666667.\n",
      "2025-04-16 11:13:21,349 - INFO - Gene cds-WP_001186206.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,350 - INFO - Gene gene-SPN23F_RS07735 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,350 - INFO - Gene cds-WP_000687454.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,350 - INFO - Gene gene-SPN23F_RS07740 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,351 - INFO - Gene cds-WP_000922099.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,351 - INFO - Gene gene-SPN23F_RS07745 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,352 - INFO - Gene cds-WP_000159973.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,352 - INFO - Gene gene-SPN23F_RS07750 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,352 - INFO - Gene cds-WP_000927607.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,353 - INFO - Gene gene-SPN23F_RS07755 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,353 - INFO - Gene cds-WP_000161549.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,354 - INFO - Gene gene-SPN23F_RS07760 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,354 - INFO - Gene cds-WP_000763178.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,355 - INFO - Gene gene-SPN23F_RS07765 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,355 - INFO - Gene cds-WP_000062049.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,356 - INFO - Gene gene-SPN23F_RS07770 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,356 - INFO - Gene cds-WP_000129838.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,356 - INFO - Gene gene-SPN23F_RS07775 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,357 - INFO - Gene cds-WP_000198230.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,357 - INFO - Gene gene-SPN23F_RS07780 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,357 - INFO - Gene cds-WP_001208881.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,358 - INFO - Gene gene-SPN23F_RS07785 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,358 - INFO - Gene cds-WP_001017359.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,359 - INFO - Gene gene-SPN23F_RS07790 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,359 - INFO - Gene cds-WP_000565932.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,360 - INFO - Gene gene-SPN23F_RS07795 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,360 - INFO - Gene cds-WP_000221834.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,360 - INFO - Gene gene-SPN23F_RS07800 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,361 - INFO - Gene cds-WP_001240344.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,361 - INFO - Gene gene-SPN23F_RS07805 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,362 - INFO - Gene cds-WP_001139818.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,362 - INFO - Gene gene-SPN23F_RS07810 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,362 - INFO - Gene cds-WP_000896122.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,363 - INFO - Gene gene-SPN23F_RS07815 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,363 - INFO - Gene cds-WP_000730218.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,364 - INFO - Gene gene-SPN23F_RS13580 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,364 - INFO - Gene cds-WP_000169861.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,364 - INFO - Gene gene-SPN23F_RS07820 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,365 - INFO - Gene cds-WP_000051860.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,365 - INFO - Gene gene-SPN23F_RS07825 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,366 - INFO - Gene cds-WP_000143806.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,367 - INFO - Gene gene-SPN23F_RS07830 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,368 - INFO - Gene cds-WP_001136857.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,368 - INFO - Gene gene-SPN23F_RS07835 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,369 - INFO - Gene cds-WP_000057536.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,369 - INFO - Gene gene-SPN23F_RS07840 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,370 - INFO - Gene cds-WP_000425577.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,370 - INFO - Gene gene-SPN23F_RS07845 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,370 - INFO - Gene cds-WP_000162611.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,371 - INFO - Gene gene-SPN23F_RS07850 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,371 - INFO - Gene cds-WP_001067626.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,371 - INFO - Gene gene-SPN23F_RS07855 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,372 - INFO - Gene cds-WP_000612390.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,372 - INFO - Gene gene-SPN23F_RS13290 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,373 - INFO - Gene cds-WP_158302876.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,373 - INFO - Gene gene-SPN23F_RS07860 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,373 - INFO - Gene cds-WP_000194857.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,374 - INFO - Gene gene-SPN23F_RS07865 has a mutation rate of 0.2733333333333333.\n",
      "2025-04-16 11:13:21,385 - INFO - Gene cds-WP_001021764.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,404 - INFO - Gene gene-SPN23F_RS07870 has a mutation rate of 0.16333333333333333.\n",
      "2025-04-16 11:13:21,405 - INFO - Gene cds-WP_000969681.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,405 - INFO - Gene gene-SPN23F_RS07875 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,405 - INFO - Gene cds-WP_000573750.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,406 - INFO - Gene gene-SPN23F_RS07880 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,406 - INFO - Gene cds-WP_000667209.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,406 - INFO - Gene gene-SPN23F_RS07885 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,414 - INFO - Gene cds-WP_001286809.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,415 - INFO - Gene gene-SPN23F_RS07890 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,415 - INFO - Gene cds-WP_000206142.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,415 - INFO - Gene gene-SPN23F_RS07895 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,416 - INFO - Gene cds-WP_000358623.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,416 - INFO - Gene gene-SPN23F_RS07900 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,417 - INFO - Gene cds-WP_000161128.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,417 - INFO - Gene gene-SPN23F_RS07905 has a mutation rate of 0.05333333333333334.\n",
      "2025-04-16 11:13:21,418 - INFO - Gene cds-WP_000670026.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,418 - INFO - Gene gene-SPN23F_RS07910 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,419 - INFO - Gene cds-WP_000609575.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,419 - INFO - Gene gene-SPN23F_RS13585 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,419 - INFO - Gene cds-WP_001203349.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,420 - INFO - Gene gene-SPN23F_RS07915 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,420 - INFO - Gene cds-WP_000237811.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,421 - INFO - Gene gene-SPN23F_RS07920 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,421 - INFO - Gene cds-WP_000774582.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,422 - INFO - Gene gene-SPN23F_RS07925 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,422 - INFO - Gene cds-WP_001134282.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,422 - INFO - Gene gene-SPN23F_RS07930 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,423 - INFO - Gene cds-WP_000717373.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,423 - INFO - Gene gene-SPN23F_RS07935 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,424 - INFO - Gene cds-WP_000450965.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,424 - INFO - Gene gene-SPN23F_RS07940 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,424 - INFO - Gene cds-WP_001814284.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,425 - INFO - Gene gene-SPN23F_RS07945 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,426 - INFO - Gene cds-WP_001112906.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,426 - INFO - Gene gene-SPN23F_RS07950 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,426 - INFO - Gene cds-WP_000464163.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,427 - INFO - Gene gene-SPN23F_RS07955 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,427 - INFO - Gene cds-WP_000136430.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,428 - INFO - Gene gene-SPN23F_RS07960 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,428 - INFO - Gene cds-WP_001020945.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,429 - INFO - Gene gene-SPN23F_RS07965 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,429 - INFO - Gene cds-WP_000266853.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,429 - INFO - Gene gene-SPN23F_RS07970 has a mutation rate of 0.3433333333333333.\n",
      "2025-04-16 11:13:21,430 - INFO - Gene cds-WP_000011276.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,430 - INFO - Gene gene-SPN23F_RS07975 has a mutation rate of 0.47.\n",
      "2025-04-16 11:13:21,431 - INFO - Gene cds-WP_001231086.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,431 - INFO - Gene gene-SPN23F_RS07980 has a mutation rate of 0.3333333333333333.\n",
      "2025-04-16 11:13:21,432 - INFO - Gene cds-WP_000163035.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,432 - INFO - Gene gene-SPN23F_RS07985 has a mutation rate of 0.07333333333333333.\n",
      "2025-04-16 11:13:21,432 - INFO - Gene cds-WP_001140412.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,433 - INFO - Gene gene-SPN23F_RS07990 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:21,433 - INFO - Gene cds-WP_000422605.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,434 - INFO - Gene gene-SPN23F_RS07995 has a mutation rate of 0.13666666666666666.\n",
      "2025-04-16 11:13:21,434 - INFO - Gene cds-WP_000106354.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,435 - INFO - Gene gene-SPN23F_RS13590 has a mutation rate of 0.05.\n",
      "2025-04-16 11:13:21,435 - INFO - Gene cds-WP_000442276.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,436 - INFO - Gene gene-SPN23F_RS08005 has a mutation rate of 0.32666666666666666.\n",
      "2025-04-16 11:13:21,436 - INFO - Gene cds-WP_000162437.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,437 - INFO - Gene gene-SPN23F_RS08010 has a mutation rate of 0.06666666666666667.\n",
      "2025-04-16 11:13:21,437 - INFO - Gene cds-WP_000229856.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,438 - INFO - Gene gene-SPN23F_RS08015 has a mutation rate of 0.8166666666666667.\n",
      "2025-04-16 11:13:21,438 - INFO - Gene cds-WP_012386833.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,438 - INFO - Gene gene-SPN23F_RS08020 has a mutation rate of 0.15.\n",
      "2025-04-16 11:13:21,439 - INFO - Gene cds-WP_000087897.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,439 - INFO - Gene gene-SPN23F_RS08025 has a mutation rate of 0.6933333333333334.\n",
      "2025-04-16 11:13:21,440 - INFO - Gene cds-WP_000221614.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,440 - INFO - Gene gene-SPN23F_RS08030 has a mutation rate of 0.5066666666666667.\n",
      "2025-04-16 11:13:21,440 - INFO - Gene cds-WP_001122712.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,441 - INFO - Gene gene-SPN23F_RS08035 has a mutation rate of 0.12666666666666668.\n",
      "2025-04-16 11:13:21,441 - INFO - Gene cds-WP_001049323.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,442 - INFO - Gene gene-SPN23F_RS08040 has a mutation rate of 0.51.\n",
      "2025-04-16 11:13:21,442 - INFO - Gene cds-WP_001287230.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,443 - INFO - Gene gene-SPN23F_RS13295 has a mutation rate of 0.5966666666666667.\n",
      "2025-04-16 11:13:21,446 - INFO - Gene cds-WP_000403103.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,473 - INFO - Gene gene-SPN23F_RS08050 has a mutation rate of 0.22.\n",
      "2025-04-16 11:13:21,474 - INFO - Gene cds-WP_000229959.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,474 - INFO - Gene gene-SPN23F_RS08055 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,475 - INFO - Gene gene-SPN23F_RS08060 has a mutation rate of 0.16666666666666666.\n",
      "2025-04-16 11:13:21,475 - INFO - Gene cds-WP_000158722.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,483 - INFO - Gene gene-SPN23F_RS08065 has a mutation rate of 0.16333333333333333.\n",
      "2025-04-16 11:13:21,483 - INFO - Gene cds-WP_000940733.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,484 - INFO - Gene gene-SPN23F_RS08070 has a mutation rate of 0.38666666666666666.\n",
      "2025-04-16 11:13:21,484 - INFO - Gene cds-WP_000671107.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,485 - INFO - Gene gene-SPN23F_RS08075 has a mutation rate of 0.6466666666666666.\n",
      "2025-04-16 11:13:21,485 - INFO - Gene cds-WP_000841572.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,486 - INFO - Gene gene-SPN23F_RS08080 has a mutation rate of 0.65.\n",
      "2025-04-16 11:13:21,486 - INFO - Gene cds-WP_000958925.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,486 - INFO - Gene gene-SPN23F_RS08085 has a mutation rate of 0.5333333333333333.\n",
      "2025-04-16 11:13:21,487 - INFO - Gene cds-WP_001050251.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,487 - INFO - Gene gene-SPN23F_RS08090 has a mutation rate of 0.13.\n",
      "2025-04-16 11:13:21,488 - INFO - Gene cds-WP_000263202.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,488 - INFO - Gene gene-SPN23F_RS08095 has a mutation rate of 0.3566666666666667.\n",
      "2025-04-16 11:13:21,488 - INFO - Gene cds-WP_000040907.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,489 - INFO - Gene cds-SPN23F_RS14260 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,489 - INFO - Gene cds-SPN23F_RS13595 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,490 - INFO - Gene gene-SPN23F_RS08115 has a mutation rate of 0.26.\n",
      "2025-04-16 11:13:21,490 - INFO - Gene cds-WP_000814982.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,490 - INFO - Gene gene-SPN23F_RS08120 has a mutation rate of 0.22666666666666666.\n",
      "2025-04-16 11:13:21,491 - INFO - Gene cds-WP_000794775.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,491 - INFO - Gene gene-SPN23F_RS08125 has a mutation rate of 0.6233333333333333.\n",
      "2025-04-16 11:13:21,492 - INFO - Gene cds-WP_000199207.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,492 - INFO - Gene gene-SPN23F_RS08135 has a mutation rate of 0.23333333333333334.\n",
      "2025-04-16 11:13:21,493 - INFO - Gene cds-WP_000808711.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,493 - INFO - Gene gene-SPN23F_RS08140 has a mutation rate of 0.46.\n",
      "2025-04-16 11:13:21,493 - INFO - Gene cds-WP_000033951.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,494 - INFO - Gene gene-SPN23F_RS08145 has a mutation rate of 0.46.\n",
      "2025-04-16 11:13:21,494 - INFO - Gene cds-WP_001061594.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,494 - INFO - Gene gene-SPN23F_RS08150 has a mutation rate of 0.20333333333333334.\n",
      "2025-04-16 11:13:21,495 - INFO - Gene cds-WP_000849378.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,495 - INFO - Gene gene-SPN23F_RS08155 has a mutation rate of 0.08333333333333333.\n",
      "2025-04-16 11:13:21,495 - INFO - Gene cds-WP_001040912.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,495 - INFO - Gene gene-SPN23F_RS12485 has a mutation rate of 0.08666666666666667.\n",
      "2025-04-16 11:13:21,496 - INFO - Gene cds-WP_000705155.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,496 - INFO - Gene gene-SPN23F_RS08165 has a mutation rate of 0.7433333333333333.\n",
      "2025-04-16 11:13:21,497 - INFO - Gene cds-WP_001814295.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,497 - INFO - Gene gene-SPN23F_RS08170 has a mutation rate of 0.3433333333333333.\n",
      "2025-04-16 11:13:21,497 - INFO - Gene cds-WP_001156525.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,497 - INFO - Gene gene-SPN23F_RS14135 has a mutation rate of 0.17.\n",
      "2025-04-16 11:13:21,498 - INFO - Gene cds-WP_001812292.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,498 - INFO - Gene gene-SPN23F_RS08175 has a mutation rate of 0.9933333333333333.\n",
      "2025-04-16 11:13:21,499 - INFO - Gene cds-WP_000754652.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,500 - INFO - Gene gene-SPN23F_RS08180 has a mutation rate of 0.5466666666666666.\n",
      "2025-04-16 11:13:21,500 - INFO - Gene cds-WP_000881171.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,501 - INFO - Gene gene-SPN23F_RS08185 has a mutation rate of 0.5333333333333333.\n",
      "2025-04-16 11:13:21,501 - INFO - Gene cds-WP_000624092.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,502 - INFO - Gene gene-SPN23F_RS08190 has a mutation rate of 0.9.\n",
      "2025-04-16 11:13:21,502 - INFO - Gene cds-WP_000360036.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,503 - INFO - Gene gene-SPN23F_RS08195 has a mutation rate of 0.15.\n",
      "2025-04-16 11:13:21,503 - INFO - Gene cds-WP_000500194.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,503 - INFO - Gene gene-SPN23F_RS08200 has a mutation rate of 0.1.\n",
      "2025-04-16 11:13:21,504 - INFO - Gene cds-WP_000531723.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,504 - INFO - Gene gene-SPN23F_RS08205 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,505 - INFO - Gene cds-WP_001018251.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,505 - INFO - Gene gene-SPN23F_RS08210 has a mutation rate of 0.94.\n",
      "2025-04-16 11:13:21,505 - INFO - Gene cds-WP_000762099.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,506 - INFO - Gene gene-SPN23F_RS08215 has a mutation rate of 0.5733333333333334.\n",
      "2025-04-16 11:13:21,506 - INFO - Gene cds-WP_001814296.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,506 - INFO - Gene gene-SPN23F_RS08220 has a mutation rate of 0.74.\n",
      "2025-04-16 11:13:21,506 - INFO - Gene cds-WP_000422781.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,507 - INFO - Gene gene-SPN23F_RS08225 has a mutation rate of 0.43333333333333335.\n",
      "2025-04-16 11:13:21,507 - INFO - Gene cds-WP_000608357.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,507 - INFO - Gene gene-SPN23F_RS08230 has a mutation rate of 0.32.\n",
      "2025-04-16 11:13:21,508 - INFO - Gene cds-WP_000887954.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,508 - INFO - Gene gene-SPN23F_RS08235 has a mutation rate of 0.32666666666666666.\n",
      "2025-04-16 11:13:21,508 - INFO - Gene cds-WP_000548961.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,509 - INFO - Gene gene-SPN23F_RS08240 has a mutation rate of 0.7233333333333334.\n",
      "2025-04-16 11:13:21,509 - INFO - Gene cds-WP_001273817.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,510 - INFO - Gene gene-SPN23F_RS08245 has a mutation rate of 0.72.\n",
      "2025-04-16 11:13:21,510 - INFO - Gene cds-WP_000281608.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,511 - INFO - Gene gene-SPN23F_RS08250 has a mutation rate of 0.06333333333333334.\n",
      "2025-04-16 11:13:21,511 - INFO - Gene cds-WP_001166786.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,512 - INFO - Gene gene-SPN23F_RS08255 has a mutation rate of 0.75.\n",
      "2025-04-16 11:13:21,512 - INFO - Gene cds-WP_000160934.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,513 - INFO - Gene gene-SPN23F_RS08260 has a mutation rate of 0.63.\n",
      "2025-04-16 11:13:21,513 - INFO - Gene cds-WP_000188505.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,513 - INFO - Gene gene-SPN23F_RS08270 has a mutation rate of 0.6766666666666666.\n",
      "2025-04-16 11:13:21,514 - INFO - Gene cds-WP_001814297.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,514 - INFO - Gene gene-SPN23F_RS08275 has a mutation rate of 0.5133333333333333.\n",
      "2025-04-16 11:13:21,514 - INFO - Gene cds-WP_000737917.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,515 - INFO - Gene gene-SPN23F_RS08280 has a mutation rate of 0.08.\n",
      "2025-04-16 11:13:21,515 - INFO - Gene cds-WP_000720032.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,516 - INFO - Gene gene-SPN23F_RS08285 has a mutation rate of 0.38666666666666666.\n",
      "2025-04-16 11:13:21,516 - INFO - Gene cds-WP_000691400.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,517 - INFO - Gene gene-SPN23F_RS08290 has a mutation rate of 0.4633333333333333.\n",
      "2025-04-16 11:13:21,547 - INFO - Gene cds-WP_001123803.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,548 - INFO - Gene gene-SPN23F_RS08300 has a mutation rate of 0.3933333333333333.\n",
      "2025-04-16 11:13:21,548 - INFO - Gene cds-WP_000690854.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,549 - INFO - Gene gene-SPN23F_RS08305 has a mutation rate of 0.4866666666666667.\n",
      "2025-04-16 11:13:21,549 - INFO - Gene cds-WP_000199277.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,550 - INFO - Gene gene-SPN23F_RS08310 has a mutation rate of 0.06333333333333334.\n",
      "2025-04-16 11:13:21,551 - INFO - Gene cds-WP_000619143.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,554 - INFO - Gene gene-SPN23F_RS08315 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:21,554 - INFO - Gene cds-WP_000559784.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,555 - INFO - Gene gene-SPN23F_RS08320 has a mutation rate of 0.11666666666666667.\n",
      "2025-04-16 11:13:21,556 - INFO - Gene cds-WP_000733056.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,556 - INFO - Gene gene-SPN23F_RS08325 has a mutation rate of 0.42.\n",
      "2025-04-16 11:13:21,556 - INFO - Gene cds-WP_000256028.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,557 - INFO - Gene gene-SPN23F_RS08330 has a mutation rate of 0.9933333333333333.\n",
      "2025-04-16 11:13:21,557 - INFO - Gene cds-WP_000595843.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,558 - INFO - Gene gene-SPN23F_RS08335 has a mutation rate of 0.5733333333333334.\n",
      "2025-04-16 11:13:21,558 - INFO - Gene cds-WP_000323365.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,558 - INFO - Gene gene-SPN23F_RS08340 has a mutation rate of 0.9733333333333334.\n",
      "2025-04-16 11:13:21,559 - INFO - Gene cds-WP_000620757.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,559 - INFO - Gene gene-SPN23F_RS08345 has a mutation rate of 0.023333333333333334.\n",
      "2025-04-16 11:13:21,560 - INFO - Gene cds-WP_000240132.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,560 - INFO - Gene gene-SPN23F_RS13640 has a mutation rate of 0.8.\n",
      "2025-04-16 11:13:21,560 - INFO - Gene cds-WP_000572965.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,561 - INFO - Gene gene-SPN23F_RS08360 has a mutation rate of 0.7533333333333333.\n",
      "2025-04-16 11:13:21,561 - INFO - Gene cds-WP_000769849.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,561 - INFO - Gene gene-SPN23F_RS08365 has a mutation rate of 0.79.\n",
      "2025-04-16 11:13:21,562 - INFO - Gene cds-WP_000768038.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,562 - INFO - Gene gene-SPN23F_RS08380 has a mutation rate of 0.47333333333333333.\n",
      "2025-04-16 11:13:21,563 - INFO - Gene cds-WP_001810114.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,563 - INFO - Gene gene-SPN23F_RS08385 has a mutation rate of 0.57.\n",
      "2025-04-16 11:13:21,563 - INFO - Gene cds-WP_001032415.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,564 - INFO - Gene gene-SPN23F_RS08390 has a mutation rate of 0.03666666666666667.\n",
      "2025-04-16 11:13:21,564 - INFO - Gene cds-WP_000576513.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,564 - INFO - Gene gene-SPN23F_RS08395 has a mutation rate of 0.5333333333333333.\n",
      "2025-04-16 11:13:21,565 - INFO - Gene cds-WP_000053385.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,565 - INFO - Gene gene-SPN23F_RS08400 has a mutation rate of 0.42.\n",
      "2025-04-16 11:13:21,565 - INFO - Gene cds-WP_001101082.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,566 - INFO - Gene gene-SPN23F_RS08405 has a mutation rate of 0.3566666666666667.\n",
      "2025-04-16 11:13:21,566 - INFO - Gene cds-WP_000144280.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,566 - INFO - Gene gene-SPN23F_RS08410 has a mutation rate of 0.29.\n",
      "2025-04-16 11:13:21,566 - INFO - Gene cds-WP_000090397.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,567 - INFO - Gene gene-SPN23F_RS08415 has a mutation rate of 0.2833333333333333.\n",
      "2025-04-16 11:13:21,567 - INFO - Gene cds-WP_001055251.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,568 - INFO - Gene gene-SPN23F_RS08420 has a mutation rate of 0.8766666666666667.\n",
      "2025-04-16 11:13:21,568 - INFO - Gene cds-WP_000994194.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,568 - INFO - Gene gene-SPN23F_RS08425 has a mutation rate of 0.9333333333333333.\n",
      "2025-04-16 11:13:21,568 - INFO - Gene cds-WP_000778689.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,569 - INFO - Gene gene-SPN23F_RS08430 has a mutation rate of 0.64.\n",
      "2025-04-16 11:13:21,569 - INFO - Gene cds-WP_000814694.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,569 - INFO - Gene gene-SPN23F_RS08435 has a mutation rate of 0.21666666666666667.\n",
      "2025-04-16 11:13:21,570 - INFO - Gene cds-WP_000966746.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,570 - INFO - Gene gene-SPN23F_RS08440 has a mutation rate of 0.7566666666666667.\n",
      "2025-04-16 11:13:21,570 - INFO - Gene cds-WP_078134415.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,570 - INFO - Gene gene-SPN23F_RS08445 has a mutation rate of 0.24666666666666667.\n",
      "2025-04-16 11:13:21,571 - INFO - Gene cds-WP_000360349.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,571 - INFO - Gene gene-SPN23F_RS08450 has a mutation rate of 0.26666666666666666.\n",
      "2025-04-16 11:13:21,571 - INFO - Gene cds-WP_000222979.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,572 - INFO - Gene gene-SPN23F_RS08455 has a mutation rate of 0.5633333333333334.\n",
      "2025-04-16 11:13:21,572 - INFO - Gene cds-WP_001281530.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,572 - INFO - Gene gene-SPN23F_RS08460 has a mutation rate of 0.6033333333333334.\n",
      "2025-04-16 11:13:21,572 - INFO - Gene cds-WP_000057490.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,573 - INFO - Gene gene-SPN23F_RS13955 has a mutation rate of 0.9466666666666667.\n",
      "2025-04-16 11:13:21,573 - INFO - Gene cds-WP_001814705.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,573 - INFO - Gene gene-SPN23F_RS08470 has a mutation rate of 0.9466666666666667.\n",
      "2025-04-16 11:13:21,573 - INFO - Gene cds-WP_000254914.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,574 - INFO - Gene gene-SPN23F_RS08475 has a mutation rate of 0.6266666666666667.\n",
      "2025-04-16 11:13:21,574 - INFO - Gene cds-WP_000575466.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,574 - INFO - Gene gene-SPN23F_RS08480 has a mutation rate of 0.016666666666666666.\n",
      "2025-04-16 11:13:21,574 - INFO - Gene cds-WP_001192007.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,575 - INFO - Gene gene-SPN23F_RS08485 has a mutation rate of 0.016666666666666666.\n",
      "2025-04-16 11:13:21,575 - INFO - Gene cds-WP_161802579.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,575 - INFO - Gene gene-SPN23F_RS08490 has a mutation rate of 0.06666666666666667.\n",
      "2025-04-16 11:13:21,576 - INFO - Gene cds-WP_000672136.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,576 - INFO - Gene gene-SPN23F_RS08495 has a mutation rate of 0.49666666666666665.\n",
      "2025-04-16 11:13:21,576 - INFO - Gene cds-WP_000477439.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,577 - INFO - Gene gene-SPN23F_RS08500 has a mutation rate of 0.023333333333333334.\n",
      "2025-04-16 11:13:21,577 - INFO - Gene cds-WP_001135651.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,577 - INFO - Gene gene-SPN23F_RS08505 has a mutation rate of 0.19333333333333333.\n",
      "2025-04-16 11:13:21,578 - INFO - Gene cds-WP_000241719.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,578 - INFO - Gene gene-SPN23F_RS08510 has a mutation rate of 0.5933333333333334.\n",
      "2025-04-16 11:13:21,578 - INFO - Gene cds-WP_001042145.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,579 - INFO - Gene gene-SPN23F_RS08515 has a mutation rate of 0.35333333333333333.\n",
      "2025-04-16 11:13:21,579 - INFO - Gene cds-WP_000730470.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,580 - INFO - Gene gene-SPN23F_RS08520 has a mutation rate of 0.12333333333333334.\n",
      "2025-04-16 11:13:21,580 - INFO - Gene cds-WP_000237989.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,581 - INFO - Gene gene-SPN23F_RS08525 has a mutation rate of 0.12333333333333334.\n",
      "2025-04-16 11:13:21,581 - INFO - Gene cds-WP_000738769.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,581 - INFO - Gene gene-SPN23F_RS08530 has a mutation rate of 0.43666666666666665.\n",
      "2025-04-16 11:13:21,582 - INFO - Gene cds-WP_000587470.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,612 - INFO - Gene gene-SPN23F_RS08535 has a mutation rate of 0.9666666666666667.\n",
      "2025-04-16 11:13:21,612 - INFO - Gene cds-WP_012636588.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,613 - INFO - Gene gene-SPN23F_RS08540 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,613 - INFO - Gene cds-WP_000576349.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,613 - INFO - Gene gene-SPN23F_RS13300 has a mutation rate of 0.2833333333333333.\n",
      "2025-04-16 11:13:21,614 - INFO - Gene cds-WP_000361700.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,614 - INFO - Gene gene-SPN23F_RS08550 has a mutation rate of 0.72.\n",
      "2025-04-16 11:13:21,615 - INFO - Gene cds-WP_000795116.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,615 - INFO - Gene gene-SPN23F_RS08555 has a mutation rate of 0.8433333333333334.\n",
      "2025-04-16 11:13:21,619 - INFO - Gene cds-WP_001048748.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,621 - INFO - Gene gene-SPN23F_RS08560 has a mutation rate of 0.4533333333333333.\n",
      "2025-04-16 11:13:21,622 - INFO - Gene cds-WP_000648075.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,622 - INFO - Gene gene-SPN23F_RS08565 has a mutation rate of 0.09.\n",
      "2025-04-16 11:13:21,623 - INFO - Gene cds-WP_000635008.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,623 - INFO - Gene gene-SPN23F_RS08570 has a mutation rate of 0.16666666666666666.\n",
      "2025-04-16 11:13:21,623 - INFO - Gene cds-WP_000864060.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,624 - INFO - Gene gene-SPN23F_RS08575 has a mutation rate of 0.11333333333333333.\n",
      "2025-04-16 11:13:21,624 - INFO - Gene cds-WP_000232209.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,624 - INFO - Gene gene-SPN23F_RS08580 has a mutation rate of 0.22333333333333333.\n",
      "2025-04-16 11:13:21,625 - INFO - Gene cds-WP_001274091.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,625 - INFO - Gene gene-SPN23F_RS08585 has a mutation rate of 0.4766666666666667.\n",
      "2025-04-16 11:13:21,626 - INFO - Gene cds-WP_000940707.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,626 - INFO - Gene gene-SPN23F_RS08590 has a mutation rate of 0.4166666666666667.\n",
      "2025-04-16 11:13:21,626 - INFO - Gene cds-WP_001235287.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,627 - INFO - Gene gene-SPN23F_RS08595 has a mutation rate of 0.7233333333333334.\n",
      "2025-04-16 11:13:21,627 - INFO - Gene cds-WP_000475771.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,628 - INFO - Gene gene-SPN23F_RS08600 has a mutation rate of 0.5666666666666667.\n",
      "2025-04-16 11:13:21,628 - INFO - Gene cds-WP_000701265.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,628 - INFO - Gene gene-SPN23F_RS08605 has a mutation rate of 0.023333333333333334.\n",
      "2025-04-16 11:13:21,628 - INFO - Gene cds-WP_000694288.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,629 - INFO - Gene gene-SPN23F_RS08610 has a mutation rate of 0.1.\n",
      "2025-04-16 11:13:21,629 - INFO - Gene cds-WP_001267153.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,629 - INFO - Gene gene-SPN23F_RS08615 has a mutation rate of 0.5966666666666667.\n",
      "2025-04-16 11:13:21,629 - INFO - Gene cds-WP_000400504.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,630 - INFO - Gene gene-SPN23F_RS08620 has a mutation rate of 0.13.\n",
      "2025-04-16 11:13:21,630 - INFO - Gene cds-WP_000400493.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,630 - INFO - Gene gene-SPN23F_RS08625 has a mutation rate of 0.11333333333333333.\n",
      "2025-04-16 11:13:21,631 - INFO - Gene cds-WP_001207696.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,631 - INFO - Gene gene-SPN23F_RS08630 has a mutation rate of 0.31666666666666665.\n",
      "2025-04-16 11:13:21,631 - INFO - Gene cds-WP_000140849.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,632 - INFO - Gene gene-SPN23F_RS08635 has a mutation rate of 0.5766666666666667.\n",
      "2025-04-16 11:13:21,632 - INFO - Gene cds-WP_000446527.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,632 - INFO - Gene gene-SPN23F_RS08640 has a mutation rate of 0.8866666666666667.\n",
      "2025-04-16 11:13:21,633 - INFO - Gene cds-WP_000802391.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,633 - INFO - Gene gene-SPN23F_RS08645 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,633 - INFO - Gene cds-WP_001203672.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,634 - INFO - Gene gene-SPN23F_RS08650 has a mutation rate of 0.04666666666666667.\n",
      "2025-04-16 11:13:21,634 - INFO - Gene cds-WP_000119153.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,635 - INFO - Gene gene-SPN23F_RS08655 has a mutation rate of 0.06.\n",
      "2025-04-16 11:13:21,635 - INFO - Gene cds-WP_000055451.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,635 - INFO - Gene gene-SPN23F_RS08660 has a mutation rate of 0.16.\n",
      "2025-04-16 11:13:21,636 - INFO - Gene cds-WP_000498761.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,636 - INFO - Gene gene-SPN23F_RS08665 has a mutation rate of 0.5833333333333334.\n",
      "2025-04-16 11:13:21,636 - INFO - Gene cds-WP_001245352.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,637 - INFO - Gene gene-SPN23F_RS08670 has a mutation rate of 0.37.\n",
      "2025-04-16 11:13:21,637 - INFO - Gene cds-WP_000895287.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,637 - INFO - Gene gene-SPN23F_RS13960 has a mutation rate of 0.54.\n",
      "2025-04-16 11:13:21,638 - INFO - Gene cds-WP_000693430.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,638 - INFO - Gene cds-SPN23F_RS13060 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,639 - INFO - Gene gene-SPN23F_RS08685 has a mutation rate of 0.08666666666666667.\n",
      "2025-04-16 11:13:21,639 - INFO - Gene cds-WP_000076185.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,639 - INFO - Gene gene-SPN23F_RS08690 has a mutation rate of 0.49.\n",
      "2025-04-16 11:13:21,640 - INFO - Gene cds-WP_000164313.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,640 - INFO - Gene gene-SPN23F_RS08695 has a mutation rate of 0.4166666666666667.\n",
      "2025-04-16 11:13:21,706 - INFO - Gene cds-WP_001064552.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,707 - INFO - Gene gene-SPN23F_RS08705 has a mutation rate of 0.6233333333333333.\n",
      "2025-04-16 11:13:21,707 - INFO - Gene cds-WP_000455313.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,707 - INFO - Gene gene-SPN23F_RS08710 has a mutation rate of 0.09.\n",
      "2025-04-16 11:13:21,708 - INFO - Gene cds-WP_000225980.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,708 - INFO - Gene gene-SPN23F_RS08715 has a mutation rate of 0.44.\n",
      "2025-04-16 11:13:21,709 - INFO - Gene cds-WP_000704388.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,709 - INFO - Gene gene-SPN23F_RS08720 has a mutation rate of 0.56.\n",
      "2025-04-16 11:13:21,710 - INFO - Gene cds-WP_000151957.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,710 - INFO - Gene gene-SPN23F_RS08725 has a mutation rate of 0.5133333333333333.\n",
      "2025-04-16 11:13:21,710 - INFO - Gene cds-WP_001814299.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,711 - INFO - Gene cds-SPN23F_RS12525 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,711 - INFO - Gene gene-SPN23F_RS08740 has a mutation rate of 0.04.\n",
      "2025-04-16 11:13:21,711 - INFO - Gene cds-WP_000176924.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,712 - INFO - Gene gene-SPN23F_RS08745 has a mutation rate of 0.07.\n",
      "2025-04-16 11:13:21,712 - INFO - Gene cds-WP_000392877.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,712 - INFO - Gene gene-SPN23F_RS08750 has a mutation rate of 0.5533333333333333.\n",
      "2025-04-16 11:13:21,713 - INFO - Gene cds-WP_000614540.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,713 - INFO - Gene gene-SPN23F_RS08755 has a mutation rate of 0.24666666666666667.\n",
      "2025-04-16 11:13:21,713 - INFO - Gene cds-WP_000406248.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,714 - INFO - Gene gene-SPN23F_RS08760 has a mutation rate of 0.72.\n",
      "2025-04-16 11:13:21,714 - INFO - Gene cds-WP_001812475.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,714 - INFO - Gene gene-SPN23F_RS08765 has a mutation rate of 0.5133333333333333.\n",
      "2025-04-16 11:13:21,715 - INFO - Gene cds-WP_000163704.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,715 - INFO - Gene gene-SPN23F_RS08770 has a mutation rate of 0.8533333333333334.\n",
      "2025-04-16 11:13:21,715 - INFO - Gene cds-WP_001152092.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,716 - INFO - Gene gene-SPN23F_RS08775 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:21,716 - INFO - Gene cds-WP_000979235.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,717 - INFO - Gene gene-SPN23F_RS08780 has a mutation rate of 0.15.\n",
      "2025-04-16 11:13:21,717 - INFO - Gene cds-WP_000775037.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,718 - INFO - Gene gene-SPN23F_RS08785 has a mutation rate of 0.10666666666666667.\n",
      "2025-04-16 11:13:21,718 - INFO - Gene cds-WP_000404940.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,718 - INFO - Gene cds-SPN23F_RS08790 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,719 - INFO - Gene gene-SPN23F_RS08795 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,719 - INFO - Gene cds-WP_000669600.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,720 - INFO - Gene gene-SPN23F_RS13600 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,720 - INFO - Gene cds-WP_000899898.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,720 - INFO - Gene cds-SPN23F_RS14225 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,721 - INFO - Gene cds-SPN23F_RS14265 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,721 - INFO - Gene cds-SPN23F_RS12540 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,722 - INFO - Gene gene-SPN23F_RS08820 has a mutation rate of 0.99.\n",
      "2025-04-16 11:13:21,722 - INFO - Gene cds-WP_000156346.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,722 - INFO - Gene gene-SPN23F_RS08825 has a mutation rate of 0.31666666666666665.\n",
      "2025-04-16 11:13:21,723 - INFO - Gene cds-WP_000257066.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,723 - INFO - Gene gene-SPN23F_RS08830 has a mutation rate of 0.06.\n",
      "2025-04-16 11:13:21,724 - INFO - Gene cds-WP_001003001.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,724 - INFO - Gene gene-SPN23F_RS08835 has a mutation rate of 0.68.\n",
      "2025-04-16 11:13:21,725 - INFO - Gene cds-WP_000844707.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,725 - INFO - Gene gene-SPN23F_RS08840 has a mutation rate of 0.8466666666666667.\n",
      "2025-04-16 11:13:21,726 - INFO - Gene cds-WP_000121651.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,726 - INFO - Gene gene-SPN23F_RS08845 has a mutation rate of 0.43333333333333335.\n",
      "2025-04-16 11:13:21,727 - INFO - Gene cds-WP_000151313.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,727 - INFO - Gene gene-SPN23F_RS08850 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,727 - INFO - Gene cds-WP_000060169.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,728 - INFO - Gene gene-SPN23F_RS08855 has a mutation rate of 0.5033333333333333.\n",
      "2025-04-16 11:13:21,728 - INFO - Gene cds-WP_000391203.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,729 - INFO - Gene gene-SPN23F_RS08860 has a mutation rate of 0.12.\n",
      "2025-04-16 11:13:21,730 - INFO - Gene cds-WP_000963728.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,730 - INFO - Gene gene-SPN23F_RS08865 has a mutation rate of 0.36666666666666664.\n",
      "2025-04-16 11:13:21,730 - INFO - Gene cds-WP_000242240.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,731 - INFO - Gene gene-SPN23F_RS08870 has a mutation rate of 0.16666666666666666.\n",
      "2025-04-16 11:13:21,731 - INFO - Gene cds-WP_001168885.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,732 - INFO - Gene cds-SPN23F_RS08875 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,732 - INFO - Gene gene-SPN23F_RS08880 has a mutation rate of 0.36333333333333334.\n",
      "2025-04-16 11:13:21,733 - INFO - Gene cds-WP_000028165.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,733 - INFO - Gene gene-SPN23F_RS13970 has a mutation rate of 0.12666666666666668.\n",
      "2025-04-16 11:13:21,733 - INFO - Gene cds-WP_000693219.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,734 - INFO - Gene gene-SPN23F_RS08890 has a mutation rate of 0.24.\n",
      "2025-04-16 11:13:21,734 - INFO - Gene cds-WP_000569970.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,735 - INFO - Gene gene-SPN23F_RS08895 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,735 - INFO - Gene cds-WP_001287962.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,735 - INFO - Gene gene-SPN23F_RS08900 has a mutation rate of 0.21.\n",
      "2025-04-16 11:13:21,736 - INFO - Gene cds-WP_000571755.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,736 - INFO - Gene gene-SPN23F_RS08905 has a mutation rate of 0.07.\n",
      "2025-04-16 11:13:21,762 - INFO - Gene cds-WP_000158459.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,766 - INFO - Gene gene-SPN23F_RS08910 has a mutation rate of 0.28.\n",
      "2025-04-16 11:13:21,766 - INFO - Gene cds-WP_000489783.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,767 - INFO - Gene gene-SPN23F_RS08915 has a mutation rate of 0.17666666666666667.\n",
      "2025-04-16 11:13:21,768 - INFO - Gene cds-WP_000587538.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,768 - INFO - Gene gene-SPN23F_RS08920 has a mutation rate of 0.12333333333333334.\n",
      "2025-04-16 11:13:21,768 - INFO - Gene cds-WP_000032115.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,769 - INFO - Gene gene-SPN23F_RS08925 has a mutation rate of 0.17666666666666667.\n",
      "2025-04-16 11:13:21,772 - INFO - Gene cds-WP_000291153.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,775 - INFO - Gene gene-SPN23F_RS08930 has a mutation rate of 0.2.\n",
      "2025-04-16 11:13:21,776 - INFO - Gene cds-WP_000161879.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,777 - INFO - Gene gene-SPN23F_RS08935 has a mutation rate of 0.2.\n",
      "2025-04-16 11:13:21,777 - INFO - Gene cds-WP_001292322.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,777 - INFO - Gene gene-SPN23F_RS08940 has a mutation rate of 0.18.\n",
      "2025-04-16 11:13:21,778 - INFO - Gene cds-WP_001232158.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,778 - INFO - Gene gene-SPN23F_RS08945 has a mutation rate of 0.043333333333333335.\n",
      "2025-04-16 11:13:21,778 - INFO - Gene cds-WP_000938701.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,779 - INFO - Gene gene-SPN23F_RS08950 has a mutation rate of 0.11333333333333333.\n",
      "2025-04-16 11:13:21,780 - INFO - Gene cds-WP_000653913.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,780 - INFO - Gene gene-SPN23F_RS08955 has a mutation rate of 0.22333333333333333.\n",
      "2025-04-16 11:13:21,780 - INFO - Gene cds-WP_000598035.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,781 - INFO - Gene cds-SPN23F_RS08960 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,781 - INFO - Gene cds-SPN23F_RS08965 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,781 - INFO - Gene gene-SPN23F_RS08975 has a mutation rate of 0.09.\n",
      "2025-04-16 11:13:21,782 - INFO - Gene cds-WP_001093470.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,782 - INFO - Gene gene-SPN23F_RS08980 has a mutation rate of 0.21333333333333335.\n",
      "2025-04-16 11:13:21,782 - INFO - Gene cds-WP_000341102.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,783 - INFO - Gene gene-SPN23F_RS12555 has a mutation rate of 0.26.\n",
      "2025-04-16 11:13:21,783 - INFO - Gene cds-WP_079199710.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,784 - INFO - Gene gene-SPN23F_RS08995 has a mutation rate of 0.15.\n",
      "2025-04-16 11:13:21,784 - INFO - Gene cds-WP_000514203.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,784 - INFO - Gene gene-SPN23F_RS14270 has a mutation rate of 0.24666666666666667.\n",
      "2025-04-16 11:13:21,785 - INFO - Gene cds-WP_001861345.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,785 - INFO - Gene gene-SPN23F_RS09000 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:21,786 - INFO - Gene cds-WP_000221627.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,786 - INFO - Gene gene-SPN23F_RS09005 has a mutation rate of 0.06666666666666667.\n",
      "2025-04-16 11:13:21,787 - INFO - Gene cds-WP_000570244.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,787 - INFO - Gene gene-SPN23F_RS09010 has a mutation rate of 0.08333333333333333.\n",
      "2025-04-16 11:13:21,788 - INFO - Gene cds-WP_001029580.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,788 - INFO - Gene gene-SPN23F_RS09015 has a mutation rate of 0.72.\n",
      "2025-04-16 11:13:21,789 - INFO - Gene cds-WP_000828755.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,789 - INFO - Gene gene-SPN23F_RS13435 has a mutation rate of 0.6266666666666667.\n",
      "2025-04-16 11:13:21,790 - INFO - Gene cds-WP_000464401.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,790 - INFO - Gene gene-SPN23F_RS09025 has a mutation rate of 0.47333333333333333.\n",
      "2025-04-16 11:13:21,791 - INFO - Gene cds-WP_000714837.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,791 - INFO - Gene gene-SPN23F_RS09030 has a mutation rate of 0.9866666666666667.\n",
      "2025-04-16 11:13:21,791 - INFO - Gene cds-WP_000284095.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,791 - INFO - Gene gene-SPN23F_RS13440 has a mutation rate of 0.056666666666666664.\n",
      "2025-04-16 11:13:21,792 - INFO - Gene cds-WP_000523770.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,792 - INFO - Gene gene-SPN23F_RS09040 has a mutation rate of 0.5466666666666666.\n",
      "2025-04-16 11:13:21,793 - INFO - Gene cds-WP_000435550.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,794 - INFO - Gene gene-SPN23F_RS09045 has a mutation rate of 0.8966666666666666.\n",
      "2025-04-16 11:13:21,794 - INFO - Gene cds-WP_001188182.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,794 - INFO - Gene gene-SPN23F_RS09050 has a mutation rate of 0.98.\n",
      "2025-04-16 11:13:21,795 - INFO - Gene cds-WP_000451122.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,795 - INFO - Gene gene-SPN23F_RS09055 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,796 - INFO - Gene cds-WP_001028058.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,796 - INFO - Gene gene-SPN23F_RS09060 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,796 - INFO - Gene cds-WP_000529017.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,797 - INFO - Gene gene-SPN23F_RS09065 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,798 - INFO - Gene cds-WP_000747915.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,798 - INFO - Gene cds-SPN23F_RS09070 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,799 - INFO - Gene gene-SPN23F_RS09075 has a mutation rate of 0.36666666666666664.\n",
      "2025-04-16 11:13:21,799 - INFO - Gene cds-WP_000719700.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,800 - INFO - Gene gene-SPN23F_RS09080 has a mutation rate of 0.7133333333333334.\n",
      "2025-04-16 11:13:21,800 - INFO - Gene cds-WP_000257084.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,801 - INFO - Gene gene-SPN23F_RS09085 has a mutation rate of 0.14333333333333334.\n",
      "2025-04-16 11:13:21,801 - INFO - Gene cds-WP_000961810.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,803 - INFO - Gene gene-SPN23F_RS09090 has a mutation rate of 0.5766666666666667.\n",
      "2025-04-16 11:13:21,809 - INFO - Gene cds-WP_001132287.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,827 - INFO - Gene gene-SPN23F_RS09095 has a mutation rate of 0.04.\n",
      "2025-04-16 11:13:21,832 - INFO - Gene cds-WP_001048906.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,833 - INFO - Gene gene-SPN23F_RS09100 has a mutation rate of 0.13666666666666666.\n",
      "2025-04-16 11:13:21,833 - INFO - Gene cds-WP_000818080.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,834 - INFO - Gene gene-SPN23F_RS09105 has a mutation rate of 0.57.\n",
      "2025-04-16 11:13:21,835 - INFO - Gene cds-WP_001113205.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,841 - INFO - Gene gene-SPN23F_RS09110 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,842 - INFO - Gene gene-SPN23F_RS09115 has a mutation rate of 0.9866666666666667.\n",
      "2025-04-16 11:13:21,843 - INFO - Gene cds-WP_000502535.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,844 - INFO - Gene gene-SPN23F_RS09120 has a mutation rate of 0.45666666666666667.\n",
      "2025-04-16 11:13:21,845 - INFO - Gene cds-WP_000669751.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,845 - INFO - Gene gene-SPN23F_RS09125 has a mutation rate of 0.07666666666666666.\n",
      "2025-04-16 11:13:21,846 - INFO - Gene cds-WP_000797961.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,846 - INFO - Gene gene-SPN23F_RS09130 has a mutation rate of 0.16.\n",
      "2025-04-16 11:13:21,847 - INFO - Gene cds-WP_001084979.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,847 - INFO - Gene gene-SPN23F_RS09135 has a mutation rate of 0.30333333333333334.\n",
      "2025-04-16 11:13:21,847 - INFO - Gene cds-WP_000230202.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,848 - INFO - Gene gene-SPN23F_RS09140 has a mutation rate of 0.28.\n",
      "2025-04-16 11:13:21,849 - INFO - Gene cds-WP_001205280.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,849 - INFO - Gene gene-SPN23F_RS09145 has a mutation rate of 0.3566666666666667.\n",
      "2025-04-16 11:13:21,850 - INFO - Gene cds-WP_001809002.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,851 - INFO - Gene gene-SPN23F_RS09150 has a mutation rate of 0.13666666666666666.\n",
      "2025-04-16 11:13:21,851 - INFO - Gene cds-WP_000030213.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,851 - INFO - Gene gene-SPN23F_RS09155 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,852 - INFO - Gene cds-WP_000455066.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,852 - INFO - Gene gene-SPN23F_RS09160 has a mutation rate of 0.07333333333333333.\n",
      "2025-04-16 11:13:21,853 - INFO - Gene cds-WP_000064115.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,853 - INFO - Gene gene-SPN23F_RS09165 has a mutation rate of 0.01.\n",
      "2025-04-16 11:13:21,853 - INFO - Gene cds-WP_000109957.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,854 - INFO - Gene gene-SPN23F_RS09170 has a mutation rate of 0.45666666666666667.\n",
      "2025-04-16 11:13:21,854 - INFO - Gene cds-WP_307777729.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,855 - INFO - Gene gene-SPN23F_RS12575 has a mutation rate of 0.32.\n",
      "2025-04-16 11:13:21,855 - INFO - Gene cds-WP_048814460.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,856 - INFO - Gene gene-SPN23F_RS09180 has a mutation rate of 0.48.\n",
      "2025-04-16 11:13:21,857 - INFO - Gene cds-WP_000229339.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,857 - INFO - Gene gene-SPN23F_RS09185 has a mutation rate of 0.5266666666666666.\n",
      "2025-04-16 11:13:21,858 - INFO - Gene cds-WP_000782429.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,858 - INFO - Gene gene-SPN23F_RS09190 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,859 - INFO - Gene cds-WP_000250870.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,860 - INFO - Gene gene-SPN23F_RS09195 has a mutation rate of 0.37.\n",
      "2025-04-16 11:13:21,860 - INFO - Gene cds-WP_001094994.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,860 - INFO - Gene gene-SPN23F_RS09200 has a mutation rate of 0.79.\n",
      "2025-04-16 11:13:21,861 - INFO - Gene cds-WP_001005881.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,861 - INFO - Gene gene-SPN23F_RS09205 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,862 - INFO - Gene cds-WP_001151646.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,863 - INFO - Gene gene-SPN23F_RS09210 has a mutation rate of 0.6233333333333333.\n",
      "2025-04-16 11:13:21,863 - INFO - Gene cds-WP_000825665.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,864 - INFO - Gene gene-SPN23F_RS09215 has a mutation rate of 0.6433333333333333.\n",
      "2025-04-16 11:13:21,864 - INFO - Gene cds-WP_000413347.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,865 - INFO - Gene gene-SPN23F_RS09220 has a mutation rate of 0.23666666666666666.\n",
      "2025-04-16 11:13:21,865 - INFO - Gene cds-WP_000175876.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,866 - INFO - Gene gene-SPN23F_RS09225 has a mutation rate of 0.6566666666666666.\n",
      "2025-04-16 11:13:21,866 - INFO - Gene cds-WP_000155434.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,867 - INFO - Gene gene-SPN23F_RS09230 has a mutation rate of 0.8266666666666667.\n",
      "2025-04-16 11:13:21,867 - INFO - Gene cds-WP_000565006.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,868 - INFO - Gene gene-SPN23F_RS09235 has a mutation rate of 0.23.\n",
      "2025-04-16 11:13:21,869 - INFO - Gene cds-WP_000481756.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,869 - INFO - Gene gene-SPN23F_RS09240 has a mutation rate of 0.23666666666666666.\n",
      "2025-04-16 11:13:21,870 - INFO - Gene cds-WP_000934017.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,870 - INFO - Gene cds-SPN23F_RS12580 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,871 - INFO - Gene gene-SPN23F_RS09245 has a mutation rate of 0.9066666666666666.\n",
      "2025-04-16 11:13:21,871 - INFO - Gene cds-WP_001126993.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,871 - INFO - Gene gene-SPN23F_RS09250 has a mutation rate of 0.41333333333333333.\n",
      "2025-04-16 11:13:21,872 - INFO - Gene cds-WP_000331298.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,873 - INFO - Gene gene-SPN23F_RS09255 has a mutation rate of 0.9133333333333333.\n",
      "2025-04-16 11:13:21,873 - INFO - Gene cds-WP_000169888.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,874 - INFO - Gene gene-SPN23F_RS09260 has a mutation rate of 0.53.\n",
      "2025-04-16 11:13:21,874 - INFO - Gene cds-WP_000076548.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,894 - INFO - Gene gene-SPN23F_RS09265 has a mutation rate of 0.2866666666666667.\n",
      "2025-04-16 11:13:21,904 - INFO - Gene cds-WP_000658682.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,905 - INFO - Gene gene-SPN23F_RS09270 has a mutation rate of 0.69.\n",
      "2025-04-16 11:13:21,906 - INFO - Gene cds-WP_000601919.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,906 - INFO - Gene gene-SPN23F_RS09275 has a mutation rate of 0.9733333333333334.\n",
      "2025-04-16 11:13:21,913 - INFO - Gene cds-WP_000439654.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,913 - INFO - Gene gene-SPN23F_RS13645 has a mutation rate of 0.13333333333333333.\n",
      "2025-04-16 11:13:21,914 - INFO - Gene cds-WP_001814313.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,914 - INFO - Gene gene-SPN23F_RS09285 has a mutation rate of 0.07666666666666666.\n",
      "2025-04-16 11:13:21,915 - INFO - Gene cds-WP_000789949.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,916 - INFO - Gene gene-SPN23F_RS09290 has a mutation rate of 0.39.\n",
      "2025-04-16 11:13:21,917 - INFO - Gene cds-WP_000656286.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,918 - INFO - Gene gene-SPN23F_RS09295 has a mutation rate of 0.07.\n",
      "2025-04-16 11:13:21,919 - INFO - Gene cds-WP_000127552.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,919 - INFO - Gene gene-SPN23F_RS09300 has a mutation rate of 0.08.\n",
      "2025-04-16 11:13:21,920 - INFO - Gene cds-WP_001216179.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,920 - INFO - Gene gene-SPN23F_RS09305 has a mutation rate of 0.30666666666666664.\n",
      "2025-04-16 11:13:21,921 - INFO - Gene cds-WP_000589142.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,922 - INFO - Gene gene-SPN23F_RS09310 has a mutation rate of 0.4866666666666667.\n",
      "2025-04-16 11:13:21,923 - INFO - Gene cds-WP_000738372.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,923 - INFO - Gene gene-SPN23F_RS09315 has a mutation rate of 0.3333333333333333.\n",
      "2025-04-16 11:13:21,924 - INFO - Gene cds-WP_000793807.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,925 - INFO - Gene gene-SPN23F_RS09320 has a mutation rate of 0.59.\n",
      "2025-04-16 11:13:21,925 - INFO - Gene cds-WP_000996558.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,926 - INFO - Gene gene-SPN23F_RS09325 has a mutation rate of 0.48333333333333334.\n",
      "2025-04-16 11:13:21,926 - INFO - Gene cds-WP_000848802.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,927 - INFO - Gene gene-SPN23F_RS09330 has a mutation rate of 0.48333333333333334.\n",
      "2025-04-16 11:13:21,927 - INFO - Gene cds-WP_000945327.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,928 - INFO - Gene gene-SPN23F_RS13975 has a mutation rate of 0.4666666666666667.\n",
      "2025-04-16 11:13:21,929 - INFO - Gene cds-WP_001813172.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,930 - INFO - Gene gene-SPN23F_RS09340 has a mutation rate of 0.47.\n",
      "2025-04-16 11:13:21,931 - INFO - Gene cds-WP_000239022.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,931 - INFO - Gene gene-SPN23F_RS09345 has a mutation rate of 0.6666666666666666.\n",
      "2025-04-16 11:13:21,932 - INFO - Gene cds-WP_010976622.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,934 - INFO - Gene gene-SPN23F_RS09350 has a mutation rate of 0.62.\n",
      "2025-04-16 11:13:21,934 - INFO - Gene cds-WP_001280260.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,934 - INFO - Gene gene-SPN23F_RS09360 has a mutation rate of 0.49.\n",
      "2025-04-16 11:13:21,936 - INFO - Gene cds-WP_001134492.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,937 - INFO - Gene gene-SPN23F_RS09365 has a mutation rate of 0.5166666666666667.\n",
      "2025-04-16 11:13:21,937 - INFO - Gene cds-WP_000922204.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,938 - INFO - Gene gene-SPN23F_RS09370 has a mutation rate of 0.74.\n",
      "2025-04-16 11:13:21,939 - INFO - Gene cds-WP_001180173.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,940 - INFO - Gene gene-SPN23F_RS09375 has a mutation rate of 0.9066666666666666.\n",
      "2025-04-16 11:13:21,940 - INFO - Gene cds-WP_000022608.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,941 - INFO - Gene cds-SPN23F_RS09380 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,942 - INFO - Gene cds-SPN23F_RS13320 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,943 - INFO - Gene gene-SPN23F_RS09395 has a mutation rate of 0.6166666666666667.\n",
      "2025-04-16 11:13:21,943 - INFO - Gene cds-WP_000767469.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,943 - INFO - Gene cds-SPN23F_RS12600 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,944 - INFO - Gene gene-SPN23F_RS09400 has a mutation rate of 0.48333333333333334.\n",
      "2025-04-16 11:13:21,944 - INFO - Gene cds-WP_001810061.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,944 - INFO - Gene gene-SPN23F_RS09405 has a mutation rate of 0.16666666666666666.\n",
      "2025-04-16 11:13:21,945 - INFO - Gene cds-WP_000770408.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,946 - INFO - Gene gene-SPN23F_RS09410 has a mutation rate of 0.7666666666666667.\n",
      "2025-04-16 11:13:21,947 - INFO - Gene cds-WP_001194485.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,947 - INFO - Gene gene-SPN23F_RS09415 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,949 - INFO - Gene cds-WP_000372608.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,977 - INFO - Gene gene-SPN23F_RS09420 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,978 - INFO - Gene cds-WP_000865861.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,979 - INFO - Gene gene-SPN23F_RS09425 has a mutation rate of 0.12666666666666668.\n",
      "2025-04-16 11:13:21,980 - INFO - Gene cds-WP_000651211.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,989 - INFO - Gene gene-SPN23F_RS09430 has a mutation rate of 0.89.\n",
      "2025-04-16 11:13:21,990 - INFO - Gene cds-WP_000604338.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,991 - INFO - Gene gene-SPN23F_RS09435 has a mutation rate of 0.6066666666666667.\n",
      "2025-04-16 11:13:21,994 - INFO - Gene cds-WP_000046998.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,995 - INFO - Gene gene-SPN23F_RS09440 has a mutation rate of 0.5933333333333334.\n",
      "2025-04-16 11:13:21,996 - INFO - Gene cds-WP_000212522.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,996 - INFO - Gene gene-SPN23F_RS09445 has a mutation rate of 0.45.\n",
      "2025-04-16 11:13:21,997 - INFO - Gene cds-WP_000833735.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:21,998 - INFO - Gene gene-SPN23F_RS09450 has a mutation rate of 0.13666666666666666.\n",
      "2025-04-16 11:13:21,998 - INFO - Gene cds-WP_001022629.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,001 - INFO - Gene gene-SPN23F_RS09455 has a mutation rate of 0.58.\n",
      "2025-04-16 11:13:22,001 - INFO - Gene cds-WP_000645106.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,001 - INFO - Gene gene-SPN23F_RS09460 has a mutation rate of 0.6933333333333334.\n",
      "2025-04-16 11:13:22,003 - INFO - Gene cds-WP_000181689.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,003 - INFO - Gene gene-SPN23F_RS09465 has a mutation rate of 0.21.\n",
      "2025-04-16 11:13:22,004 - INFO - Gene cds-WP_000557779.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,004 - INFO - Gene gene-SPN23F_RS09470 has a mutation rate of 0.9466666666666667.\n",
      "2025-04-16 11:13:22,005 - INFO - Gene cds-WP_000182976.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,008 - INFO - Gene gene-SPN23F_RS09475 has a mutation rate of 0.30333333333333334.\n",
      "2025-04-16 11:13:22,008 - INFO - Gene cds-WP_000575210.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,009 - INFO - Gene gene-SPN23F_RS09480 has a mutation rate of 0.9.\n",
      "2025-04-16 11:13:22,009 - INFO - Gene cds-WP_000082879.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,010 - INFO - Gene gene-SPN23F_RS09485 has a mutation rate of 0.32.\n",
      "2025-04-16 11:13:22,010 - INFO - Gene cds-WP_000651443.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,011 - INFO - Gene gene-SPN23F_RS09490 has a mutation rate of 0.65.\n",
      "2025-04-16 11:13:22,012 - INFO - Gene cds-WP_000230153.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,012 - INFO - Gene gene-SPN23F_RS09495 has a mutation rate of 0.31333333333333335.\n",
      "2025-04-16 11:13:22,013 - INFO - Gene cds-WP_000209199.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,013 - INFO - Gene gene-SPN23F_RS09500 has a mutation rate of 0.8066666666666666.\n",
      "2025-04-16 11:13:22,014 - INFO - Gene cds-WP_000167830.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,014 - INFO - Gene gene-SPN23F_RS09505 has a mutation rate of 0.06.\n",
      "2025-04-16 11:13:22,014 - INFO - Gene cds-WP_001108391.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,015 - INFO - Gene gene-SPN23F_RS09510 has a mutation rate of 0.36.\n",
      "2025-04-16 11:13:22,016 - INFO - Gene cds-WP_000777248.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,017 - INFO - Gene gene-SPN23F_RS09515 has a mutation rate of 0.5433333333333333.\n",
      "2025-04-16 11:13:22,018 - INFO - Gene cds-WP_001193668.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,020 - INFO - Gene gene-SPN23F_RS09520 has a mutation rate of 0.4266666666666667.\n",
      "2025-04-16 11:13:22,020 - INFO - Gene cds-WP_000764300.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,021 - INFO - Gene gene-SPN23F_RS09525 has a mutation rate of 0.7933333333333333.\n",
      "2025-04-16 11:13:22,022 - INFO - Gene cds-WP_000858249.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,023 - INFO - Gene gene-SPN23F_RS09530 has a mutation rate of 0.35333333333333333.\n",
      "2025-04-16 11:13:22,025 - INFO - Gene cds-WP_000821640.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,025 - INFO - Gene gene-SPN23F_RS09535 has a mutation rate of 0.06333333333333334.\n",
      "2025-04-16 11:13:22,026 - INFO - Gene cds-WP_001222228.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,027 - INFO - Gene gene-SPN23F_RS09540 has a mutation rate of 0.05.\n",
      "2025-04-16 11:13:22,029 - INFO - Gene cds-WP_000105310.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,029 - INFO - Gene gene-SPN23F_RS09545 has a mutation rate of 0.18666666666666668.\n",
      "2025-04-16 11:13:22,030 - INFO - Gene cds-WP_000351907.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,031 - INFO - Gene gene-SPN23F_RS09550 has a mutation rate of 0.44666666666666666.\n",
      "2025-04-16 11:13:22,033 - INFO - Gene cds-WP_001808689.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,033 - INFO - Gene gene-SPN23F_RS09555 has a mutation rate of 0.03.\n",
      "2025-04-16 11:13:22,034 - INFO - Gene cds-WP_000560725.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,047 - INFO - Gene gene-SPN23F_RS09560 has a mutation rate of 0.27.\n",
      "2025-04-16 11:13:22,063 - INFO - Gene cds-WP_001118962.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,065 - INFO - Gene gene-SPN23F_RS09565 has a mutation rate of 0.32666666666666666.\n",
      "2025-04-16 11:13:22,066 - INFO - Gene cds-WP_012386884.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,075 - INFO - Gene gene-SPN23F_RS09570 has a mutation rate of 0.12.\n",
      "2025-04-16 11:13:22,076 - INFO - Gene cds-WP_000370376.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,077 - INFO - Gene gene-SPN23F_RS09575 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,078 - INFO - Gene cds-WP_000364990.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,079 - INFO - Gene gene-SPN23F_RS09580 has a mutation rate of 0.7933333333333333.\n",
      "2025-04-16 11:13:22,080 - INFO - Gene cds-WP_000171211.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,080 - INFO - Gene gene-SPN23F_RS09585 has a mutation rate of 0.68.\n",
      "2025-04-16 11:13:22,081 - INFO - Gene cds-WP_000514317.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,082 - INFO - Gene gene-SPN23F_RS09590 has a mutation rate of 0.18333333333333332.\n",
      "2025-04-16 11:13:22,082 - INFO - Gene cds-WP_000760670.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,082 - INFO - Gene gene-SPN23F_RS09595 has a mutation rate of 0.27666666666666667.\n",
      "2025-04-16 11:13:22,083 - INFO - Gene cds-WP_000202629.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,084 - INFO - Gene gene-SPN23F_RS09600 has a mutation rate of 0.22666666666666666.\n",
      "2025-04-16 11:13:22,085 - INFO - Gene cds-WP_001012089.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,086 - INFO - Gene cds-SPN23F_RS09605 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,086 - INFO - Gene gene-SPN23F_RS09610 has a mutation rate of 0.33.\n",
      "2025-04-16 11:13:22,087 - INFO - Gene cds-WP_001291293.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,089 - INFO - Gene gene-SPN23F_RS09615 has a mutation rate of 0.6533333333333333.\n",
      "2025-04-16 11:13:22,089 - INFO - Gene cds-WP_000159556.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,090 - INFO - Gene gene-SPN23F_RS09620 has a mutation rate of 0.10666666666666667.\n",
      "2025-04-16 11:13:22,091 - INFO - Gene cds-WP_000103700.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,092 - INFO - Gene cds-SPN23F_RS09625 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,093 - INFO - Gene gene-SPN23F_RS09635 has a mutation rate of 0.5966666666666667.\n",
      "2025-04-16 11:13:22,094 - INFO - Gene cds-WP_000742234.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,096 - INFO - Gene gene-SPN23F_RS09640 has a mutation rate of 0.22.\n",
      "2025-04-16 11:13:22,096 - INFO - Gene cds-WP_000835622.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,096 - INFO - Gene gene-SPN23F_RS09645 has a mutation rate of 0.79.\n",
      "2025-04-16 11:13:22,097 - INFO - Gene cds-WP_001122348.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,098 - INFO - Gene gene-SPN23F_RS13980 has a mutation rate of 0.35333333333333333.\n",
      "2025-04-16 11:13:22,099 - INFO - Gene cds-WP_000692999.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,099 - INFO - Gene gene-SPN23F_RS09655 has a mutation rate of 0.37.\n",
      "2025-04-16 11:13:22,100 - INFO - Gene cds-WP_000806565.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,101 - INFO - Gene gene-SPN23F_RS09660 has a mutation rate of 0.84.\n",
      "2025-04-16 11:13:22,103 - INFO - Gene cds-WP_000757162.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,104 - INFO - Gene gene-SPN23F_RS09665 has a mutation rate of 0.68.\n",
      "2025-04-16 11:13:22,105 - INFO - Gene cds-WP_000455417.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,106 - INFO - Gene gene-SPN23F_RS09670 has a mutation rate of 0.8766666666666667.\n",
      "2025-04-16 11:13:22,106 - INFO - Gene cds-WP_000538915.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,107 - INFO - Gene gene-SPN23F_RS09675 has a mutation rate of 0.78.\n",
      "2025-04-16 11:13:22,108 - INFO - Gene cds-WP_000959957.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,111 - INFO - Gene gene-SPN23F_RS09680 has a mutation rate of 0.8266666666666667.\n",
      "2025-04-16 11:13:22,111 - INFO - Gene cds-WP_000844516.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,112 - INFO - Gene cds-SPN23F_RS12620 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,119 - INFO - Gene gene-SPN23F_RS09695 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,144 - INFO - Gene gene-SPN23F_RS09700 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,144 - INFO - Gene gene-SPN23F_RS09705 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,152 - INFO - Gene gene-SPN23F_RS09710 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,154 - INFO - Gene gene-SPN23F_RS09715 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,154 - INFO - Gene gene-SPN23F_RS09720 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,155 - INFO - Gene gene-SPN23F_RS09725 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,155 - INFO - Gene gene-SPN23F_RS09730 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,157 - INFO - Gene gene-SPN23F_RS09735 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,158 - INFO - Gene gene-SPN23F_RS09740 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,159 - INFO - Gene gene-SPN23F_RS09745 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,159 - INFO - Gene gene-SPN23F_RS09750 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,159 - INFO - Gene gene-SPN23F_RS09755 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,160 - INFO - Gene gene-SPN23F_RS09760 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,160 - INFO - Gene gene-SPN23F_RS09765 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,160 - INFO - Gene gene-SPN23F_RS09770 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,160 - INFO - Gene gene-SPN23F_RS09775 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,161 - INFO - Gene gene-SPN23F_RS09780 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,161 - INFO - Gene gene-SPN23F_RS09785 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,161 - INFO - Gene gene-SPN23F_RS09790 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,162 - INFO - Gene gene-SPN23F_RS09795 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,162 - INFO - Gene gene-SPN23F_RS09800 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,162 - INFO - Gene gene-SPN23F_RS09805 has a mutation rate of 0.34.\n",
      "2025-04-16 11:13:22,163 - INFO - Gene cds-WP_001050309.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,163 - INFO - Gene gene-SPN23F_RS09810 has a mutation rate of 0.6733333333333333.\n",
      "2025-04-16 11:13:22,163 - INFO - Gene cds-WP_000705101.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,164 - INFO - Gene gene-SPN23F_RS09815 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,164 - INFO - Gene cds-WP_000775323.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,164 - INFO - Gene gene-SPN23F_RS13450 has a mutation rate of 0.3433333333333333.\n",
      "2025-04-16 11:13:22,165 - INFO - Gene cds-WP_000775827.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,165 - INFO - Gene cds-SPN23F_RS12630 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,165 - INFO - Gene gene-SPN23F_RS09840 has a mutation rate of 0.3233333333333333.\n",
      "2025-04-16 11:13:22,166 - INFO - Gene cds-WP_000031573.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,166 - INFO - Gene gene-SPN23F_RS09845 has a mutation rate of 0.49.\n",
      "2025-04-16 11:13:22,166 - INFO - Gene cds-WP_000917299.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,167 - INFO - Gene gene-SPN23F_RS09850 has a mutation rate of 0.49.\n",
      "2025-04-16 11:13:22,167 - INFO - Gene cds-WP_000282443.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,167 - INFO - Gene gene-SPN23F_RS09855 has a mutation rate of 0.5266666666666666.\n",
      "2025-04-16 11:13:22,168 - INFO - Gene cds-WP_001107758.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,168 - INFO - Gene gene-SPN23F_RS09860 has a mutation rate of 0.4666666666666667.\n",
      "2025-04-16 11:13:22,168 - INFO - Gene cds-WP_000578313.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,169 - INFO - Gene gene-SPN23F_RS09865 has a mutation rate of 0.43333333333333335.\n",
      "2025-04-16 11:13:22,169 - INFO - Gene cds-WP_000615767.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,169 - INFO - Gene gene-SPN23F_RS09870 has a mutation rate of 0.38333333333333336.\n",
      "2025-04-16 11:13:22,169 - INFO - Gene cds-WP_001013974.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,170 - INFO - Gene gene-SPN23F_RS13985 has a mutation rate of 0.5266666666666666.\n",
      "2025-04-16 11:13:22,170 - INFO - Gene cds-WP_223200442.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,170 - INFO - Gene gene-SPN23F_RS09875 has a mutation rate of 0.3333333333333333.\n",
      "2025-04-16 11:13:22,171 - INFO - Gene cds-WP_000291826.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,171 - INFO - Gene gene-SPN23F_RS09880 has a mutation rate of 0.6766666666666666.\n",
      "2025-04-16 11:13:22,171 - INFO - Gene cds-WP_000725749.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,171 - INFO - Gene gene-SPN23F_RS09885 has a mutation rate of 0.22666666666666666.\n",
      "2025-04-16 11:13:22,172 - INFO - Gene cds-WP_000776589.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,172 - INFO - Gene gene-SPN23F_RS09890 has a mutation rate of 0.5833333333333334.\n",
      "2025-04-16 11:13:22,172 - INFO - Gene cds-WP_000800947.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,172 - INFO - Gene gene-SPN23F_RS09895 has a mutation rate of 0.47.\n",
      "2025-04-16 11:13:22,173 - INFO - Gene cds-WP_000078805.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,173 - INFO - Gene cds-SPN23F_RS13990 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,173 - INFO - Gene gene-SPN23F_RS13995 has a mutation rate of 0.9733333333333334.\n",
      "2025-04-16 11:13:22,174 - INFO - Gene cds-WP_001826799.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,174 - INFO - Gene cds-SPN23F_RS09915 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,174 - INFO - Gene gene-SPN23F_RS09920 has a mutation rate of 0.06.\n",
      "2025-04-16 11:13:22,175 - INFO - Gene cds-WP_000532876.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,175 - INFO - Gene gene-SPN23F_RS09925 has a mutation rate of 0.07333333333333333.\n",
      "2025-04-16 11:13:22,175 - INFO - Gene cds-WP_001284359.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,176 - INFO - Gene gene-SPN23F_RS09930 has a mutation rate of 0.5.\n",
      "2025-04-16 11:13:22,176 - INFO - Gene cds-WP_000595762.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,177 - INFO - Gene gene-SPN23F_RS09935 has a mutation rate of 0.03333333333333333.\n",
      "2025-04-16 11:13:22,177 - INFO - Gene cds-WP_000083614.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,177 - INFO - Gene gene-SPN23F_RS09940 has a mutation rate of 0.02666666666666667.\n",
      "2025-04-16 11:13:22,178 - INFO - Gene cds-WP_000186955.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,178 - INFO - Gene cds-SPN23F_RS12655 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,178 - INFO - Gene gene-SPN23F_RS09955 has a mutation rate of 0.8366666666666667.\n",
      "2025-04-16 11:13:22,179 - INFO - Gene cds-WP_001209198.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,179 - INFO - Gene gene-SPN23F_RS09960 has a mutation rate of 0.9633333333333334.\n",
      "2025-04-16 11:13:22,179 - INFO - Gene cds-WP_001111248.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,180 - INFO - Gene gene-SPN23F_RS09965 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:22,180 - INFO - Gene cds-WP_000890146.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,180 - INFO - Gene gene-SPN23F_RS09970 has a mutation rate of 0.73.\n",
      "2025-04-16 11:13:22,181 - INFO - Gene cds-WP_001101205.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,181 - INFO - Gene gene-SPN23F_RS09975 has a mutation rate of 0.64.\n",
      "2025-04-16 11:13:22,181 - INFO - Gene cds-WP_000204064.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,182 - INFO - Gene gene-SPN23F_RS09980 has a mutation rate of 0.52.\n",
      "2025-04-16 11:13:22,182 - INFO - Gene cds-WP_001814321.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,182 - INFO - Gene gene-SPN23F_RS09985 has a mutation rate of 0.56.\n",
      "2025-04-16 11:13:22,212 - INFO - Gene cds-WP_000366093.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,212 - INFO - Gene gene-SPN23F_RS09990 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:22,213 - INFO - Gene cds-WP_000289348.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,213 - INFO - Gene gene-SPN23F_RS09995 has a mutation rate of 0.11.\n",
      "2025-04-16 11:13:22,214 - INFO - Gene cds-WP_000405240.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,214 - INFO - Gene gene-SPN23F_RS10000 has a mutation rate of 0.93.\n",
      "2025-04-16 11:13:22,214 - INFO - Gene cds-WP_001036278.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,215 - INFO - Gene gene-SPN23F_RS10005 has a mutation rate of 0.15666666666666668.\n",
      "2025-04-16 11:13:22,216 - INFO - Gene cds-WP_001085462.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,221 - INFO - Gene gene-SPN23F_RS10010 has a mutation rate of 0.72.\n",
      "2025-04-16 11:13:22,222 - INFO - Gene cds-WP_000642705.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,222 - INFO - Gene gene-SPN23F_RS10015 has a mutation rate of 0.8933333333333333.\n",
      "2025-04-16 11:13:22,222 - INFO - Gene cds-WP_000239284.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,223 - INFO - Gene gene-SPN23F_RS10020 has a mutation rate of 0.57.\n",
      "2025-04-16 11:13:22,223 - INFO - Gene cds-WP_000455537.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,223 - INFO - Gene gene-SPN23F_RS10025 has a mutation rate of 0.52.\n",
      "2025-04-16 11:13:22,224 - INFO - Gene cds-WP_000288232.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,224 - INFO - Gene gene-SPN23F_RS10030 has a mutation rate of 0.19666666666666666.\n",
      "2025-04-16 11:13:22,224 - INFO - Gene cds-WP_000839908.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,225 - INFO - Gene gene-SPN23F_RS10035 has a mutation rate of 0.67.\n",
      "2025-04-16 11:13:22,225 - INFO - Gene cds-WP_001093641.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,225 - INFO - Gene gene-SPN23F_RS14140 has a mutation rate of 0.22.\n",
      "2025-04-16 11:13:22,226 - INFO - Gene cds-WP_001227862.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,227 - INFO - Gene gene-SPN23F_RS10045 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,227 - INFO - Gene cds-WP_000181087.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,227 - INFO - Gene gene-SPN23F_RS10050 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,227 - INFO - Gene cds-WP_000786759.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,228 - INFO - Gene gene-SPN23F_RS10055 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,228 - INFO - Gene cds-WP_000875569.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,228 - INFO - Gene gene-SPN23F_RS10060 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,229 - INFO - Gene cds-WP_001033170.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,229 - INFO - Gene gene-SPN23F_RS10065 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,229 - INFO - Gene cds-WP_000912954.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,230 - INFO - Gene gene-SPN23F_RS10070 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,230 - INFO - Gene cds-WP_000701298.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,230 - INFO - Gene gene-SPN23F_RS10075 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,231 - INFO - Gene cds-WP_000699969.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,231 - INFO - Gene gene-SPN23F_RS10080 has a mutation rate of 0.55.\n",
      "2025-04-16 11:13:22,231 - INFO - Gene cds-WP_000078794.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,232 - INFO - Gene gene-SPN23F_RS10085 has a mutation rate of 0.44666666666666666.\n",
      "2025-04-16 11:13:22,232 - INFO - Gene cds-WP_000571415.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,232 - INFO - Gene gene-SPN23F_RS10095 has a mutation rate of 0.06.\n",
      "2025-04-16 11:13:22,233 - INFO - Gene cds-WP_000438285.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,233 - INFO - Gene gene-SPN23F_RS10100 has a mutation rate of 0.15666666666666668.\n",
      "2025-04-16 11:13:22,233 - INFO - Gene cds-WP_000228756.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,234 - INFO - Gene gene-SPN23F_RS10105 has a mutation rate of 0.07666666666666666.\n",
      "2025-04-16 11:13:22,234 - INFO - Gene cds-WP_000907135.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,235 - INFO - Gene gene-SPN23F_RS10110 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,235 - INFO - Gene gene-SPN23F_RS10115 has a mutation rate of 0.3333333333333333.\n",
      "2025-04-16 11:13:22,235 - INFO - Gene cds-WP_000390046.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,236 - INFO - Gene gene-SPN23F_RS10120 has a mutation rate of 0.37333333333333335.\n",
      "2025-04-16 11:13:22,236 - INFO - Gene cds-WP_001036779.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,236 - INFO - Gene gene-SPN23F_RS10125 has a mutation rate of 0.46.\n",
      "2025-04-16 11:13:22,237 - INFO - Gene cds-WP_001036519.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,237 - INFO - Gene gene-SPN23F_RS10130 has a mutation rate of 0.15333333333333332.\n",
      "2025-04-16 11:13:22,237 - INFO - Gene cds-WP_000358033.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,238 - INFO - Gene gene-SPN23F_RS10135 has a mutation rate of 0.45666666666666667.\n",
      "2025-04-16 11:13:22,238 - INFO - Gene cds-WP_000615437.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,238 - INFO - Gene gene-SPN23F_RS10140 has a mutation rate of 0.38.\n",
      "2025-04-16 11:13:22,239 - INFO - Gene cds-WP_000727280.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,239 - INFO - Gene gene-SPN23F_RS10145 has a mutation rate of 0.65.\n",
      "2025-04-16 11:13:22,239 - INFO - Gene cds-WP_001280766.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,240 - INFO - Gene gene-SPN23F_RS10150 has a mutation rate of 0.8766666666666667.\n",
      "2025-04-16 11:13:22,240 - INFO - Gene cds-WP_000706960.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,240 - INFO - Gene gene-SPN23F_RS10155 has a mutation rate of 0.12333333333333334.\n",
      "2025-04-16 11:13:22,241 - INFO - Gene cds-WP_000747993.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,241 - INFO - Gene gene-SPN23F_RS10165 has a mutation rate of 0.013333333333333334.\n",
      "2025-04-16 11:13:22,241 - INFO - Gene cds-WP_001016913.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,242 - INFO - Gene gene-SPN23F_RS10170 has a mutation rate of 0.8.\n",
      "2025-04-16 11:13:22,242 - INFO - Gene cds-WP_000152701.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,243 - INFO - Gene gene-SPN23F_RS10175 has a mutation rate of 0.16333333333333333.\n",
      "2025-04-16 11:13:22,243 - INFO - Gene cds-WP_001174613.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,243 - INFO - Gene gene-SPN23F_RS10180 has a mutation rate of 0.38666666666666666.\n",
      "2025-04-16 11:13:22,244 - INFO - Gene cds-WP_000835948.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,244 - INFO - Gene gene-SPN23F_RS10185 has a mutation rate of 0.29333333333333333.\n",
      "2025-04-16 11:13:22,244 - INFO - Gene cds-WP_001288287.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,245 - INFO - Gene gene-SPN23F_RS10190 has a mutation rate of 0.81.\n",
      "2025-04-16 11:13:22,245 - INFO - Gene cds-WP_000856523.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,245 - INFO - Gene gene-SPN23F_RS10195 has a mutation rate of 0.04.\n",
      "2025-04-16 11:13:22,246 - INFO - Gene cds-WP_001809310.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,246 - INFO - Gene gene-SPN23F_RS10200 has a mutation rate of 0.5633333333333334.\n",
      "2025-04-16 11:13:22,246 - INFO - Gene cds-WP_000703180.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,247 - INFO - Gene gene-SPN23F_RS10205 has a mutation rate of 0.8833333333333333.\n",
      "2025-04-16 11:13:22,247 - INFO - Gene cds-WP_000444687.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,247 - INFO - Gene gene-SPN23F_RS10210 has a mutation rate of 0.71.\n",
      "2025-04-16 11:13:22,267 - INFO - Gene cds-WP_000006144.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,279 - INFO - Gene gene-SPN23F_RS10215 has a mutation rate of 0.4166666666666667.\n",
      "2025-04-16 11:13:22,280 - INFO - Gene cds-WP_000085026.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,280 - INFO - Gene gene-SPN23F_RS10220 has a mutation rate of 0.53.\n",
      "2025-04-16 11:13:22,281 - INFO - Gene cds-WP_001161981.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,281 - INFO - Gene gene-SPN23F_RS10225 has a mutation rate of 0.7066666666666667.\n",
      "2025-04-16 11:13:22,281 - INFO - Gene cds-WP_001216856.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,282 - INFO - Gene gene-SPN23F_RS10230 has a mutation rate of 0.49333333333333335.\n",
      "2025-04-16 11:13:22,282 - INFO - Gene cds-WP_000333253.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,282 - INFO - Gene gene-SPN23F_RS10235 has a mutation rate of 0.08666666666666667.\n",
      "2025-04-16 11:13:22,286 - INFO - Gene cds-WP_000565531.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,287 - INFO - Gene gene-SPN23F_RS10240 has a mutation rate of 0.8866666666666667.\n",
      "2025-04-16 11:13:22,287 - INFO - Gene cds-WP_000825254.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,288 - INFO - Gene gene-SPN23F_RS10245 has a mutation rate of 0.29.\n",
      "2025-04-16 11:13:22,288 - INFO - Gene cds-WP_000815392.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,289 - INFO - Gene gene-SPN23F_RS10250 has a mutation rate of 0.9466666666666667.\n",
      "2025-04-16 11:13:22,289 - INFO - Gene cds-WP_000904834.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,289 - INFO - Gene gene-SPN23F_RS10255 has a mutation rate of 0.9666666666666667.\n",
      "2025-04-16 11:13:22,290 - INFO - Gene cds-WP_000660009.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,290 - INFO - Gene gene-SPN23F_RS10260 has a mutation rate of 0.6166666666666667.\n",
      "2025-04-16 11:13:22,291 - INFO - Gene cds-WP_000575597.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,291 - INFO - Gene cds-SPN23F_RS10265 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,291 - INFO - Gene gene-SPN23F_RS10270 has a mutation rate of 0.016666666666666666.\n",
      "2025-04-16 11:13:22,292 - INFO - Gene cds-WP_000831905.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,292 - INFO - Gene gene-SPN23F_RS10275 has a mutation rate of 0.22333333333333333.\n",
      "2025-04-16 11:13:22,292 - INFO - Gene cds-WP_000666469.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,293 - INFO - Gene gene-SPN23F_RS10280 has a mutation rate of 0.14333333333333334.\n",
      "2025-04-16 11:13:22,293 - INFO - Gene cds-WP_000977368.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,294 - INFO - Gene gene-SPN23F_RS10285 has a mutation rate of 0.18.\n",
      "2025-04-16 11:13:22,294 - INFO - Gene cds-WP_000079148.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,294 - INFO - Gene gene-SPN23F_RS10290 has a mutation rate of 0.48333333333333334.\n",
      "2025-04-16 11:13:22,295 - INFO - Gene cds-WP_000453143.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,295 - INFO - Gene gene-SPN23F_RS10295 has a mutation rate of 0.5833333333333334.\n",
      "2025-04-16 11:13:22,295 - INFO - Gene cds-WP_001124777.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,296 - INFO - Gene gene-SPN23F_RS10300 has a mutation rate of 0.41.\n",
      "2025-04-16 11:13:22,296 - INFO - Gene cds-WP_000990607.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,297 - INFO - Gene gene-SPN23F_RS10305 has a mutation rate of 0.7933333333333333.\n",
      "2025-04-16 11:13:22,297 - INFO - Gene cds-WP_000866719.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,297 - INFO - Gene gene-SPN23F_RS10310 has a mutation rate of 1.0.\n",
      "2025-04-16 11:13:22,298 - INFO - Gene cds-WP_000893500.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,298 - INFO - Gene gene-SPN23F_RS10315 has a mutation rate of 0.94.\n",
      "2025-04-16 11:13:22,298 - INFO - Gene cds-WP_000832916.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,299 - INFO - Gene gene-SPN23F_RS10320 has a mutation rate of 0.5333333333333333.\n",
      "2025-04-16 11:13:22,299 - INFO - Gene cds-WP_001055978.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,299 - INFO - Gene gene-SPN23F_RS10325 has a mutation rate of 0.6033333333333334.\n",
      "2025-04-16 11:13:22,300 - INFO - Gene cds-WP_000820189.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,300 - INFO - Gene cds-SPN23F_RS14000 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,300 - INFO - Gene gene-SPN23F_RS10335 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,301 - INFO - Gene gene-SPN23F_RS10340 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,301 - INFO - Gene gene-SPN23F_RS10345 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,301 - INFO - Gene gene-SPN23F_RS10350 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,302 - INFO - Gene gene-SPN23F_RS10355 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,302 - INFO - Gene gene-SPN23F_RS10360 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,302 - INFO - Gene gene-SPN23F_RS10365 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,303 - INFO - Gene gene-SPN23F_RS10370 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,303 - INFO - Gene gene-SPN23F_RS10375 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,303 - INFO - Gene gene-SPN23F_RS10380 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,304 - INFO - Gene gene-SPN23F_RS10385 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,304 - INFO - Gene gene-SPN23F_RS10390 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,304 - INFO - Gene gene-SPN23F_RS10395 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,304 - INFO - Gene gene-SPN23F_RS10400 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,305 - INFO - Gene gene-SPN23F_RS10405 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,305 - INFO - Gene cds-WP_000588897.1-2 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,306 - INFO - Gene gene-SPN23F_RS10410 has a mutation rate of 0.1.\n",
      "2025-04-16 11:13:22,306 - INFO - Gene cds-WP_000376736.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,306 - INFO - Gene gene-SPN23F_RS10415 has a mutation rate of 0.016666666666666666.\n",
      "2025-04-16 11:13:22,307 - INFO - Gene cds-WP_001210991.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,307 - INFO - Gene gene-SPN23F_RS10420 has a mutation rate of 0.056666666666666664.\n",
      "2025-04-16 11:13:22,307 - INFO - Gene cds-WP_001809375.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,307 - INFO - Gene gene-SPN23F_RS10425 has a mutation rate of 0.8766666666666667.\n",
      "2025-04-16 11:13:22,308 - INFO - Gene cds-WP_000762607.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,308 - INFO - Gene gene-SPN23F_RS10430 has a mutation rate of 0.49.\n",
      "2025-04-16 11:13:22,309 - INFO - Gene cds-WP_001861296.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,309 - INFO - Gene gene-SPN23F_RS10435 has a mutation rate of 0.21.\n",
      "2025-04-16 11:13:22,309 - INFO - Gene cds-WP_000260685.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,310 - INFO - Gene cds-SPN23F_RS10440 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,310 - INFO - Gene cds-SPN23F_RS12695 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,311 - INFO - Gene gene-SPN23F_RS10455 has a mutation rate of 0.17333333333333334.\n",
      "2025-04-16 11:13:22,311 - INFO - Gene cds-WP_000022748.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,311 - INFO - Gene gene-SPN23F_RS10460 has a mutation rate of 0.4666666666666667.\n",
      "2025-04-16 11:13:22,312 - INFO - Gene cds-WP_000581530.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,342 - INFO - Gene gene-SPN23F_RS14275 has a mutation rate of 0.36333333333333334.\n",
      "2025-04-16 11:13:22,342 - INFO - Gene cds-WP_001814328.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,343 - INFO - Gene gene-SPN23F_RS10465 has a mutation rate of 0.12666666666666668.\n",
      "2025-04-16 11:13:22,343 - INFO - Gene cds-WP_000676515.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,343 - INFO - Gene gene-SPN23F_RS10470 has a mutation rate of 0.07666666666666666.\n",
      "2025-04-16 11:13:22,344 - INFO - Gene cds-WP_000105270.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,344 - INFO - Gene gene-SPN23F_RS10475 has a mutation rate of 0.7366666666666667.\n",
      "2025-04-16 11:13:22,344 - INFO - Gene cds-WP_000728312.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,345 - INFO - Gene gene-SPN23F_RS10480 has a mutation rate of 0.28.\n",
      "2025-04-16 11:13:22,345 - INFO - Gene cds-WP_000798146.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,345 - INFO - Gene gene-SPN23F_RS10485 has a mutation rate of 0.06.\n",
      "2025-04-16 11:13:22,350 - INFO - Gene cds-WP_000809625.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,351 - INFO - Gene gene-SPN23F_RS10490 has a mutation rate of 0.09.\n",
      "2025-04-16 11:13:22,351 - INFO - Gene cds-WP_000134124.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,352 - INFO - Gene gene-SPN23F_RS10495 has a mutation rate of 0.7033333333333334.\n",
      "2025-04-16 11:13:22,352 - INFO - Gene cds-WP_000763967.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,353 - INFO - Gene gene-SPN23F_RS10500 has a mutation rate of 0.2.\n",
      "2025-04-16 11:13:22,353 - INFO - Gene cds-WP_000429499.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,353 - INFO - Gene gene-SPN23F_RS10505 has a mutation rate of 0.36.\n",
      "2025-04-16 11:13:22,354 - INFO - Gene cds-WP_000737438.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,354 - INFO - Gene gene-SPN23F_RS10510 has a mutation rate of 0.03.\n",
      "2025-04-16 11:13:22,355 - INFO - Gene cds-WP_001069051.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,355 - INFO - Gene gene-SPN23F_RS10515 has a mutation rate of 0.7433333333333333.\n",
      "2025-04-16 11:13:22,355 - INFO - Gene cds-WP_000067859.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,356 - INFO - Gene gene-SPN23F_RS10520 has a mutation rate of 0.25333333333333335.\n",
      "2025-04-16 11:13:22,356 - INFO - Gene cds-WP_001134220.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,356 - INFO - Gene gene-SPN23F_RS10525 has a mutation rate of 0.5833333333333334.\n",
      "2025-04-16 11:13:22,357 - INFO - Gene cds-WP_000242112.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,357 - INFO - Gene gene-SPN23F_RS10530 has a mutation rate of 0.55.\n",
      "2025-04-16 11:13:22,357 - INFO - Gene cds-WP_001077556.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,358 - INFO - Gene cds-SPN23F_RS10535 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,358 - INFO - Gene gene-SPN23F_RS10540 has a mutation rate of 0.5533333333333333.\n",
      "2025-04-16 11:13:22,358 - INFO - Gene cds-WP_000166727.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,359 - INFO - Gene gene-SPN23F_RS10545 has a mutation rate of 0.5233333333333333.\n",
      "2025-04-16 11:13:22,359 - INFO - Gene cds-WP_001049932.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,359 - INFO - Gene gene-SPN23F_RS10550 has a mutation rate of 0.12333333333333334.\n",
      "2025-04-16 11:13:22,360 - INFO - Gene cds-WP_000241469.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,360 - INFO - Gene gene-SPN23F_RS10555 has a mutation rate of 0.7466666666666667.\n",
      "2025-04-16 11:13:22,360 - INFO - Gene cds-WP_000454427.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,361 - INFO - Gene gene-SPN23F_RS10560 has a mutation rate of 0.9633333333333334.\n",
      "2025-04-16 11:13:22,361 - INFO - Gene cds-WP_000932661.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,361 - INFO - Gene gene-SPN23F_RS10565 has a mutation rate of 0.69.\n",
      "2025-04-16 11:13:22,362 - INFO - Gene cds-WP_000260018.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,362 - INFO - Gene gene-SPN23F_RS10570 has a mutation rate of 0.20666666666666667.\n",
      "2025-04-16 11:13:22,362 - INFO - Gene cds-WP_000727904.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,362 - INFO - Gene gene-SPN23F_RS10575 has a mutation rate of 0.17333333333333334.\n",
      "2025-04-16 11:13:22,363 - INFO - Gene cds-WP_000739246.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,363 - INFO - Gene gene-SPN23F_RS14145 has a mutation rate of 0.04.\n",
      "2025-04-16 11:13:22,364 - INFO - Gene cds-WP_000768904.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,364 - INFO - Gene gene-SPN23F_RS10580 has a mutation rate of 0.42333333333333334.\n",
      "2025-04-16 11:13:22,364 - INFO - Gene cds-WP_000167766.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,365 - INFO - Gene gene-SPN23F_RS10585 has a mutation rate of 0.9066666666666666.\n",
      "2025-04-16 11:13:22,365 - INFO - Gene cds-WP_000345133.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,365 - INFO - Gene cds-SPN23F_RS12710 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,366 - INFO - Gene gene-SPN23F_RS10600 has a mutation rate of 0.8566666666666667.\n",
      "2025-04-16 11:13:22,366 - INFO - Gene cds-WP_001811371.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,367 - INFO - Gene gene-SPN23F_RS10605 has a mutation rate of 0.84.\n",
      "2025-04-16 11:13:22,367 - INFO - Gene cds-WP_000250539.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,367 - INFO - Gene gene-SPN23F_RS10610 has a mutation rate of 0.19666666666666666.\n",
      "2025-04-16 11:13:22,368 - INFO - Gene cds-WP_000413382.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,368 - INFO - Gene gene-SPN23F_RS10615 has a mutation rate of 0.79.\n",
      "2025-04-16 11:13:22,368 - INFO - Gene cds-WP_000588030.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,369 - INFO - Gene gene-SPN23F_RS10620 has a mutation rate of 0.6333333333333333.\n",
      "2025-04-16 11:13:22,369 - INFO - Gene cds-WP_000738625.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,370 - INFO - Gene gene-SPN23F_RS10625 has a mutation rate of 0.99.\n",
      "2025-04-16 11:13:22,370 - INFO - Gene cds-WP_078134418.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,370 - INFO - Gene gene-SPN23F_RS10630 has a mutation rate of 0.56.\n",
      "2025-04-16 11:13:22,371 - INFO - Gene cds-WP_000249573.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,371 - INFO - Gene gene-SPN23F_RS10635 has a mutation rate of 0.05.\n",
      "2025-04-16 11:13:22,371 - INFO - Gene cds-WP_000286415.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,372 - INFO - Gene gene-SPN23F_RS10640 has a mutation rate of 0.47.\n",
      "2025-04-16 11:13:22,372 - INFO - Gene cds-WP_000649468.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,372 - INFO - Gene gene-SPN23F_RS10645 has a mutation rate of 0.08333333333333333.\n",
      "2025-04-16 11:13:22,373 - INFO - Gene cds-WP_001134457.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,373 - INFO - Gene gene-SPN23F_RS10650 has a mutation rate of 0.7966666666666666.\n",
      "2025-04-16 11:13:22,374 - INFO - Gene cds-WP_001220865.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,374 - INFO - Gene gene-SPN23F_RS10655 has a mutation rate of 0.17666666666666667.\n",
      "2025-04-16 11:13:22,374 - INFO - Gene cds-WP_001285241.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,375 - INFO - Gene gene-SPN23F_RS10660 has a mutation rate of 0.5633333333333334.\n",
      "2025-04-16 11:13:22,375 - INFO - Gene cds-WP_001108852.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,395 - INFO - Gene gene-SPN23F_RS10665 has a mutation rate of 0.78.\n",
      "2025-04-16 11:13:22,405 - INFO - Gene cds-WP_000866913.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,405 - INFO - Gene gene-SPN23F_RS10670 has a mutation rate of 0.42.\n",
      "2025-04-16 11:13:22,406 - INFO - Gene cds-WP_000022518.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,406 - INFO - Gene gene-SPN23F_RS10675 has a mutation rate of 0.35.\n",
      "2025-04-16 11:13:22,406 - INFO - Gene cds-WP_000570116.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,407 - INFO - Gene gene-SPN23F_RS10680 has a mutation rate of 0.7533333333333333.\n",
      "2025-04-16 11:13:22,407 - INFO - Gene cds-WP_000746992.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,408 - INFO - Gene gene-SPN23F_RS10685 has a mutation rate of 0.43333333333333335.\n",
      "2025-04-16 11:13:22,408 - INFO - Gene cds-WP_001172823.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,409 - INFO - Gene gene-SPN23F_RS10690 has a mutation rate of 0.48333333333333334.\n",
      "2025-04-16 11:13:22,409 - INFO - Gene cds-WP_000473965.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,410 - INFO - Gene gene-SPN23F_RS10695 has a mutation rate of 0.6833333333333333.\n",
      "2025-04-16 11:13:22,410 - INFO - Gene cds-WP_000177121.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,410 - INFO - Gene gene-SPN23F_RS10700 has a mutation rate of 0.5933333333333334.\n",
      "2025-04-16 11:13:22,411 - INFO - Gene cds-WP_001277932.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,411 - INFO - Gene gene-SPN23F_RS10705 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,417 - INFO - Gene gene-SPN23F_RS10710 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,417 - INFO - Gene gene-SPN23F_RS10715 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,418 - INFO - Gene gene-SPN23F_RS10720 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,418 - INFO - Gene gene-SPN23F_RS10725 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,418 - INFO - Gene gene-SPN23F_RS10730 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,419 - INFO - Gene gene-SPN23F_RS10735 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,419 - INFO - Gene gene-SPN23F_RS10740 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,420 - INFO - Gene gene-SPN23F_RS10745 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,420 - INFO - Gene gene-SPN23F_RS10750 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,420 - INFO - Gene gene-SPN23F_RS10755 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,421 - INFO - Gene gene-SPN23F_RS10760 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,421 - INFO - Gene gene-SPN23F_RS10765 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,421 - INFO - Gene gene-SPN23F_RS10770 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,422 - INFO - Gene gene-SPN23F_RS10775 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,422 - INFO - Gene gene-SPN23F_RS10780 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,422 - INFO - Gene gene-SPN23F_RS10785 has a mutation rate of 0.4033333333333333.\n",
      "2025-04-16 11:13:22,423 - INFO - Gene cds-WP_000355927.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,423 - INFO - Gene gene-SPN23F_RS10790 has a mutation rate of 0.5033333333333333.\n",
      "2025-04-16 11:13:22,423 - INFO - Gene cds-WP_000031090.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,424 - INFO - Gene gene-SPN23F_RS10795 has a mutation rate of 0.08666666666666667.\n",
      "2025-04-16 11:13:22,424 - INFO - Gene cds-WP_000018263.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,424 - INFO - Gene gene-SPN23F_RS10800 has a mutation rate of 0.07333333333333333.\n",
      "2025-04-16 11:13:22,425 - INFO - Gene cds-WP_000400702.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,425 - INFO - Gene gene-SPN23F_RS10805 has a mutation rate of 0.31666666666666665.\n",
      "2025-04-16 11:13:22,426 - INFO - Gene cds-WP_000124748.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,426 - INFO - Gene gene-SPN23F_RS10810 has a mutation rate of 0.42.\n",
      "2025-04-16 11:13:22,426 - INFO - Gene cds-WP_000859856.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,426 - INFO - Gene cds-SPN23F_RS14010 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,427 - INFO - Gene gene-SPN23F_RS10820 has a mutation rate of 0.5.\n",
      "2025-04-16 11:13:22,427 - INFO - Gene cds-WP_000908137.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,427 - INFO - Gene gene-SPN23F_RS10825 has a mutation rate of 0.5766666666666667.\n",
      "2025-04-16 11:13:22,428 - INFO - Gene cds-WP_000963680.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,428 - INFO - Gene gene-SPN23F_RS10830 has a mutation rate of 0.21333333333333335.\n",
      "2025-04-16 11:13:22,428 - INFO - Gene cds-WP_001231472.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,429 - INFO - Gene gene-SPN23F_RS10835 has a mutation rate of 0.69.\n",
      "2025-04-16 11:13:22,429 - INFO - Gene cds-WP_001092739.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,429 - INFO - Gene cds-SPN23F_RS12720 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,430 - INFO - Gene gene-SPN23F_RS10850 has a mutation rate of 0.5733333333333334.\n",
      "2025-04-16 11:13:22,430 - INFO - Gene cds-WP_000356602.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,430 - INFO - Gene gene-SPN23F_RS10855 has a mutation rate of 0.09.\n",
      "2025-04-16 11:13:22,431 - INFO - Gene cds-WP_000166475.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,431 - INFO - Gene gene-SPN23F_RS10860 has a mutation rate of 0.64.\n",
      "2025-04-16 11:13:22,432 - INFO - Gene cds-WP_000833277.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,432 - INFO - Gene gene-SPN23F_RS10865 has a mutation rate of 0.23666666666666666.\n",
      "2025-04-16 11:13:22,432 - INFO - Gene cds-WP_000669493.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,433 - INFO - Gene gene-SPN23F_RS10870 has a mutation rate of 0.5833333333333334.\n",
      "2025-04-16 11:13:22,433 - INFO - Gene cds-WP_000595180.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,433 - INFO - Gene gene-SPN23F_RS10875 has a mutation rate of 0.02666666666666667.\n",
      "2025-04-16 11:13:22,434 - INFO - Gene cds-WP_000049768.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,434 - INFO - Gene gene-SPN23F_RS10880 has a mutation rate of 0.27666666666666667.\n",
      "2025-04-16 11:13:22,434 - INFO - Gene cds-WP_000536449.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,435 - INFO - Gene gene-SPN23F_RS10885 has a mutation rate of 0.2866666666666667.\n",
      "2025-04-16 11:13:22,435 - INFO - Gene cds-WP_001245781.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,436 - INFO - Gene cds-SPN23F_RS10890 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,436 - INFO - Gene gene-SPN23F_RS10895 has a mutation rate of 0.38333333333333336.\n",
      "2025-04-16 11:13:22,436 - INFO - Gene cds-WP_001206584.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,437 - INFO - Gene gene-SPN23F_RS10900 has a mutation rate of 0.5633333333333334.\n",
      "2025-04-16 11:13:22,437 - INFO - Gene cds-WP_000415108.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,437 - INFO - Gene gene-SPN23F_RS10905 has a mutation rate of 0.7133333333333334.\n",
      "2025-04-16 11:13:22,438 - INFO - Gene cds-WP_000202235.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,438 - INFO - Gene gene-SPN23F_RS10910 has a mutation rate of 0.21666666666666667.\n",
      "2025-04-16 11:13:22,461 - INFO - Gene cds-WP_000658498.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,469 - INFO - Gene gene-SPN23F_RS10915 has a mutation rate of 0.9933333333333333.\n",
      "2025-04-16 11:13:22,469 - INFO - Gene cds-WP_000834301.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,470 - INFO - Gene gene-SPN23F_RS10920 has a mutation rate of 0.73.\n",
      "2025-04-16 11:13:22,470 - INFO - Gene cds-WP_000885102.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,471 - INFO - Gene gene-SPN23F_RS10925 has a mutation rate of 0.3566666666666667.\n",
      "2025-04-16 11:13:22,471 - INFO - Gene cds-WP_000127466.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,471 - INFO - Gene gene-SPN23F_RS10930 has a mutation rate of 0.44666666666666666.\n",
      "2025-04-16 11:13:22,472 - INFO - Gene cds-WP_001812545.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,476 - INFO - Gene gene-SPN23F_RS10935 has a mutation rate of 0.9066666666666666.\n",
      "2025-04-16 11:13:22,478 - INFO - Gene cds-WP_001180961.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,478 - INFO - Gene gene-SPN23F_RS10940 has a mutation rate of 0.4166666666666667.\n",
      "2025-04-16 11:13:22,478 - INFO - Gene cds-WP_000546887.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,479 - INFO - Gene gene-SPN23F_RS10945 has a mutation rate of 0.84.\n",
      "2025-04-16 11:13:22,479 - INFO - Gene cds-WP_000008795.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,480 - INFO - Gene gene-SPN23F_RS10950 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:22,480 - INFO - Gene cds-WP_000910909.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,481 - INFO - Gene gene-SPN23F_RS10955 has a mutation rate of 0.37333333333333335.\n",
      "2025-04-16 11:13:22,481 - INFO - Gene cds-WP_001095515.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,481 - INFO - Gene gene-SPN23F_RS14015 has a mutation rate of 0.43333333333333335.\n",
      "2025-04-16 11:13:22,482 - INFO - Gene cds-WP_000915304.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,482 - INFO - Gene gene-SPN23F_RS10965 has a mutation rate of 0.76.\n",
      "2025-04-16 11:13:22,482 - INFO - Gene cds-WP_001037912.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,483 - INFO - Gene gene-SPN23F_RS10975 has a mutation rate of 0.65.\n",
      "2025-04-16 11:13:22,483 - INFO - Gene cds-WP_000950152.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,483 - INFO - Gene cds-SPN23F_RS10980 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,484 - INFO - Gene gene-SPN23F_RS10985 has a mutation rate of 0.30333333333333334.\n",
      "2025-04-16 11:13:22,484 - INFO - Gene cds-WP_000095485.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,484 - INFO - Gene gene-SPN23F_RS10990 has a mutation rate of 0.03.\n",
      "2025-04-16 11:13:22,485 - INFO - Gene cds-WP_000414968.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,485 - INFO - Gene gene-SPN23F_RS10995 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,486 - INFO - Gene cds-WP_001065636.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,486 - INFO - Gene gene-SPN23F_RS11000 has a mutation rate of 0.76.\n",
      "2025-04-16 11:13:22,486 - INFO - Gene cds-WP_000938210.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,487 - INFO - Gene gene-SPN23F_RS11005 has a mutation rate of 0.07666666666666666.\n",
      "2025-04-16 11:13:22,487 - INFO - Gene cds-WP_001145439.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,487 - INFO - Gene gene-SPN23F_RS11010 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,488 - INFO - Gene cds-WP_000436644.1-3 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,488 - INFO - Gene gene-SPN23F_RS14020 has a mutation rate of 0.5066666666666667.\n",
      "2025-04-16 11:13:22,488 - INFO - Gene cds-WP_012636593.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,488 - INFO - Gene gene-SPN23F_RS11020 has a mutation rate of 0.27666666666666667.\n",
      "2025-04-16 11:13:22,489 - INFO - Gene cds-WP_000814947.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,489 - INFO - Gene gene-SPN23F_RS11025 has a mutation rate of 0.86.\n",
      "2025-04-16 11:13:22,490 - INFO - Gene cds-WP_000830880.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,490 - INFO - Gene gene-SPN23F_RS14025 has a mutation rate of 0.53.\n",
      "2025-04-16 11:13:22,490 - INFO - Gene cds-WP_000693028.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,490 - INFO - Gene gene-SPN23F_RS11035 has a mutation rate of 0.09666666666666666.\n",
      "2025-04-16 11:13:22,491 - INFO - Gene cds-WP_000832180.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,491 - INFO - Gene gene-SPN23F_RS11040 has a mutation rate of 0.6033333333333334.\n",
      "2025-04-16 11:13:22,491 - INFO - Gene cds-WP_000771232.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,492 - INFO - Gene gene-SPN23F_RS11045 has a mutation rate of 0.19.\n",
      "2025-04-16 11:13:22,492 - INFO - Gene cds-WP_000730285.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,493 - INFO - Gene gene-SPN23F_RS11050 has a mutation rate of 0.48333333333333334.\n",
      "2025-04-16 11:13:22,493 - INFO - Gene cds-WP_001046141.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,493 - INFO - Gene gene-SPN23F_RS11055 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:22,494 - INFO - Gene cds-WP_001176227.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,494 - INFO - Gene gene-SPN23F_RS11060 has a mutation rate of 0.8833333333333333.\n",
      "2025-04-16 11:13:22,494 - INFO - Gene cds-WP_000566080.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,495 - INFO - Gene gene-SPN23F_RS14030 has a mutation rate of 0.2966666666666667.\n",
      "2025-04-16 11:13:22,495 - INFO - Gene cds-WP_000692976.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,496 - INFO - Gene gene-SPN23F_RS11070 has a mutation rate of 0.9466666666666667.\n",
      "2025-04-16 11:13:22,496 - INFO - Gene cds-WP_000775873.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,496 - INFO - Gene gene-SPN23F_RS11075 has a mutation rate of 0.7866666666666666.\n",
      "2025-04-16 11:13:22,497 - INFO - Gene cds-WP_000770608.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,497 - INFO - Gene gene-SPN23F_RS11080 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,497 - INFO - Gene cds-WP_001808790.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,498 - INFO - Gene gene-SPN23F_RS11085 has a mutation rate of 0.77.\n",
      "2025-04-16 11:13:22,498 - INFO - Gene cds-WP_000434983.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,498 - INFO - Gene gene-SPN23F_RS11090 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,499 - INFO - Gene cds-WP_001155337.1-2 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,499 - INFO - Gene gene-SPN23F_RS11095 has a mutation rate of 0.08666666666666667.\n",
      "2025-04-16 11:13:22,499 - INFO - Gene cds-WP_001204054.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,500 - INFO - Gene gene-SPN23F_RS11100 has a mutation rate of 0.5933333333333334.\n",
      "2025-04-16 11:13:22,500 - INFO - Gene cds-WP_000137358.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,501 - INFO - Gene gene-SPN23F_RS11105 has a mutation rate of 0.8866666666666667.\n",
      "2025-04-16 11:13:22,501 - INFO - Gene cds-WP_000986376.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,501 - INFO - Gene gene-SPN23F_RS11110 has a mutation rate of 0.5533333333333333.\n",
      "2025-04-16 11:13:22,502 - INFO - Gene cds-WP_000603788.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,502 - INFO - Gene gene-SPN23F_RS11115 has a mutation rate of 0.49333333333333335.\n",
      "2025-04-16 11:13:22,521 - INFO - Gene cds-WP_000975298.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,530 - INFO - Gene gene-SPN23F_RS11120 has a mutation rate of 0.06333333333333334.\n",
      "2025-04-16 11:13:22,534 - INFO - Gene cds-WP_000912476.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,535 - INFO - Gene gene-SPN23F_RS11125 has a mutation rate of 0.6066666666666667.\n",
      "2025-04-16 11:13:22,535 - INFO - Gene cds-WP_000460927.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,535 - INFO - Gene gene-SPN23F_RS11130 has a mutation rate of 0.13.\n",
      "2025-04-16 11:13:22,536 - INFO - Gene cds-WP_000136847.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,536 - INFO - Gene gene-SPN23F_RS11135 has a mutation rate of 0.03666666666666667.\n",
      "2025-04-16 11:13:22,536 - INFO - Gene cds-WP_000796544.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,537 - INFO - Gene gene-SPN23F_RS11140 has a mutation rate of 0.17666666666666667.\n",
      "2025-04-16 11:13:22,537 - INFO - Gene cds-WP_000290417.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,540 - INFO - Gene gene-SPN23F_RS11145 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,542 - INFO - Gene cds-WP_001265622.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,542 - INFO - Gene gene-SPN23F_RS11150 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,542 - INFO - Gene cds-WP_000436237.1-4 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,543 - INFO - Gene gene-SPN23F_RS11155 has a mutation rate of 0.5.\n",
      "2025-04-16 11:13:22,544 - INFO - Gene cds-WP_000753884.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,544 - INFO - Gene gene-SPN23F_RS11160 has a mutation rate of 0.06666666666666667.\n",
      "2025-04-16 11:13:22,544 - INFO - Gene cds-WP_001809622.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,545 - INFO - Gene cds-SPN23F_RS12745 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,545 - INFO - Gene gene-SPN23F_RS11175 has a mutation rate of 0.91.\n",
      "2025-04-16 11:13:22,546 - INFO - Gene cds-WP_409349413.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,546 - INFO - Gene gene-SPN23F_RS11180 has a mutation rate of 0.9066666666666666.\n",
      "2025-04-16 11:13:22,546 - INFO - Gene cds-WP_000251395.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,547 - INFO - Gene gene-SPN23F_RS11185 has a mutation rate of 0.64.\n",
      "2025-04-16 11:13:22,547 - INFO - Gene cds-WP_000150722.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,547 - INFO - Gene gene-SPN23F_RS11190 has a mutation rate of 0.9533333333333334.\n",
      "2025-04-16 11:13:22,548 - INFO - Gene cds-WP_000432770.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,548 - INFO - Gene gene-SPN23F_RS11195 has a mutation rate of 0.9933333333333333.\n",
      "2025-04-16 11:13:22,548 - INFO - Gene cds-WP_000640057.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,549 - INFO - Gene gene-SPN23F_RS11200 has a mutation rate of 1.0.\n",
      "2025-04-16 11:13:22,549 - INFO - Gene cds-WP_000803440.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,549 - INFO - Gene gene-SPN23F_RS11205 has a mutation rate of 0.9966666666666667.\n",
      "2025-04-16 11:13:22,550 - INFO - Gene cds-WP_000721498.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,550 - INFO - Gene gene-SPN23F_RS11210 has a mutation rate of 0.74.\n",
      "2025-04-16 11:13:22,550 - INFO - Gene cds-WP_000094616.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,551 - INFO - Gene gene-SPN23F_RS11215 has a mutation rate of 0.09.\n",
      "2025-04-16 11:13:22,551 - INFO - Gene cds-WP_000185363.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,552 - INFO - Gene gene-SPN23F_RS11220 has a mutation rate of 0.5033333333333333.\n",
      "2025-04-16 11:13:22,552 - INFO - Gene cds-WP_000013420.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,552 - INFO - Gene gene-SPN23F_RS11225 has a mutation rate of 0.31666666666666665.\n",
      "2025-04-16 11:13:22,553 - INFO - Gene cds-WP_001290376.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,553 - INFO - Gene gene-SPN23F_RS11230 has a mutation rate of 0.8033333333333333.\n",
      "2025-04-16 11:13:22,553 - INFO - Gene cds-WP_000686945.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,554 - INFO - Gene cds-SPN23F_RS14280 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,554 - INFO - Gene gene-SPN23F_RS11240 has a mutation rate of 0.5066666666666667.\n",
      "2025-04-16 11:13:22,554 - INFO - Gene cds-WP_000501061.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,555 - INFO - Gene gene-SPN23F_RS11245 has a mutation rate of 0.5466666666666666.\n",
      "2025-04-16 11:13:22,555 - INFO - Gene cds-WP_000657764.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,555 - INFO - Gene gene-SPN23F_RS11250 has a mutation rate of 0.03.\n",
      "2025-04-16 11:13:22,556 - INFO - Gene cds-WP_001808444.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,556 - INFO - Gene gene-SPN23F_RS11255 has a mutation rate of 0.7733333333333333.\n",
      "2025-04-16 11:13:22,557 - INFO - Gene cds-WP_000190404.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,557 - INFO - Gene gene-SPN23F_RS11260 has a mutation rate of 0.8666666666666667.\n",
      "2025-04-16 11:13:22,557 - INFO - Gene cds-WP_000614250.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,558 - INFO - Gene gene-SPN23F_RS11265 has a mutation rate of 0.3566666666666667.\n",
      "2025-04-16 11:13:22,558 - INFO - Gene cds-WP_001031113.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,558 - INFO - Gene gene-SPN23F_RS11270 has a mutation rate of 0.31.\n",
      "2025-04-16 11:13:22,558 - INFO - Gene cds-WP_000780608.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,559 - INFO - Gene gene-SPN23F_RS11275 has a mutation rate of 0.05333333333333334.\n",
      "2025-04-16 11:13:22,559 - INFO - Gene cds-WP_000981721.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,560 - INFO - Gene gene-SPN23F_RS11280 has a mutation rate of 0.01.\n",
      "2025-04-16 11:13:22,560 - INFO - Gene cds-WP_000899623.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,561 - INFO - Gene gene-SPN23F_RS11285 has a mutation rate of 0.03666666666666667.\n",
      "2025-04-16 11:13:22,561 - INFO - Gene cds-WP_000020992.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,561 - INFO - Gene gene-SPN23F_RS11290 has a mutation rate of 0.31333333333333335.\n",
      "2025-04-16 11:13:22,561 - INFO - Gene cds-WP_000483180.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,562 - INFO - Gene gene-SPN23F_RS11295 has a mutation rate of 0.16666666666666666.\n",
      "2025-04-16 11:13:22,562 - INFO - Gene cds-WP_000912113.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,563 - INFO - Gene gene-SPN23F_RS11300 has a mutation rate of 0.25666666666666665.\n",
      "2025-04-16 11:13:22,563 - INFO - Gene cds-WP_001286139.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,563 - INFO - Gene gene-SPN23F_RS11305 has a mutation rate of 0.5633333333333334.\n",
      "2025-04-16 11:13:22,563 - INFO - Gene cds-WP_000388542.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,564 - INFO - Gene gene-SPN23F_RS11310 has a mutation rate of 0.44666666666666666.\n",
      "2025-04-16 11:13:22,564 - INFO - Gene cds-WP_000588548.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,565 - INFO - Gene gene-SPN23F_RS11315 has a mutation rate of 0.5733333333333334.\n",
      "2025-04-16 11:13:22,565 - INFO - Gene cds-WP_000724067.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,566 - INFO - Gene gene-SPN23F_RS11320 has a mutation rate of 0.08666666666666667.\n",
      "2025-04-16 11:13:22,586 - INFO - Gene cds-WP_000950018.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,596 - INFO - Gene gene-SPN23F_RS11325 has a mutation rate of 0.6633333333333333.\n",
      "2025-04-16 11:13:22,597 - INFO - Gene cds-WP_001269488.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,597 - INFO - Gene gene-SPN23F_RS11330 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:22,597 - INFO - Gene cds-WP_001249309.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,598 - INFO - Gene gene-SPN23F_RS11335 has a mutation rate of 0.16.\n",
      "2025-04-16 11:13:22,598 - INFO - Gene cds-WP_000919857.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,599 - INFO - Gene gene-SPN23F_RS11340 has a mutation rate of 0.03666666666666667.\n",
      "2025-04-16 11:13:22,599 - INFO - Gene cds-WP_000351967.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,599 - INFO - Gene gene-SPN23F_RS11345 has a mutation rate of 0.5566666666666666.\n",
      "2025-04-16 11:13:22,601 - INFO - Gene cds-WP_000969733.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,605 - INFO - Gene gene-SPN23F_RS11350 has a mutation rate of 0.72.\n",
      "2025-04-16 11:13:22,606 - INFO - Gene cds-WP_000066731.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,606 - INFO - Gene gene-SPN23F_RS11355 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,607 - INFO - Gene cds-WP_000813667.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,607 - INFO - Gene gene-SPN23F_RS11360 has a mutation rate of 0.5366666666666666.\n",
      "2025-04-16 11:13:22,607 - INFO - Gene cds-WP_000071468.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,608 - INFO - Gene cds-SPN23F_RS14035 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,608 - INFO - Gene gene-SPN23F_RS14235 has a mutation rate of 0.056666666666666664.\n",
      "2025-04-16 11:13:22,609 - INFO - Gene cds-WP_000930581.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,609 - INFO - Gene gene-SPN23F_RS11380 has a mutation rate of 0.20333333333333334.\n",
      "2025-04-16 11:13:22,609 - INFO - Gene cds-WP_001808792.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,610 - INFO - Gene gene-SPN23F_RS11385 has a mutation rate of 0.32.\n",
      "2025-04-16 11:13:22,610 - INFO - Gene cds-WP_001127306.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,610 - INFO - Gene gene-SPN23F_RS11390 has a mutation rate of 0.7.\n",
      "2025-04-16 11:13:22,611 - INFO - Gene cds-WP_000980829.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,611 - INFO - Gene gene-SPN23F_RS11395 has a mutation rate of 0.93.\n",
      "2025-04-16 11:13:22,612 - INFO - Gene cds-WP_000395926.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,612 - INFO - Gene gene-SPN23F_RS11400 has a mutation rate of 0.27666666666666667.\n",
      "2025-04-16 11:13:22,613 - INFO - Gene cds-WP_000076765.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,613 - INFO - Gene gene-SPN23F_RS11405 has a mutation rate of 0.5333333333333333.\n",
      "2025-04-16 11:13:22,613 - INFO - Gene cds-WP_000280548.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,614 - INFO - Gene gene-SPN23F_RS11410 has a mutation rate of 0.62.\n",
      "2025-04-16 11:13:22,614 - INFO - Gene cds-WP_000357847.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,614 - INFO - Gene gene-SPN23F_RS11415 has a mutation rate of 0.4.\n",
      "2025-04-16 11:13:22,615 - INFO - Gene cds-WP_000183311.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,615 - INFO - Gene gene-SPN23F_RS12760 has a mutation rate of 0.9866666666666667.\n",
      "2025-04-16 11:13:22,616 - INFO - Gene cds-WP_000458085.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,616 - INFO - Gene gene-SPN23F_RS11425 has a mutation rate of 0.8233333333333334.\n",
      "2025-04-16 11:13:22,616 - INFO - Gene cds-WP_000241928.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,616 - INFO - Gene gene-SPN23F_RS11430 has a mutation rate of 0.16666666666666666.\n",
      "2025-04-16 11:13:22,617 - INFO - Gene cds-WP_000594667.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,617 - INFO - Gene gene-SPN23F_RS11435 has a mutation rate of 0.42.\n",
      "2025-04-16 11:13:22,618 - INFO - Gene cds-WP_001024075.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,618 - INFO - Gene gene-SPN23F_RS11445 has a mutation rate of 0.7766666666666666.\n",
      "2025-04-16 11:13:22,619 - INFO - Gene cds-WP_001109685.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,619 - INFO - Gene gene-SPN23F_RS11450 has a mutation rate of 0.45666666666666667.\n",
      "2025-04-16 11:13:22,619 - INFO - Gene cds-WP_001211273.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,620 - INFO - Gene gene-SPN23F_RS11455 has a mutation rate of 0.12666666666666668.\n",
      "2025-04-16 11:13:22,620 - INFO - Gene cds-WP_000136224.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,620 - INFO - Gene gene-SPN23F_RS11460 has a mutation rate of 0.07333333333333333.\n",
      "2025-04-16 11:13:22,621 - INFO - Gene cds-WP_000754546.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,621 - INFO - Gene gene-SPN23F_RS11465 has a mutation rate of 0.58.\n",
      "2025-04-16 11:13:22,621 - INFO - Gene cds-WP_001813866.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,622 - INFO - Gene gene-SPN23F_RS11470 has a mutation rate of 0.3466666666666667.\n",
      "2025-04-16 11:13:22,622 - INFO - Gene cds-WP_000647690.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,623 - INFO - Gene gene-SPN23F_RS11475 has a mutation rate of 0.7733333333333333.\n",
      "2025-04-16 11:13:22,623 - INFO - Gene cds-WP_000698561.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,623 - INFO - Gene gene-SPN23F_RS11480 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,623 - INFO - Gene cds-WP_001278167.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,624 - INFO - Gene gene-SPN23F_RS11485 has a mutation rate of 0.17333333333333334.\n",
      "2025-04-16 11:13:22,624 - INFO - Gene cds-WP_000852486.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,704 - INFO - Gene gene-SPN23F_RS11490 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,704 - INFO - Gene cds-WP_000864220.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,705 - INFO - Gene gene-SPN23F_RS11495 has a mutation rate of 0.42.\n",
      "2025-04-16 11:13:22,705 - INFO - Gene cds-WP_000715123.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,706 - INFO - Gene gene-SPN23F_RS11500 has a mutation rate of 0.02.\n",
      "2025-04-16 11:13:22,706 - INFO - Gene cds-WP_000599107.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,706 - INFO - Gene gene-SPN23F_RS11505 has a mutation rate of 0.16666666666666666.\n",
      "2025-04-16 11:13:22,707 - INFO - Gene cds-WP_000649974.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,707 - INFO - Gene gene-SPN23F_RS11510 has a mutation rate of 0.9266666666666666.\n",
      "2025-04-16 11:13:22,707 - INFO - Gene cds-WP_000867608.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,708 - INFO - Gene gene-SPN23F_RS11515 has a mutation rate of 0.31666666666666665.\n",
      "2025-04-16 11:13:22,708 - INFO - Gene cds-WP_000395739.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,708 - INFO - Gene gene-SPN23F_RS11520 has a mutation rate of 0.59.\n",
      "2025-04-16 11:13:22,708 - INFO - Gene cds-WP_000029866.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,709 - INFO - Gene cds-SPN23F_RS11525 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,709 - INFO - Gene cds-SPN23F_RS11530 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,710 - INFO - Gene gene-SPN23F_RS11535 has a mutation rate of 0.54.\n",
      "2025-04-16 11:13:22,710 - INFO - Gene cds-WP_000973408.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,710 - INFO - Gene gene-SPN23F_RS11540 has a mutation rate of 0.18333333333333332.\n",
      "2025-04-16 11:13:22,711 - INFO - Gene cds-WP_000808063.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,711 - INFO - Gene gene-SPN23F_RS11545 has a mutation rate of 0.28.\n",
      "2025-04-16 11:13:22,711 - INFO - Gene cds-WP_000268466.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,711 - INFO - Gene gene-SPN23F_RS11550 has a mutation rate of 0.20666666666666667.\n",
      "2025-04-16 11:13:22,712 - INFO - Gene cds-WP_000727012.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,712 - INFO - Gene gene-SPN23F_RS11555 has a mutation rate of 0.17.\n",
      "2025-04-16 11:13:22,712 - INFO - Gene cds-WP_001249454.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,713 - INFO - Gene gene-SPN23F_RS11560 has a mutation rate of 0.12.\n",
      "2025-04-16 11:13:22,713 - INFO - Gene cds-WP_001078983.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,713 - INFO - Gene gene-SPN23F_RS11565 has a mutation rate of 0.11.\n",
      "2025-04-16 11:13:22,713 - INFO - Gene cds-WP_000377893.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,714 - INFO - Gene gene-SPN23F_RS11570 has a mutation rate of 0.2633333333333333.\n",
      "2025-04-16 11:13:22,714 - INFO - Gene cds-WP_000510412.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,714 - INFO - Gene gene-SPN23F_RS11575 has a mutation rate of 0.36333333333333334.\n",
      "2025-04-16 11:13:22,714 - INFO - Gene cds-WP_000835715.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,715 - INFO - Gene gene-SPN23F_RS11580 has a mutation rate of 0.25666666666666665.\n",
      "2025-04-16 11:13:22,715 - INFO - Gene cds-WP_000712132.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,715 - INFO - Gene gene-SPN23F_RS11585 has a mutation rate of 0.17666666666666667.\n",
      "2025-04-16 11:13:22,716 - INFO - Gene cds-WP_001227958.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,716 - INFO - Gene gene-SPN23F_RS11590 has a mutation rate of 0.86.\n",
      "2025-04-16 11:13:22,716 - INFO - Gene cds-WP_000170191.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,717 - INFO - Gene gene-SPN23F_RS11595 has a mutation rate of 0.6966666666666667.\n",
      "2025-04-16 11:13:22,717 - INFO - Gene cds-WP_000424267.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,718 - INFO - Gene gene-SPN23F_RS11600 has a mutation rate of 0.17666666666666667.\n",
      "2025-04-16 11:13:22,718 - INFO - Gene cds-WP_000455903.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,718 - INFO - Gene gene-SPN23F_RS11605 has a mutation rate of 0.34.\n",
      "2025-04-16 11:13:22,719 - INFO - Gene cds-WP_000266669.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,719 - INFO - Gene gene-SPN23F_RS11610 has a mutation rate of 0.15.\n",
      "2025-04-16 11:13:22,719 - INFO - Gene cds-WP_000073428.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,720 - INFO - Gene gene-SPN23F_RS11615 has a mutation rate of 0.07666666666666666.\n",
      "2025-04-16 11:13:22,720 - INFO - Gene cds-WP_000165444.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,720 - INFO - Gene gene-SPN23F_RS11620 has a mutation rate of 0.03.\n",
      "2025-04-16 11:13:22,721 - INFO - Gene cds-WP_000958769.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,721 - INFO - Gene gene-SPN23F_RS11625 has a mutation rate of 0.85.\n",
      "2025-04-16 11:13:22,721 - INFO - Gene cds-WP_000834663.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,722 - INFO - Gene cds-SPN23F_RS14040 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,722 - INFO - Gene gene-SPN23F_RS11645 has a mutation rate of 0.16666666666666666.\n",
      "2025-04-16 11:13:22,722 - INFO - Gene cds-WP_001158266.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,723 - INFO - Gene gene-SPN23F_RS11650 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,723 - INFO - Gene gene-SPN23F_RS11655 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,723 - INFO - Gene gene-SPN23F_RS11660 has a mutation rate of 0.21.\n",
      "2025-04-16 11:13:22,724 - INFO - Gene cds-WP_000866065.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,724 - INFO - Gene gene-SPN23F_RS11665 has a mutation rate of 0.86.\n",
      "2025-04-16 11:13:22,724 - INFO - Gene cds-WP_000364843.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,725 - INFO - Gene gene-SPN23F_RS11670 has a mutation rate of 0.023333333333333334.\n",
      "2025-04-16 11:13:22,725 - INFO - Gene cds-WP_000799694.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,726 - INFO - Gene gene-SPN23F_RS11675 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,726 - INFO - Gene gene-SPN23F_RS11680 has a mutation rate of 0.04666666666666667.\n",
      "2025-04-16 11:13:22,726 - INFO - Gene cds-WP_000695929.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,727 - INFO - Gene gene-SPN23F_RS11685 has a mutation rate of 0.30666666666666664.\n",
      "2025-04-16 11:13:22,727 - INFO - Gene cds-WP_000681598.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,727 - INFO - Gene gene-SPN23F_RS11690 has a mutation rate of 0.25333333333333335.\n",
      "2025-04-16 11:13:22,727 - INFO - Gene cds-WP_000410383.1 has no mutations in the population. Skipping this gene.\n",
      "2025-04-16 11:13:22,728 - INFO - Integrating 1775 genes to run GWAS on pyseer.\n",
      "2025-04-16 11:13:22,729 - INFO - Skipped 2536 genes with no mutations.\n",
      "2025-04-16 11:13:22,730 - INFO - Converted mutation summary file pyseer_dataset/gene_mutation_summary.txt to numeric format and saved as pyseer_dataset/gene_mutation_summary_filtered_numeric.txt.\n",
      "2025-04-16 11:13:22,731 - INFO - Running pyseer with command: pyseer --lmm --phenotypes pyseer_dataset/resistances.subset.txt --similarity pyseer_dataset/phylogeny_distances.tsv --pres pyseer_dataset/gene_mutation_summary_filtered_numeric.txt > pyseer_dataset/pyseer_results.txt\n",
      "Read 300 phenotypes\n",
      "Detected binary phenotype\n",
      "Setting up LMM\n",
      "Similarity matrix has dimension (616, 616)\n",
      "Analysing 300 samples found in both phenotype and similarity matrix\n",
      "h^2 = 0.87\n",
      "1775 loaded variants\n",
      "19 pre-filtered variants\n",
      "1756 tested variants\n",
      "1756 printed variants\n",
      "2025-04-16 11:13:25,331 - INFO - Pyseer run completed successfully.\n",
      "2025-04-16 11:13:25,332 - INFO - GWAS results are saved in pyseer_dataset\n",
      "2025-04-16 11:13:25,332 - INFO - Script completed in 0 hours, 0 minutes, and 6 seconds.\n",
      "2025-04-16 11:13:25,333 - INFO - Finished running pyseer.\n",
      "2025-04-16 11:13:25,333 - INFO - GWAS analysis completed.\n"
     ]
    }
   ],
   "source": [
    "from muggwas import run_pyseer\n",
    "\n",
    "phenotype_file = 'pyseer_dataset/resistances.subset.txt'\n",
    "mutation_file = 'pyseer_dataset/gene_mutation_summary.txt'\n",
    "similarity_matrix_file = 'pyseer_dataset/phylogeny_distances.tsv'\n",
    "output_dir = 'pyseer_dataset/'\n",
    "\n",
    "# Run pyseer\n",
    "run_pyseer(phenotype_file, similarity_matrix_file, mutation_file, output_dir)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## **Future Direction**\n",
    "\n",
    "1. Add statistical significance threshold to the GWAS result:\n",
    "- The Bonferroni threshold depends on how many genes are tested.\n",
    "\n",
    "2. Add a plotting module:\n",
    "- Input: `gwas_result.txt`\n",
    "- Output:\n",
    "    1. Q-Q plot: `gwas_result.qq.png`\n",
    "    2. manhattan plot: `gwas_result.man.png`\n",
    "- Functionality: Plot Q-Q plots and manhattan plots."
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "muggwas",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.22"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
